
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="NeSI Support Documentation">
      
      
      
        <link rel="canonical" href="https://github.com/nesi/support-docs-concept/assets/glossary/Glossary/">
      
      
        <link rel="prev" href="../">
      
      
        <link rel="next" href="../dictionary/">
      
      
      <link rel="icon" href="../../icons/favicon.ico">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.6">
    
    
      
        <title>Glossary - NeSI Support Center</title>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/main.50c56a3b.min.css">
      
        
        <link rel="stylesheet" href="../../stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/neoteroi-mkdocs.css">
    
      <link rel="stylesheet" href="../../stylesheets/footer.css">
    
      <link rel="stylesheet" href="../../stylesheets/custom_admonations.css">
    
      <link rel="stylesheet" href="../../stylesheets/theme.css">
    
    <script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function n(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],n("js",new Date),n("config","G-TVQR4R1F40"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&n("event","search",{search_term:this.value})}),document$.subscribe(function(){var a=document.forms.feedback;if(void 0!==a)for(var e of a.querySelectorAll("[type=submit]"))e.addEventListener("click",function(e){e.preventDefault();var t=document.location.pathname,e=this.getAttribute("data-md-value");n("event","feedback",{page:t,data:e}),a.firstElementChild.disabled=!0;e=a.querySelector(".md-feedback__note [data-md-value='"+e+"']");e&&(e.hidden=!1)}),a.hidden=!1}),location$.subscribe(function(e){n("config","G-TVQR4R1F40",{page_path:e.pathname})})});var e=document.createElement("script");e.async=!0,e.src="https://www.googletagmanager.com/gtag/js?id=G-TVQR4R1F40",document.getElementById("__analytics").insertAdjacentElement("afterEnd",e)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
<style>
#banner{
    font-size: larger;
    text-align: center;
    background:  var(--md-warning-bg-color);
}
#banner a:link {
    color: blue;
}
#banner a:visited {
    color: purple;
}
#banner a:focus,
#banner a:hover {
    border-bottom: 1px solid;
}
#banner p{
    color: black;
}
</style>

<div id="banner">
    <p>This site is in Beta. All content is a duplication of  
    <a href="https://support.nesi.org.nz/hc/en-gb">offical NeSI support documentation.</a>
    </p>
</div>

      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="NeSI Support Center" class="md-header__button md-logo" aria-label="NeSI Support Center" data-md-component="logo">
      
  <img src="../../icons/logo.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            NeSI Support Center
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Glossary
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2a7 7 0 0 0-7 7c0 2.38 1.19 4.47 3 5.74V17a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1v-2.26c1.81-1.27 3-3.36 3-5.74a7 7 0 0 0-7-7M9 21a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1v-1H9v1Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="custom" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2a7 7 0 0 1 7 7c0 2.38-1.19 4.47-3 5.74V17a1 1 0 0 1-1 1H9a1 1 0 0 1-1-1v-2.26C6.19 13.47 5 11.38 5 9a7 7 0 0 1 7-7M9 21v-1h6v1a1 1 0 0 1-1 1h-4a1 1 0 0 1-1-1m3-17a5 5 0 0 0-5 5c0 2.05 1.23 3.81 3 4.58V16h4v-2.42c1.77-.77 3-2.53 3-4.58a5 5 0 0 0-5-5Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/nesi/support-docs-concept" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    


    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="NeSI Support Center" class="md-nav__button md-logo" aria-label="NeSI Support Center" data-md-component="logo">
      
  <img src="../../icons/logo.svg" alt="logo">

    </a>
    NeSI Support Center
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/nesi/support-docs-concept" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../../../Getting_Started/Accounts-Projects_and_Allocations/Creating_a_NeSI_Account_Profile/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Getting Started
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../../../General/Announcements/Accessing_NeSI_Support_during_the_holiday_break/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    General
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../../../Scientific_Computing/Training/Introduction_to_computing_on_the_NeSI_HPC/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Scientific Computing
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../../../Storage/File_Systems_and_Quotas/NeSI_File_Systems_and_Quotas/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Storage
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                

                  

  
    <a href="https://github.com/nesi/support-docs-concept/edit/main/docs/assets/glossary/Glossary.md" title="Edit this page" class="md-content__button md-icon">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"/></svg>
    </a>
  
  


  <h1>Glossary</h1>

<p><strong>ABAQUS:</strong></p>
<p>Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA.</p>
<p><strong>ABRicate:</strong></p>
<p>Mass screening of contigs for antimicrobial and virulence genes</p>
<p><strong>ABySS:</strong></p>
<p>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</p>
<p><strong>ACTC:</strong></p>
<p>ACTC converts independent triangles into triangle strips or fans.</p>
<p><strong>AGAT:</strong></p>
<p>Suite of tools to handle gene annotations in any GTF/GFF format.</p>
<p><strong>AGE:</strong></p>
<p>Alignment of sequences with structural variants.</p>
<p><strong>AMOS:</strong></p>
<p>Collection of tools for genome assembly</p>
<p><strong>AMRFinderPlus:</strong></p>
<p>NCBI Antimicrobial Resistance Gene Finder Plus</p>
<p><strong>ANIcalculator:</strong></p>
<p>Calculate the bidirectional average nucleotide identity (gANI) and</p>
<p><strong>ANNOVAR:</strong></p>
<p>Efficient software tool to utilize update-to-date information to functionally</p>
<p><strong>ANTLR:</strong></p>
<p>ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.</p>
<p><strong>ANTS:</strong></p>
<p>Ants is a versatile, easy to use Python library for developing ancillary applications.                   This package is restricted to the UM user group.</p>
<p><strong>ANTs:</strong></p>
<p>ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,</p>
<p><strong>AOCC:</strong></p>
<p>AMD Optimized C/C++ &amp; Fortran compilers (AOCC) based on LLVM 13.0</p>
<p><strong>AOCL-BLIS:</strong></p>
<p>Optimized version of BLIS for AMD EPYC family of processors..</p>
<p><strong>AOCL-FFTW:</strong></p>
<p>Optimized version of FFTW for AMD EPYC family of processors.</p>
<p><strong>AOCL-ScaLAPACK:</strong></p>
<p>Optimized version of ScaLAPACK for AMD EPYC family of processors.</p>
<p><strong>APR:</strong></p>
<p>Apache Portable Runtime (APR) libraries.</p>
<p><strong>APR-util:</strong></p>
<p>Apache Portable Runtime (APR) util libraries.</p>
<p><strong>ARCSI:</strong></p>
<p>The Atmospheric and Radiometric Correction of Satellite Imagery (ARCSI) software provides a command line tool for the atmospheric correction of Earth Observation imagery. The aim of ARCSI is to provide as automatic as possible method of retrieving the atmospheric correction parameters and using them to parameterise 6S.Universal Command Line Environment for AWS.</p>
<p><strong>ARIBA:</strong></p>
<p>Antimicrobial Resistance Identification By Assembly</p>
<p><strong>ATK:</strong></p>
<p>ATK provides the set of accessibility interfaces that are implemented by other toolkits and applications.</p>
<p><strong>AUGUSTUS:</strong></p>
<p>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences</p>
<p><strong>AdapterRemoval:</strong></p>
<p>Ssearches for and removes remnant adapter sequences</p>
<p><strong>Advisor:</strong></p>
<p>Vectorization Optimization and Thread Prototyping</p>
<p><strong>AlphaFold:</strong></p>
<p>AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known</p>
<p><strong>AlphaFold2DB:</strong></p>
<p>AlphaFold2 databases</p>
<p><strong>AlwaysIntelMKL:</strong></p>
<p>Overrides the MKL internal utility function mkl_serv_intel_cpu_true</p>
<p><strong>Amber:</strong></p>
<p>Amber (originally Assisted Model Building with Energy</p>
<p><strong>Anaconda2:</strong></p>
<p>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.  IMPORTANT: This version of Anaconda Python comes with Intel MKL support to speed up            certain types of mathematical computations, such as linear algebra or FFT.            The module sets             MKL_NUM_THREADS=1             to run MKL on a single thread by default, avoiding accidental oversubscription            of cores. The number of threads can be increased for large problems, please            refer to the Intel MKL documentation for guidance.</p>
<p><strong>Anaconda3:</strong></p>
<p>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.</p>
<p><strong>Apptainer:</strong></p>
<p>Apptainer is a portable application stack packaging and runtime utility.</p>
<p><strong>Armadillo:</strong></p>
<p>C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use.</p>
<p><strong>Arrow:</strong></p>
<p>Apache Arrow, a cross-language development platform for in-memory data.</p>
<p><strong>Aspera-CLI:</strong></p>
<p>IBM Aspera Command-Line Interface (the Aspera CLI) is</p>
<p><strong>Augustus:</strong></p>
<p>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</p>
<p><strong>AutoDock-GPU:</strong></p>
<p>OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly</p>
<p><strong>AutoDock_Vina:</strong></p>
<p>AutoDock Vina is an open-source program for doing molecular docking.</p>
<p><strong>BBMap:</strong></p>
<p>BBMap short read aligner, and other bioinformatic tools.</p>
<p><strong>BCFtools:</strong></p>
<p>Manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF.</p>
<p><strong>BCL-Convert:</strong></p>
<p>Converts per cycle binary data output by Illumina sequencers containing basecall</p>
<p><strong>BEAST:</strong></p>
<p>Bayesian MCMC phylogenetic analysis of molecular sequences for reconstructing</p>
<p><strong>BEDOPS:</strong></p>
<p>BEDOPS is an open-source command-line toolkit that performs highly</p>
<p><strong>BEDTools:</strong></p>
<p>The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps</p>
<p><strong>BEEF:</strong></p>
<p>BEEF is a library implementing the Bayesian Error Estimation Functional, a description of which can be found here:  http://dx.doi.org/10.1103/PhysRevB.85.235149</p>
<p><strong>BLASR:</strong></p>
<p>BLASR (Basic Local Alignment with Successive Refinement) rapidly maps</p>
<p><strong>BLAST:</strong></p>
<p>Basic Local Alignment Search Tool, or BLAST, is an algorithm</p>
<p><strong>BLASTDB:</strong></p>
<p>BLAST databases downloaded from NCBI.</p>
<p><strong>BLAT:</strong></p>
<p>BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more.</p>
<p><strong>BLIS:</strong></p>
<p>BLIS is a portable software framework for instantiating high-performance</p>
<p><strong>BOLT-LMM:</strong></p>
<p>The BOLT-LMM algorithm for mixed model association testing,</p>
<p><strong>BRAKER:</strong></p>
<p>Pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET</p>
<p><strong>BUSCO:</strong></p>
<p>Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs</p>
<p><strong>BWA:</strong></p>
<p>Burrows-Wheeler Aligner (BWA) is an efficient program that aligns</p>
<p><strong>BamTools:</strong></p>
<p>BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.</p>
<p><strong>Bandage:</strong></p>
<p>Bandage is a program for visualising de novo assembly graphs</p>
<p><strong>Basilisk:</strong></p>
<p>Basilisk is a Free Software program for the solution of partial differential equations on adaptive Cartesian meshes.</p>
<p><strong>BayPass:</strong></p>
<p>Genome-Wide Scan for Adaptive Differentiation and Association Analysis with population-specific covariables</p>
<p><strong>BayeScan:</strong></p>
<p>Identify candidate loci under natural selection from genetic data,</p>
<p><strong>BayesAss:</strong></p>
<p>Program for inference of recent immigration rates between populations using unlinked multilocus genotypes</p>
<p><strong>Bazel:</strong></p>
<p>Bazel is a build tool that builds code quickly and reliably.</p>
<p><strong>Beagle:</strong></p>
<p>Package for phasing genotypes and for imputing ungenotyped markers.</p>
<p><strong>BerkeleyGW:</strong></p>
<p>The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle</p>
<p><strong>Bifrost:</strong></p>
<p>Highly parallel construction, indexing and querying of colored and compacted de Bruijn graphs.</p>
<p><strong>Bio-DB-BigFile:</strong></p>
<p>Read BigWig and BigBed genome feature databases</p>
<p><strong>Bio-DB-HTS:</strong></p>
<p>Read files using HTSlib including BAM/CRAM, Tabix and BCF database files</p>
<p><strong>BioConductor:</strong></p>
<p>Tools for the analysis of high-throughput genomic data in R.</p>
<p><strong>Bismark:</strong></p>
<p>A tool to map bisulfite converted sequence reads and</p>
<p><strong>Bison:</strong></p>
<p>Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.</p>
<p><strong>BlenderPy:</strong></p>
<p>Blender provides a pipeline for 3D modeling, rigging, animation, simulation, rendering,</p>
<p><strong>Boost:</strong></p>
<p>Boost provides free peer-reviewed portable C++ source libraries.</p>
<p><strong>Bowtie:</strong></p>
<p>Ultrafast, memory-efficient short read aligner.</p>
<p><strong>Bowtie2:</strong></p>
<p>Ultrafast and memory-efficient tool for aligning</p>
<p><strong>Bpipe:</strong></p>
<p>A platform for running big bioinformatics jobs that consist of a series of processing stages</p>
<p><strong>Bracken:</strong></p>
<p>Hghly accurate statistical method that computes the abundance of</p>
<p><strong>BreakDancer:</strong></p>
<p>Genome-wide detection of structural variants from next generation paired-end sequencing reads.</p>
<p><strong>BreakSeq2:</strong></p>
<p>Nucleotide-resolution analysis of structural variants</p>
<p><strong>CCL:</strong></p>
<p>Clozure CL (often called CCL for short) is a free Common Lisp implementation</p>
<p><strong>CD-HIT:</strong></p>
<p>CD-HIT is a very widely used program for clustering and</p>
<p><strong>CDO:</strong></p>
<p>CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.</p>
<p><strong>CFITSIO:</strong></p>
<p>CFITSIO is a library of C and Fortran subroutines for reading and writing data files in</p>
<p><strong>CGAL:</strong></p>
<p>The goal of the CGAL Open Source Project is to provide easy access to efficient</p>
<p><strong>CMake:</strong></p>
<p>CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.</p>
<p><strong>CNVnator:</strong></p>
<p>Copy Number Variation discovery and genotyping from depth of read mapping.</p>
<p><strong>CNVpytor:</strong></p>
<p>Python package and command line tool for CNV/CNA analysis from depth-of-coverage by mapped read</p>
<p><strong>COMSOL:</strong></p>
<p>COMSOL is a multiphysics solver that provides a unified workflow for electrical, mechanical, fluid, and chemical applications.</p>
<p><strong>CONCOCT:</strong></p>
<p>Program for unsupervised binning of metagenomic contigs by using nucleotide composition,</p>
<p><strong>CP2K:</strong></p>
<p>CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular</p>
<p><strong>CPU:</strong></p>
<p>Slectronic circuitry executes instructions of a computer program.</p>
<p><strong>CRAMINO:</strong></p>
<p>A tool for quick quality assessment of cram and bam files, intended for long read sequencing</p>
<p><strong>CTPL:</strong></p>
<p>C++ Thread Pool Library</p>
<p><strong>CUDA:</strong></p>
<p>CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.</p>
<p><strong>CUnit:</strong></p>
<p>Automated testing framework for C.</p>
<p><strong>Canu:</strong></p>
<p>Sequence assembler designed for high-noise single-molecule sequencing.</p>
<p><strong>CapnProto:</strong></p>
<p>Fast data interchange format and capability-based RPC system.</p>
<p><strong>Catch2:</strong></p>
<p>A modern, C++-native, header-only,  test framework for unit-tests, TDD and BDD</p>
<p><strong>CellRanger:</strong></p>
<p>Cell Ranger is a set of analysis pipelines that process Chromium</p>
<p><strong>Centrifuge:</strong></p>
<p>Classifier for metagenomic sequences</p>
<p><strong>Cereal:</strong></p>
<p>C++11 serialization library</p>
<p><strong>Charm++:</strong></p>
<p>An asynchronous message-driven C++ communication library targeted towards tightly coupled, high-performance parallel machines.</p>
<p><strong>CheckM:</strong></p>
<p>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,</p>
<p><strong>CheckM2:</strong></p>
<p>Rapid assessment of genome bin quality using machine learning</p>
<p><strong>CheckV:</strong></p>
<p>Assess the quality of metagenome-assembled viral genomes.</p>
<p><strong>Circlator:</strong></p>
<p>A tool to circularize genome assemblies</p>
<p><strong>Circos:</strong></p>
<p>Package for visualizing data in a circular layout - this makes Circos ideal for exploring</p>
<p><strong>Clair3:</strong></p>
<p>Syumphonizing pileup and full-alignment for high-performance long-read variant calling.</p>
<p><strong>Clustal-Omega:</strong></p>
<p>Clustal Omega is a multiple sequence alignment</p>
<p><strong>ClustalW2:</strong></p>
<p>ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.</p>
<p><strong>Corset:</strong></p>
<p>Clusters contigs and counts reads from de novo assembled transcriptomes.</p>
<p><strong>CoverM:</strong></p>
<p>DNA read coverage and relative abundance calculator focused on metagenomics applications</p>
<p><strong>CppUnit:</strong></p>
<p>C++ port of the JUnit framework for unit testing.</p>
<p><strong>CrayCCE:</strong></p>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: February 2023).</p>
<p><strong>CrayGNU:</strong></p>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: 23.02).</p>
<p><strong>CrayIntel:</strong></p>
<p>Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: February 2023 with Intel 19 compiler).</p>
<p><strong>CubeGUI:</strong></p>
<p>Graphical report explorer report explorer for Scalasca and Score-P</p>
<p><strong>CubeLib:</strong></p>
<p>Cube general purpose C++ library component and command-line tools.</p>
<p><strong>CubeWriter:</strong></p>
<p>Cube high-performance C writer library component.</p>
<p><strong>Cufflinks:</strong></p>
<p>Transcript assembly, differential expression, and differential regulation for RNA-Seq</p>
<p><strong>Cytoscape:</strong></p>
<p>Cytoscape is an open source software platform for visualizing molecular interaction networks and</p>
<p><strong>D-Genies:</strong></p>
<p>D-Genies also allows to display dot plots from other aligners by uploading their PAF or MAF alignment file.</p>
<p><strong>DAS_Tool:</strong></p>
<p>DAS Tool is an automated method that integrates the results of a flexible number of binning</p>
<p><strong>DB:</strong></p>
<p>Berkeley DB enables the development of custom data management solutions,</p>
<p><strong>DBG2OLC:</strong></p>
<p>DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation</p>
<p><strong>DIAMOND:</strong></p>
<p>Sequence aligner for protein and translated DNA searches</p>
<p><strong>DISCOVARdenovo:</strong></p>
<p>Assembler suitable for large genomes based on Illumina reads of length 250 or longer.</p>
<p><strong>DRAM:</strong></p>
<p>Tool for annotating metagenomic assembled genomes and VirSorter identified viral contigs..</p>
<p><strong>DaliLite:</strong></p>
<p>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,</p>
<p><strong>DeconSeq:</strong></p>
<p>A tool that can be used to automatically detect and efficiently remove sequence contaminations</p>
<p><strong>DeepLabCut:</strong></p>
<p>Efficient method for 3D markerless pose estimation based on transfer learning with deep neural networks.</p>
<p><strong>DefaultModules:</strong></p>
<p>Defines the set of modules loaded by default</p>
<p><strong>Delft3D:</strong></p>
<p>Integrated simulation of sediment transport and morphology, waves, water quality and ecology.</p>
<p><strong>Delft3D_FM:</strong></p>
<p>3D modeling suite to investigate hydrodynamics, sediment transport and morphology and water quality for fluvial, estuarine and coastal environments</p>
<p><strong>Delly:</strong></p>
<p>Structural variant discovery by integrated paired-end and split-read analysis</p>
<p><strong>Dorado:</strong></p>
<p>High-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.</p>
<p><strong>Dsuite:</strong></p>
<p>Fast calculation of the ABBA-BABA statistics across many populations/species</p>
<p><strong>EDTA:</strong></p>
<p>Automated whole-genome de-novo TE annotation and benchmarking the annotation performance of TE libraries.</p>
<p><strong>EIGENSOFT:</strong></p>
<p>The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.</p>
<p><strong>ELPA:</strong></p>
<p>Eigenvalue SoLvers for Petaflop-Applications .</p>
<p><strong>EMAN:</strong></p>
<p>EMAN is a powerful image processing library as well as a complete software suite</p>
<p><strong>EMAN2:</strong></p>
<p>Greyscale scientific image processing suite with a primary focus on processing data from transmission electron microscopes</p>
<p><strong>EMBOSS:</strong></p>
<p>EMBOSS is 'The European Molecular Biology Open Software Suite'.</p>
<p><strong>EMIRGE:</strong></p>
<p>Reconstructs full length ribosomal genes from short read</p>
<p><strong>ENMTML:</strong></p>
<p>R package for integrated construction of Ecological Niche Models.</p>
<p><strong>ESMF:</strong></p>
<p>The Earth System Modeling Framework (ESMF) is software for building and coupling weather,  climate, and related models.</p>
<p><strong>ETE:</strong></p>
<p>A Python framework for the analysis and visualization of phylogenetic trees</p>
<p><strong>EasyBuild:</strong></p>
<p>EasyBuild is a software build and installation framework</p>
<p><strong>Eigen:</strong></p>
<p>Eigen is a C++ template library for linear algebra:</p>
<p><strong>Elmer:</strong></p>
<p>Elmer is an open source multiphysical simulation software mainly developed by</p>
<p><strong>Embree:</strong></p>
<p>Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree's performance-optimized ray tracing kernels.</p>
<p><strong>EnergyPlus:</strong></p>
<p>Energy simulation program used to model energy consumption and water use in buildings.</p>
<p><strong>ErlangOTP:</strong></p>
<p>Erlang is a programming language used to build massively scalable</p>
<p><strong>EukRep-EukCC:</strong></p>
<p>Completeness and contamination estimator for metagenomic assembled microbial eukaryotic genomes.</p>
<p><strong>ExaBayes:</strong></p>
<p>Bayesian tree inference, particularly suitable for large-scale analyses.</p>
<p><strong>ExaML:</strong></p>
<p>Exascale Maximum Likelihood for phylogenetic inference using MPI.</p>
<p><strong>Extrae:</strong></p>
<p>Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches</p>
<p><strong>FALCON:</strong></p>
<p>Falcon: a set of tools for fast aligning long reads for consensus and assembly</p>
<p><strong>FASTX-Toolkit:</strong></p>
<p>Tools for Short-Reads FASTA/FASTQ files preprocessing.</p>
<p><strong>FCM:</strong></p>
<p>FCM Build - A powerful build system for modern Fortran software applications. FCM Version Control - Wrappers to the Subversion version control system, usage conventions and processes for scientific software development.</p>
<p><strong>FDS:</strong></p>
<p>Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.</p>
<p><strong>FFTW:</strong></p>
<p>FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.</p>
<p><strong>FFmpeg:</strong></p>
<p>A complete, cross-platform solution to record, convert and stream audio and video.</p>
<p><strong>FLTK:</strong></p>
<p>FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,</p>
<p><strong>FTGL:</strong></p>
<p>FTGL is a free cross-platform Open Source C++ library that uses Freetype2 to simplify rendering fonts in OpenGL applications. FTGL supports bitmaps, pixmaps, texture maps, outlines, polygon mesh, and extruded polygon rendering modes.</p>
<p><strong>FastANI:</strong></p>
<p>Tool for fast alignment-free computation of</p>
<p><strong>FastME:</strong></p>
<p>FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.</p>
<p><strong>FastQC:</strong></p>
<p>A set of tools (in Java) for working with next generation sequencing data in the BAM format.</p>
<p><strong>FastQ_Screen:</strong></p>
<p>FastQ Screen allows you to screen a library of sequences in FastQ</p>
<p><strong>FastTree:</strong></p>
<p>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide</p>
<p><strong>File-Rename:</strong></p>
<p>A Perl version of the rename utility, with support for regular expressions.</p>
<p><strong>Filtlong:</strong></p>
<p>Tool for filtering long reads by quality.</p>
<p><strong>FimTyper:</strong></p>
<p>Identifies the FimH type in total or partial sequenced isolates of E. coli..</p>
<p><strong>FlexiBLAS:</strong></p>
<p>FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation</p>
<p><strong>Flye:</strong></p>
<p>Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio</p>
<p><strong>FoX:</strong></p>
<p>FoX is an XML library written in Fortran 95.</p>
<p><strong>FragGeneScan:</strong></p>
<p>FragGeneScan is an application for finding (fragmented) genes in short reads.</p>
<p><strong>FreeBayes:</strong></p>
<p>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.</p>
<p><strong>FreeSurfer:</strong></p>
<p>FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.</p>
<p><strong>FriBidi:</strong></p>
<p>Free Implementation of the Unicode Bidirectional Algorithm.</p>
<p><strong>GATK:</strong></p>
<p>The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute</p>
<p><strong>GCC:</strong></p>
<p>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,</p>
<p><strong>GCCcore:</strong></p>
<p>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,</p>
<p><strong>GD:</strong></p>
<p>Interface to Gd Graphics Library</p>
<p><strong>GDAL:</strong></p>
<p>GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful command-line utilities for  data translation and processing.  NOTE: The GDAL IO cache by default uses 5% of total memory. This seems not necessary. This module sets GDAL_CACHEMAX=256 (256MB),   which should have no performance impact. Feel free to change if necessary, using 'export GDAL_CACHEMAX=xxx' (in your job script)   after loading the GDAL module.</p>
<p><strong>GEMMA:</strong></p>
<p>Genome-wide Efficient Mixed Model Association</p>
<p><strong>GEOS:</strong></p>
<p>GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)</p>
<p><strong>GLM:</strong></p>
<p>OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on</p>
<p><strong>GLPK:</strong></p>
<p>GNU Linear Programming Kit is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.</p>
<p><strong>GLib:</strong></p>
<p>GLib is one of the base libraries of the GTK+ project</p>
<p><strong>GMAP-GSNAP:</strong></p>
<p>GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences</p>
<p><strong>GMP:</strong></p>
<p>GMP is a free library for arbitrary precision arithmetic,</p>
<p><strong>GMT:</strong></p>
<p>GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools.</p>
<p><strong>GOLD:</strong></p>
<p>A genetic algorithm for docking flexible ligands into protein binding sites</p>
<p><strong>GObject-Introspection:</strong></p>
<p>GObject introspection is a middleware layer between C libraries</p>
<p><strong>GPFS:</strong></p>
<p>High-performance clustered file system software developed by IBM.</p>
<p><strong>GRADS:</strong></p>
<p>The Grid Analysis and Display System (GrADS) is an interactive desktop tool that is used for easy access, manipulation, and visualization of earth science data.</p>
<p><strong>GRASS:</strong></p>
<p>The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization</p>
<p><strong>GRIDSS:</strong></p>
<p>GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements.</p>
<p><strong>GROMACS:</strong></p>
<p>GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.</p>
<p><strong>GSL:</strong></p>
<p>The GNU Scientific Library (GSL) is a numerical library for C and C++  programmers.  The library provides a wide range of mathematical routines  such as random number generators, special functions and least-squares fitting.</p>
<p><strong>GST-plugins-base:</strong></p>
<p>GStreamer plug-ins and elements.</p>
<p><strong>GStreamer:</strong></p>
<p>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.</p>
<p><strong>GTDB-Tk:</strong></p>
<p>A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.</p>
<p><strong>GTK+:</strong></p>
<p>GTK+ is the primary library used to construct user interfaces in GNOME.</p>
<p><strong>GTS:</strong></p>
<p>GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.</p>
<p><strong>GUI:</strong></p>
<p>A digital interface in which a user interacts with graphical components such as icons, buttons, and menus.</p>
<p><strong>GeneMark-ES:</strong></p>
<p>Eukaryotic gene prediction suite with automatic training</p>
<p><strong>GenomeThreader:</strong></p>
<p>GenomeThreader is a software tool to compute gene structure predictions.</p>
<p><strong>Gerris:</strong></p>
<p>Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow. This module also includes GfsView, a visualisation application for Gerris output.</p>
<p><strong>GetOrganelle:</strong></p>
<p>Toolkit to assemble organelle genome from genomic skimming data.</p>
<p><strong>GlimmerHMM:</strong></p>
<p>Gene finder based on a Generalized Hidden Markov Model.</p>
<p><strong>Go:</strong></p>
<p>An open source programming language</p>
<p><strong>Graphviz:</strong></p>
<p>Graphviz is open source graph visualization software. Graph visualization</p>
<p><strong>Gubbins:</strong></p>
<p>Genealogies Unbiased By recomBinations In Nucleotide Sequences</p>
<p><strong>Guile:</strong></p>
<p>Guile is the GNU Ubiquitous Intelligent Language for Extensions,</p>
<p><strong>HDF:</strong></p>
<p>HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines.</p>
<p><strong>HDF-EOS:</strong></p>
<p>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.</p>
<p><strong>HDF-EOS5:</strong></p>
<p>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.</p>
<p><strong>HDF5:</strong></p>
<p>HDF5 is a unique technology suite that makes possible the management of</p>
<p><strong>HISAT2:</strong></p>
<p>HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads</p>
<p><strong>HMMER:</strong></p>
<p>HMMER is used for searching sequence databases for homologs of protein sequences,</p>
<p><strong>HMMER2:</strong></p>
<p>HMMER is used for searching sequence databases for homologs of protein sequences,</p>
<p><strong>HOPS:</strong></p>
<p>Pipeline which focuses on screening MALT data for the presence of a user-specified list of target species.</p>
<p><strong>HPC:</strong></p>
<p>Like a regular computer, but larger. Primarily used for heating data centers.</p>
<p><strong>HTSeq:</strong></p>
<p>HTSeq is a Python library to facilitate processing and analysis</p>
<p><strong>HTSlib:</strong></p>
<p>A C library for reading/writing high-throughput sequencing data.</p>
<p><strong>HarfBuzz:</strong></p>
<p>HarfBuzz is an OpenType text shaping engine.</p>
<p><strong>HpcGridRunner:</strong></p>
<p>HPC GridRunner is a simple command-line interface to high throughput computing using a variety of different grid computing platforms, including LSF, SGE, SLURM, and PBS.</p>
<p><strong>Humann:</strong></p>
<p>Pipeline for efficiently and accurately determining the coverage and abundance of microbial pathways in a community from metagenomic data.</p>
<p><strong>HybPiper:</strong></p>
<p>Extracting Coding Sequence and Introns for Phylogenetics from High-Throughput Sequencing Reads Using Target Enrichment.</p>
<p><strong>Hypre:</strong></p>
<p>Hypre is a library for solving large, sparse linear systems of equations on massively</p>
<p><strong>ICU:</strong></p>
<p>ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.</p>
<p><strong>IDBA:</strong></p>
<p>Iterative de Bruijn graph assembler for second-generation sequencing reads.</p>
<p><strong>IDBA-UD:</strong></p>
<p>IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads</p>
<p><strong>IDL:</strong></p>
<p>IDL is the trusted scientific programming language used across disciplines to extract meaningful visualizations from complex numerical data.</p>
<p><strong>IGV:</strong></p>
<p>The Integrative Genomics Viewer (IGV) is a high-performance visualization</p>
<p><strong>IMPUTE:</strong></p>
<p>Genotype imputation and haplotype phasing.</p>
<p><strong>IQ-TREE:</strong></p>
<p>Efficient phylogenomic software by maximum likelihood</p>
<p><strong>IRkernel:</strong></p>
<p>R packages for providing R kernel for Jupyter.</p>
<p><strong>ISA-L:</strong></p>
<p>Intelligent Storage Acceleration Library</p>
<p><strong>ImageMagick:</strong></p>
<p>ImageMagick is a software suite to create, edit, compose, or convert bitmap images</p>
<p><strong>Infernal:</strong></p>
<p>Infernal ('INFERence of RNA ALignment') is for searching DNA sequence databases</p>
<p><strong>Inspector:</strong></p>
<p>Intel Inspector XE is an easy to use memory error checker and thread checker for serial</p>
<p><strong>InterProScan:</strong></p>
<p>Sequence analysis application (nucleotide and protein sequences) that combines</p>
<p><strong>JAGS:</strong></p>
<p>Just Another Gibbs Sampler - a program for the statistical analysis of Bayesian hierarchical models by Markov Chain Monte Carlo.</p>
<p><strong>JUnit:</strong></p>
<p>A programmer-oriented testing framework for Java.</p>
<p><strong>JasPer:</strong></p>
<p>The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard.</p>
<p><strong>Java:</strong></p>
<p>Java Platform, Standard Edition (Java SE) lets you develop and deploy  Java applications on desktops and servers.</p>
<p><strong>Jellyfish:</strong></p>
<p>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.</p>
<p><strong>JsonCpp:</strong></p>
<p>JsonCpp is a C++ library that allows manipulating JSON values,</p>
<p><strong>Julia:</strong></p>
<p>A high-level, high-performance dynamic language for technical computing.  This version was compiled from source with USE_INTEL_JITEVENTS=1 to enable profiling with VTune.</p>
<p><strong>JupyterLab:</strong></p>
<p>An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture.</p>
<p><strong>KAT:</strong></p>
<p>The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.</p>
<p><strong>KEALib:</strong></p>
<p>KEALib provides an implementation of the GDAL data model. The format supports raster attribute tables, image pyramids, meta-data and in-built statistics while also handling very large files and compression throughout. Based on the HDF5 standard, it also provides a base from which other formats can be derived and is a good choice for long term data archiving. An independent software library (libkea) provides complete access to the KEA image format and a GDAL driver allowing KEA images to be used from any GDAL supported software.</p>
<p><strong>KMC:</strong></p>
<p>Disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.</p>
<p><strong>Kaiju:</strong></p>
<p>Kaiju is a program for sensitive taxonomic classification of high-throughput</p>
<p><strong>Kent_tools:</strong></p>
<p>Collection of tools used by the UCSC genome browser.</p>
<p><strong>KorfSNAP:</strong></p>
<p>Semi-HMM-based Nucleic Acid Parser</p>
<p><strong>Kraken2:</strong></p>
<p>Taxonomic sequence classifier.</p>
<p><strong>KronaTools:</strong></p>
<p>Krona Tools is a set of scripts to create Krona charts from</p>
<p><strong>KyotoCabinet:</strong></p>
<p>Library of routines for managing a database.</p>
<p><strong>LAME:</strong></p>
<p>LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.</p>
<p><strong>LAMMPS:</strong></p>
<p>LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality.</p>
<p><strong>LAST:</strong></p>
<p>LAST finds similar regions between sequences.</p>
<p><strong>LASTZ:</strong></p>
<p>LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to</p>
<p><strong>LDC:</strong></p>
<p>D programming language compiler</p>
<p><strong>LEfSe:</strong></p>
<p>Determines the features most likely to explain differences between classes by coupling standard tests for statistical significance</p>
<p><strong>LINKS:</strong></p>
<p>Alignment-free scaffolding of genome assembly drafts with long reads</p>
<p><strong>LLVM:</strong></p>
<p>The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  ("LLVM IR"). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.   This build includes the clang C/C++ compiler frontend.</p>
<p><strong>LMDB:</strong></p>
<p>LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance</p>
<p><strong>LSD2:</strong></p>
<p>Least-squares methods to estimate rates and dates from phylogenies</p>
<p><strong>LTR_retriever:</strong></p>
<p>Highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package.</p>
<p><strong>LUMPY:</strong></p>
<p>A probabilistic framework for structural variant discovery.</p>
<p><strong>LZO:</strong></p>
<p>Portable lossless data compression library</p>
<p><strong>LibTIFF:</strong></p>
<p>tiff: Library and tools for reading and writing TIFF data files</p>
<p><strong>Libav:</strong></p>
<p>Libraries for dealing with multimedia formats of all sorts.  Forked from FFmpeg</p>
<p><strong>Libint:</strong></p>
<p>Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.</p>
<p><strong>LittleCMS:</strong></p>
<p>Color management engine.</p>
<p><strong>Loki:</strong></p>
<p>Loki is a C++ library of designs, containing flexible implementations of common design patterns and</p>
<p><strong>LongStitch:</strong></p>
<p>A genome assembly correction and scaffolding pipeline using long reads</p>
<p><strong>M4:</strong></p>
<p>GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible</p>
<p><strong>MAFFT:</strong></p>
<p>Multiple sequence alignment program offering a range of methods.</p>
<p><strong>MAGMA:</strong></p>
<p>Tool for gene analysis and generalized gene-set analysis of GWAS data.</p>
<p><strong>MAKER:</strong></p>
<p>Genome annotation pipeline</p>
<p><strong>MATIO:</strong></p>
<p>matio is an C library for reading and writing Matlab MAT files.</p>
<p><strong>MATLAB:</strong></p>
<p>A high-level language and interactive environment for numerical computing.</p>
<p><strong>MCL:</strong></p>
<p>The MCL algorithm is short for the Markov Cluster Algorithm, a fast</p>
<p><strong>MCR:</strong></p>
<p>The Matlab Compiler Runtime is required for running compiled MATLAB executables without MATLAB itself.</p>
<p><strong>MEGAHIT:</strong></p>
<p>An ultra-fast single-node solution for large and complex</p>
<p><strong>METABOLIC:</strong></p>
<p>Metabolic And Biogeochemistry anaLyses In microbes</p>
<p><strong>METIS:</strong></p>
<p>METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,</p>
<p><strong>MMseqs2:</strong></p>
<p>MMseqs2: ultra fast and sensitive search and clustering suite</p>
<p><strong>MOB-suite:</strong></p>
<p>Software tools for clustering, reconstruction and typing of plasmids from draft assemblies.</p>
<p><strong>MODFLOW:</strong></p>
<p>MODFLOW is the U.S. Geological Survey modular finite-difference flow model, which is a computer code that solves the groundwater flow equation. The program is used by hydrogeologists to simulate the flow of groundwater through aquifers.</p>
<p><strong>MPFR:</strong></p>
<p>The MPFR library is a C library for multiple-precision</p>
<p><strong>MPI:</strong></p>
<p>A standardised message-passing standard designed to function on parallel computing architectures.</p>
<p><strong>MSMC:</strong></p>
<p>Multiple Sequentially Markovian Coalescent, infers population size and gene flow from multiple genome sequences</p>
<p><strong>MUMPS:</strong></p>
<p>A parallel sparse direct solver</p>
<p><strong>MUMmer:</strong></p>
<p>MUMmer is a system for rapidly aligning entire genomes,</p>
<p><strong>MUSCLE:</strong></p>
<p>MUSCLE is a program for creating multiple alignments of amino acid or nucleotide</p>
<p><strong>MUST:</strong></p>
<p>MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.</p>
<p><strong>MaSuRCA:</strong></p>
<p>MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph</p>
<p><strong>Magma:</strong></p>
<p>Magma is a large, well-supported software package designed for computations in algebra, number theory, algebraic geometry and algebraic combinatorics. It provides a mathematically rigorous environment     for defining and working with structures such as groups, rings, fields, modules, algebras, schemes, curves, graphs, designs, codes and many others. Magma also supports a number of databases designed     to aid computational research in those areas of mathematics which are algebraic in nature.</p>
<p><strong>MarkerMiner:</strong></p>
<p>Workflow for effective discovery of SCN loci in flowering plants angiosperms</p>
<p><strong>Mash:</strong></p>
<p>Fast genome and metagenome distance estimation using MinHash</p>
<p><strong>Maven:</strong></p>
<p>Binary maven install, Apache Maven is a software project management and comprehension tool. Based on</p>
<p><strong>MaxBin:</strong></p>
<p>MaxBin is software for binning assembled metagenomic sequences based on</p>
<p><strong>Meraculous:</strong></p>
<p>Eukaryotic genome assembler for Illumina sequence data.</p>
<p><strong>Mesa:</strong></p>
<p>Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.   Note that this build enables CPU-based rendering with OpenSWR and LLVM.  The module is intended to be used with visualisation software, such as  ParaView, on nodes where no GPU hardware is available.   Both on-screen and off-screen rendering are supported.  IMPORTANT: The OpenSWR software rasteriser can use multiple threads for            best performance. The number of threads is controlled by the            environment variable KNOB_MAX_WORKER_THREADS. The module sets             KNOB_MAX_WORKER_THREADS=1             by default to avoid accidental oversubscription of nodes.</p>
<p><strong>Meson:</strong></p>
<p>Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.</p>
<p><strong>MetaBAT:</strong></p>
<p>An efficient tool for accurately reconstructing single genomes from complex microbial communities</p>
<p><strong>MetaEuk:</strong></p>
<p>MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic</p>
<p><strong>MetaGeneAnnotator:</strong></p>
<p>MetaGeneAnnotator is a gene-finding program for prokaryote and phage.</p>
<p><strong>MetaPhlAn:</strong></p>
<p>MetaPhlAn is a computational tool for profiling the composition of microbial</p>
<p><strong>MetaPhlAn2:</strong></p>
<p>MetaPhlAn is a computational tool for profiling the composition of microbial</p>
<p><strong>MetaSV:</strong></p>
<p>Structural-variant caller</p>
<p><strong>MetaVelvet:</strong></p>
<p>A short read assember for metagenomics</p>
<p><strong>Metaxa2:</strong></p>
<p>Taxonomic classification of rRNA.</p>
<p><strong>Miniconda3:</strong></p>
<p>A platform for Python-based data analytics</p>
<p><strong>Minimac3:</strong></p>
<p>Low memory and more computationally efficient implementation of the genotype imputation algorithms.</p>
<p><strong>MitoZ:</strong></p>
<p>Toolkit which aims to automatically filter pair-end raw data,</p>
<p><strong>Molcas:</strong></p>
<p>Molcas is an ab initio quantum chemistry software package</p>
<p><strong>Molpro:</strong></p>
<p>Molpro is a complete system of ab initio programs for molecular electronic structure calculations.</p>
<p><strong>Mono:</strong></p>
<p>An open source, cross-platform, implementation of C# and the CLR that is</p>
<p><strong>Monocle3:</strong></p>
<p>An analysis toolkit for single-cell RNA-seq.</p>
<p><strong>Mothur:</strong></p>
<p>Mothur is a single piece of open-source, expandable software</p>
<p><strong>MotionCorr:</strong></p>
<p>Motion Correction for Dose-Fractionation Stack, by Dr. Xueming Li of the Cheng Laboratory at UCSF. The actual executable is named dosefgpu_driftcorr</p>
<p><strong>MrBayes:</strong></p>
<p>MrBayes is a program for the Bayesian estimation of phylogeny.</p>
<p><strong>Mule:</strong></p>
<p>Mule is an API written in Python which allows you to access and manipulate files produced by the UM (Unified Model, of the Met Office (UK)).</p>
<p><strong>MultiQC:</strong></p>
<p>Aggregate results from bioinformatics analyses across many samples into a single</p>
<p><strong>NAMD:</strong></p>
<p>NAMD is a parallel molecular dynamics code designed for high-performance simulation of</p>
<p><strong>NASM:</strong></p>
<p>NASM: General-purpose x86 assembler</p>
<p><strong>NCARG:</strong></p>
<p>NCAR Graphics is a Fortran and C based software package for</p>
<p><strong>NCCL:</strong></p>
<p>The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective</p>
<p><strong>NCL:</strong></p>
<p>NCL is an interpreted language designed specifically for scientific data analysis and   visualization.</p>
<p><strong>NCO:</strong></p>
<p>manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5</p>
<p><strong>NCVIEW:</strong></p>
<p>Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc.</p>
<p><strong>NGS:</strong></p>
<p>NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from</p>
<p><strong>NIWA:</strong></p>
<p>Crown Research Institute, conducts research across a broad range of disciplines in the environmental sciences.</p>
<p><strong>NLopt:</strong></p>
<p>NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms</p>
<p><strong>NSPR:</strong></p>
<p>Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level</p>
<p><strong>NSS:</strong></p>
<p>Network Security Services (NSS) is a set of libraries designed to support cross-platform development</p>
<p><strong>NVHPC:</strong></p>
<p>C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)</p>
<p><strong>NWChem:</strong></p>
<p>NWChem aims to provide its users with computational chemistry tools that are scalable both in</p>
<p><strong>NanoComp:</strong></p>
<p>Comparing runs of Oxford Nanopore sequencing data and alignments</p>
<p><strong>NanoLyse:</strong></p>
<p>Removing reads mapping to the lambda genome.</p>
<p><strong>NanoPlot:</strong></p>
<p>Plotting suite for Oxford Nanopore sequencing data and alignments.</p>
<p><strong>NanoStat:</strong></p>
<p>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.</p>
<p><strong>NeSI:</strong></p>
<p>New Zealand national high performance computing platform.</p>
<p><strong>Newton-X:</strong></p>
<p>NX is a general-purpose program package for simulating the dynamics of electronically excited molecules and molecular assemblies.</p>
<p><strong>NextGenMap:</strong></p>
<p>NextGenMap is a flexible highly sensitive short read mapping tool that</p>
<p><strong>Nextflow:</strong></p>
<p>Nextflow is a reactive workflow framework and a programming DSL</p>
<p><strong>Ninja:</strong></p>
<p>Ninja is a small build system with a focus on speed.</p>
<p><strong>Nsight-Compute:</strong></p>
<p>NVIDIA Nsight Compute is an interactive kernel profiler for CUDA applications. It provides detailed</p>
<p><strong>Nsight-Systems:</strong></p>
<p>NVIDIA Nsight Systems is a system-wide performance analysis tool designed to visualize an</p>
<p><strong>OASIS3-MCT:</strong></p>
<p>The OASIS coupler is a software allowing synchronized exchanges of coupling information between numerical codes representing different components of the climate system.</p>
<p><strong>OBITools:</strong></p>
<p>Manipulate various data and sequence files.</p>
<p><strong>OCI:</strong></p>
<p>Oracle Call Interface (OCI) is the comprehensive, high performance, native C language interface to Oracle Database for custom or packaged applications.  NOTE: This package is only available on Maui Ancil nodes that provide database access.</p>
<p><strong>OMA:</strong></p>
<p>Orthologous MAtrix project is a method and database for the inference</p>
<p><strong>OPARI2:</strong></p>
<p>source-to-source instrumentation tool for OpenMP and hybrid codes.</p>
<p><strong>ORCA:</strong></p>
<p>ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry</p>
<p><strong>OSPRay:</strong></p>
<p>OSPRay features interactive CPU rendering capabilities geared towards Scientific Visualization applications. Advanced shading effects such as Ambient Occlusion, shadows, and transparency can be rendered interactively, enabling new insights into data exploration.</p>
<p><strong>OSU-Micro-Benchmarks:</strong></p>
<p>OSU Micro-Benchmarks for MPI</p>
<p><strong>Octave:</strong></p>
<p>GNU Octave is a high-level interpreted language, primarily intended for numerical computations.</p>
<p><strong>Octopus:</strong></p>
<p>Octopus is a scientific program aimed at the ab initio virtual experimentation</p>
<p><strong>OpenBLAS:</strong></p>
<p>OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.</p>
<p><strong>OpenBabel:</strong></p>
<p>Open Babel is a chemical toolbox designed to speak the many</p>
<p><strong>OpenCMISS:</strong></p>
<p>OpenCMISS is a set of libraries and applications which provide modelling and</p>
<p><strong>OpenCV:</strong></p>
<p>OpenCV (Open Source Computer Vision Library) is an open source computer vision</p>
<p><strong>OpenFAST:</strong></p>
<p>Wind turbine multiphysics simulation tool</p>
<p><strong>OpenFOAM:</strong></p>
<p>OpenFOAM is a free, open source CFD software package.</p>
<p><strong>OpenJPEG:</strong></p>
<p>An open-source JPEG 2000 codec written in C</p>
<p><strong>OpenMPI:</strong></p>
<p>The Open MPI Project is an open source MPI-3 implementation. This version is built with CUDA support enabled.</p>
<p><strong>OpenSSL:</strong></p>
<p>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,</p>
<p><strong>OpenSees:</strong></p>
<p>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.</p>
<p><strong>OpenSeesPy:</strong></p>
<p>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.</p>
<p><strong>OpenSlide:</strong></p>
<p>OpenSlide is a C library that provides a simple interface to</p>
<p><strong>OrfM:</strong></p>
<p>A simple and not slow open reading frame (ORF) caller.</p>
<p><strong>OrthoFiller:</strong></p>
<p>Identifies missing annotations for evolutionarily conserved genes.</p>
<p><strong>OrthoFinder:</strong></p>
<p>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics</p>
<p><strong>OrthoMCL:</strong></p>
<p>Genome-scale algorithm for grouping orthologous protein sequences.</p>
<p><strong>PALEOMIX:</strong></p>
<p>pipelines and tools designed to aid the rapid processing of High-Throughput Sequencing (HTS) data.</p>
<p><strong>PAML:</strong></p>
<p>PAML is a package of programs for phylogenetic</p>
<p><strong>PAPI:</strong></p>
<p>PAPI provides the tool designer and application engineer with a consistent interface and</p>
<p><strong>PBJelly:</strong></p>
<p>PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or</p>
<p><strong>PCRE:</strong></p>
<p>The PCRE library is a set of functions that implement regular expression pattern matching using</p>
<p><strong>PCRE2:</strong></p>
<p>The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5.</p>
<p><strong>PDT:</strong></p>
<p>Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program</p>
<p><strong>PEAR:</strong></p>
<p>Memory-efficient,fully parallelized and highly accurate pair-end read merger.</p>
<p><strong>PEST++:</strong></p>
<p>PEST++ is a software suite aimed at supporting</p>
<p><strong>PETSc:</strong></p>
<p>PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the</p>
<p><strong>PFFT:</strong></p>
<p>PFFT is a software library for computing massively parallel, fast Fourier</p>
<p><strong>PGI:</strong></p>
<p>C, C++ and Fortran compilers from The Portland Group - PGI</p>
<p><strong>PHASIUS:</strong></p>
<p>A tool to visualize phase block structure from (many) BAM or CRAM files together with BED annotation</p>
<p><strong>PLINK:</strong></p>
<p>PLINK is a free, open-source whole genome association analysis toolset,</p>
<p><strong>PLUMED:</strong></p>
<p>PLUMED is an open source library for free energy calculations in molecular systems which</p>
<p><strong>POSIX:</strong></p>
<p>A set of standard operating system interfaces based on the Unix operating system</p>
<p><strong>PRANK:</strong></p>
<p>Probabilistic multiple alignment program for DNA, codon and amino-acid sequences. .</p>
<p><strong>PROJ:</strong></p>
<p>Program proj is a standard Unix filter function which converts  geographic longitude and latitude coordinates into cartesian coordinates</p>
<p><strong>Pango:</strong></p>
<p>Pango is a library for laying out and rendering of text, with an emphasis on internationalization.</p>
<p><strong>ParMETIS:</strong></p>
<p>ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs,</p>
<p><strong>ParaView:</strong></p>
<p>ParaView is a scientific parallel visualizer.  This version supports CPU-only rendering without X context using the OSMesa library, it does not support GPU rendering, and it does not provide a GUI.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.  Ray tracing using the OSPRay library is also supported.</p>
<p><strong>Parallel:</strong></p>
<p>parallel: Build and execute shell commands in parallel</p>
<p><strong>ParallelIO:</strong></p>
<p>A high-level Parallel I/O Library for structured grid applications</p>
<p><strong>Paraver:</strong></p>
<p>Performance visualization and analysis tool based on traces.</p>
<p><strong>Peregrine:</strong></p>
<p>Genome assembler for long reads (length &gt; 10kb, accuracy &gt; 99%).</p>
<p><strong>Perl:</strong></p>
<p>Larry Wall's Practical Extraction and Report Language</p>
<p><strong>PhyML:</strong></p>
<p>Phylogenetic estimation using Maximum Likelihood</p>
<p><strong>PhyloPhlAn:</strong></p>
<p>Integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes.</p>
<p><strong>Pilon:</strong></p>
<p>Pilon is an automated genome assembly improvement and variant detection tool</p>
<p><strong>PnetCDF:</strong></p>
<p>Parallel netCDF: A Parallel I/O Library for NetCDF File Access</p>
<p><strong>Porechop:</strong></p>
<p>Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.</p>
<p><strong>PostgreSQL:</strong></p>
<p>Client-side programs and libraries for accessing PostgreSQL databases.</p>
<p><strong>Prodigal:</strong></p>
<p>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)</p>
<p><strong>ProtHint:</strong></p>
<p>Pipeline for predicting and scoring hints (in the form of introns, start and</p>
<p><strong>Proteinortho:</strong></p>
<p>Proteinortho is a tool to detect orthologous genes within different species.</p>
<p><strong>PyOpenGL:</strong></p>
<p>PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.</p>
<p><strong>PyQt:</strong></p>
<p>PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.</p>
<p><strong>PyTorch:</strong></p>
<p>Tensors and Dynamic neural networks in Python with strong GPU acceleration.</p>
<p><strong>Python:</strong></p>
<p>Python is a programming language that lets you work more quickly and integrate your systems more effectively.</p>
<p><strong>Python-GPU:</strong></p>
<p>The python packages which depend on CUDA: pycuda, pygpu, and scikit-cuda.</p>
<p><strong>Python-Geo:</strong></p>
<p>Python packages for geospatial data I/O, mostly based on the OSGEO libraries GDAL and OGR</p>
<p><strong>QIIME2:</strong></p>
<p>An open-source bioinformatics pipeline for microbiome analysis</p>
<p><strong>QUAST:</strong></p>
<p>Evaluates genome assemblies</p>
<p><strong>Qt5:</strong></p>
<p>Qt is a comprehensive cross-platform C++ application framework.</p>
<p><strong>QuantumESPRESSO:</strong></p>
<p>Quantum ESPRESSO  is an integrated suite of computer codes</p>
<p><strong>QuickTree:</strong></p>
<p>Efficient implementation of the Neighbor-Joining algorithm, capable of reconstructing phylogenies from huge alignments .</p>
<p><strong>R:</strong></p>
<p>R is a free software environment for statistical computing and graphics.</p>
<p><strong>R-Geo:</strong></p>
<p>R packages for Geometric and Geospatial data which depend</p>
<p><strong>R-bundle-Bioconductor:</strong></p>
<p>Bioconductor provides tools for the analysis and comprehension</p>
<p><strong>RANGS-GSHHS:</strong></p>
<p>A binary file set RANGS (Regionally Accessible Nested Global Shorelines) based on GSHHS (Global Self-consistent Hierarchical High-resolution Shorelines) data.  Note: RANGS-GSHHS is used with NCL.</p>
<p><strong>RAxML:</strong></p>
<p>RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.</p>
<p><strong>RAxML-NG:</strong></p>
<p>RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)</p>
<p><strong>RDP-Classifier:</strong></p>
<p>The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic</p>
<p><strong>RECON:</strong></p>
<p>De novo identification and classification of repeat sequence families from genomic sequences</p>
<p><strong>RMBlast:</strong></p>
<p>RMBlast supports RepeatMasker searches by adding a few necessary features to the stock NCBI blastn program. These include:</p>
<p><strong>RNAmmer:</strong></p>
<p>consistent and rapid annotation of ribosomal RNA genes.</p>
<p><strong>ROCm:</strong></p>
<p>Platform for GPU Enabled HPC and UltraScale Computing</p>
<p><strong>ROOT:</strong></p>
<p>The ROOT system provides a set of OO frameworks with all the functionality</p>
<p><strong>RSEM:</strong></p>
<p>Estimates gene and isoform expression levels from RNA-Seq data</p>
<p><strong>RSGISLib:</strong></p>
<p>The Remote Sensing and GIS software library (RSGISLib) is a</p>
<p><strong>Racon:</strong></p>
<p>Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.</p>
<p><strong>Ragout:</strong></p>
<p>Tool for chromosome assembly using multiple references.</p>
<p><strong>RapidNJ:</strong></p>
<p>An algorithmic engineered implementation of canonical neighbour-joining.</p>
<p><strong>Raven:</strong></p>
<p>De novo genome assembler for long uncorrected reads.</p>
<p><strong>Rcorrector:</strong></p>
<p>kmer-based error correction method for RNA-seq data.</p>
<p><strong>Relion:</strong></p>
<p>RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on)</p>
<p><strong>RepeatMasker:</strong></p>
<p>Screens DNA sequences for interspersed repeats and low complexity DNA sequences.</p>
<p><strong>RepeatModeler:</strong></p>
<p>De novo transposable element (TE) family identification and modeling package.</p>
<p><strong>RepeatScout:</strong></p>
<p>De novo identification of repeat families in large genomes</p>
<p><strong>Riskscape:</strong></p>
<p>RiskScape is an open-source spatial data processing application used for multi-hazard risk analysis. RiskScape is highly customisable, letting modellers tailor the risk analysis to suit the problem domain and input data being modelled.</p>
<p><strong>Roary:</strong></p>
<p>Rapid large-scale prokaryote pan genome analysis</p>
<p><strong>Rosetta:</strong></p>
<p>Rosetta is the premier software suite for modeling macromolecular structures. As a flexible,</p>
<p><strong>Rstudio:</strong></p>
<p>RStudio is a set of integrated tools designed to help you be more productive with R.</p>
<p><strong>Ruby:</strong></p>
<p>Ruby is a dynamic, open source programming language with</p>
<p><strong>Rust:</strong></p>
<p>Systems programming language that runs blazingly fast, prevents segfaults,</p>
<p><strong>SAGE:</strong></p>
<p>Ppackage containing programs for use in the genetic analysis of</p>
<p><strong>SAMtools:</strong></p>
<p>Samtools is a suite of programs for interacting with high-throughput sequencing data.</p>
<p><strong>SAS:</strong></p>
<p>SAS is a statistical software suite developed by SAS Institute for data management, advanced analytics, multivariate analysis, business intelligence, criminal investigation, and predictive analytics. - Homepage: https://www.sas.com/en_nz/home.html/</p>
<p><strong>SCOTCH:</strong></p>
<p>Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.</p>
<p><strong>SCP:</strong></p>
<p>Means of securely transferring files between over an SSH connection.</p>
<p><strong>SCons:</strong></p>
<p>SCons is a software construction tool.</p>
<p><strong>SDL2:</strong></p>
<p>Simple DirectMedia Layer, a cross-platform multimedia library</p>
<p><strong>SEPP:</strong></p>
<p>SATe-enabled Phylogenetic Placement. Phylogenetic placement of short reads into reference alignments and trees.</p>
<p><strong>SHAPEIT4:</strong></p>
<p>Estimation of haplotypes (aka phasing)</p>
<p><strong>SIONlib:</strong></p>
<p>Scalable I/O library for parallel access to task-local files.</p>
<p><strong>SIP:</strong></p>
<p>SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.</p>
<p><strong>SKESA:</strong></p>
<p>SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.</p>
<p><strong>SMRT-Link:</strong></p>
<p>PacBios open-source software suite is designed for use with Single Molecule,</p>
<p><strong>SNVoter-NanoMethPhase:</strong></p>
<p>SNVoter - A top up tool to enhance SNV calling from Nanopore sequencing data &amp;</p>
<p><strong>SOAPdenovo2:</strong></p>
<p>Short Oligonucleotide Analysis Package - novel short-read assembly</p>
<p><strong>SOCI:</strong></p>
<p>Database access library for C++ that makes the illusion of embedding SQL queries in the</p>
<p><strong>SPAdes:</strong></p>
<p>Genome assembler for single-cell and isolates data sets</p>
<p><strong>SPECFEM3D:</strong></p>
<p>SPECFEM3D Cartesian simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic, poroelastic or seismic wave propagation in any type of conforming mesh of hexahedra (structured or not.) It can, for instance, model seismic waves propagating in sedimentary basins or any other regional geological model following earthquakes. It can also be used for non-destructive testing or for ocean acoustics.</p>
<p><strong>SPIDER:</strong></p>
<p>System for Processing Image Data from Electron microscopy and Related fields</p>
<p><strong>SQLite:</strong></p>
<p>SQLite: SQL Database Engine in a C Library</p>
<p><strong>SQLplus:</strong></p>
<p>SQL*Plus is an interactive and batch query tool that is installed with every Oracle Database installation. It has a command-line user interface and a Windows Graphical User Interface (GUI).</p>
<p><strong>SSAHA2:</strong></p>
<p>Pairwise sequence alignment program designed for the efficient mapping of sequencing</p>
<p><strong>SSH:</strong></p>
<p>A network communication protocol that enables two computers to communicate</p>
<p><strong>STAR:</strong></p>
<p>Fast universal RNA-seq aligner</p>
<p><strong>STAR-Fusion:</strong></p>
<p>Processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set</p>
<p><strong>SUNDIALS:</strong></p>
<p>SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers</p>
<p><strong>SURVIVOR:</strong></p>
<p>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,</p>
<p><strong>SWIG:</strong></p>
<p>SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.</p>
<p><strong>Salmon:</strong></p>
<p>Salmon is a wicked-fast program to produce a highly-accurate,</p>
<p><strong>Sambamba:</strong></p>
<p>Tools for working with SAM/BAM data</p>
<p><strong>ScaLAPACK:</strong></p>
<p>The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines</p>
<p><strong>Scalasca:</strong></p>
<p>Tool that supports the performance optimization of</p>
<p><strong>Score-P:</strong></p>
<p>Measurement infrastructure is a highly scalable and easy-to-use</p>
<p><strong>SeqAn3:</strong></p>
<p>C++ library of efficient algorithms and data structures for the</p>
<p><strong>SeqKit:</strong></p>
<p>Ultrafast toolkit for FASTA/Q file manipulation</p>
<p><strong>SeqMonk:</strong></p>
<p>A tool to visualise and analyse high throughput mapped sequence data.</p>
<p><strong>SiBELia:</strong></p>
<p>A comparative genomics tool for analysing genomic variations that correlate with pathogens, or</p>
<p><strong>Siesta:</strong></p>
<p>SIESTA is both a method and its computer program implementation, to perform efficient electronic</p>
<p><strong>SignalP:</strong></p>
<p>SignalP predicts the presence  and  location of signal peptide  cleavage sites</p>
<p><strong>Sniffles:</strong></p>
<p>A fast structural variant caller for long-read sequencing.</p>
<p><strong>SortMeRNA:</strong></p>
<p>SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</p>
<p><strong>SourceTracker:</strong></p>
<p>SourceTracker is a Bayesian approach to estimating the proportion of a novel community that comes</p>
<p><strong>Spack:</strong></p>
<p>Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific</p>
<p><strong>Spark:</strong></p>
<p>Spark is Hadoop MapReduce done in memory</p>
<p><strong>Spectrum Scale:</strong></p>
<p>High-performance clustered file system software developed by IBM.</p>
<p><strong>SqueezeMeta:</strong></p>
<p>fully automated metagenomics pipeline, from reads to bins.</p>
<p><strong>Stacks:</strong></p>
<p>Stacks is a software pipeline for building loci from short-read sequences, such as those generated on</p>
<p><strong>StringTie:</strong></p>
<p>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.</p>
<p><strong>Structure:</strong></p>
<p>The program structure is a free software package for using multi-locus genotype data to investigate</p>
<p><strong>Subread:</strong></p>
<p>High performance read alignment, quantification and mutation discovery</p>
<p><strong>Subversion:</strong></p>
<p>Subversion is an open source version control system.</p>
<p><strong>SuiteSparse:</strong></p>
<p>SuiteSparse is a collection of libraries manipulate sparse matrices.</p>
<p><strong>SuperLU:</strong></p>
<p>Solution of large, sparse, nonsymmetric systems of linear equations.</p>
<p><strong>Supernova:</strong></p>
<p>Supernova is a software package for de novo assembly from Chromium Linked-Reads</p>
<p><strong>TMHMM:</strong></p>
<p>Prediction of transmembrane helices in proteins</p>
<p><strong>TSEBRA:</strong></p>
<p>Transcript Selector for BRAKER</p>
<p><strong>TURBOMOLE:</strong></p>
<p>Program Package For Electronic Structure Calculations.</p>
<p><strong>TWL-NINJA:</strong></p>
<p>Nearly Infinite Neighbor Joining Application.</p>
<p><strong>Tcl:</strong></p>
<p>Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language,</p>
<p><strong>TensorFlow:</strong></p>
<p>An open-source software library for Machine Intelligence</p>
<p><strong>TensorRT:</strong></p>
<p>NVIDIA TensorRT is a platform for high-performance deep learning inference</p>
<p><strong>Theano:</strong></p>
<p>Theano is a Python library that allows you to define, optimize,</p>
<p><strong>Tk:</strong></p>
<p>Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for</p>
<p><strong>TopHat:</strong></p>
<p>TopHat is a fast splice junction mapper for RNA-Seq reads.</p>
<p><strong>TransDecoder:</strong></p>
<p>TransDecoder identifies candidate coding regions within transcript sequences.</p>
<p><strong>TreeMix:</strong></p>
<p>TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a</p>
<p><strong>Trilinos:</strong></p>
<p>The Trilinos Project is an effort to develop algorithms and enabling technologies</p>
<p><strong>TrimGalore:</strong></p>
<p>A wrapper of FastQC and cutadapt to automate quality and adapter trimming</p>
<p><strong>Trimmomatic:</strong></p>
<p>Trimmomatic performs a variety of useful trimming tasks for illumina</p>
<p><strong>Trinity:</strong></p>
<p>Trinity represents a novel method for the efficient and robust de novo reconstruction</p>
<p><strong>Trinotate:</strong></p>
<p>C++ library of efficient algorithms and data structures for the</p>
<p><strong>Trycycler:</strong></p>
<p>Tool for generating consensus long-read assemblies for bacterial genomes.</p>
<p><strong>TuiView:</strong></p>
<p>TuiView is a lightweight raster GIS with powerful raster attribute table manipulation</p>
<p><strong>TurboVNC:</strong></p>
<p>TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide</p>
<p><strong>UCX:</strong></p>
<p>Unified Communication X</p>
<p><strong>UDUNITS:</strong></p>
<p>UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.</p>
<p><strong>USEARCH:</strong></p>
<p>USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are</p>
<p><strong>Unicycler:</strong></p>
<p>Assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets</p>
<p><strong>VASP:</strong></p>
<p>The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.</p>
<p><strong>VCF-kit:</strong></p>
<p>VCF-kit is a command-line based collection of utilities for performing analysis on</p>
<p><strong>VCFtools:</strong></p>
<p>The aim of VCFtools is to provide</p>
<p><strong>VEP:</strong></p>
<p>Variant Effect Predictor (VEP) determines the effect of your</p>
<p><strong>VESTA:</strong></p>
<p>VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies. Run 'VESTA-gui' to launch.</p>
<p><strong>VIBRANT:</strong></p>
<p>Virus Identification By iteRative ANnoTation</p>
<p><strong>VMD:</strong></p>
<p>VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular</p>
<p><strong>VPN:</strong></p>
<p>Method of extending access to a private network.</p>
<p><strong>VSEARCH:</strong></p>
<p>An open source alternative to the metagenomics tool USEARCH.</p>
<p><strong>VTK:</strong></p>
<p>The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.</p>
<p><strong>VTune:</strong></p>
<p>Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.</p>
<p><strong>Valgrind:</strong></p>
<p>Valgrind: Debugging and profiling tools</p>
<p><strong>VarScan:</strong></p>
<p>Variant calling and somatic mutation/CNV detection for next-generation sequencing data</p>
<p><strong>Velvet:</strong></p>
<p>Sequence assembler for very short reads</p>
<p><strong>VelvetOptimiser:</strong></p>
<p>Perl script for optimising the three primary parameter options of the Velvet de novo sequence assembler.</p>
<p><strong>ViennaRNA:</strong></p>
<p>The Vienna RNA Package consists of a C code library and several</p>
<p><strong>Vim:</strong></p>
<p>Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set.</p>
<p><strong>VirHostMatcher:</strong></p>
<p>Tools for computing various oligonucleotide frequency (ONF) based distance/dissimialrity measures.</p>
<p><strong>VirSorter:</strong></p>
<p>VirSorter: mining viral signal from microbial genomic data.</p>
<p><strong>VirtualGL:</strong></p>
<p>VirtualGL is an open source toolkit that gives any Linux or</p>
<p><strong>VisIt:</strong></p>
<p>VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool.  This version supports interactive CPU-only rendering with the VisIt GUI using the Mesa library. It does not support GPU rendering.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.</p>
<p><strong>WAAFLE:</strong></p>
<p>Workflow to Annotate Assemblies and Find LGT Events.</p>
<p><strong>WhatsHap:</strong></p>
<p>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.</p>
<p><strong>Wise2:</strong></p>
<p>Aligning proteins or protein HMMs to DNA</p>
<p><strong>XCONV:</strong></p>
<p>Xconv is a program designed to convert model output into a format suitable for use in various plotting packages. Xconv is designed to be simple to use with a point and click, windows based interface.</p>
<p><strong>XGKS:</strong></p>
<p>XGKS is a level 2C implementation of the ANSI Graphical Kernel System</p>
<p><strong>XHMM:</strong></p>
<p>Calls copy number variation (CNV) from normalized read-depth data from exome capture or other targeted sequencing experiments.</p>
<p><strong>XIOS:</strong></p>
<p>XIOS stands for XML-IO-Server and is a library dedicated to I/O management in climate codes. This version uses netCDF4 with the parallel HDF5 library, and it contains a patch for various problems in the mesh connectivity algorithm</p>
<p><strong>XMDS2:</strong></p>
<p>Fast integrator of stochastic partial differential equations.</p>
<p><strong>XSD:</strong></p>
<p>CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.</p>
<p><strong>XVFB:</strong></p>
<p>A display server implementing the X11 display server protocol, XVFB performs all graphical operations in virtual memory without showing any screen output.
This allows applications that 'require' a GUI to run in a command line environment. Can be invoked with <code>xvfb-run</code>.</p>
<p><strong>XZ:</strong></p>
<p>xz: XZ utilities</p>
<p><strong>Xerces-C++:</strong></p>
<p>Xerces-C++ is a validating XML parser written in a portable</p>
<p><strong>YAXT:</strong></p>
<p>Yet Another eXchange Tool - Library that performs halo exchange with MPI for domain decomposed simulations.</p>
<p><strong>Yade:</strong></p>
<p>Yade is an extensible open-source framework for discrete numerical models,</p>
<p><strong>Yasm:</strong></p>
<p>Yasm: Complete rewrite of the NASM assembler with BSD license</p>
<p><strong>ZeroMQ:</strong></p>
<p>ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.</p>
<p><strong>Zip:</strong></p>
<p>Zip is a compression and file packaging/archive utility.</p>
<p><strong>Zonation:</strong></p>
<p>Spatial conservation prioritisation framework for large-scale conservation planning.</p>
<p><strong>angsd:</strong></p>
<p>Program for analysing NGS data.</p>
<p><strong>ant:</strong></p>
<p>Apache Ant is a Java library and command-line tool whose mission is to drive processes described in</p>
<p><strong>antiSMASH:</strong></p>
<p>antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary</p>
<p><strong>any2fasta:</strong></p>
<p>Convert various sequence formats to FASTA</p>
<p><strong>argtable:</strong></p>
<p>Argtable is an ANSI C library for parsing GNU style</p>
<p><strong>aria2:</strong></p>
<p>aria2 is a lightweight multi-protocol &amp; multi-source command-line download utility.</p>
<p><strong>arpack-ng:</strong></p>
<p>ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.</p>
<p><strong>at-spi2-atk:</strong></p>
<p>AT-SPI 2 toolkit bridge</p>
<p><strong>attr:</strong></p>
<p>Commands for Manipulating Filesystem Extended Attributes</p>
<p><strong>azul-zulu:</strong></p>
<p>Java Development Kit (JDK), and a compliant implementation of the Java Standard Edition (SE) specification.</p>
<p><strong>bamUtil:</strong></p>
<p>Repository that contains several programs</p>
<p><strong>barrnap:</strong></p>
<p>Barrnap predicts the location of ribosomal RNA genes in genomes.</p>
<p><strong>bcl2fastq2:</strong></p>
<p>bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by</p>
<p><strong>binutils:</strong></p>
<p>binutils: GNU binary utilities</p>
<p><strong>bioawk:</strong></p>
<p>An extension to awk, adding the support of several common biological data formats</p>
<p><strong>blasr_libcpp:</strong></p>
<p>Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for</p>
<p><strong>breseq:</strong></p>
<p>breseq is a computational pipeline for the analysis of short-read re-sequencing data</p>
<p><strong>bsddb3:</strong></p>
<p>bsddb3 is a nearly complete Python binding of the</p>
<p><strong>bzip2:</strong></p>
<p>bzip2 is a freely available, patent free, high-quality data compressor. It typically</p>
<p><strong>c-ares:</strong></p>
<p>c-ares is a C library for asynchronous DNS requests (including name resolves)</p>
<p><strong>cairo:</strong></p>
<p>Cairo is a 2D graphics library with support for multiple output devices.</p>
<p><strong>cdbfasta:</strong></p>
<p>Fasta file indexing and retrival tool</p>
<p><strong>chewBBACA:</strong></p>
<p>A complete suite for gene-by-gene schema creation and strain identification..</p>
<p><strong>chopper:</strong></p>
<p>Rust implementation of NanoFilt+NanoLyse</p>
<p><strong>cromwell:</strong></p>
<p>Workflow Management System geared towards scientific workflows.</p>
<p><strong>ctags:</strong></p>
<p>Ctags generates an index (or tag) file of language objects found in source files that allows these</p>
<p><strong>ctffind:</strong></p>
<p>ctffind is a program for finding CTFs of electron micrographs</p>
<p><strong>cuDNN:</strong></p>
<p>The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for     deep neural networks.</p>
<p><strong>cutadapt:</strong></p>
<p>cutadapt removes adapter sequences</p>
<p><strong>cuteSV:</strong></p>
<p>Fast and scalable long-read-based SV detection</p>
<p><strong>cwltool:</strong></p>
<p>Common Workflow Language tool description reference implementation</p>
<p><strong>cyvcf2:</strong></p>
<p>cython + htslib == fast VCF and BCF processing</p>
<p><strong>dammit:</strong></p>
<p>de novo transcriptome annotator..</p>
<p><strong>deepTools:</strong></p>
<p>deepTools is a suite of python tools particularly developed for the efficient analysis of</p>
<p><strong>devtools:</strong></p>
<p>R functions that simplify and expedite common tasks in package development.</p>
<p><strong>double-conversion:</strong></p>
<p>Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.</p>
<p><strong>drep:</strong></p>
<p>Rapid and accurate comparison and de-replication of microbial genomes</p>
<p><strong>dtcmp:</strong></p>
<p>DTCMP Library provides pre-defined and user-defined</p>
<p><strong>duphold:</strong></p>
<p>uphold your DUP and DEL calls</p>
<p><strong>duplex-tools:</strong></p>
<p>Range of tools to support operations on Duplex Sequencing read pairs.</p>
<p><strong>eDNA:</strong></p>
<p>A suite of tools to conduct metabarcoding analyses targeting any group of organisms. Includes utilities</p>
<p><strong>ecCodes:</strong></p>
<p>ecCodes is a package developed by ECMWF which provides an application programming interface and</p>
<p><strong>eccodes:</strong></p>
<p>ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats:  - WMO FM-92 GRIB edition 1 and edition 2 - WMO FM-94 BUFR edition 3 and edition 4  - WMO GTS abbreviated header (only decoding)  A useful set of command line tools provide quick access to the messages.</p>
<p><strong>ectyper:</strong></p>
<p>Standalone versatile serotyping module for Escherichia coli..</p>
<p><strong>edlib:</strong></p>
<p>Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.</p>
<p><strong>eggnog-mapper:</strong></p>
<p>Tool for fast functional annotation of novel sequences (genes or proteins)</p>
<p><strong>ensmallen:</strong></p>
<p>C++ header-only library for numerical optimization</p>
<p><strong>entrez-direct:</strong></p>
<p>an advanced method for accessing the NCBI's set of interconnected databases</p>
<p><strong>exonerate:</strong></p>
<p>Generic tool for pairwise sequence comparison</p>
<p><strong>expat:</strong></p>
<p>Expat is an XML parser library written in C. It is a stream-oriented parser  in which an application registers handlers for things the parser might find  in the XML document (like start tags)</p>
<p><strong>fastStructure:</strong></p>
<p>fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x.</p>
<p><strong>fastp:</strong></p>
<p>A tool designed to provide fast all-in-one preprocessing for FastQ files.</p>
<p><strong>fastq-tools:</strong></p>
<p>A collection of small and efficient programs for performing some common and</p>
<p><strong>fcGENE:</strong></p>
<p>Format converting tool for genotype Data.</p>
<p><strong>fgbio:</strong></p>
<p>A set of tools to analyze genomic data with a focus on Next Generation Sequencing.</p>
<p><strong>fineRADstructure:</strong></p>
<p>A package for population structure inference from RAD-seq data</p>
<p><strong>flatbuffers:</strong></p>
<p>FlatBuffers: Memory Efficient Serialization Library</p>
<p><strong>flex:</strong></p>
<p>Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text.</p>
<p><strong>fmlrc:</strong></p>
<p>Tool for performing  hybrid correction of long read sequencing</p>
<p><strong>fmt:</strong></p>
<p>Formatting library providing a fast and safe alternative to C stdio and C++ iostreams.</p>
<p><strong>forge:</strong></p>
<p>Arm Forge combines Arm DDT, the leading debugger for time-saving high performance application debugging, and Arm MAP, the trusted performance profiler for invaluable optimization advice.</p>
<p><strong>foss:</strong></p>
<p>GNU Compiler Collection (GCC) based compiler toolchain, including</p>
<p><strong>funcx-endpoint:</strong></p>
<p>funcX is a distributed Function as a Service (FaaS) platform that enables flexible,</p>
<p><strong>fxtract:</strong></p>
<p>Extract sequences from a fastx (fasta or fastq) file given a subsequence.</p>
<p><strong>g2clib:</strong></p>
<p>Library contains GRIB2 encoder/decoder ('C' version).</p>
<p><strong>g2lib:</strong></p>
<p>Library contains GRIB2 encoder/decoder and search/indexing routines.</p>
<p><strong>ga4gh:</strong></p>
<p>A reference implementation of the GA4GH API</p>
<p><strong>geany:</strong></p>
<p>A GTK+ based text editor with with basic features of an integrated development environment.</p>
<p><strong>genometools:</strong></p>
<p>GenomeTools: A Comprehensive Software Library for Efficient Processing of</p>
<p><strong>gettext:</strong></p>
<p>GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation</p>
<p><strong>gffread:</strong></p>
<p>GFF/GTF parsing utility providing format conversions,</p>
<p><strong>giflib:</strong></p>
<p>giflib is a library for reading and writing gif images.</p>
<p><strong>gimkl:</strong></p>
<p>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL</p>
<p><strong>gimpi:</strong></p>
<p>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.</p>
<p><strong>git:</strong></p>
<p>Git is a free and open source distributed version control system designed</p>
<p><strong>globus-automate-client:</strong></p>
<p>Client for the Globus Flows service.</p>
<p><strong>globus-compute-endpoint:</strong></p>
<p>Globus Compute is a distributed Function as a Service (FaaS) platform that enables flexible,</p>
<p><strong>gmsh:</strong></p>
<p>Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor..</p>
<p><strong>gnuplot:</strong></p>
<p>Portable interactive, function plotting utility</p>
<p><strong>gompi:</strong></p>
<p>GNU Compiler Collection (GCC) based compiler toolchain,</p>
<p><strong>google-sparsehash:</strong></p>
<p>An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library</p>
<p><strong>googletest:</strong></p>
<p>Google's C++ test framework</p>
<p><strong>gperf:</strong></p>
<p>Pperfect hash function generator.</p>
<p><strong>grib_api:</strong></p>
<p>The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of command line tools is also provided to give quick access to GRIB messages.  Note that JPEG and Python support have been disabled for this build.</p>
<p><strong>grive2:</strong></p>
<p>Command line tool for Google Drive.</p>
<p><strong>gsort:</strong></p>
<p>Tool to sort genomic files according to a genomefile.</p>
<p><strong>h5pp:</strong></p>
<p>A simple C++17 wrapper for HDF5.</p>
<p><strong>help2man:</strong></p>
<p>help2man produces simple manual pages from the '--help' and '--version' output of other commands.</p>
<p><strong>hifiasm:</strong></p>
<p>Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.</p>
<p><strong>hunspell:</strong></p>
<p>Spell checker and morphological analyzer library and program designed for languages</p>
<p><strong>hypothesis:</strong></p>
<p>Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized</p>
<p><strong>icc:</strong></p>
<p>Intel C and C++ compilers</p>
<p><strong>iccifort:</strong></p>
<p>Intel C, C++ &amp; Fortran compilers</p>
<p><strong>ifort:</strong></p>
<p>Intel Fortran compiler</p>
<p><strong>iimpi:</strong></p>
<p>Intel C/C++ and Fortran compilers, alongside Intel MPI.</p>
<p><strong>imkl:</strong></p>
<p>Intel Math Kernel Library is a library of highly optimized,</p>
<p><strong>imkl-FFTW:</strong></p>
<p>FFTW interfaces using Intel oneAPI Math Kernel Library</p>
<p><strong>impi:</strong></p>
<p>Intel MPI Library, compatible with MPICH ABI</p>
<p><strong>intel:</strong></p>
<p>Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).</p>
<p><strong>intel-compilers:</strong></p>
<p>Intel C, C++ &amp; Fortran compilers (classic and oneAPI)</p>
<p><strong>ipyrad:</strong></p>
<p>ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic</p>
<p><strong>ispc:</strong></p>
<p>Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for 'single program, multiple data' (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.</p>
<p><strong>jbigkit:</strong></p>
<p>JBIG-KIT is a software implementation of the JBIG1 data compression standard</p>
<p><strong>jcvi:</strong></p>
<p>Collection of Python libraries to parse bioinformatics files, or perform computation related to assembly, annotation, and comparative genomics.</p>
<p><strong>jemalloc:</strong></p>
<p>A general purpose malloc(3) implementation that emphasizes fragmentation avoidance and</p>
<p><strong>jq:</strong></p>
<p>Lightweight and flexible command-line JSON processor.</p>
<p><strong>json-c:</strong></p>
<p>JSON-C implements a reference counting object model that allows you to easily construct JSON objects</p>
<p><strong>jvarkit:</strong></p>
<p>Java utilities for Bioinformatics</p>
<p><strong>kalign2:</strong></p>
<p>Kalign is a fast multiple sequence alignment program for biological sequences.</p>
<p><strong>kallisto:</strong></p>
<p>kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally</p>
<p><strong>kineto:</strong></p>
<p>A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters</p>
<p><strong>kma:</strong></p>
<p>KMA is a mapping method designed to map raw reads directly against redundant databases,</p>
<p><strong>libFLAME:</strong></p>
<p>libFLAME is a portable library for dense matrix computations,</p>
<p><strong>libGLU:</strong></p>
<p>The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.</p>
<p><strong>libKML:</strong></p>
<p>Reference implementation of OGC KML 2.2</p>
<p><strong>libcircle:</strong></p>
<p>API for distributing embarrassingly parallel workloads using self-stabilization.</p>
<p><strong>libconfig:</strong></p>
<p>A Library for processing structured configuration files</p>
<p><strong>libdeflate:</strong></p>
<p>Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.</p>
<p><strong>libdrm:</strong></p>
<p>Direct Rendering Manager runtime library.</p>
<p><strong>libdwarf:</strong></p>
<p>The DWARF Debugging Information Format is of interest to programmers working on compilers</p>
<p><strong>libepoxy:</strong></p>
<p>Library for handling OpenGL function pointer management</p>
<p><strong>libevent:</strong></p>
<p>The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.</p>
<p><strong>libffi:</strong></p>
<p>The libffi library provides a portable, high level programming interface to various calling</p>
<p><strong>libgcrypt:</strong></p>
<p>Libgpg-error is a small library that defines common error values for all GnuPG components.</p>
<p><strong>libgd:</strong></p>
<p>GD is an open source code library for the dynamic creation of images by programmers.</p>
<p><strong>libgeotiff:</strong></p>
<p>Library for reading and writing coordinate system information from/to GeoTIFF files</p>
<p><strong>libglvnd:</strong></p>
<p>libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.</p>
<p><strong>libgpg-error:</strong></p>
<p>Libgpg-error is a small library that defines common error values for all GnuPG components.</p>
<p><strong>libgpuarray:</strong></p>
<p>Arrays on GPU device memory, for Theano</p>
<p><strong>libgtextutils:</strong></p>
<p>ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream</p>
<p><strong>libiconv:</strong></p>
<p>Libiconv converts from one character encoding to another through Unicode conversion</p>
<p><strong>libjpeg-turbo:</strong></p>
<p>libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG</p>
<p><strong>libmatheval:</strong></p>
<p>GNU libmatheval is a library (callable from C and Fortran) to parse</p>
<p><strong>libpciaccess:</strong></p>
<p>Generic PCI access library.</p>
<p><strong>libpng:</strong></p>
<p>libpng is the official PNG reference library</p>
<p><strong>libreadline:</strong></p>
<p>The GNU Readline library provides a set of functions for use by applications that</p>
<p><strong>libspatialite:</strong></p>
<p>SpatiaLite is an open source library intended to extend the SQLite core to support</p>
<p><strong>libtool:</strong></p>
<p>GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries</p>
<p><strong>libunistring:</strong></p>
<p>This library provides functions for manipulating Unicode strings and for manipulating C strings</p>
<p><strong>libunwind:</strong></p>
<p>Define a portable and efficient C programming API to determine the call-chain of a program.</p>
<p><strong>libvpx:</strong></p>
<p>The WebM Project is dedicated to developing a high-quality, open video format for the web that's freely available to everyone.</p>
<p><strong>libxc:</strong></p>
<p>Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.</p>
<p><strong>libxslt:</strong></p>
<p>Libxslt is the XSLT C library developed for the GNOME project</p>
<p><strong>libxsmm:</strong></p>
<p>LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications</p>
<p><strong>libzstd:</strong></p>
<p>Fast lossless compression algorithm.</p>
<p><strong>lighttpd:</strong></p>
<p>A web server.</p>
<p><strong>likwid:</strong></p>
<p>Command line tools for Linux to support programmers in developing high</p>
<p><strong>lp_solve:</strong></p>
<p>Mixed Integer Linear Programming (MILP) solver</p>
<p><strong>lwgrp:</strong></p>
<p>The light-weight group library defines data structures and collective operations to</p>
<p><strong>magma:</strong></p>
<p>The MAGMA project aims to develop a dense linear algebra library similar to</p>
<p><strong>makedepend:</strong></p>
<p>The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.</p>
<p><strong>manta:</strong></p>
<p>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.</p>
<p><strong>mapDamage:</strong></p>
<p>tracks and quantifies DNA damage patterns among ancient</p>
<p><strong>meRanTK:</strong></p>
<p>High performance toolkit for complete analysis of methylated RNA data.</p>
<p><strong>medaka:</strong></p>
<p>Medaka is a tool to create a consensus sequence from nanopore sequencing data.</p>
<p><strong>megalodon:</strong></p>
<p>Tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads</p>
<p><strong>metaWRAP:</strong></p>
<p>Flexible pipeline for genome-resolved metagenomic data analysis.</p>
<p><strong>miRDeep2:</strong></p>
<p>Completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs</p>
<p><strong>midl:</strong></p>
<p>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).</p>
<p><strong>mimalloc:</strong></p>
<p>mimalloc is a general purpose allocator with excellent performance characteristics.</p>
<p><strong>miniasm:</strong></p>
<p>Fast OLC-based de novo assembler for noisy long reads.</p>
<p><strong>minieigen:</strong></p>
<p>A small wrapper for core parts of Eigen, c++ library for linear algebra.</p>
<p><strong>minimap2:</strong></p>
<p>Minimap2 is a fast sequence mapping and alignment</p>
<p><strong>mjpegtools:</strong></p>
<p>The mjpeg programs are a set of tools that can do recording of videos and playback, simple cut-and-paste editing and the MPEG compression of audio and video under Linux.</p>
<p><strong>mlpack:</strong></p>
<p>Fast, and flexible C++ machine learning library with bindings to other languages</p>
<p><strong>mo_tidl:</strong></p>
<p>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).</p>
<p><strong>modbam2bed:</strong></p>
<p>A program to aggregate modified base counts stored in a modified-base BAM file to a bedMethyl file.</p>
<p><strong>mpi4py:</strong></p>
<p>This package provides Python bindings for the Message Passing Interface (MPI) standard. It is implemented on top of the MPI-1/2/3 specification and exposes an API which grounds on the standard MPI-2 C++ bindings.</p>
<p><strong>mpifileutils:</strong></p>
<p>MPI-Based File Utilities For Distributed Systems</p>
<p><strong>muParser:</strong></p>
<p>muParser is an extensible high performance math expression</p>
<p><strong>nano:</strong></p>
<p>nano is a text editor.</p>
<p><strong>nanoQC:</strong></p>
<p>Create fastQC-like plots for Oxford Nanopore sequencing data.</p>
<p><strong>nanofilt:</strong></p>
<p>Filtering and trimming of long read sequencing data.</p>
<p><strong>nanoget:</strong></p>
<p>Functions to extract information from Oxford Nanopore sequencing data and alignments</p>
<p><strong>nanomath:</strong></p>
<p>A few simple math function for other Oxford Nanopore processing scripts</p>
<p><strong>nanopolish:</strong></p>
<p>Software package for signal-level analysis of Oxford Nanopore sequencing data.</p>
<p><strong>ncbi-vdb:</strong></p>
<p>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for</p>
<p><strong>nccmp:</strong></p>
<p>nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage). Parallel comparisons are done in local memory without requiring temporary files. Highly recommended for regression testing scientific models or datasets in a test-driven development environment.</p>
<p><strong>ncview:</strong></p>
<p>Visual browser for netCDF format files.</p>
<p><strong>ne:</strong></p>
<p>ne is a free (GPL'd) text editor based on the POSIX standard</p>
<p><strong>netCDF:</strong></p>
<p>NetCDF (network Common Data Form) is a set of software libraries</p>
<p><strong>netCDF-C++:</strong></p>
<p>NetCDF (network Common Data Form) is a set of software libraries</p>
<p><strong>netCDF-C++4:</strong></p>
<p>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.</p>
<p><strong>netCDF-Fortran:</strong></p>
<p>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.</p>
<p><strong>nettle:</strong></p>
<p>Nettle is a cryptographic library that is designed to fit easily</p>
<p><strong>networkx:</strong></p>
<p>NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,</p>
<p><strong>nodejs:</strong></p>
<p>Node.js is a platform built on Chrome's JavaScript runtime  for easily building fast, scalable network applications. Node.js uses an  event-driven, non-blocking I/O model that makes it lightweight and efficient,  perfect for data-intensive real-time applications that run across distributed devices.</p>
<p><strong>nseg:</strong></p>
<p>Used to mask nucleic acid sequences</p>
<p><strong>nsync:</strong></p>
<p>nsync is a C library that exports various synchronization primitives, such as mutexes</p>
<p><strong>nullarbor:</strong></p>
<p>Reads to report pipeline for bacterial isolate NGS data.</p>
<p><strong>ont-guppy-gpu:</strong></p>
<p>Data processing toolkit that contains the Oxford Nanopore Technologies' basecalling algorithms,</p>
<p><strong>padloc:</strong></p>
<p>Prokaryotic Antiviral Defence LOCator</p>
<p><strong>pairtools:</strong></p>
<p>CLI tools to process mapped Hi-C data</p>
<p><strong>panaroo:</strong></p>
<p>A pangenome analysis pipeline.</p>
<p><strong>pandoc:</strong></p>
<p>Almost universal document converter</p>
<p><strong>parallel-fastq-dump:</strong></p>
<p>parallel fastq-dump wrapper</p>
<p><strong>parasail:</strong></p>
<p>parasail is a SIMD C (C99) library containing implementations</p>
<p><strong>patchelf:</strong></p>
<p>PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.</p>
<p><strong>pauvre:</strong></p>
<p>Tools for plotting Oxford Nanopore and other long-read data.</p>
<p><strong>pfunit:</strong></p>
<p>pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.</p>
<p><strong>phyx:</strong></p>
<p>phyx performs phylogenetics analyses on trees and sequences.</p>
<p><strong>picard:</strong></p>
<p>A set of tools (in Java) for working with next generation sequencing data in the BAM format.</p>
<p><strong>pixman:</strong></p>
<p>Pixman is a low-level software library for pixel manipulation, providing features such as image</p>
<p><strong>pplacer:</strong></p>
<p>Places query sequences on a fixed reference phylogenetic tree</p>
<p><strong>preseq:</strong></p>
<p>Software for predicting library complexity and genome coverage in high-throughput sequencing.</p>
<p><strong>prodigal:</strong></p>
<p>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)</p>
<p><strong>prodigal-gv:</strong></p>
<p>A fork of Prodigal meant to improve gene calling</p>
<p><strong>prokka:</strong></p>
<p>Prokka is a software tool for the rapid annotation of prokaryotic genomes.</p>
<p><strong>proovread:</strong></p>
<p>PacBio hybrid error correction through iterative short read consensus</p>
<p><strong>protobuf:</strong></p>
<p>Google Protocol Buffers</p>
<p><strong>protobuf-python:</strong></p>
<p>Python Protocol Buffers runtime library.</p>
<p><strong>psmc:</strong></p>
<p>Infers population size history from a diploid sequence using the PSMC model.</p>
<p><strong>pstoedit:</strong></p>
<p>pstoedit translates PostScript and PDF graphics into other vector formats.</p>
<p><strong>pullseq:</strong></p>
<p>Utility program for extracting sequences from a fasta/fastq file</p>
<p><strong>purge_dups:</strong></p>
<p>purge haplotigs and overlaps in an assembly based on read depth</p>
<p><strong>purge_haplotigs:</strong></p>
<p>Pipeline to help with curating heterozygous diploid genome assemblies</p>
<p><strong>pv:</strong></p>
<p>Monitors the progress of data through a unix pipeline.</p>
<p><strong>pyani:</strong></p>
<p>Whole-genome classification using Average Nucleotide Identity</p>
<p><strong>pycoQC:</strong></p>
<p>Computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.</p>
<p><strong>pymol-open-source:</strong></p>
<p>PyMOL (open source version) molecular visualization system.</p>
<p><strong>pyspoa:</strong></p>
<p>Python bindings to spoa.</p>
<p><strong>qcat:</strong></p>
<p>Command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files</p>
<p><strong>rDock:</strong></p>
<p>rDock is a fast and versatile Open Source docking program that</p>
<p><strong>randfold:</strong></p>
<p>Minimum free energy of folding randomization test software</p>
<p><strong>rasusa:</strong></p>
<p>Randomly subsample sequencing reads to a specified coverage.</p>
<p><strong>razers3:</strong></p>
<p>Tool for mapping millions of short genomic reads onto a reference genome.</p>
<p><strong>re2c:</strong></p>
<p>re2c is a free and open-source lexer generator for C and C++. Its main goal is generating</p>
<p><strong>rnaQUAST:</strong></p>
<p>Tool for evaluating RNA-Seq assemblies using reference genome and gene database</p>
<p><strong>rsync:</strong></p>
<p>rsync is an open source utility that provides fast incremental file transfer.  rsync is freely available under the GNU General Public License</p>
<p><strong>rust-fmlrc:</strong></p>
<p>FM-index Long Read Corrector (Rust implementation)</p>
<p><strong>samblaster:</strong></p>
<p>samblaster is a fast and flexible program for marking duplicates in read-id grouped paired-end SAM files.</p>
<p><strong>samclip:</strong></p>
<p>Filter SAM file for soft and hard clipped alignments.</p>
<p><strong>sbt:</strong></p>
<p>sbt is a build tool for Scala, Java, and more.</p>
<p><strong>seqmagick:</strong></p>
<p>Seqmagick is a utility built in the spirit of imagemagick to expose the</p>
<p><strong>seqtk:</strong></p>
<p>Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.</p>
<p><strong>shumlib:</strong></p>
<p>Shumlib is the collective name for a set of libraries which are used by the UM; the UK Met Office's Unified Model, that may be of use to external tools or applications where identical functionality is desired. The hope of the project is to enable developers to quickly and easily access parts of the UM code that are commonly duplicated elsewhere, at the same time benefiting from any improvements or optimisations that might be made in support of the UM itself.</p>
<p><strong>sismonr:</strong></p>
<p>Simulation of In Silico Multi-Omic Networks R package.</p>
<p><strong>slow5tools:</strong></p>
<p>Toolkit for converting (FAST5 &lt;-&gt; SLOW5), compressing, viewing, indexing</p>
<p><strong>smafa:</strong></p>
<p>Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences</p>
<p><strong>smoove:</strong></p>
<p>simplifies and speeds calling and genotyping SVs for short reads.</p>
<p><strong>snakemake:</strong></p>
<p>The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.</p>
<p><strong>snaphu:</strong></p>
<p>SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping</p>
<p><strong>snappy:</strong></p>
<p>Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.</p>
<p><strong>snp-sites:</strong></p>
<p>Finds SNP sites from a multi-FASTA alignment file.</p>
<p><strong>snpEff:</strong></p>
<p>SnpEff is a variant annotation and effect prediction tool.</p>
<p><strong>somalier:</strong></p>
<p>extract informative sites, evaluate relatedness, and</p>
<p><strong>spaln:</strong></p>
<p>Stand-alone program that maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.</p>
<p><strong>spdlog:</strong></p>
<p>Fast C++ logging library.</p>
<p><strong>spglib:</strong></p>
<p>Spglib is a C library for finding and handling crystal symmetries.</p>
<p><strong>splat:</strong></p>
<p>Splat deploys (builds and installs) the components of heterogenous systems such as cylc suites, described by splat config files, into target directory trees.</p>
<p><strong>spoa:</strong></p>
<p>c++ implementation of the partial order alignment (POA) algorithm</p>
<p><strong>sratoolkit:</strong></p>
<p>The SRA Toolkit, and the source-code SRA System Development</p>
<p><strong>srun-wrapper:</strong></p>
<p>Provides an mpirun script which merely wraps the srun command.</p>
<p><strong>sublime:</strong></p>
<p>Text editor with GUI</p>
<p><strong>supercomputer:</strong></p>
<p>Like a regular computer, but larger. Primarily used for heating data centers.</p>
<p><strong>supercomputing:</strong></p>
<p>Like a regular computer, but larger. Primarily used for heating data centers.</p>
<p><strong>swarm:</strong></p>
<p>A robust and fast clustering method for amplicon-based studies.</p>
<p><strong>swissknife:</strong></p>
<p>Perl module for reading and writing UniProtKB data in plain text format.</p>
<p><strong>tRNAscan-SE:</strong></p>
<p>Transfer RNA detection</p>
<p><strong>tabix:</strong></p>
<p>Generic indexer for TAB-delimited genome position files</p>
<p><strong>tabixpp:</strong></p>
<p>C++ wrapper to tabix indexer</p>
<p><strong>tbb:</strong></p>
<p>Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that</p>
<p><strong>tbl2asn:</strong></p>
<p>Command-line program that automates the creation of</p>
<p><strong>tmux:</strong></p>
<p>tmux is a terminal multiplexer. It lets you switch easily</p>
<p><strong>trf:</strong></p>
<p>Locates tandem repeats in DNA sequences.</p>
<p><strong>trimAl:</strong></p>
<p>Tool for automated alignment trimming in large-scale phylogenetic analyses</p>
<p><strong>um2netcdf:</strong></p>
<p>um2netcdf converts Unified Model output files in fieldsfile format into netCDF format.</p>
<p><strong>unimap:</strong></p>
<p>Fork of minimap2 optimized for assembly-to-reference</p>
<p><strong>unrar:</strong></p>
<p>RAR is a powerful archive manager.</p>
<p><strong>util-linux:</strong></p>
<p>Set of Linux utilities</p>
<p><strong>vcflib:</strong></p>
<p>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.</p>
<p><strong>vg:</strong></p>
<p>variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods</p>
<p><strong>wgsim:</strong></p>
<p>Wgsim is a small tool for simulating sequence reads from a reference genome.</p>
<p><strong>wheel:</strong></p>
<p>A built-package format for Python.</p>
<p><strong>wtdbg:</strong></p>
<p>de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies.</p>
<p><strong>wxWidgets:</strong></p>
<p>widget toolkit and tools library for creating graphical user interfaces for cross-platform applications.</p>
<p><strong>xbitmaps:</strong></p>
<p>provides bitmaps for x</p>
<p><strong>xkbcommon:</strong></p>
<p>keyboard keymap compiler and support library</p>
<p><strong>yacrd:</strong></p>
<p>Chimeric Read Detector for long reads</p>
<p><strong>yajl:</strong></p>
<p>Yet Another JSON Library. Why does the world need another C library for parsing JSON? Good question.</p>
<p><strong>yak:</strong></p>
<p>Yet another k-mer analyzer</p>
<p><strong>yaml-cpp:</strong></p>
<p>YAML parser and emitter in C++</p>
<p><strong>zlib:</strong></p>
<p>zlib is designed to be a free, general-purpose, legally unencumbered -- that is,</p>
<p><strong>zstd:</strong></p>
<p>Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.</p>







  
    
  
  
    
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1-2.1-2M12.5 7v5.2l4 2.4-1 1L11 13V7h1.5M11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2v1.8Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">January 15, 2024</span>
  </span>

    
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Created">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M14.47 15.08 11 13V7h1.5v5.25l3.08 1.83c-.41.28-.79.62-1.11 1m-1.39 4.84c-.36.05-.71.08-1.08.08-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8c0 .37-.03.72-.08 1.08.69.1 1.33.32 1.92.64.1-.56.16-1.13.16-1.72 0-5.5-4.5-10-10-10S2 6.5 2 12s4.47 10 10 10c.59 0 1.16-.06 1.72-.16-.32-.59-.54-1.23-.64-1.92M18 15v3h-3v2h3v3h2v-3h3v-2h-3v-3h-2Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">December 14, 2022</span>
  </span>

    
    
    
  </aside>


  




                



              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
    <div id="new-footer">
        <div id="partners">
          <div id="logos">
            <a href="https://nesi.org.nz" class="nesi-footer-logo" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/LogoNeSI-footer.png"></a>
            <a href="http://www.mbie.govt.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoMBIE.png"></a>
            <a href="https://www.auckland.ac.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoUoA.png"></a>
            <a href="https://www.niwa.co.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoNIWA.png"></a>
            <a href="https://www.otago.ac.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoUoO.png"></a>
            <a href="http://www.landcareresearch.co.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/LogoManaakiWhenuaLandcareResearch.png"></a>
          </div>
        </div>
        <div id="inner">
          <span id="links"><a href="https://www.nesi.org.nz/copyright" target="_blank">Copyright  <span id="year">2023</span>  New Zealand eScience Infrastructure</a> | <a href="https://www.nesi.org.nz//disclaimer" target="_blank">Disclaimer</a> | <a href="https://www.nesi.org.nz//privacy" target="_blank">Privacy</a>
          </span>
        </div>
    </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../../..", "features": ["navigation.indexes", "navigation.top", "navigation.prune", "content.code.copy", "content.action.edit", "navigation.path"], "search": "../../javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../javascripts/bundle.e1c3ead8.min.js"></script>
      
        <script src="../../javascripts/general.js"></script>
      
    
  </body>
</html>