
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="NeSI Support Documentation">
      
      
      
        <link rel="canonical" href="https://github.com/nesi/support-docs-concept/Glossary/">
      
      
        <link rel="prev" href="../FORMAT/">
      
      
        <link rel="next" href="../MACROS/">
      
      
      <link rel="icon" href="../assets/icons/favicon.ico">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.7">
    
    
      
        <title>Glossary - NeSI Support Center</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.f2e4d321.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/neoteroi-mkdocs.css">
    
      <link rel="stylesheet" href="../assets/stylesheets/footer.css">
    
      <link rel="stylesheet" href="../assets/stylesheets/custom_admonations.css">
    
      <link rel="stylesheet" href="../assets/stylesheets/theme.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function n(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],n("js",new Date),n("config","G-TVQR4R1F40"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&n("event","search",{search_term:this.value})}),document$.subscribe(function(){var a=document.forms.feedback;if(void 0!==a)for(var e of a.querySelectorAll("[type=submit]"))e.addEventListener("click",function(e){e.preventDefault();var t=document.location.pathname,e=this.getAttribute("data-md-value");n("event","feedback",{page:t,data:e}),a.firstElementChild.disabled=!0;e=a.querySelector(".md-feedback__note [data-md-value='"+e+"']");e&&(e.hidden=!1)}),a.hidden=!1}),location$.subscribe(function(e){n("config","G-TVQR4R1F40",{page_path:e.pathname})})});var e=document.createElement("script");e.async=!0,e.src="https://www.googletagmanager.com/gtag/js?id=G-TVQR4R1F40",document.getElementById("__analytics").insertAdjacentElement("afterEnd",e)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#cpus" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
<style>
#banner{
    font-size: larger;
    text-align: center;
    background:  var(--md-warning-bg-color);
}
#banner a:link {
    color: blue;
}
#banner a:visited {
    color: purple;
}
#banner a:focus,
#banner a:hover {
    border-bottom: 1px solid;
}
#banner p{
    color: black;
}
</style>

<div id="banner">
    <p>This site is in Beta. All content is a duplication of  
    <a href="https://support.nesi.org.nz/hc/en-gb">offical NeSI support documentation.</a>
    </p>
</div>

      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="NeSI Support Center" class="md-header__button md-logo" aria-label="NeSI Support Center" data-md-component="logo">
      
  <img src="../assets/icons/logo.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            NeSI Support Center
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Glossary
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2a7 7 0 0 0-7 7c0 2.38 1.19 4.47 3 5.74V17a1 1 0 0 0 1 1h6a1 1 0 0 0 1-1v-2.26c1.81-1.27 3-3.36 3-5.74a7 7 0 0 0-7-7M9 21a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1v-1H9v1Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="custom" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2a7 7 0 0 1 7 7c0 2.38-1.19 4.47-3 5.74V17a1 1 0 0 1-1 1H9a1 1 0 0 1-1-1v-2.26C6.19 13.47 5 11.38 5 9a7 7 0 0 1 7-7M9 21v-1h6v1a1 1 0 0 1-1 1h-4a1 1 0 0 1-1-1m3-17a5 5 0 0 0-5 5c0 2.05 1.23 3.81 3 4.58V16h4v-2.42c1.77-.77 3-2.53 3-4.58a5 5 0 0 0-5-5Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/nesi/support-docs-concept" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    


    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="NeSI Support Center" class="md-nav__button md-logo" aria-label="NeSI Support Center" data-md-component="logo">
      
  <img src="../assets/icons/logo.svg" alt="logo">

    </a>
    NeSI Support Center
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/nesi/support-docs-concept" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../Getting_Started/Accounts-Projects_and_Allocations/Creating_a_NeSI_Account_Profile/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Getting Started
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../General/Announcements/Accessing_NeSI_Support_during_the_holiday_break/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    General
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../Scientific_Computing/Supported_Applications/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Scientific Computing
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--pruned md-nav__item--nested">
      
        
  
  
    
  
  
    <a href="../Storage/File_Systems_and_Quotas/NeSI_File_Systems_and_Quotas/" class="md-nav__link">
      
  
  <span class="md-ellipsis">
    Storage
  </span>
  

      
        <span class="md-nav__icon md-icon"></span>
      
    </a>
  

  

      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#cpus" class="md-nav__link">
    <span class="md-ellipsis">
      CPUs:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cpus_1" class="md-nav__link">
    <span class="md-ellipsis">
      CPU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cpu" class="md-nav__link">
    <span class="md-ellipsis">
      CPU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cuda" class="md-nav__link">
    <span class="md-ellipsis">
      CUDA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf" class="md-nav__link">
    <span class="md-ellipsis">
      HDF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hpcs" class="md-nav__link">
    <span class="md-ellipsis">
      HPCs:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hpc" class="md-nav__link">
    <span class="md-ellipsis">
      HPC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpi" class="md-nav__link">
    <span class="md-ellipsis">
      MPI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nesis" class="md-nav__link">
    <span class="md-ellipsis">
      NeSI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nesi" class="md-nav__link">
    <span class="md-ellipsis">
      NeSI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#niwas" class="md-nav__link">
    <span class="md-ellipsis">
      NIWA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#niwa" class="md-nav__link">
    <span class="md-ellipsis">
      NIWA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scp" class="md-nav__link">
    <span class="md-ellipsis">
      SCP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ssh" class="md-nav__link">
    <span class="md-ellipsis">
      SSH:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vpn" class="md-nav__link">
    <span class="md-ellipsis">
      VPN:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supercomputers" class="md-nav__link">
    <span class="md-ellipsis">
      supercomputers:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supercomputer" class="md-nav__link">
    <span class="md-ellipsis">
      supercomputer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supercomputings" class="md-nav__link">
    <span class="md-ellipsis">
      supercomputings:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supercomputing" class="md-nav__link">
    <span class="md-ellipsis">
      supercomputing:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abaquss" class="md-nav__link">
    <span class="md-ellipsis">
      ABAQUS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abaqus" class="md-nav__link">
    <span class="md-ellipsis">
      ABAQUS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abricates" class="md-nav__link">
    <span class="md-ellipsis">
      ABRicate's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abricate" class="md-nav__link">
    <span class="md-ellipsis">
      ABRicate:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abysss" class="md-nav__link">
    <span class="md-ellipsis">
      ABySS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#abyss" class="md-nav__link">
    <span class="md-ellipsis">
      ABySS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#actcs" class="md-nav__link">
    <span class="md-ellipsis">
      ACTC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#actc" class="md-nav__link">
    <span class="md-ellipsis">
      ACTC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#agats" class="md-nav__link">
    <span class="md-ellipsis">
      AGAT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#agat" class="md-nav__link">
    <span class="md-ellipsis">
      AGAT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ages" class="md-nav__link">
    <span class="md-ellipsis">
      AGE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#age" class="md-nav__link">
    <span class="md-ellipsis">
      AGE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amoss" class="md-nav__link">
    <span class="md-ellipsis">
      AMOS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amos" class="md-nav__link">
    <span class="md-ellipsis">
      AMOS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amrfinderpluss" class="md-nav__link">
    <span class="md-ellipsis">
      AMRFinderPlus's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amrfinderplus" class="md-nav__link">
    <span class="md-ellipsis">
      AMRFinderPlus:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anicalculators" class="md-nav__link">
    <span class="md-ellipsis">
      ANIcalculator's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anicalculator" class="md-nav__link">
    <span class="md-ellipsis">
      ANIcalculator:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#annovars" class="md-nav__link">
    <span class="md-ellipsis">
      ANNOVAR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#annovar" class="md-nav__link">
    <span class="md-ellipsis">
      ANNOVAR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antlrs" class="md-nav__link">
    <span class="md-ellipsis">
      ANTLR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antlr" class="md-nav__link">
    <span class="md-ellipsis">
      ANTLR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antss" class="md-nav__link">
    <span class="md-ellipsis">
      ANTS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ants" class="md-nav__link">
    <span class="md-ellipsis">
      ANTS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antss_1" class="md-nav__link">
    <span class="md-ellipsis">
      ANTs's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ants_1" class="md-nav__link">
    <span class="md-ellipsis">
      ANTs:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aoccs" class="md-nav__link">
    <span class="md-ellipsis">
      AOCC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocc" class="md-nav__link">
    <span class="md-ellipsis">
      AOCC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-bliss" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-BLIS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-blis" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-BLIS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-fftws" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-FFTW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-fftw" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-FFTW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-scalapacks" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-ScaLAPACK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aocl-scalapack" class="md-nav__link">
    <span class="md-ellipsis">
      AOCL-ScaLAPACK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aprs" class="md-nav__link">
    <span class="md-ellipsis">
      APR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#apr" class="md-nav__link">
    <span class="md-ellipsis">
      APR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#apr-utils" class="md-nav__link">
    <span class="md-ellipsis">
      APR-util's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#apr-util" class="md-nav__link">
    <span class="md-ellipsis">
      APR-util:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arcsis" class="md-nav__link">
    <span class="md-ellipsis">
      ARCSI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arcsi" class="md-nav__link">
    <span class="md-ellipsis">
      ARCSI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aribas" class="md-nav__link">
    <span class="md-ellipsis">
      ARIBA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ariba" class="md-nav__link">
    <span class="md-ellipsis">
      ARIBA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#atks" class="md-nav__link">
    <span class="md-ellipsis">
      ATK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#atk" class="md-nav__link">
    <span class="md-ellipsis">
      ATK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#augustuss" class="md-nav__link">
    <span class="md-ellipsis">
      AUGUSTUS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#augustus" class="md-nav__link">
    <span class="md-ellipsis">
      AUGUSTUS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#adapterremovals" class="md-nav__link">
    <span class="md-ellipsis">
      AdapterRemoval's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#adapterremoval" class="md-nav__link">
    <span class="md-ellipsis">
      AdapterRemoval:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#advisors" class="md-nav__link">
    <span class="md-ellipsis">
      Advisor's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#advisor" class="md-nav__link">
    <span class="md-ellipsis">
      Advisor:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alphafolds" class="md-nav__link">
    <span class="md-ellipsis">
      AlphaFold's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alphafold" class="md-nav__link">
    <span class="md-ellipsis">
      AlphaFold:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alphafold2dbs" class="md-nav__link">
    <span class="md-ellipsis">
      AlphaFold2DB's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alphafold2db" class="md-nav__link">
    <span class="md-ellipsis">
      AlphaFold2DB:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alwaysintelmkls" class="md-nav__link">
    <span class="md-ellipsis">
      AlwaysIntelMKL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#alwaysintelmkl" class="md-nav__link">
    <span class="md-ellipsis">
      AlwaysIntelMKL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ambers" class="md-nav__link">
    <span class="md-ellipsis">
      Amber's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#amber" class="md-nav__link">
    <span class="md-ellipsis">
      Amber:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anaconda2s" class="md-nav__link">
    <span class="md-ellipsis">
      Anaconda2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anaconda2" class="md-nav__link">
    <span class="md-ellipsis">
      Anaconda2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anaconda3s" class="md-nav__link">
    <span class="md-ellipsis">
      Anaconda3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#anaconda3" class="md-nav__link">
    <span class="md-ellipsis">
      Anaconda3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#apptainers" class="md-nav__link">
    <span class="md-ellipsis">
      Apptainer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#apptainer" class="md-nav__link">
    <span class="md-ellipsis">
      Apptainer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#armadillos" class="md-nav__link">
    <span class="md-ellipsis">
      Armadillo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#armadillo" class="md-nav__link">
    <span class="md-ellipsis">
      Armadillo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arrows" class="md-nav__link">
    <span class="md-ellipsis">
      Arrow's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arrow" class="md-nav__link">
    <span class="md-ellipsis">
      Arrow:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aspera-clis" class="md-nav__link">
    <span class="md-ellipsis">
      Aspera-CLI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aspera-cli" class="md-nav__link">
    <span class="md-ellipsis">
      Aspera-CLI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#augustuss_1" class="md-nav__link">
    <span class="md-ellipsis">
      Augustus's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#augustus_1" class="md-nav__link">
    <span class="md-ellipsis">
      Augustus:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#autodock-gpus" class="md-nav__link">
    <span class="md-ellipsis">
      AutoDock-GPU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#autodock-gpu" class="md-nav__link">
    <span class="md-ellipsis">
      AutoDock-GPU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#autodock_vinas" class="md-nav__link">
    <span class="md-ellipsis">
      AutoDock_Vina's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#autodock_vina" class="md-nav__link">
    <span class="md-ellipsis">
      AutoDock_Vina:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bbmaps" class="md-nav__link">
    <span class="md-ellipsis">
      BBMap's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bbmap" class="md-nav__link">
    <span class="md-ellipsis">
      BBMap:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcftoolss" class="md-nav__link">
    <span class="md-ellipsis">
      BCFtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcftools" class="md-nav__link">
    <span class="md-ellipsis">
      BCFtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcl-converts" class="md-nav__link">
    <span class="md-ellipsis">
      BCL-Convert's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcl-convert" class="md-nav__link">
    <span class="md-ellipsis">
      BCL-Convert:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beasts" class="md-nav__link">
    <span class="md-ellipsis">
      BEAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beast" class="md-nav__link">
    <span class="md-ellipsis">
      BEAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedopss" class="md-nav__link">
    <span class="md-ellipsis">
      BEDOPS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedops" class="md-nav__link">
    <span class="md-ellipsis">
      BEDOPS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      BEDTools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bedtools" class="md-nav__link">
    <span class="md-ellipsis">
      BEDTools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beefs" class="md-nav__link">
    <span class="md-ellipsis">
      BEEF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beef" class="md-nav__link">
    <span class="md-ellipsis">
      BEEF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blasrs" class="md-nav__link">
    <span class="md-ellipsis">
      BLASR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blasr" class="md-nav__link">
    <span class="md-ellipsis">
      BLASR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blasts" class="md-nav__link">
    <span class="md-ellipsis">
      BLAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blast" class="md-nav__link">
    <span class="md-ellipsis">
      BLAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blastdbs" class="md-nav__link">
    <span class="md-ellipsis">
      BLASTDB's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blastdb" class="md-nav__link">
    <span class="md-ellipsis">
      BLASTDB:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blats" class="md-nav__link">
    <span class="md-ellipsis">
      BLAT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blat" class="md-nav__link">
    <span class="md-ellipsis">
      BLAT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bliss" class="md-nav__link">
    <span class="md-ellipsis">
      BLIS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blis" class="md-nav__link">
    <span class="md-ellipsis">
      BLIS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bolt-lmms" class="md-nav__link">
    <span class="md-ellipsis">
      BOLT-LMM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bolt-lmm" class="md-nav__link">
    <span class="md-ellipsis">
      BOLT-LMM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#brakers" class="md-nav__link">
    <span class="md-ellipsis">
      BRAKER's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#braker" class="md-nav__link">
    <span class="md-ellipsis">
      BRAKER:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#buscos" class="md-nav__link">
    <span class="md-ellipsis">
      BUSCO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#busco" class="md-nav__link">
    <span class="md-ellipsis">
      BUSCO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwas" class="md-nav__link">
    <span class="md-ellipsis">
      BWA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bwa" class="md-nav__link">
    <span class="md-ellipsis">
      BWA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bamtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      BamTools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bamtools" class="md-nav__link">
    <span class="md-ellipsis">
      BamTools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bandages" class="md-nav__link">
    <span class="md-ellipsis">
      Bandage's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bandage" class="md-nav__link">
    <span class="md-ellipsis">
      Bandage:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#basilisks" class="md-nav__link">
    <span class="md-ellipsis">
      Basilisk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#basilisk" class="md-nav__link">
    <span class="md-ellipsis">
      Basilisk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#baypasss" class="md-nav__link">
    <span class="md-ellipsis">
      BayPass's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#baypass" class="md-nav__link">
    <span class="md-ellipsis">
      BayPass:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayescans" class="md-nav__link">
    <span class="md-ellipsis">
      BayeScan's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayescan" class="md-nav__link">
    <span class="md-ellipsis">
      BayeScan:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayesasss" class="md-nav__link">
    <span class="md-ellipsis">
      BayesAss's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bayesass" class="md-nav__link">
    <span class="md-ellipsis">
      BayesAss:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bazels" class="md-nav__link">
    <span class="md-ellipsis">
      Bazel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bazel" class="md-nav__link">
    <span class="md-ellipsis">
      Bazel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beagles" class="md-nav__link">
    <span class="md-ellipsis">
      Beagle's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#beagle" class="md-nav__link">
    <span class="md-ellipsis">
      Beagle:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#berkeleygws" class="md-nav__link">
    <span class="md-ellipsis">
      BerkeleyGW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#berkeleygw" class="md-nav__link">
    <span class="md-ellipsis">
      BerkeleyGW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bifrosts" class="md-nav__link">
    <span class="md-ellipsis">
      Bifrost's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bifrost" class="md-nav__link">
    <span class="md-ellipsis">
      Bifrost:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bio-db-bigfiles" class="md-nav__link">
    <span class="md-ellipsis">
      Bio-DB-BigFile's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bio-db-bigfile" class="md-nav__link">
    <span class="md-ellipsis">
      Bio-DB-BigFile:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bio-db-htss" class="md-nav__link">
    <span class="md-ellipsis">
      Bio-DB-HTS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bio-db-hts" class="md-nav__link">
    <span class="md-ellipsis">
      Bio-DB-HTS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bioconductors" class="md-nav__link">
    <span class="md-ellipsis">
      BioConductor's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bioconductor" class="md-nav__link">
    <span class="md-ellipsis">
      BioConductor:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bismarks" class="md-nav__link">
    <span class="md-ellipsis">
      Bismark's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bismark" class="md-nav__link">
    <span class="md-ellipsis">
      Bismark:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bisons" class="md-nav__link">
    <span class="md-ellipsis">
      Bison's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bison" class="md-nav__link">
    <span class="md-ellipsis">
      Bison:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blenderpys" class="md-nav__link">
    <span class="md-ellipsis">
      BlenderPy's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blenderpy" class="md-nav__link">
    <span class="md-ellipsis">
      BlenderPy:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#boosts" class="md-nav__link">
    <span class="md-ellipsis">
      Boost's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#boost" class="md-nav__link">
    <span class="md-ellipsis">
      Boost:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowties" class="md-nav__link">
    <span class="md-ellipsis">
      Bowtie's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie" class="md-nav__link">
    <span class="md-ellipsis">
      Bowtie:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie2s" class="md-nav__link">
    <span class="md-ellipsis">
      Bowtie2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bowtie2" class="md-nav__link">
    <span class="md-ellipsis">
      Bowtie2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bpipes" class="md-nav__link">
    <span class="md-ellipsis">
      Bpipe's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bpipe" class="md-nav__link">
    <span class="md-ellipsis">
      Bpipe:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#brackens" class="md-nav__link">
    <span class="md-ellipsis">
      Bracken's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bracken" class="md-nav__link">
    <span class="md-ellipsis">
      Bracken:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breakdancers" class="md-nav__link">
    <span class="md-ellipsis">
      BreakDancer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breakdancer" class="md-nav__link">
    <span class="md-ellipsis">
      BreakDancer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breakseq2s" class="md-nav__link">
    <span class="md-ellipsis">
      BreakSeq2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breakseq2" class="md-nav__link">
    <span class="md-ellipsis">
      BreakSeq2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ccls" class="md-nav__link">
    <span class="md-ellipsis">
      CCL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ccl" class="md-nav__link">
    <span class="md-ellipsis">
      CCL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cd-hits" class="md-nav__link">
    <span class="md-ellipsis">
      CD-HIT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cd-hit" class="md-nav__link">
    <span class="md-ellipsis">
      CD-HIT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cdos" class="md-nav__link">
    <span class="md-ellipsis">
      CDO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cdo" class="md-nav__link">
    <span class="md-ellipsis">
      CDO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cfitsios" class="md-nav__link">
    <span class="md-ellipsis">
      CFITSIO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cfitsio" class="md-nav__link">
    <span class="md-ellipsis">
      CFITSIO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cgals" class="md-nav__link">
    <span class="md-ellipsis">
      CGAL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cgal" class="md-nav__link">
    <span class="md-ellipsis">
      CGAL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cmakes" class="md-nav__link">
    <span class="md-ellipsis">
      CMake's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cmake" class="md-nav__link">
    <span class="md-ellipsis">
      CMake:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnvnators" class="md-nav__link">
    <span class="md-ellipsis">
      CNVnator's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnvnator" class="md-nav__link">
    <span class="md-ellipsis">
      CNVnator:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnvpytors" class="md-nav__link">
    <span class="md-ellipsis">
      CNVpytor's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cnvpytor" class="md-nav__link">
    <span class="md-ellipsis">
      CNVpytor:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comsols" class="md-nav__link">
    <span class="md-ellipsis">
      COMSOL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#comsol" class="md-nav__link">
    <span class="md-ellipsis">
      COMSOL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#concocts" class="md-nav__link">
    <span class="md-ellipsis">
      CONCOCT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#concoct" class="md-nav__link">
    <span class="md-ellipsis">
      CONCOCT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cp2ks" class="md-nav__link">
    <span class="md-ellipsis">
      CP2K's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cp2k" class="md-nav__link">
    <span class="md-ellipsis">
      CP2K:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#craminos" class="md-nav__link">
    <span class="md-ellipsis">
      CRAMINO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cramino" class="md-nav__link">
    <span class="md-ellipsis">
      CRAMINO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctpls" class="md-nav__link">
    <span class="md-ellipsis">
      CTPL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctpl" class="md-nav__link">
    <span class="md-ellipsis">
      CTPL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cudas" class="md-nav__link">
    <span class="md-ellipsis">
      CUDA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cunits" class="md-nav__link">
    <span class="md-ellipsis">
      CUnit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cunit" class="md-nav__link">
    <span class="md-ellipsis">
      CUnit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#canus" class="md-nav__link">
    <span class="md-ellipsis">
      Canu's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#canu" class="md-nav__link">
    <span class="md-ellipsis">
      Canu:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#capnprotos" class="md-nav__link">
    <span class="md-ellipsis">
      CapnProto's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#capnproto" class="md-nav__link">
    <span class="md-ellipsis">
      CapnProto:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#catch2s" class="md-nav__link">
    <span class="md-ellipsis">
      Catch2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#catch2" class="md-nav__link">
    <span class="md-ellipsis">
      Catch2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cellrangers" class="md-nav__link">
    <span class="md-ellipsis">
      CellRanger's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cellranger" class="md-nav__link">
    <span class="md-ellipsis">
      CellRanger:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#centrifuges" class="md-nav__link">
    <span class="md-ellipsis">
      Centrifuge's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#centrifuge" class="md-nav__link">
    <span class="md-ellipsis">
      Centrifuge:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cereals" class="md-nav__link">
    <span class="md-ellipsis">
      Cereal's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cereal" class="md-nav__link">
    <span class="md-ellipsis">
      Cereal:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#charms" class="md-nav__link">
    <span class="md-ellipsis">
      Charm++'s:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#charm" class="md-nav__link">
    <span class="md-ellipsis">
      Charm++:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkms" class="md-nav__link">
    <span class="md-ellipsis">
      CheckM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkm" class="md-nav__link">
    <span class="md-ellipsis">
      CheckM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkm2s" class="md-nav__link">
    <span class="md-ellipsis">
      CheckM2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkm2" class="md-nav__link">
    <span class="md-ellipsis">
      CheckM2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkvs" class="md-nav__link">
    <span class="md-ellipsis">
      CheckV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#checkv" class="md-nav__link">
    <span class="md-ellipsis">
      CheckV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#circlators" class="md-nav__link">
    <span class="md-ellipsis">
      Circlator's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#circlator" class="md-nav__link">
    <span class="md-ellipsis">
      Circlator:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#circoss" class="md-nav__link">
    <span class="md-ellipsis">
      Circos's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#circos" class="md-nav__link">
    <span class="md-ellipsis">
      Circos:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clair3s" class="md-nav__link">
    <span class="md-ellipsis">
      Clair3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clair3" class="md-nav__link">
    <span class="md-ellipsis">
      Clair3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustal-omegas" class="md-nav__link">
    <span class="md-ellipsis">
      Clustal-Omega's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustal-omega" class="md-nav__link">
    <span class="md-ellipsis">
      Clustal-Omega:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustalw2s" class="md-nav__link">
    <span class="md-ellipsis">
      ClustalW2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#clustalw2" class="md-nav__link">
    <span class="md-ellipsis">
      ClustalW2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#corsets" class="md-nav__link">
    <span class="md-ellipsis">
      Corset's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#corset" class="md-nav__link">
    <span class="md-ellipsis">
      Corset:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#coverms" class="md-nav__link">
    <span class="md-ellipsis">
      CoverM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#coverm" class="md-nav__link">
    <span class="md-ellipsis">
      CoverM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cppunits" class="md-nav__link">
    <span class="md-ellipsis">
      CppUnit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cppunit" class="md-nav__link">
    <span class="md-ellipsis">
      CppUnit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#craycces" class="md-nav__link">
    <span class="md-ellipsis">
      CrayCCE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#craycce" class="md-nav__link">
    <span class="md-ellipsis">
      CrayCCE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#craygnus" class="md-nav__link">
    <span class="md-ellipsis">
      CrayGNU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#craygnu" class="md-nav__link">
    <span class="md-ellipsis">
      CrayGNU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#crayintels" class="md-nav__link">
    <span class="md-ellipsis">
      CrayIntel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#crayintel" class="md-nav__link">
    <span class="md-ellipsis">
      CrayIntel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubeguis" class="md-nav__link">
    <span class="md-ellipsis">
      CubeGUI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubegui" class="md-nav__link">
    <span class="md-ellipsis">
      CubeGUI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubelibs" class="md-nav__link">
    <span class="md-ellipsis">
      CubeLib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubelib" class="md-nav__link">
    <span class="md-ellipsis">
      CubeLib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubewriters" class="md-nav__link">
    <span class="md-ellipsis">
      CubeWriter's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cubewriter" class="md-nav__link">
    <span class="md-ellipsis">
      CubeWriter:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cufflinkss" class="md-nav__link">
    <span class="md-ellipsis">
      Cufflinks's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cufflinks" class="md-nav__link">
    <span class="md-ellipsis">
      Cufflinks:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cytoscapes" class="md-nav__link">
    <span class="md-ellipsis">
      Cytoscape's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cytoscape" class="md-nav__link">
    <span class="md-ellipsis">
      Cytoscape:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#d-geniess" class="md-nav__link">
    <span class="md-ellipsis">
      D-Genies's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#d-genies" class="md-nav__link">
    <span class="md-ellipsis">
      D-Genies:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#das_tools" class="md-nav__link">
    <span class="md-ellipsis">
      DAS_Tool's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#das_tool" class="md-nav__link">
    <span class="md-ellipsis">
      DAS_Tool:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dbs" class="md-nav__link">
    <span class="md-ellipsis">
      DB's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#db" class="md-nav__link">
    <span class="md-ellipsis">
      DB:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dbg2olcs" class="md-nav__link">
    <span class="md-ellipsis">
      DBG2OLC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dbg2olc" class="md-nav__link">
    <span class="md-ellipsis">
      DBG2OLC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#diamonds" class="md-nav__link">
    <span class="md-ellipsis">
      DIAMOND's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#diamond" class="md-nav__link">
    <span class="md-ellipsis">
      DIAMOND:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#discovardenovos" class="md-nav__link">
    <span class="md-ellipsis">
      DISCOVARdenovo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#discovardenovo" class="md-nav__link">
    <span class="md-ellipsis">
      DISCOVARdenovo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#drams" class="md-nav__link">
    <span class="md-ellipsis">
      DRAM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dram" class="md-nav__link">
    <span class="md-ellipsis">
      DRAM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dalilites" class="md-nav__link">
    <span class="md-ellipsis">
      DaliLite's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dalilite" class="md-nav__link">
    <span class="md-ellipsis">
      DaliLite:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deconseqs" class="md-nav__link">
    <span class="md-ellipsis">
      DeconSeq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deconseq" class="md-nav__link">
    <span class="md-ellipsis">
      DeconSeq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deeplabcuts" class="md-nav__link">
    <span class="md-ellipsis">
      DeepLabCut's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deeplabcut" class="md-nav__link">
    <span class="md-ellipsis">
      DeepLabCut:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#defaultmoduless" class="md-nav__link">
    <span class="md-ellipsis">
      DefaultModules's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#defaultmodules" class="md-nav__link">
    <span class="md-ellipsis">
      DefaultModules:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#delft3ds" class="md-nav__link">
    <span class="md-ellipsis">
      Delft3D's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#delft3d" class="md-nav__link">
    <span class="md-ellipsis">
      Delft3D:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#delft3d_fms" class="md-nav__link">
    <span class="md-ellipsis">
      Delft3D_FM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#delft3d_fm" class="md-nav__link">
    <span class="md-ellipsis">
      Delft3D_FM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dellys" class="md-nav__link">
    <span class="md-ellipsis">
      Delly's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#delly" class="md-nav__link">
    <span class="md-ellipsis">
      Delly:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dorados" class="md-nav__link">
    <span class="md-ellipsis">
      Dorado's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dorado" class="md-nav__link">
    <span class="md-ellipsis">
      Dorado:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dsuites" class="md-nav__link">
    <span class="md-ellipsis">
      Dsuite's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dsuite" class="md-nav__link">
    <span class="md-ellipsis">
      Dsuite:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#edtas" class="md-nav__link">
    <span class="md-ellipsis">
      EDTA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#edta" class="md-nav__link">
    <span class="md-ellipsis">
      EDTA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eigensofts" class="md-nav__link">
    <span class="md-ellipsis">
      EIGENSOFT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eigensoft" class="md-nav__link">
    <span class="md-ellipsis">
      EIGENSOFT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#elpas" class="md-nav__link">
    <span class="md-ellipsis">
      ELPA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#elpa" class="md-nav__link">
    <span class="md-ellipsis">
      ELPA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emans" class="md-nav__link">
    <span class="md-ellipsis">
      EMAN's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eman" class="md-nav__link">
    <span class="md-ellipsis">
      EMAN:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eman2s" class="md-nav__link">
    <span class="md-ellipsis">
      EMAN2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eman2" class="md-nav__link">
    <span class="md-ellipsis">
      EMAN2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#embosss" class="md-nav__link">
    <span class="md-ellipsis">
      EMBOSS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emboss" class="md-nav__link">
    <span class="md-ellipsis">
      EMBOSS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emirges" class="md-nav__link">
    <span class="md-ellipsis">
      EMIRGE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#emirge" class="md-nav__link">
    <span class="md-ellipsis">
      EMIRGE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enmtmls" class="md-nav__link">
    <span class="md-ellipsis">
      ENMTML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enmtml" class="md-nav__link">
    <span class="md-ellipsis">
      ENMTML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#esmfs" class="md-nav__link">
    <span class="md-ellipsis">
      ESMF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#esmf" class="md-nav__link">
    <span class="md-ellipsis">
      ESMF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#etes" class="md-nav__link">
    <span class="md-ellipsis">
      ETE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ete" class="md-nav__link">
    <span class="md-ellipsis">
      ETE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#easybuilds" class="md-nav__link">
    <span class="md-ellipsis">
      EasyBuild's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#easybuild" class="md-nav__link">
    <span class="md-ellipsis">
      EasyBuild:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eigens" class="md-nav__link">
    <span class="md-ellipsis">
      Eigen's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eigen" class="md-nav__link">
    <span class="md-ellipsis">
      Eigen:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#elmers" class="md-nav__link">
    <span class="md-ellipsis">
      Elmer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#elmer" class="md-nav__link">
    <span class="md-ellipsis">
      Elmer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#embrees" class="md-nav__link">
    <span class="md-ellipsis">
      Embree's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#embree" class="md-nav__link">
    <span class="md-ellipsis">
      Embree:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#energypluss" class="md-nav__link">
    <span class="md-ellipsis">
      EnergyPlus's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#energyplus" class="md-nav__link">
    <span class="md-ellipsis">
      EnergyPlus:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#erlangotps" class="md-nav__link">
    <span class="md-ellipsis">
      ErlangOTP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#erlangotp" class="md-nav__link">
    <span class="md-ellipsis">
      ErlangOTP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eukrep-eukccs" class="md-nav__link">
    <span class="md-ellipsis">
      EukRep-EukCC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eukrep-eukcc" class="md-nav__link">
    <span class="md-ellipsis">
      EukRep-EukCC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exabayess" class="md-nav__link">
    <span class="md-ellipsis">
      ExaBayes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exabayes" class="md-nav__link">
    <span class="md-ellipsis">
      ExaBayes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#examls" class="md-nav__link">
    <span class="md-ellipsis">
      ExaML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#examl" class="md-nav__link">
    <span class="md-ellipsis">
      ExaML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#extraes" class="md-nav__link">
    <span class="md-ellipsis">
      Extrae's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#extrae" class="md-nav__link">
    <span class="md-ellipsis">
      Extrae:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#falcons" class="md-nav__link">
    <span class="md-ellipsis">
      FALCON's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#falcon" class="md-nav__link">
    <span class="md-ellipsis">
      FALCON:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastx-toolkits" class="md-nav__link">
    <span class="md-ellipsis">
      FASTX-Toolkit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastx-toolkit" class="md-nav__link">
    <span class="md-ellipsis">
      FASTX-Toolkit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fcms" class="md-nav__link">
    <span class="md-ellipsis">
      FCM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fcm" class="md-nav__link">
    <span class="md-ellipsis">
      FCM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fdss" class="md-nav__link">
    <span class="md-ellipsis">
      FDS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fds" class="md-nav__link">
    <span class="md-ellipsis">
      FDS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fftws" class="md-nav__link">
    <span class="md-ellipsis">
      FFTW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fftw" class="md-nav__link">
    <span class="md-ellipsis">
      FFTW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ffmpegs" class="md-nav__link">
    <span class="md-ellipsis">
      FFmpeg's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ffmpeg" class="md-nav__link">
    <span class="md-ellipsis">
      FFmpeg:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fltks" class="md-nav__link">
    <span class="md-ellipsis">
      FLTK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fltk" class="md-nav__link">
    <span class="md-ellipsis">
      FLTK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ftgls" class="md-nav__link">
    <span class="md-ellipsis">
      FTGL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ftgl" class="md-nav__link">
    <span class="md-ellipsis">
      FTGL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastanis" class="md-nav__link">
    <span class="md-ellipsis">
      FastANI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastani" class="md-nav__link">
    <span class="md-ellipsis">
      FastANI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastmes" class="md-nav__link">
    <span class="md-ellipsis">
      FastME's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastme" class="md-nav__link">
    <span class="md-ellipsis">
      FastME:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastqcs" class="md-nav__link">
    <span class="md-ellipsis">
      FastQC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastqc" class="md-nav__link">
    <span class="md-ellipsis">
      FastQC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastq_screens" class="md-nav__link">
    <span class="md-ellipsis">
      FastQ_Screen's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastq_screen" class="md-nav__link">
    <span class="md-ellipsis">
      FastQ_Screen:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fasttrees" class="md-nav__link">
    <span class="md-ellipsis">
      FastTree's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fasttree" class="md-nav__link">
    <span class="md-ellipsis">
      FastTree:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#file-renames" class="md-nav__link">
    <span class="md-ellipsis">
      File-Rename's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#file-rename" class="md-nav__link">
    <span class="md-ellipsis">
      File-Rename:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#filtlongs" class="md-nav__link">
    <span class="md-ellipsis">
      Filtlong's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#filtlong" class="md-nav__link">
    <span class="md-ellipsis">
      Filtlong:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fimtypers" class="md-nav__link">
    <span class="md-ellipsis">
      FimTyper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fimtyper" class="md-nav__link">
    <span class="md-ellipsis">
      FimTyper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flexiblass" class="md-nav__link">
    <span class="md-ellipsis">
      FlexiBLAS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flexiblas" class="md-nav__link">
    <span class="md-ellipsis">
      FlexiBLAS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flyes" class="md-nav__link">
    <span class="md-ellipsis">
      Flye's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flye" class="md-nav__link">
    <span class="md-ellipsis">
      Flye:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#foxs" class="md-nav__link">
    <span class="md-ellipsis">
      FoX's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fox" class="md-nav__link">
    <span class="md-ellipsis">
      FoX:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fraggenescans" class="md-nav__link">
    <span class="md-ellipsis">
      FragGeneScan's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fraggenescan" class="md-nav__link">
    <span class="md-ellipsis">
      FragGeneScan:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freebayess" class="md-nav__link">
    <span class="md-ellipsis">
      FreeBayes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freebayes" class="md-nav__link">
    <span class="md-ellipsis">
      FreeBayes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freesurfers" class="md-nav__link">
    <span class="md-ellipsis">
      FreeSurfer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#freesurfer" class="md-nav__link">
    <span class="md-ellipsis">
      FreeSurfer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fribidis" class="md-nav__link">
    <span class="md-ellipsis">
      FriBidi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fribidi" class="md-nav__link">
    <span class="md-ellipsis">
      FriBidi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gatks" class="md-nav__link">
    <span class="md-ellipsis">
      GATK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gatk" class="md-nav__link">
    <span class="md-ellipsis">
      GATK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gccs" class="md-nav__link">
    <span class="md-ellipsis">
      GCC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gcc" class="md-nav__link">
    <span class="md-ellipsis">
      GCC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gcccores" class="md-nav__link">
    <span class="md-ellipsis">
      GCCcore's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gcccore" class="md-nav__link">
    <span class="md-ellipsis">
      GCCcore:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gds" class="md-nav__link">
    <span class="md-ellipsis">
      GD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gd" class="md-nav__link">
    <span class="md-ellipsis">
      GD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gdals" class="md-nav__link">
    <span class="md-ellipsis">
      GDAL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gdal" class="md-nav__link">
    <span class="md-ellipsis">
      GDAL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gemmas" class="md-nav__link">
    <span class="md-ellipsis">
      GEMMA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gemma" class="md-nav__link">
    <span class="md-ellipsis">
      GEMMA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geoss" class="md-nav__link">
    <span class="md-ellipsis">
      GEOS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geos" class="md-nav__link">
    <span class="md-ellipsis">
      GEOS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glms" class="md-nav__link">
    <span class="md-ellipsis">
      GLM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glm" class="md-nav__link">
    <span class="md-ellipsis">
      GLM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glpks" class="md-nav__link">
    <span class="md-ellipsis">
      GLPK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glpk" class="md-nav__link">
    <span class="md-ellipsis">
      GLPK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glibs" class="md-nav__link">
    <span class="md-ellipsis">
      GLib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glib" class="md-nav__link">
    <span class="md-ellipsis">
      GLib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmap-gsnaps" class="md-nav__link">
    <span class="md-ellipsis">
      GMAP-GSNAP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmap-gsnap" class="md-nav__link">
    <span class="md-ellipsis">
      GMAP-GSNAP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmps" class="md-nav__link">
    <span class="md-ellipsis">
      GMP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmp" class="md-nav__link">
    <span class="md-ellipsis">
      GMP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmts" class="md-nav__link">
    <span class="md-ellipsis">
      GMT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmt" class="md-nav__link">
    <span class="md-ellipsis">
      GMT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#golds" class="md-nav__link">
    <span class="md-ellipsis">
      GOLD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gold" class="md-nav__link">
    <span class="md-ellipsis">
      GOLD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gobject-introspections" class="md-nav__link">
    <span class="md-ellipsis">
      GObject-Introspection's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gobject-introspection" class="md-nav__link">
    <span class="md-ellipsis">
      GObject-Introspection:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gradss" class="md-nav__link">
    <span class="md-ellipsis">
      GRADS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grads" class="md-nav__link">
    <span class="md-ellipsis">
      GRADS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grasss" class="md-nav__link">
    <span class="md-ellipsis">
      GRASS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grass" class="md-nav__link">
    <span class="md-ellipsis">
      GRASS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gridsss" class="md-nav__link">
    <span class="md-ellipsis">
      GRIDSS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gridss" class="md-nav__link">
    <span class="md-ellipsis">
      GRIDSS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gromacss" class="md-nav__link">
    <span class="md-ellipsis">
      GROMACS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gromacs" class="md-nav__link">
    <span class="md-ellipsis">
      GROMACS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gsls" class="md-nav__link">
    <span class="md-ellipsis">
      GSL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gsl" class="md-nav__link">
    <span class="md-ellipsis">
      GSL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gst-plugins-bases" class="md-nav__link">
    <span class="md-ellipsis">
      GST-plugins-base's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gst-plugins-base" class="md-nav__link">
    <span class="md-ellipsis">
      GST-plugins-base:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gstreamers" class="md-nav__link">
    <span class="md-ellipsis">
      GStreamer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gstreamer" class="md-nav__link">
    <span class="md-ellipsis">
      GStreamer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gtdb-tks" class="md-nav__link">
    <span class="md-ellipsis">
      GTDB-Tk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gtdb-tk" class="md-nav__link">
    <span class="md-ellipsis">
      GTDB-Tk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gtks" class="md-nav__link">
    <span class="md-ellipsis">
      GTK+'s:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gtk" class="md-nav__link">
    <span class="md-ellipsis">
      GTK+:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gtss" class="md-nav__link">
    <span class="md-ellipsis">
      GTS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gts" class="md-nav__link">
    <span class="md-ellipsis">
      GTS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genemark-ess" class="md-nav__link">
    <span class="md-ellipsis">
      GeneMark-ES's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genemark-es" class="md-nav__link">
    <span class="md-ellipsis">
      GeneMark-ES:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genomethreaders" class="md-nav__link">
    <span class="md-ellipsis">
      GenomeThreader's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genomethreader" class="md-nav__link">
    <span class="md-ellipsis">
      GenomeThreader:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gerriss" class="md-nav__link">
    <span class="md-ellipsis">
      Gerris's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gerris" class="md-nav__link">
    <span class="md-ellipsis">
      Gerris:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#getorganelles" class="md-nav__link">
    <span class="md-ellipsis">
      GetOrganelle's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#getorganelle" class="md-nav__link">
    <span class="md-ellipsis">
      GetOrganelle:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glimmerhmms" class="md-nav__link">
    <span class="md-ellipsis">
      GlimmerHMM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#glimmerhmm" class="md-nav__link">
    <span class="md-ellipsis">
      GlimmerHMM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gos" class="md-nav__link">
    <span class="md-ellipsis">
      Go's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#go" class="md-nav__link">
    <span class="md-ellipsis">
      Go:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#graphvizs" class="md-nav__link">
    <span class="md-ellipsis">
      Graphviz's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#graphviz" class="md-nav__link">
    <span class="md-ellipsis">
      Graphviz:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gubbinss" class="md-nav__link">
    <span class="md-ellipsis">
      Gubbins's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gubbins" class="md-nav__link">
    <span class="md-ellipsis">
      Gubbins:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#guiles" class="md-nav__link">
    <span class="md-ellipsis">
      Guile's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#guile" class="md-nav__link">
    <span class="md-ellipsis">
      Guile:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdfs" class="md-nav__link">
    <span class="md-ellipsis">
      HDF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf-eoss" class="md-nav__link">
    <span class="md-ellipsis">
      HDF-EOS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf-eos" class="md-nav__link">
    <span class="md-ellipsis">
      HDF-EOS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf-eos5s" class="md-nav__link">
    <span class="md-ellipsis">
      HDF-EOS5's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf-eos5" class="md-nav__link">
    <span class="md-ellipsis">
      HDF-EOS5:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf5s" class="md-nav__link">
    <span class="md-ellipsis">
      HDF5's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hdf5" class="md-nav__link">
    <span class="md-ellipsis">
      HDF5:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hisat2s" class="md-nav__link">
    <span class="md-ellipsis">
      HISAT2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hisat2" class="md-nav__link">
    <span class="md-ellipsis">
      HISAT2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmers" class="md-nav__link">
    <span class="md-ellipsis">
      HMMER's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmer" class="md-nav__link">
    <span class="md-ellipsis">
      HMMER:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmer2s" class="md-nav__link">
    <span class="md-ellipsis">
      HMMER2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hmmer2" class="md-nav__link">
    <span class="md-ellipsis">
      HMMER2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hopss" class="md-nav__link">
    <span class="md-ellipsis">
      HOPS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hops" class="md-nav__link">
    <span class="md-ellipsis">
      HOPS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#htseqs" class="md-nav__link">
    <span class="md-ellipsis">
      HTSeq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#htseq" class="md-nav__link">
    <span class="md-ellipsis">
      HTSeq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#htslibs" class="md-nav__link">
    <span class="md-ellipsis">
      HTSlib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#htslib" class="md-nav__link">
    <span class="md-ellipsis">
      HTSlib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#harfbuzzs" class="md-nav__link">
    <span class="md-ellipsis">
      HarfBuzz's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#harfbuzz" class="md-nav__link">
    <span class="md-ellipsis">
      HarfBuzz:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hpcgridrunners" class="md-nav__link">
    <span class="md-ellipsis">
      HpcGridRunner's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hpcgridrunner" class="md-nav__link">
    <span class="md-ellipsis">
      HpcGridRunner:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#humanns" class="md-nav__link">
    <span class="md-ellipsis">
      Humann's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#humann" class="md-nav__link">
    <span class="md-ellipsis">
      Humann:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hybpipers" class="md-nav__link">
    <span class="md-ellipsis">
      HybPiper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hybpiper" class="md-nav__link">
    <span class="md-ellipsis">
      HybPiper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hypres" class="md-nav__link">
    <span class="md-ellipsis">
      Hypre's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hypre" class="md-nav__link">
    <span class="md-ellipsis">
      Hypre:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#icus" class="md-nav__link">
    <span class="md-ellipsis">
      ICU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#icu" class="md-nav__link">
    <span class="md-ellipsis">
      ICU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idbas" class="md-nav__link">
    <span class="md-ellipsis">
      IDBA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idba" class="md-nav__link">
    <span class="md-ellipsis">
      IDBA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idba-uds" class="md-nav__link">
    <span class="md-ellipsis">
      IDBA-UD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idba-ud" class="md-nav__link">
    <span class="md-ellipsis">
      IDBA-UD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idls" class="md-nav__link">
    <span class="md-ellipsis">
      IDL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#idl" class="md-nav__link">
    <span class="md-ellipsis">
      IDL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#igvs" class="md-nav__link">
    <span class="md-ellipsis">
      IGV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#igv" class="md-nav__link">
    <span class="md-ellipsis">
      IGV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imputes" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#impute" class="md-nav__link">
    <span class="md-ellipsis">
      IMPUTE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iq-trees" class="md-nav__link">
    <span class="md-ellipsis">
      IQ-TREE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iq-tree" class="md-nav__link">
    <span class="md-ellipsis">
      IQ-TREE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#irkernels" class="md-nav__link">
    <span class="md-ellipsis">
      IRkernel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#irkernel" class="md-nav__link">
    <span class="md-ellipsis">
      IRkernel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#isa-ls" class="md-nav__link">
    <span class="md-ellipsis">
      ISA-L's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#isa-l" class="md-nav__link">
    <span class="md-ellipsis">
      ISA-L:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imagemagicks" class="md-nav__link">
    <span class="md-ellipsis">
      ImageMagick's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imagemagick" class="md-nav__link">
    <span class="md-ellipsis">
      ImageMagick:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#infernals" class="md-nav__link">
    <span class="md-ellipsis">
      Infernal's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#infernal" class="md-nav__link">
    <span class="md-ellipsis">
      Infernal:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#inspectors" class="md-nav__link">
    <span class="md-ellipsis">
      Inspector's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#inspector" class="md-nav__link">
    <span class="md-ellipsis">
      Inspector:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#interproscans" class="md-nav__link">
    <span class="md-ellipsis">
      InterProScan's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#interproscan" class="md-nav__link">
    <span class="md-ellipsis">
      InterProScan:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jagss" class="md-nav__link">
    <span class="md-ellipsis">
      JAGS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jags" class="md-nav__link">
    <span class="md-ellipsis">
      JAGS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#junits" class="md-nav__link">
    <span class="md-ellipsis">
      JUnit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#junit" class="md-nav__link">
    <span class="md-ellipsis">
      JUnit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jaspers" class="md-nav__link">
    <span class="md-ellipsis">
      JasPer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jasper" class="md-nav__link">
    <span class="md-ellipsis">
      JasPer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#javas" class="md-nav__link">
    <span class="md-ellipsis">
      Java's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#java" class="md-nav__link">
    <span class="md-ellipsis">
      Java:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jellyfishs" class="md-nav__link">
    <span class="md-ellipsis">
      Jellyfish's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jellyfish" class="md-nav__link">
    <span class="md-ellipsis">
      Jellyfish:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jsoncpps" class="md-nav__link">
    <span class="md-ellipsis">
      JsonCpp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jsoncpp" class="md-nav__link">
    <span class="md-ellipsis">
      JsonCpp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#julias" class="md-nav__link">
    <span class="md-ellipsis">
      Julia's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#julia" class="md-nav__link">
    <span class="md-ellipsis">
      Julia:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jupyterlabs" class="md-nav__link">
    <span class="md-ellipsis">
      JupyterLab's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jupyterlab" class="md-nav__link">
    <span class="md-ellipsis">
      JupyterLab:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kats" class="md-nav__link">
    <span class="md-ellipsis">
      KAT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kat" class="md-nav__link">
    <span class="md-ellipsis">
      KAT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kealibs" class="md-nav__link">
    <span class="md-ellipsis">
      KEALib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kealib" class="md-nav__link">
    <span class="md-ellipsis">
      KEALib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kmcs" class="md-nav__link">
    <span class="md-ellipsis">
      KMC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kmc" class="md-nav__link">
    <span class="md-ellipsis">
      KMC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kaijus" class="md-nav__link">
    <span class="md-ellipsis">
      Kaiju's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kaiju" class="md-nav__link">
    <span class="md-ellipsis">
      Kaiju:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kent_toolss" class="md-nav__link">
    <span class="md-ellipsis">
      Kent_tools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kent_tools" class="md-nav__link">
    <span class="md-ellipsis">
      Kent_tools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#korfsnaps" class="md-nav__link">
    <span class="md-ellipsis">
      KorfSNAP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#korfsnap" class="md-nav__link">
    <span class="md-ellipsis">
      KorfSNAP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken2s" class="md-nav__link">
    <span class="md-ellipsis">
      Kraken2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kraken2" class="md-nav__link">
    <span class="md-ellipsis">
      Kraken2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kronatoolss" class="md-nav__link">
    <span class="md-ellipsis">
      KronaTools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kronatools" class="md-nav__link">
    <span class="md-ellipsis">
      KronaTools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kyotocabinets" class="md-nav__link">
    <span class="md-ellipsis">
      KyotoCabinet's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kyotocabinet" class="md-nav__link">
    <span class="md-ellipsis">
      KyotoCabinet:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lames" class="md-nav__link">
    <span class="md-ellipsis">
      LAME's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lame" class="md-nav__link">
    <span class="md-ellipsis">
      LAME:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lammpss" class="md-nav__link">
    <span class="md-ellipsis">
      LAMMPS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lammps" class="md-nav__link">
    <span class="md-ellipsis">
      LAMMPS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lasts" class="md-nav__link">
    <span class="md-ellipsis">
      LAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#last" class="md-nav__link">
    <span class="md-ellipsis">
      LAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lastzs" class="md-nav__link">
    <span class="md-ellipsis">
      LASTZ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lastz" class="md-nav__link">
    <span class="md-ellipsis">
      LASTZ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldcs" class="md-nav__link">
    <span class="md-ellipsis">
      LDC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ldc" class="md-nav__link">
    <span class="md-ellipsis">
      LDC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lefses" class="md-nav__link">
    <span class="md-ellipsis">
      LEfSe's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lefse" class="md-nav__link">
    <span class="md-ellipsis">
      LEfSe:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#linkss" class="md-nav__link">
    <span class="md-ellipsis">
      LINKS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#links" class="md-nav__link">
    <span class="md-ellipsis">
      LINKS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#llvms" class="md-nav__link">
    <span class="md-ellipsis">
      LLVM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#llvm" class="md-nav__link">
    <span class="md-ellipsis">
      LLVM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lmdbs" class="md-nav__link">
    <span class="md-ellipsis">
      LMDB's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lmdb" class="md-nav__link">
    <span class="md-ellipsis">
      LMDB:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lsd2s" class="md-nav__link">
    <span class="md-ellipsis">
      LSD2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lsd2" class="md-nav__link">
    <span class="md-ellipsis">
      LSD2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ltr_retrievers" class="md-nav__link">
    <span class="md-ellipsis">
      LTR_retriever's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ltr_retriever" class="md-nav__link">
    <span class="md-ellipsis">
      LTR_retriever:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lumpys" class="md-nav__link">
    <span class="md-ellipsis">
      LUMPY's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lumpy" class="md-nav__link">
    <span class="md-ellipsis">
      LUMPY:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lzos" class="md-nav__link">
    <span class="md-ellipsis">
      LZO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lzo" class="md-nav__link">
    <span class="md-ellipsis">
      LZO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libtiffs" class="md-nav__link">
    <span class="md-ellipsis">
      LibTIFF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libtiff" class="md-nav__link">
    <span class="md-ellipsis">
      LibTIFF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libavs" class="md-nav__link">
    <span class="md-ellipsis">
      Libav's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libav" class="md-nav__link">
    <span class="md-ellipsis">
      Libav:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libints" class="md-nav__link">
    <span class="md-ellipsis">
      Libint's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libint" class="md-nav__link">
    <span class="md-ellipsis">
      Libint:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#littlecmss" class="md-nav__link">
    <span class="md-ellipsis">
      LittleCMS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#littlecms" class="md-nav__link">
    <span class="md-ellipsis">
      LittleCMS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lokis" class="md-nav__link">
    <span class="md-ellipsis">
      Loki's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#loki" class="md-nav__link">
    <span class="md-ellipsis">
      Loki:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#longstitchs" class="md-nav__link">
    <span class="md-ellipsis">
      LongStitch's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#longstitch" class="md-nav__link">
    <span class="md-ellipsis">
      LongStitch:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#m4s" class="md-nav__link">
    <span class="md-ellipsis">
      M4's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#m4" class="md-nav__link">
    <span class="md-ellipsis">
      M4:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#maffts" class="md-nav__link">
    <span class="md-ellipsis">
      MAFFT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mafft" class="md-nav__link">
    <span class="md-ellipsis">
      MAFFT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magmas" class="md-nav__link">
    <span class="md-ellipsis">
      MAGMA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magma" class="md-nav__link">
    <span class="md-ellipsis">
      MAGMA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#makers" class="md-nav__link">
    <span class="md-ellipsis">
      MAKER's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#maker" class="md-nav__link">
    <span class="md-ellipsis">
      MAKER:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#matios" class="md-nav__link">
    <span class="md-ellipsis">
      MATIO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#matio" class="md-nav__link">
    <span class="md-ellipsis">
      MATIO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#matlabs" class="md-nav__link">
    <span class="md-ellipsis">
      MATLAB's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#matlab" class="md-nav__link">
    <span class="md-ellipsis">
      MATLAB:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcls" class="md-nav__link">
    <span class="md-ellipsis">
      MCL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcl" class="md-nav__link">
    <span class="md-ellipsis">
      MCL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcrs" class="md-nav__link">
    <span class="md-ellipsis">
      MCR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mcr" class="md-nav__link">
    <span class="md-ellipsis">
      MCR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megahits" class="md-nav__link">
    <span class="md-ellipsis">
      MEGAHIT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megahit" class="md-nav__link">
    <span class="md-ellipsis">
      MEGAHIT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabolics" class="md-nav__link">
    <span class="md-ellipsis">
      METABOLIC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabolic" class="md-nav__link">
    <span class="md-ellipsis">
      METABOLIC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metiss" class="md-nav__link">
    <span class="md-ellipsis">
      METIS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metis" class="md-nav__link">
    <span class="md-ellipsis">
      METIS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mmseqs2s" class="md-nav__link">
    <span class="md-ellipsis">
      MMseqs2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mmseqs2" class="md-nav__link">
    <span class="md-ellipsis">
      MMseqs2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mob-suites" class="md-nav__link">
    <span class="md-ellipsis">
      MOB-suite's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mob-suite" class="md-nav__link">
    <span class="md-ellipsis">
      MOB-suite:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modflows" class="md-nav__link">
    <span class="md-ellipsis">
      MODFLOW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modflow" class="md-nav__link">
    <span class="md-ellipsis">
      MODFLOW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpfrs" class="md-nav__link">
    <span class="md-ellipsis">
      MPFR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpfr" class="md-nav__link">
    <span class="md-ellipsis">
      MPFR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#msmcs" class="md-nav__link">
    <span class="md-ellipsis">
      MSMC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#msmc" class="md-nav__link">
    <span class="md-ellipsis">
      MSMC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mumpss" class="md-nav__link">
    <span class="md-ellipsis">
      MUMPS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mumps" class="md-nav__link">
    <span class="md-ellipsis">
      MUMPS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummers" class="md-nav__link">
    <span class="md-ellipsis">
      MUMmer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mummer" class="md-nav__link">
    <span class="md-ellipsis">
      MUMmer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muscles" class="md-nav__link">
    <span class="md-ellipsis">
      MUSCLE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muscle" class="md-nav__link">
    <span class="md-ellipsis">
      MUSCLE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#musts" class="md-nav__link">
    <span class="md-ellipsis">
      MUST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#must" class="md-nav__link">
    <span class="md-ellipsis">
      MUST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#masurcas" class="md-nav__link">
    <span class="md-ellipsis">
      MaSuRCA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#masurca" class="md-nav__link">
    <span class="md-ellipsis">
      MaSuRCA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magmas_1" class="md-nav__link">
    <span class="md-ellipsis">
      Magma's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magma_1" class="md-nav__link">
    <span class="md-ellipsis">
      Magma:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#markerminers" class="md-nav__link">
    <span class="md-ellipsis">
      MarkerMiner's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#markerminer" class="md-nav__link">
    <span class="md-ellipsis">
      MarkerMiner:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mashs" class="md-nav__link">
    <span class="md-ellipsis">
      Mash's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mash" class="md-nav__link">
    <span class="md-ellipsis">
      Mash:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mavens" class="md-nav__link">
    <span class="md-ellipsis">
      Maven's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#maven" class="md-nav__link">
    <span class="md-ellipsis">
      Maven:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#maxbins" class="md-nav__link">
    <span class="md-ellipsis">
      MaxBin's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#maxbin" class="md-nav__link">
    <span class="md-ellipsis">
      MaxBin:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meraculouss" class="md-nav__link">
    <span class="md-ellipsis">
      Meraculous's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meraculous" class="md-nav__link">
    <span class="md-ellipsis">
      Meraculous:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mesas" class="md-nav__link">
    <span class="md-ellipsis">
      Mesa's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mesa" class="md-nav__link">
    <span class="md-ellipsis">
      Mesa:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mesons" class="md-nav__link">
    <span class="md-ellipsis">
      Meson's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#meson" class="md-nav__link">
    <span class="md-ellipsis">
      Meson:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabats" class="md-nav__link">
    <span class="md-ellipsis">
      MetaBAT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metabat" class="md-nav__link">
    <span class="md-ellipsis">
      MetaBAT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaeuks" class="md-nav__link">
    <span class="md-ellipsis">
      MetaEuk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaeuk" class="md-nav__link">
    <span class="md-ellipsis">
      MetaEuk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metageneannotators" class="md-nav__link">
    <span class="md-ellipsis">
      MetaGeneAnnotator's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metageneannotator" class="md-nav__link">
    <span class="md-ellipsis">
      MetaGeneAnnotator:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaphlans" class="md-nav__link">
    <span class="md-ellipsis">
      MetaPhlAn's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaphlan" class="md-nav__link">
    <span class="md-ellipsis">
      MetaPhlAn:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaphlan2s" class="md-nav__link">
    <span class="md-ellipsis">
      MetaPhlAn2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaphlan2" class="md-nav__link">
    <span class="md-ellipsis">
      MetaPhlAn2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metasvs" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metasv" class="md-nav__link">
    <span class="md-ellipsis">
      MetaSV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metavelvets" class="md-nav__link">
    <span class="md-ellipsis">
      MetaVelvet's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metavelvet" class="md-nav__link">
    <span class="md-ellipsis">
      MetaVelvet:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaxa2s" class="md-nav__link">
    <span class="md-ellipsis">
      Metaxa2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metaxa2" class="md-nav__link">
    <span class="md-ellipsis">
      Metaxa2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniconda3s" class="md-nav__link">
    <span class="md-ellipsis">
      Miniconda3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniconda3" class="md-nav__link">
    <span class="md-ellipsis">
      Miniconda3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimac3s" class="md-nav__link">
    <span class="md-ellipsis">
      Minimac3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimac3" class="md-nav__link">
    <span class="md-ellipsis">
      Minimac3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mitozs" class="md-nav__link">
    <span class="md-ellipsis">
      MitoZ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mitoz" class="md-nav__link">
    <span class="md-ellipsis">
      MitoZ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#molcass" class="md-nav__link">
    <span class="md-ellipsis">
      Molcas's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#molcas" class="md-nav__link">
    <span class="md-ellipsis">
      Molcas:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#molpros" class="md-nav__link">
    <span class="md-ellipsis">
      Molpro's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#molpro" class="md-nav__link">
    <span class="md-ellipsis">
      Molpro:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#monos" class="md-nav__link">
    <span class="md-ellipsis">
      Mono's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mono" class="md-nav__link">
    <span class="md-ellipsis">
      Mono:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#monocle3s" class="md-nav__link">
    <span class="md-ellipsis">
      Monocle3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#monocle3" class="md-nav__link">
    <span class="md-ellipsis">
      Monocle3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mothurs" class="md-nav__link">
    <span class="md-ellipsis">
      Mothur's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mothur" class="md-nav__link">
    <span class="md-ellipsis">
      Mothur:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#motioncorrs" class="md-nav__link">
    <span class="md-ellipsis">
      MotionCorr's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#motioncorr" class="md-nav__link">
    <span class="md-ellipsis">
      MotionCorr:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mrbayess" class="md-nav__link">
    <span class="md-ellipsis">
      MrBayes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mrbayes" class="md-nav__link">
    <span class="md-ellipsis">
      MrBayes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mules" class="md-nav__link">
    <span class="md-ellipsis">
      Mule's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mule" class="md-nav__link">
    <span class="md-ellipsis">
      Mule:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multiqcs" class="md-nav__link">
    <span class="md-ellipsis">
      MultiQC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#multiqc" class="md-nav__link">
    <span class="md-ellipsis">
      MultiQC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#namds" class="md-nav__link">
    <span class="md-ellipsis">
      NAMD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#namd" class="md-nav__link">
    <span class="md-ellipsis">
      NAMD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nasms" class="md-nav__link">
    <span class="md-ellipsis">
      NASM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nasm" class="md-nav__link">
    <span class="md-ellipsis">
      NASM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncargs" class="md-nav__link">
    <span class="md-ellipsis">
      NCARG's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncarg" class="md-nav__link">
    <span class="md-ellipsis">
      NCARG:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nccls" class="md-nav__link">
    <span class="md-ellipsis">
      NCCL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nccl" class="md-nav__link">
    <span class="md-ellipsis">
      NCCL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncls" class="md-nav__link">
    <span class="md-ellipsis">
      NCL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncl" class="md-nav__link">
    <span class="md-ellipsis">
      NCL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncos" class="md-nav__link">
    <span class="md-ellipsis">
      NCO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nco" class="md-nav__link">
    <span class="md-ellipsis">
      NCO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncviews" class="md-nav__link">
    <span class="md-ellipsis">
      NCVIEW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncview" class="md-nav__link">
    <span class="md-ellipsis">
      NCVIEW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ngss" class="md-nav__link">
    <span class="md-ellipsis">
      NGS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ngs" class="md-nav__link">
    <span class="md-ellipsis">
      NGS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nlopts" class="md-nav__link">
    <span class="md-ellipsis">
      NLopt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nlopt" class="md-nav__link">
    <span class="md-ellipsis">
      NLopt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsprs" class="md-nav__link">
    <span class="md-ellipsis">
      NSPR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nspr" class="md-nav__link">
    <span class="md-ellipsis">
      NSPR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsss" class="md-nav__link">
    <span class="md-ellipsis">
      NSS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nss" class="md-nav__link">
    <span class="md-ellipsis">
      NSS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nvhpcs" class="md-nav__link">
    <span class="md-ellipsis">
      NVHPC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nvhpc" class="md-nav__link">
    <span class="md-ellipsis">
      NVHPC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nwchems" class="md-nav__link">
    <span class="md-ellipsis">
      NWChem's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nwchem" class="md-nav__link">
    <span class="md-ellipsis">
      NWChem:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanocomps" class="md-nav__link">
    <span class="md-ellipsis">
      NanoComp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanocomp" class="md-nav__link">
    <span class="md-ellipsis">
      NanoComp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanolyses" class="md-nav__link">
    <span class="md-ellipsis">
      NanoLyse's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanolyse" class="md-nav__link">
    <span class="md-ellipsis">
      NanoLyse:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanoplots" class="md-nav__link">
    <span class="md-ellipsis">
      NanoPlot's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanoplot" class="md-nav__link">
    <span class="md-ellipsis">
      NanoPlot:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanostats" class="md-nav__link">
    <span class="md-ellipsis">
      NanoStat's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanostat" class="md-nav__link">
    <span class="md-ellipsis">
      NanoStat:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#newton-xs" class="md-nav__link">
    <span class="md-ellipsis">
      Newton-X's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#newton-x" class="md-nav__link">
    <span class="md-ellipsis">
      Newton-X:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextgenmaps" class="md-nav__link">
    <span class="md-ellipsis">
      NextGenMap's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextgenmap" class="md-nav__link">
    <span class="md-ellipsis">
      NextGenMap:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextflows" class="md-nav__link">
    <span class="md-ellipsis">
      Nextflow's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nextflow" class="md-nav__link">
    <span class="md-ellipsis">
      Nextflow:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ninjas" class="md-nav__link">
    <span class="md-ellipsis">
      Ninja's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ninja" class="md-nav__link">
    <span class="md-ellipsis">
      Ninja:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsight-computes" class="md-nav__link">
    <span class="md-ellipsis">
      Nsight-Compute's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsight-compute" class="md-nav__link">
    <span class="md-ellipsis">
      Nsight-Compute:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsight-systemss" class="md-nav__link">
    <span class="md-ellipsis">
      Nsight-Systems's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsight-systems" class="md-nav__link">
    <span class="md-ellipsis">
      Nsight-Systems:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#oasis3-mcts" class="md-nav__link">
    <span class="md-ellipsis">
      OASIS3-MCT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#oasis3-mct" class="md-nav__link">
    <span class="md-ellipsis">
      OASIS3-MCT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obitoolss" class="md-nav__link">
    <span class="md-ellipsis">
      OBITools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obitools" class="md-nav__link">
    <span class="md-ellipsis">
      OBITools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ocis" class="md-nav__link">
    <span class="md-ellipsis">
      OCI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#oci" class="md-nav__link">
    <span class="md-ellipsis">
      OCI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#omas" class="md-nav__link">
    <span class="md-ellipsis">
      OMA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#oma" class="md-nav__link">
    <span class="md-ellipsis">
      OMA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opari2s" class="md-nav__link">
    <span class="md-ellipsis">
      OPARI2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opari2" class="md-nav__link">
    <span class="md-ellipsis">
      OPARI2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orcas" class="md-nav__link">
    <span class="md-ellipsis">
      ORCA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orca" class="md-nav__link">
    <span class="md-ellipsis">
      ORCA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#osprays" class="md-nav__link">
    <span class="md-ellipsis">
      OSPRay's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ospray" class="md-nav__link">
    <span class="md-ellipsis">
      OSPRay:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#osu-micro-benchmarkss" class="md-nav__link">
    <span class="md-ellipsis">
      OSU-Micro-Benchmarks's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#osu-micro-benchmarks" class="md-nav__link">
    <span class="md-ellipsis">
      OSU-Micro-Benchmarks:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#octaves" class="md-nav__link">
    <span class="md-ellipsis">
      Octave's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#octave" class="md-nav__link">
    <span class="md-ellipsis">
      Octave:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#octopuss" class="md-nav__link">
    <span class="md-ellipsis">
      Octopus's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#octopus" class="md-nav__link">
    <span class="md-ellipsis">
      Octopus:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openblass" class="md-nav__link">
    <span class="md-ellipsis">
      OpenBLAS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openblas" class="md-nav__link">
    <span class="md-ellipsis">
      OpenBLAS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openbabels" class="md-nav__link">
    <span class="md-ellipsis">
      OpenBabel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openbabel" class="md-nav__link">
    <span class="md-ellipsis">
      OpenBabel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opencmisss" class="md-nav__link">
    <span class="md-ellipsis">
      OpenCMISS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opencmiss" class="md-nav__link">
    <span class="md-ellipsis">
      OpenCMISS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opencvs" class="md-nav__link">
    <span class="md-ellipsis">
      OpenCV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opencv" class="md-nav__link">
    <span class="md-ellipsis">
      OpenCV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openfasts" class="md-nav__link">
    <span class="md-ellipsis">
      OpenFAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openfast" class="md-nav__link">
    <span class="md-ellipsis">
      OpenFAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openfoams" class="md-nav__link">
    <span class="md-ellipsis">
      OpenFOAM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openfoam" class="md-nav__link">
    <span class="md-ellipsis">
      OpenFOAM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openjpegs" class="md-nav__link">
    <span class="md-ellipsis">
      OpenJPEG's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openjpeg" class="md-nav__link">
    <span class="md-ellipsis">
      OpenJPEG:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openmpis" class="md-nav__link">
    <span class="md-ellipsis">
      OpenMPI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openmpi" class="md-nav__link">
    <span class="md-ellipsis">
      OpenMPI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openssls" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSSL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openssl" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSSL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openseess" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSees's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#opensees" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSees:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openseespys" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSeesPy's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openseespy" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSeesPy:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openslides" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSlide's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#openslide" class="md-nav__link">
    <span class="md-ellipsis">
      OpenSlide:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orfms" class="md-nav__link">
    <span class="md-ellipsis">
      OrfM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orfm" class="md-nav__link">
    <span class="md-ellipsis">
      OrfM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofillers" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoFiller's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofiller" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoFiller:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofinders" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoFinder's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthofinder" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoFinder:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthomcls" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoMCL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#orthomcl" class="md-nav__link">
    <span class="md-ellipsis">
      OrthoMCL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paleomixs" class="md-nav__link">
    <span class="md-ellipsis">
      PALEOMIX's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paleomix" class="md-nav__link">
    <span class="md-ellipsis">
      PALEOMIX:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pamls" class="md-nav__link">
    <span class="md-ellipsis">
      PAML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paml" class="md-nav__link">
    <span class="md-ellipsis">
      PAML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#papis" class="md-nav__link">
    <span class="md-ellipsis">
      PAPI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#papi" class="md-nav__link">
    <span class="md-ellipsis">
      PAPI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pbjellys" class="md-nav__link">
    <span class="md-ellipsis">
      PBJelly's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pbjelly" class="md-nav__link">
    <span class="md-ellipsis">
      PBJelly:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pcres" class="md-nav__link">
    <span class="md-ellipsis">
      PCRE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pcre" class="md-nav__link">
    <span class="md-ellipsis">
      PCRE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pcre2s" class="md-nav__link">
    <span class="md-ellipsis">
      PCRE2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pcre2" class="md-nav__link">
    <span class="md-ellipsis">
      PCRE2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pdts" class="md-nav__link">
    <span class="md-ellipsis">
      PDT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pdt" class="md-nav__link">
    <span class="md-ellipsis">
      PDT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pears" class="md-nav__link">
    <span class="md-ellipsis">
      PEAR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pear" class="md-nav__link">
    <span class="md-ellipsis">
      PEAR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pests" class="md-nav__link">
    <span class="md-ellipsis">
      PEST++'s:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pest" class="md-nav__link">
    <span class="md-ellipsis">
      PEST++:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#petscs" class="md-nav__link">
    <span class="md-ellipsis">
      PETSc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#petsc" class="md-nav__link">
    <span class="md-ellipsis">
      PETSc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pffts" class="md-nav__link">
    <span class="md-ellipsis">
      PFFT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pfft" class="md-nav__link">
    <span class="md-ellipsis">
      PFFT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgis" class="md-nav__link">
    <span class="md-ellipsis">
      PGI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pgi" class="md-nav__link">
    <span class="md-ellipsis">
      PGI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasiuss" class="md-nav__link">
    <span class="md-ellipsis">
      PHASIUS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phasius" class="md-nav__link">
    <span class="md-ellipsis">
      PHASIUS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plinks" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plink" class="md-nav__link">
    <span class="md-ellipsis">
      PLINK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plumeds" class="md-nav__link">
    <span class="md-ellipsis">
      PLUMED's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#plumed" class="md-nav__link">
    <span class="md-ellipsis">
      PLUMED:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pranks" class="md-nav__link">
    <span class="md-ellipsis">
      PRANK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prank" class="md-nav__link">
    <span class="md-ellipsis">
      PRANK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#projs" class="md-nav__link">
    <span class="md-ellipsis">
      PROJ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#proj" class="md-nav__link">
    <span class="md-ellipsis">
      PROJ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pangos" class="md-nav__link">
    <span class="md-ellipsis">
      Pango's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pango" class="md-nav__link">
    <span class="md-ellipsis">
      Pango:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parmetiss" class="md-nav__link">
    <span class="md-ellipsis">
      ParMETIS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parmetis" class="md-nav__link">
    <span class="md-ellipsis">
      ParMETIS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paraviews" class="md-nav__link">
    <span class="md-ellipsis">
      ParaView's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paraview" class="md-nav__link">
    <span class="md-ellipsis">
      ParaView:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallels" class="md-nav__link">
    <span class="md-ellipsis">
      Parallel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallel" class="md-nav__link">
    <span class="md-ellipsis">
      Parallel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallelios" class="md-nav__link">
    <span class="md-ellipsis">
      ParallelIO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallelio" class="md-nav__link">
    <span class="md-ellipsis">
      ParallelIO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paravers" class="md-nav__link">
    <span class="md-ellipsis">
      Paraver's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#paraver" class="md-nav__link">
    <span class="md-ellipsis">
      Paraver:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#peregrines" class="md-nav__link">
    <span class="md-ellipsis">
      Peregrine's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#peregrine" class="md-nav__link">
    <span class="md-ellipsis">
      Peregrine:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#perls" class="md-nav__link">
    <span class="md-ellipsis">
      Perl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#perl" class="md-nav__link">
    <span class="md-ellipsis">
      Perl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phymls" class="md-nav__link">
    <span class="md-ellipsis">
      PhyML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phyml" class="md-nav__link">
    <span class="md-ellipsis">
      PhyML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phylophlans" class="md-nav__link">
    <span class="md-ellipsis">
      PhyloPhlAn's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phylophlan" class="md-nav__link">
    <span class="md-ellipsis">
      PhyloPhlAn:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pilons" class="md-nav__link">
    <span class="md-ellipsis">
      Pilon's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pilon" class="md-nav__link">
    <span class="md-ellipsis">
      Pilon:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pnetcdfs" class="md-nav__link">
    <span class="md-ellipsis">
      PnetCDF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pnetcdf" class="md-nav__link">
    <span class="md-ellipsis">
      PnetCDF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#porechops" class="md-nav__link">
    <span class="md-ellipsis">
      Porechop's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#porechop" class="md-nav__link">
    <span class="md-ellipsis">
      Porechop:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#postgresqls" class="md-nav__link">
    <span class="md-ellipsis">
      PostgreSQL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#postgresql" class="md-nav__link">
    <span class="md-ellipsis">
      PostgreSQL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigals" class="md-nav__link">
    <span class="md-ellipsis">
      Prodigal's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigal" class="md-nav__link">
    <span class="md-ellipsis">
      Prodigal:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prothints" class="md-nav__link">
    <span class="md-ellipsis">
      ProtHint's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prothint" class="md-nav__link">
    <span class="md-ellipsis">
      ProtHint:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#proteinorthos" class="md-nav__link">
    <span class="md-ellipsis">
      Proteinortho's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#proteinortho" class="md-nav__link">
    <span class="md-ellipsis">
      Proteinortho:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyopengls" class="md-nav__link">
    <span class="md-ellipsis">
      PyOpenGL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyopengl" class="md-nav__link">
    <span class="md-ellipsis">
      PyOpenGL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyqts" class="md-nav__link">
    <span class="md-ellipsis">
      PyQt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyqt" class="md-nav__link">
    <span class="md-ellipsis">
      PyQt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pytorchs" class="md-nav__link">
    <span class="md-ellipsis">
      PyTorch's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pytorch" class="md-nav__link">
    <span class="md-ellipsis">
      PyTorch:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pythons" class="md-nav__link">
    <span class="md-ellipsis">
      Python's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#python" class="md-nav__link">
    <span class="md-ellipsis">
      Python:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#python-gpus" class="md-nav__link">
    <span class="md-ellipsis">
      Python-GPU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#python-gpu" class="md-nav__link">
    <span class="md-ellipsis">
      Python-GPU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#python-geos" class="md-nav__link">
    <span class="md-ellipsis">
      Python-Geo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#python-geo" class="md-nav__link">
    <span class="md-ellipsis">
      Python-Geo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qiime2s" class="md-nav__link">
    <span class="md-ellipsis">
      QIIME2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qiime2" class="md-nav__link">
    <span class="md-ellipsis">
      QIIME2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quasts" class="md-nav__link">
    <span class="md-ellipsis">
      QUAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quast" class="md-nav__link">
    <span class="md-ellipsis">
      QUAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qt5s" class="md-nav__link">
    <span class="md-ellipsis">
      Qt5's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qt5" class="md-nav__link">
    <span class="md-ellipsis">
      Qt5:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quantumespressos" class="md-nav__link">
    <span class="md-ellipsis">
      QuantumESPRESSO's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quantumespresso" class="md-nav__link">
    <span class="md-ellipsis">
      QuantumESPRESSO:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quicktrees" class="md-nav__link">
    <span class="md-ellipsis">
      QuickTree's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#quicktree" class="md-nav__link">
    <span class="md-ellipsis">
      QuickTree:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rs" class="md-nav__link">
    <span class="md-ellipsis">
      R's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r" class="md-nav__link">
    <span class="md-ellipsis">
      R:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r-geos" class="md-nav__link">
    <span class="md-ellipsis">
      R-Geo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r-geo" class="md-nav__link">
    <span class="md-ellipsis">
      R-Geo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r-bundle-bioconductors" class="md-nav__link">
    <span class="md-ellipsis">
      R-bundle-Bioconductor's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#r-bundle-bioconductor" class="md-nav__link">
    <span class="md-ellipsis">
      R-bundle-Bioconductor:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rangs-gshhss" class="md-nav__link">
    <span class="md-ellipsis">
      RANGS-GSHHS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rangs-gshhs" class="md-nav__link">
    <span class="md-ellipsis">
      RANGS-GSHHS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxmls" class="md-nav__link">
    <span class="md-ellipsis">
      RAxML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxml" class="md-nav__link">
    <span class="md-ellipsis">
      RAxML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxml-ngs" class="md-nav__link">
    <span class="md-ellipsis">
      RAxML-NG's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raxml-ng" class="md-nav__link">
    <span class="md-ellipsis">
      RAxML-NG:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rdp-classifiers" class="md-nav__link">
    <span class="md-ellipsis">
      RDP-Classifier's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rdp-classifier" class="md-nav__link">
    <span class="md-ellipsis">
      RDP-Classifier:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#recons" class="md-nav__link">
    <span class="md-ellipsis">
      RECON's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#recon" class="md-nav__link">
    <span class="md-ellipsis">
      RECON:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rmblasts" class="md-nav__link">
    <span class="md-ellipsis">
      RMBlast's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rmblast" class="md-nav__link">
    <span class="md-ellipsis">
      RMBlast:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnammers" class="md-nav__link">
    <span class="md-ellipsis">
      RNAmmer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnammer" class="md-nav__link">
    <span class="md-ellipsis">
      RNAmmer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rocms" class="md-nav__link">
    <span class="md-ellipsis">
      ROCm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rocm" class="md-nav__link">
    <span class="md-ellipsis">
      ROCm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#roots" class="md-nav__link">
    <span class="md-ellipsis">
      ROOT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#root" class="md-nav__link">
    <span class="md-ellipsis">
      ROOT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsems" class="md-nav__link">
    <span class="md-ellipsis">
      RSEM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsem" class="md-nav__link">
    <span class="md-ellipsis">
      RSEM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsgislibs" class="md-nav__link">
    <span class="md-ellipsis">
      RSGISLib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsgislib" class="md-nav__link">
    <span class="md-ellipsis">
      RSGISLib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#racons" class="md-nav__link">
    <span class="md-ellipsis">
      Racon's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#racon" class="md-nav__link">
    <span class="md-ellipsis">
      Racon:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ragouts" class="md-nav__link">
    <span class="md-ellipsis">
      Ragout's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ragout" class="md-nav__link">
    <span class="md-ellipsis">
      Ragout:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rapidnjs" class="md-nav__link">
    <span class="md-ellipsis">
      RapidNJ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rapidnj" class="md-nav__link">
    <span class="md-ellipsis">
      RapidNJ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ravens" class="md-nav__link">
    <span class="md-ellipsis">
      Raven's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#raven" class="md-nav__link">
    <span class="md-ellipsis">
      Raven:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rcorrectors" class="md-nav__link">
    <span class="md-ellipsis">
      Rcorrector's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rcorrector" class="md-nav__link">
    <span class="md-ellipsis">
      Rcorrector:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relions" class="md-nav__link">
    <span class="md-ellipsis">
      Relion's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#relion" class="md-nav__link">
    <span class="md-ellipsis">
      Relion:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmaskers" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatMasker's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmasker" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatMasker:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmodelers" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatModeler's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatmodeler" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatModeler:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatscouts" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatScout's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#repeatscout" class="md-nav__link">
    <span class="md-ellipsis">
      RepeatScout:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#riskscapes" class="md-nav__link">
    <span class="md-ellipsis">
      Riskscape's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#riskscape" class="md-nav__link">
    <span class="md-ellipsis">
      Riskscape:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#roarys" class="md-nav__link">
    <span class="md-ellipsis">
      Roary's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#roary" class="md-nav__link">
    <span class="md-ellipsis">
      Roary:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rosettas" class="md-nav__link">
    <span class="md-ellipsis">
      Rosetta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rosetta" class="md-nav__link">
    <span class="md-ellipsis">
      Rosetta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rstudios" class="md-nav__link">
    <span class="md-ellipsis">
      Rstudio's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rstudio" class="md-nav__link">
    <span class="md-ellipsis">
      Rstudio:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rubys" class="md-nav__link">
    <span class="md-ellipsis">
      Ruby's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ruby" class="md-nav__link">
    <span class="md-ellipsis">
      Ruby:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rusts" class="md-nav__link">
    <span class="md-ellipsis">
      Rust's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rust" class="md-nav__link">
    <span class="md-ellipsis">
      Rust:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sages" class="md-nav__link">
    <span class="md-ellipsis">
      SAGE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sage" class="md-nav__link">
    <span class="md-ellipsis">
      SAGE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      SAMtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samtools" class="md-nav__link">
    <span class="md-ellipsis">
      SAMtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sass" class="md-nav__link">
    <span class="md-ellipsis">
      SAS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sas" class="md-nav__link">
    <span class="md-ellipsis">
      SAS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scotchs" class="md-nav__link">
    <span class="md-ellipsis">
      SCOTCH's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scotch" class="md-nav__link">
    <span class="md-ellipsis">
      SCOTCH:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sconss" class="md-nav__link">
    <span class="md-ellipsis">
      SCons's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scons" class="md-nav__link">
    <span class="md-ellipsis">
      SCons:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sdl2s" class="md-nav__link">
    <span class="md-ellipsis">
      SDL2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sdl2" class="md-nav__link">
    <span class="md-ellipsis">
      SDL2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sepps" class="md-nav__link">
    <span class="md-ellipsis">
      SEPP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sepp" class="md-nav__link">
    <span class="md-ellipsis">
      SEPP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shapeit4s" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shapeit4" class="md-nav__link">
    <span class="md-ellipsis">
      SHAPEIT4:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sionlibs" class="md-nav__link">
    <span class="md-ellipsis">
      SIONlib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sionlib" class="md-nav__link">
    <span class="md-ellipsis">
      SIONlib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sips" class="md-nav__link">
    <span class="md-ellipsis">
      SIP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sip" class="md-nav__link">
    <span class="md-ellipsis">
      SIP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#skesas" class="md-nav__link">
    <span class="md-ellipsis">
      SKESA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#skesa" class="md-nav__link">
    <span class="md-ellipsis">
      SKESA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smrt-links" class="md-nav__link">
    <span class="md-ellipsis">
      SMRT-Link's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smrt-link" class="md-nav__link">
    <span class="md-ellipsis">
      SMRT-Link:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snvoter-nanomethphases" class="md-nav__link">
    <span class="md-ellipsis">
      SNVoter-NanoMethPhase's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snvoter-nanomethphase" class="md-nav__link">
    <span class="md-ellipsis">
      SNVoter-NanoMethPhase:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#soapdenovo2s" class="md-nav__link">
    <span class="md-ellipsis">
      SOAPdenovo2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#soapdenovo2" class="md-nav__link">
    <span class="md-ellipsis">
      SOAPdenovo2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#socis" class="md-nav__link">
    <span class="md-ellipsis">
      SOCI's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#soci" class="md-nav__link">
    <span class="md-ellipsis">
      SOCI:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spadess" class="md-nav__link">
    <span class="md-ellipsis">
      SPAdes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spades" class="md-nav__link">
    <span class="md-ellipsis">
      SPAdes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#specfem3ds" class="md-nav__link">
    <span class="md-ellipsis">
      SPECFEM3D's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#specfem3d" class="md-nav__link">
    <span class="md-ellipsis">
      SPECFEM3D:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spiders" class="md-nav__link">
    <span class="md-ellipsis">
      SPIDER's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spider" class="md-nav__link">
    <span class="md-ellipsis">
      SPIDER:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sqlites" class="md-nav__link">
    <span class="md-ellipsis">
      SQLite's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sqlite" class="md-nav__link">
    <span class="md-ellipsis">
      SQLite:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sqlpluss" class="md-nav__link">
    <span class="md-ellipsis">
      SQLplus's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sqlplus" class="md-nav__link">
    <span class="md-ellipsis">
      SQLplus:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ssaha2s" class="md-nav__link">
    <span class="md-ellipsis">
      SSAHA2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ssaha2" class="md-nav__link">
    <span class="md-ellipsis">
      SSAHA2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#stars" class="md-nav__link">
    <span class="md-ellipsis">
      STAR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#star" class="md-nav__link">
    <span class="md-ellipsis">
      STAR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#star-fusions" class="md-nav__link">
    <span class="md-ellipsis">
      STAR-Fusion's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#star-fusion" class="md-nav__link">
    <span class="md-ellipsis">
      STAR-Fusion:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sundialss" class="md-nav__link">
    <span class="md-ellipsis">
      SUNDIALS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sundials" class="md-nav__link">
    <span class="md-ellipsis">
      SUNDIALS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#survivors" class="md-nav__link">
    <span class="md-ellipsis">
      SURVIVOR's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#survivor" class="md-nav__link">
    <span class="md-ellipsis">
      SURVIVOR:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swigs" class="md-nav__link">
    <span class="md-ellipsis">
      SWIG's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swig" class="md-nav__link">
    <span class="md-ellipsis">
      SWIG:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#salmons" class="md-nav__link">
    <span class="md-ellipsis">
      Salmon's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#salmon" class="md-nav__link">
    <span class="md-ellipsis">
      Salmon:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sambambas" class="md-nav__link">
    <span class="md-ellipsis">
      Sambamba's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sambamba" class="md-nav__link">
    <span class="md-ellipsis">
      Sambamba:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scalapacks" class="md-nav__link">
    <span class="md-ellipsis">
      ScaLAPACK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scalapack" class="md-nav__link">
    <span class="md-ellipsis">
      ScaLAPACK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scalascas" class="md-nav__link">
    <span class="md-ellipsis">
      Scalasca's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#scalasca" class="md-nav__link">
    <span class="md-ellipsis">
      Scalasca:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#score-ps" class="md-nav__link">
    <span class="md-ellipsis">
      Score-P's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#score-p" class="md-nav__link">
    <span class="md-ellipsis">
      Score-P:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqan3s" class="md-nav__link">
    <span class="md-ellipsis">
      SeqAn3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqan3" class="md-nav__link">
    <span class="md-ellipsis">
      SeqAn3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqkits" class="md-nav__link">
    <span class="md-ellipsis">
      SeqKit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqkit" class="md-nav__link">
    <span class="md-ellipsis">
      SeqKit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqmonks" class="md-nav__link">
    <span class="md-ellipsis">
      SeqMonk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqmonk" class="md-nav__link">
    <span class="md-ellipsis">
      SeqMonk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sibelias" class="md-nav__link">
    <span class="md-ellipsis">
      SiBELia's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sibelia" class="md-nav__link">
    <span class="md-ellipsis">
      SiBELia:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#siestas" class="md-nav__link">
    <span class="md-ellipsis">
      Siesta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#siesta" class="md-nav__link">
    <span class="md-ellipsis">
      Siesta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#signalps" class="md-nav__link">
    <span class="md-ellipsis">
      SignalP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#signalp" class="md-nav__link">
    <span class="md-ellipsis">
      SignalP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sniffless" class="md-nav__link">
    <span class="md-ellipsis">
      Sniffles's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sniffles" class="md-nav__link">
    <span class="md-ellipsis">
      Sniffles:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sortmernas" class="md-nav__link">
    <span class="md-ellipsis">
      SortMeRNA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sortmerna" class="md-nav__link">
    <span class="md-ellipsis">
      SortMeRNA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sourcetrackers" class="md-nav__link">
    <span class="md-ellipsis">
      SourceTracker's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sourcetracker" class="md-nav__link">
    <span class="md-ellipsis">
      SourceTracker:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spacks" class="md-nav__link">
    <span class="md-ellipsis">
      Spack's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spack" class="md-nav__link">
    <span class="md-ellipsis">
      Spack:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sparks" class="md-nav__link">
    <span class="md-ellipsis">
      Spark's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spark" class="md-nav__link">
    <span class="md-ellipsis">
      Spark:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#squeezemetas" class="md-nav__link">
    <span class="md-ellipsis">
      SqueezeMeta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#squeezemeta" class="md-nav__link">
    <span class="md-ellipsis">
      SqueezeMeta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#stackss" class="md-nav__link">
    <span class="md-ellipsis">
      Stacks's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#stacks" class="md-nav__link">
    <span class="md-ellipsis">
      Stacks:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#stringties" class="md-nav__link">
    <span class="md-ellipsis">
      StringTie's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#stringtie" class="md-nav__link">
    <span class="md-ellipsis">
      StringTie:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structures" class="md-nav__link">
    <span class="md-ellipsis">
      Structure's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#structure" class="md-nav__link">
    <span class="md-ellipsis">
      Structure:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subreads" class="md-nav__link">
    <span class="md-ellipsis">
      Subread's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subread" class="md-nav__link">
    <span class="md-ellipsis">
      Subread:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subversions" class="md-nav__link">
    <span class="md-ellipsis">
      Subversion's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subversion" class="md-nav__link">
    <span class="md-ellipsis">
      Subversion:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#suitesparses" class="md-nav__link">
    <span class="md-ellipsis">
      SuiteSparse's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#suitesparse" class="md-nav__link">
    <span class="md-ellipsis">
      SuiteSparse:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#superlus" class="md-nav__link">
    <span class="md-ellipsis">
      SuperLU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#superlu" class="md-nav__link">
    <span class="md-ellipsis">
      SuperLU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supernovas" class="md-nav__link">
    <span class="md-ellipsis">
      Supernova's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#supernova" class="md-nav__link">
    <span class="md-ellipsis">
      Supernova:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tmhmms" class="md-nav__link">
    <span class="md-ellipsis">
      TMHMM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tmhmm" class="md-nav__link">
    <span class="md-ellipsis">
      TMHMM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsebras" class="md-nav__link">
    <span class="md-ellipsis">
      TSEBRA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tsebra" class="md-nav__link">
    <span class="md-ellipsis">
      TSEBRA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#turbomoles" class="md-nav__link">
    <span class="md-ellipsis">
      TURBOMOLE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#turbomole" class="md-nav__link">
    <span class="md-ellipsis">
      TURBOMOLE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#twl-ninjas" class="md-nav__link">
    <span class="md-ellipsis">
      TWL-NINJA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#twl-ninja" class="md-nav__link">
    <span class="md-ellipsis">
      TWL-NINJA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tcls" class="md-nav__link">
    <span class="md-ellipsis">
      Tcl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tcl" class="md-nav__link">
    <span class="md-ellipsis">
      Tcl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tensorflows" class="md-nav__link">
    <span class="md-ellipsis">
      TensorFlow's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tensorflow" class="md-nav__link">
    <span class="md-ellipsis">
      TensorFlow:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tensorrts" class="md-nav__link">
    <span class="md-ellipsis">
      TensorRT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tensorrt" class="md-nav__link">
    <span class="md-ellipsis">
      TensorRT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#theanos" class="md-nav__link">
    <span class="md-ellipsis">
      Theano's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#theano" class="md-nav__link">
    <span class="md-ellipsis">
      Theano:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tks" class="md-nav__link">
    <span class="md-ellipsis">
      Tk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tk" class="md-nav__link">
    <span class="md-ellipsis">
      Tk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tophats" class="md-nav__link">
    <span class="md-ellipsis">
      TopHat's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tophat" class="md-nav__link">
    <span class="md-ellipsis">
      TopHat:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#transdecoders" class="md-nav__link">
    <span class="md-ellipsis">
      TransDecoder's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#transdecoder" class="md-nav__link">
    <span class="md-ellipsis">
      TransDecoder:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#treemixs" class="md-nav__link">
    <span class="md-ellipsis">
      TreeMix's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#treemix" class="md-nav__link">
    <span class="md-ellipsis">
      TreeMix:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trilinoss" class="md-nav__link">
    <span class="md-ellipsis">
      Trilinos's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trilinos" class="md-nav__link">
    <span class="md-ellipsis">
      Trilinos:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimgalores" class="md-nav__link">
    <span class="md-ellipsis">
      TrimGalore's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimgalore" class="md-nav__link">
    <span class="md-ellipsis">
      TrimGalore:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimmomatics" class="md-nav__link">
    <span class="md-ellipsis">
      Trimmomatic's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimmomatic" class="md-nav__link">
    <span class="md-ellipsis">
      Trimmomatic:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trinitys" class="md-nav__link">
    <span class="md-ellipsis">
      Trinity's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trinity" class="md-nav__link">
    <span class="md-ellipsis">
      Trinity:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trinotates" class="md-nav__link">
    <span class="md-ellipsis">
      Trinotate's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trinotate" class="md-nav__link">
    <span class="md-ellipsis">
      Trinotate:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trycyclers" class="md-nav__link">
    <span class="md-ellipsis">
      Trycycler's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trycycler" class="md-nav__link">
    <span class="md-ellipsis">
      Trycycler:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tuiviews" class="md-nav__link">
    <span class="md-ellipsis">
      TuiView's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tuiview" class="md-nav__link">
    <span class="md-ellipsis">
      TuiView:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#turbovncs" class="md-nav__link">
    <span class="md-ellipsis">
      TurboVNC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#turbovnc" class="md-nav__link">
    <span class="md-ellipsis">
      TurboVNC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ucxs" class="md-nav__link">
    <span class="md-ellipsis">
      UCX's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ucx" class="md-nav__link">
    <span class="md-ellipsis">
      UCX:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#udunitss" class="md-nav__link">
    <span class="md-ellipsis">
      UDUNITS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#udunits" class="md-nav__link">
    <span class="md-ellipsis">
      UDUNITS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#usearchs" class="md-nav__link">
    <span class="md-ellipsis">
      USEARCH's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#usearch" class="md-nav__link">
    <span class="md-ellipsis">
      USEARCH:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unicyclers" class="md-nav__link">
    <span class="md-ellipsis">
      Unicycler's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unicycler" class="md-nav__link">
    <span class="md-ellipsis">
      Unicycler:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vasps" class="md-nav__link">
    <span class="md-ellipsis">
      VASP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vasp" class="md-nav__link">
    <span class="md-ellipsis">
      VASP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcf-kits" class="md-nav__link">
    <span class="md-ellipsis">
      VCF-kit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcf-kit" class="md-nav__link">
    <span class="md-ellipsis">
      VCF-kit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcftoolss" class="md-nav__link">
    <span class="md-ellipsis">
      VCFtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcftools" class="md-nav__link">
    <span class="md-ellipsis">
      VCFtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#veps" class="md-nav__link">
    <span class="md-ellipsis">
      VEP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vep" class="md-nav__link">
    <span class="md-ellipsis">
      VEP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vestas" class="md-nav__link">
    <span class="md-ellipsis">
      VESTA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vesta" class="md-nav__link">
    <span class="md-ellipsis">
      VESTA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vibrants" class="md-nav__link">
    <span class="md-ellipsis">
      VIBRANT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vibrant" class="md-nav__link">
    <span class="md-ellipsis">
      VIBRANT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vmds" class="md-nav__link">
    <span class="md-ellipsis">
      VMD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vmd" class="md-nav__link">
    <span class="md-ellipsis">
      VMD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vsearchs" class="md-nav__link">
    <span class="md-ellipsis">
      VSEARCH's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vsearch" class="md-nav__link">
    <span class="md-ellipsis">
      VSEARCH:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vtks" class="md-nav__link">
    <span class="md-ellipsis">
      VTK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vtk" class="md-nav__link">
    <span class="md-ellipsis">
      VTK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vtunes" class="md-nav__link">
    <span class="md-ellipsis">
      VTune's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vtune" class="md-nav__link">
    <span class="md-ellipsis">
      VTune:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#valgrinds" class="md-nav__link">
    <span class="md-ellipsis">
      Valgrind's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#valgrind" class="md-nav__link">
    <span class="md-ellipsis">
      Valgrind:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#varscans" class="md-nav__link">
    <span class="md-ellipsis">
      VarScan's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#varscan" class="md-nav__link">
    <span class="md-ellipsis">
      VarScan:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#velvets" class="md-nav__link">
    <span class="md-ellipsis">
      Velvet's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#velvet" class="md-nav__link">
    <span class="md-ellipsis">
      Velvet:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#velvetoptimisers" class="md-nav__link">
    <span class="md-ellipsis">
      VelvetOptimiser's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#velvetoptimiser" class="md-nav__link">
    <span class="md-ellipsis">
      VelvetOptimiser:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#viennarnas" class="md-nav__link">
    <span class="md-ellipsis">
      ViennaRNA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#viennarna" class="md-nav__link">
    <span class="md-ellipsis">
      ViennaRNA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vims" class="md-nav__link">
    <span class="md-ellipsis">
      Vim's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vim" class="md-nav__link">
    <span class="md-ellipsis">
      Vim:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virhostmatchers" class="md-nav__link">
    <span class="md-ellipsis">
      VirHostMatcher's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virhostmatcher" class="md-nav__link">
    <span class="md-ellipsis">
      VirHostMatcher:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virsorters" class="md-nav__link">
    <span class="md-ellipsis">
      VirSorter's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virsorter" class="md-nav__link">
    <span class="md-ellipsis">
      VirSorter:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virtualgls" class="md-nav__link">
    <span class="md-ellipsis">
      VirtualGL's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#virtualgl" class="md-nav__link">
    <span class="md-ellipsis">
      VirtualGL:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#visits" class="md-nav__link">
    <span class="md-ellipsis">
      VisIt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#visit" class="md-nav__link">
    <span class="md-ellipsis">
      VisIt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#waafles" class="md-nav__link">
    <span class="md-ellipsis">
      WAAFLE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#waafle" class="md-nav__link">
    <span class="md-ellipsis">
      WAAFLE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#whatshaps" class="md-nav__link">
    <span class="md-ellipsis">
      WhatsHap's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#whatshap" class="md-nav__link">
    <span class="md-ellipsis">
      WhatsHap:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wise2s" class="md-nav__link">
    <span class="md-ellipsis">
      Wise2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wise2" class="md-nav__link">
    <span class="md-ellipsis">
      Wise2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xconvs" class="md-nav__link">
    <span class="md-ellipsis">
      XCONV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xconv" class="md-nav__link">
    <span class="md-ellipsis">
      XCONV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xgkss" class="md-nav__link">
    <span class="md-ellipsis">
      XGKS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xgks" class="md-nav__link">
    <span class="md-ellipsis">
      XGKS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xhmms" class="md-nav__link">
    <span class="md-ellipsis">
      XHMM's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xhmm" class="md-nav__link">
    <span class="md-ellipsis">
      XHMM:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xioss" class="md-nav__link">
    <span class="md-ellipsis">
      XIOS's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xios" class="md-nav__link">
    <span class="md-ellipsis">
      XIOS:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xmds2s" class="md-nav__link">
    <span class="md-ellipsis">
      XMDS2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xmds2" class="md-nav__link">
    <span class="md-ellipsis">
      XMDS2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xsds" class="md-nav__link">
    <span class="md-ellipsis">
      XSD's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xsd" class="md-nav__link">
    <span class="md-ellipsis">
      XSD:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xzs" class="md-nav__link">
    <span class="md-ellipsis">
      XZ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xz" class="md-nav__link">
    <span class="md-ellipsis">
      XZ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xerces-cs" class="md-nav__link">
    <span class="md-ellipsis">
      Xerces-C++'s:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xerces-c" class="md-nav__link">
    <span class="md-ellipsis">
      Xerces-C++:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yaxts" class="md-nav__link">
    <span class="md-ellipsis">
      YAXT's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yaxt" class="md-nav__link">
    <span class="md-ellipsis">
      YAXT:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yades" class="md-nav__link">
    <span class="md-ellipsis">
      Yade's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yade" class="md-nav__link">
    <span class="md-ellipsis">
      Yade:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yasms" class="md-nav__link">
    <span class="md-ellipsis">
      Yasm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yasm" class="md-nav__link">
    <span class="md-ellipsis">
      Yasm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zeromqs" class="md-nav__link">
    <span class="md-ellipsis">
      ZeroMQ's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zeromq" class="md-nav__link">
    <span class="md-ellipsis">
      ZeroMQ:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zips" class="md-nav__link">
    <span class="md-ellipsis">
      Zip's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zip" class="md-nav__link">
    <span class="md-ellipsis">
      Zip:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zonations" class="md-nav__link">
    <span class="md-ellipsis">
      Zonation's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zonation" class="md-nav__link">
    <span class="md-ellipsis">
      Zonation:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#angsds" class="md-nav__link">
    <span class="md-ellipsis">
      angsd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#angsd" class="md-nav__link">
    <span class="md-ellipsis">
      angsd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ants_2" class="md-nav__link">
    <span class="md-ellipsis">
      ant's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ant" class="md-nav__link">
    <span class="md-ellipsis">
      ant:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antismashs" class="md-nav__link">
    <span class="md-ellipsis">
      antiSMASH's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#antismash" class="md-nav__link">
    <span class="md-ellipsis">
      antiSMASH:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#any2fastas" class="md-nav__link">
    <span class="md-ellipsis">
      any2fasta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#any2fasta" class="md-nav__link">
    <span class="md-ellipsis">
      any2fasta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argtables" class="md-nav__link">
    <span class="md-ellipsis">
      argtable's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#argtable" class="md-nav__link">
    <span class="md-ellipsis">
      argtable:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aria2s" class="md-nav__link">
    <span class="md-ellipsis">
      aria2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#aria2" class="md-nav__link">
    <span class="md-ellipsis">
      aria2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arpack-ngs" class="md-nav__link">
    <span class="md-ellipsis">
      arpack-ng's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#arpack-ng" class="md-nav__link">
    <span class="md-ellipsis">
      arpack-ng:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#at-spi2-atks" class="md-nav__link">
    <span class="md-ellipsis">
      at-spi2-atk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#at-spi2-atk" class="md-nav__link">
    <span class="md-ellipsis">
      at-spi2-atk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#attrs" class="md-nav__link">
    <span class="md-ellipsis">
      attr's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#attr" class="md-nav__link">
    <span class="md-ellipsis">
      attr:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#azul-zulus" class="md-nav__link">
    <span class="md-ellipsis">
      azul-zulu's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#azul-zulu" class="md-nav__link">
    <span class="md-ellipsis">
      azul-zulu:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bamutils" class="md-nav__link">
    <span class="md-ellipsis">
      bamUtil's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bamutil" class="md-nav__link">
    <span class="md-ellipsis">
      bamUtil:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#barrnaps" class="md-nav__link">
    <span class="md-ellipsis">
      barrnap's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#barrnap" class="md-nav__link">
    <span class="md-ellipsis">
      barrnap:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcl2fastq2s" class="md-nav__link">
    <span class="md-ellipsis">
      bcl2fastq2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bcl2fastq2" class="md-nav__link">
    <span class="md-ellipsis">
      bcl2fastq2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#binutilss" class="md-nav__link">
    <span class="md-ellipsis">
      binutils's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#binutils" class="md-nav__link">
    <span class="md-ellipsis">
      binutils:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bioawks" class="md-nav__link">
    <span class="md-ellipsis">
      bioawk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bioawk" class="md-nav__link">
    <span class="md-ellipsis">
      bioawk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blasr_libcpps" class="md-nav__link">
    <span class="md-ellipsis">
      blasr_libcpp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#blasr_libcpp" class="md-nav__link">
    <span class="md-ellipsis">
      blasr_libcpp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breseqs" class="md-nav__link">
    <span class="md-ellipsis">
      breseq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#breseq" class="md-nav__link">
    <span class="md-ellipsis">
      breseq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bsddb3s" class="md-nav__link">
    <span class="md-ellipsis">
      bsddb3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bsddb3" class="md-nav__link">
    <span class="md-ellipsis">
      bsddb3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bzip2s" class="md-nav__link">
    <span class="md-ellipsis">
      bzip2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bzip2" class="md-nav__link">
    <span class="md-ellipsis">
      bzip2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#c-aress" class="md-nav__link">
    <span class="md-ellipsis">
      c-ares's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#c-ares" class="md-nav__link">
    <span class="md-ellipsis">
      c-ares:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cairos" class="md-nav__link">
    <span class="md-ellipsis">
      cairo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cairo" class="md-nav__link">
    <span class="md-ellipsis">
      cairo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cdbfastas" class="md-nav__link">
    <span class="md-ellipsis">
      cdbfasta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cdbfasta" class="md-nav__link">
    <span class="md-ellipsis">
      cdbfasta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chewbbacas" class="md-nav__link">
    <span class="md-ellipsis">
      chewBBACA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chewbbaca" class="md-nav__link">
    <span class="md-ellipsis">
      chewBBACA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#choppers" class="md-nav__link">
    <span class="md-ellipsis">
      chopper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chopper" class="md-nav__link">
    <span class="md-ellipsis">
      chopper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cromwells" class="md-nav__link">
    <span class="md-ellipsis">
      cromwell's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cromwell" class="md-nav__link">
    <span class="md-ellipsis">
      cromwell:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctagss" class="md-nav__link">
    <span class="md-ellipsis">
      ctags's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctags" class="md-nav__link">
    <span class="md-ellipsis">
      ctags:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctffinds" class="md-nav__link">
    <span class="md-ellipsis">
      ctffind's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ctffind" class="md-nav__link">
    <span class="md-ellipsis">
      ctffind:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cudnns" class="md-nav__link">
    <span class="md-ellipsis">
      cuDNN's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cudnn" class="md-nav__link">
    <span class="md-ellipsis">
      cuDNN:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cutadapts" class="md-nav__link">
    <span class="md-ellipsis">
      cutadapt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cutadapt" class="md-nav__link">
    <span class="md-ellipsis">
      cutadapt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cutesvs" class="md-nav__link">
    <span class="md-ellipsis">
      cuteSV's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cutesv" class="md-nav__link">
    <span class="md-ellipsis">
      cuteSV:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cwltools" class="md-nav__link">
    <span class="md-ellipsis">
      cwltool's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cwltool" class="md-nav__link">
    <span class="md-ellipsis">
      cwltool:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cyvcf2s" class="md-nav__link">
    <span class="md-ellipsis">
      cyvcf2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cyvcf2" class="md-nav__link">
    <span class="md-ellipsis">
      cyvcf2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dammits" class="md-nav__link">
    <span class="md-ellipsis">
      dammit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dammit" class="md-nav__link">
    <span class="md-ellipsis">
      dammit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deeptoolss" class="md-nav__link">
    <span class="md-ellipsis">
      deepTools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#deeptools" class="md-nav__link">
    <span class="md-ellipsis">
      deepTools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#devtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      devtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#devtools" class="md-nav__link">
    <span class="md-ellipsis">
      devtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#double-conversions" class="md-nav__link">
    <span class="md-ellipsis">
      double-conversion's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#double-conversion" class="md-nav__link">
    <span class="md-ellipsis">
      double-conversion:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dreps" class="md-nav__link">
    <span class="md-ellipsis">
      drep's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#drep" class="md-nav__link">
    <span class="md-ellipsis">
      drep:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dtcmps" class="md-nav__link">
    <span class="md-ellipsis">
      dtcmp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dtcmp" class="md-nav__link">
    <span class="md-ellipsis">
      dtcmp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dupholds" class="md-nav__link">
    <span class="md-ellipsis">
      duphold's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#duphold" class="md-nav__link">
    <span class="md-ellipsis">
      duphold:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#duplex-toolss" class="md-nav__link">
    <span class="md-ellipsis">
      duplex-tools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#duplex-tools" class="md-nav__link">
    <span class="md-ellipsis">
      duplex-tools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ednas" class="md-nav__link">
    <span class="md-ellipsis">
      eDNA's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#edna" class="md-nav__link">
    <span class="md-ellipsis">
      eDNA:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eccodess" class="md-nav__link">
    <span class="md-ellipsis">
      ecCodes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eccodes" class="md-nav__link">
    <span class="md-ellipsis">
      ecCodes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eccodess_1" class="md-nav__link">
    <span class="md-ellipsis">
      eccodes's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eccodes_1" class="md-nav__link">
    <span class="md-ellipsis">
      eccodes:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ectypers" class="md-nav__link">
    <span class="md-ellipsis">
      ectyper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ectyper" class="md-nav__link">
    <span class="md-ellipsis">
      ectyper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#edlibs" class="md-nav__link">
    <span class="md-ellipsis">
      edlib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#edlib" class="md-nav__link">
    <span class="md-ellipsis">
      edlib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eggnog-mappers" class="md-nav__link">
    <span class="md-ellipsis">
      eggnog-mapper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#eggnog-mapper" class="md-nav__link">
    <span class="md-ellipsis">
      eggnog-mapper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ensmallens" class="md-nav__link">
    <span class="md-ellipsis">
      ensmallen's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ensmallen" class="md-nav__link">
    <span class="md-ellipsis">
      ensmallen:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#entrez-directs" class="md-nav__link">
    <span class="md-ellipsis">
      entrez-direct's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#entrez-direct" class="md-nav__link">
    <span class="md-ellipsis">
      entrez-direct:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerates" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exonerate" class="md-nav__link">
    <span class="md-ellipsis">
      exonerate:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#expats" class="md-nav__link">
    <span class="md-ellipsis">
      expat's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#expat" class="md-nav__link">
    <span class="md-ellipsis">
      expat:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#faststructures" class="md-nav__link">
    <span class="md-ellipsis">
      fastStructure's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#faststructure" class="md-nav__link">
    <span class="md-ellipsis">
      fastStructure:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastps" class="md-nav__link">
    <span class="md-ellipsis">
      fastp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastp" class="md-nav__link">
    <span class="md-ellipsis">
      fastp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastq-toolss" class="md-nav__link">
    <span class="md-ellipsis">
      fastq-tools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fastq-tools" class="md-nav__link">
    <span class="md-ellipsis">
      fastq-tools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fcgenes" class="md-nav__link">
    <span class="md-ellipsis">
      fcGENE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fcgene" class="md-nav__link">
    <span class="md-ellipsis">
      fcGENE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fgbios" class="md-nav__link">
    <span class="md-ellipsis">
      fgbio's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fgbio" class="md-nav__link">
    <span class="md-ellipsis">
      fgbio:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fineradstructures" class="md-nav__link">
    <span class="md-ellipsis">
      fineRADstructure's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fineradstructure" class="md-nav__link">
    <span class="md-ellipsis">
      fineRADstructure:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flatbufferss" class="md-nav__link">
    <span class="md-ellipsis">
      flatbuffers's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flatbuffers" class="md-nav__link">
    <span class="md-ellipsis">
      flatbuffers:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flexs" class="md-nav__link">
    <span class="md-ellipsis">
      flex's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#flex" class="md-nav__link">
    <span class="md-ellipsis">
      flex:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fmlrcs" class="md-nav__link">
    <span class="md-ellipsis">
      fmlrc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fmlrc" class="md-nav__link">
    <span class="md-ellipsis">
      fmlrc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fmts" class="md-nav__link">
    <span class="md-ellipsis">
      fmt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fmt" class="md-nav__link">
    <span class="md-ellipsis">
      fmt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#forges" class="md-nav__link">
    <span class="md-ellipsis">
      forge's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#forge" class="md-nav__link">
    <span class="md-ellipsis">
      forge:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fosss" class="md-nav__link">
    <span class="md-ellipsis">
      foss's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#foss" class="md-nav__link">
    <span class="md-ellipsis">
      foss:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#funcx-endpoints" class="md-nav__link">
    <span class="md-ellipsis">
      funcx-endpoint's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#funcx-endpoint" class="md-nav__link">
    <span class="md-ellipsis">
      funcx-endpoint:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fxtracts" class="md-nav__link">
    <span class="md-ellipsis">
      fxtract's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fxtract" class="md-nav__link">
    <span class="md-ellipsis">
      fxtract:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#g2clibs" class="md-nav__link">
    <span class="md-ellipsis">
      g2clib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#g2clib" class="md-nav__link">
    <span class="md-ellipsis">
      g2clib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#g2libs" class="md-nav__link">
    <span class="md-ellipsis">
      g2lib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#g2lib" class="md-nav__link">
    <span class="md-ellipsis">
      g2lib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ga4ghs" class="md-nav__link">
    <span class="md-ellipsis">
      ga4gh's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ga4gh" class="md-nav__link">
    <span class="md-ellipsis">
      ga4gh:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geanys" class="md-nav__link">
    <span class="md-ellipsis">
      geany's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geany" class="md-nav__link">
    <span class="md-ellipsis">
      geany:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genometoolss" class="md-nav__link">
    <span class="md-ellipsis">
      genometools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#genometools" class="md-nav__link">
    <span class="md-ellipsis">
      genometools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gettexts" class="md-nav__link">
    <span class="md-ellipsis">
      gettext's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gettext" class="md-nav__link">
    <span class="md-ellipsis">
      gettext:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gffreads" class="md-nav__link">
    <span class="md-ellipsis">
      gffread's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gffread" class="md-nav__link">
    <span class="md-ellipsis">
      gffread:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#giflibs" class="md-nav__link">
    <span class="md-ellipsis">
      giflib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#giflib" class="md-nav__link">
    <span class="md-ellipsis">
      giflib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gimkls" class="md-nav__link">
    <span class="md-ellipsis">
      gimkl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gimkl" class="md-nav__link">
    <span class="md-ellipsis">
      gimkl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gimpis" class="md-nav__link">
    <span class="md-ellipsis">
      gimpi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gimpi" class="md-nav__link">
    <span class="md-ellipsis">
      gimpi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gits" class="md-nav__link">
    <span class="md-ellipsis">
      git's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#git" class="md-nav__link">
    <span class="md-ellipsis">
      git:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#globus-automate-clients" class="md-nav__link">
    <span class="md-ellipsis">
      globus-automate-client's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#globus-automate-client" class="md-nav__link">
    <span class="md-ellipsis">
      globus-automate-client:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#globus-compute-endpoints" class="md-nav__link">
    <span class="md-ellipsis">
      globus-compute-endpoint's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#globus-compute-endpoint" class="md-nav__link">
    <span class="md-ellipsis">
      globus-compute-endpoint:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmshs" class="md-nav__link">
    <span class="md-ellipsis">
      gmsh's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gmsh" class="md-nav__link">
    <span class="md-ellipsis">
      gmsh:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gnuplots" class="md-nav__link">
    <span class="md-ellipsis">
      gnuplot's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gnuplot" class="md-nav__link">
    <span class="md-ellipsis">
      gnuplot:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gompis" class="md-nav__link">
    <span class="md-ellipsis">
      gompi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gompi" class="md-nav__link">
    <span class="md-ellipsis">
      gompi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#google-sparsehashs" class="md-nav__link">
    <span class="md-ellipsis">
      google-sparsehash's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#google-sparsehash" class="md-nav__link">
    <span class="md-ellipsis">
      google-sparsehash:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#googletests" class="md-nav__link">
    <span class="md-ellipsis">
      googletest's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#googletest" class="md-nav__link">
    <span class="md-ellipsis">
      googletest:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gperfs" class="md-nav__link">
    <span class="md-ellipsis">
      gperf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gperf" class="md-nav__link">
    <span class="md-ellipsis">
      gperf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grib_apis" class="md-nav__link">
    <span class="md-ellipsis">
      grib_api's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grib_api" class="md-nav__link">
    <span class="md-ellipsis">
      grib_api:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grive2s" class="md-nav__link">
    <span class="md-ellipsis">
      grive2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#grive2" class="md-nav__link">
    <span class="md-ellipsis">
      grive2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gsorts" class="md-nav__link">
    <span class="md-ellipsis">
      gsort's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gsort" class="md-nav__link">
    <span class="md-ellipsis">
      gsort:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#h5pps" class="md-nav__link">
    <span class="md-ellipsis">
      h5pp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#h5pp" class="md-nav__link">
    <span class="md-ellipsis">
      h5pp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#help2mans" class="md-nav__link">
    <span class="md-ellipsis">
      help2man's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#help2man" class="md-nav__link">
    <span class="md-ellipsis">
      help2man:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hifiasms" class="md-nav__link">
    <span class="md-ellipsis">
      hifiasm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hifiasm" class="md-nav__link">
    <span class="md-ellipsis">
      hifiasm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hunspells" class="md-nav__link">
    <span class="md-ellipsis">
      hunspell's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hunspell" class="md-nav__link">
    <span class="md-ellipsis">
      hunspell:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hypothesiss" class="md-nav__link">
    <span class="md-ellipsis">
      hypothesis's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hypothesis" class="md-nav__link">
    <span class="md-ellipsis">
      hypothesis:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iccs" class="md-nav__link">
    <span class="md-ellipsis">
      icc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#icc" class="md-nav__link">
    <span class="md-ellipsis">
      icc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#icciforts" class="md-nav__link">
    <span class="md-ellipsis">
      iccifort's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iccifort" class="md-nav__link">
    <span class="md-ellipsis">
      iccifort:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iforts" class="md-nav__link">
    <span class="md-ellipsis">
      ifort's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ifort" class="md-nav__link">
    <span class="md-ellipsis">
      ifort:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iimpis" class="md-nav__link">
    <span class="md-ellipsis">
      iimpi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#iimpi" class="md-nav__link">
    <span class="md-ellipsis">
      iimpi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imkls" class="md-nav__link">
    <span class="md-ellipsis">
      imkl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imkl" class="md-nav__link">
    <span class="md-ellipsis">
      imkl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imkl-fftws" class="md-nav__link">
    <span class="md-ellipsis">
      imkl-FFTW's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#imkl-fftw" class="md-nav__link">
    <span class="md-ellipsis">
      imkl-FFTW:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#impis" class="md-nav__link">
    <span class="md-ellipsis">
      impi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#impi" class="md-nav__link">
    <span class="md-ellipsis">
      impi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intels" class="md-nav__link">
    <span class="md-ellipsis">
      intel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel" class="md-nav__link">
    <span class="md-ellipsis">
      intel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-compilerss" class="md-nav__link">
    <span class="md-ellipsis">
      intel-compilers's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#intel-compilers" class="md-nav__link">
    <span class="md-ellipsis">
      intel-compilers:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ipyrads" class="md-nav__link">
    <span class="md-ellipsis">
      ipyrad's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ipyrad" class="md-nav__link">
    <span class="md-ellipsis">
      ipyrad:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ispcs" class="md-nav__link">
    <span class="md-ellipsis">
      ispc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ispc" class="md-nav__link">
    <span class="md-ellipsis">
      ispc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jbigkits" class="md-nav__link">
    <span class="md-ellipsis">
      jbigkit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jbigkit" class="md-nav__link">
    <span class="md-ellipsis">
      jbigkit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jcvis" class="md-nav__link">
    <span class="md-ellipsis">
      jcvi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jcvi" class="md-nav__link">
    <span class="md-ellipsis">
      jcvi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jemallocs" class="md-nav__link">
    <span class="md-ellipsis">
      jemalloc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jemalloc" class="md-nav__link">
    <span class="md-ellipsis">
      jemalloc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jqs" class="md-nav__link">
    <span class="md-ellipsis">
      jq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jq" class="md-nav__link">
    <span class="md-ellipsis">
      jq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#json-cs" class="md-nav__link">
    <span class="md-ellipsis">
      json-c's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#json-c" class="md-nav__link">
    <span class="md-ellipsis">
      json-c:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jvarkits" class="md-nav__link">
    <span class="md-ellipsis">
      jvarkit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#jvarkit" class="md-nav__link">
    <span class="md-ellipsis">
      jvarkit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kalign2s" class="md-nav__link">
    <span class="md-ellipsis">
      kalign2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kalign2" class="md-nav__link">
    <span class="md-ellipsis">
      kalign2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kallistos" class="md-nav__link">
    <span class="md-ellipsis">
      kallisto's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kallisto" class="md-nav__link">
    <span class="md-ellipsis">
      kallisto:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kinetos" class="md-nav__link">
    <span class="md-ellipsis">
      kineto's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kineto" class="md-nav__link">
    <span class="md-ellipsis">
      kineto:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kmas" class="md-nav__link">
    <span class="md-ellipsis">
      kma's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#kma" class="md-nav__link">
    <span class="md-ellipsis">
      kma:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libflames" class="md-nav__link">
    <span class="md-ellipsis">
      libFLAME's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libflame" class="md-nav__link">
    <span class="md-ellipsis">
      libFLAME:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libglus" class="md-nav__link">
    <span class="md-ellipsis">
      libGLU's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libglu" class="md-nav__link">
    <span class="md-ellipsis">
      libGLU:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libkmls" class="md-nav__link">
    <span class="md-ellipsis">
      libKML's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libkml" class="md-nav__link">
    <span class="md-ellipsis">
      libKML:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libcircles" class="md-nav__link">
    <span class="md-ellipsis">
      libcircle's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libcircle" class="md-nav__link">
    <span class="md-ellipsis">
      libcircle:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libconfigs" class="md-nav__link">
    <span class="md-ellipsis">
      libconfig's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libconfig" class="md-nav__link">
    <span class="md-ellipsis">
      libconfig:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdeflates" class="md-nav__link">
    <span class="md-ellipsis">
      libdeflate's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdeflate" class="md-nav__link">
    <span class="md-ellipsis">
      libdeflate:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdrms" class="md-nav__link">
    <span class="md-ellipsis">
      libdrm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdrm" class="md-nav__link">
    <span class="md-ellipsis">
      libdrm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdwarfs" class="md-nav__link">
    <span class="md-ellipsis">
      libdwarf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libdwarf" class="md-nav__link">
    <span class="md-ellipsis">
      libdwarf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libepoxys" class="md-nav__link">
    <span class="md-ellipsis">
      libepoxy's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libepoxy" class="md-nav__link">
    <span class="md-ellipsis">
      libepoxy:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libevents" class="md-nav__link">
    <span class="md-ellipsis">
      libevent's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libevent" class="md-nav__link">
    <span class="md-ellipsis">
      libevent:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libffis" class="md-nav__link">
    <span class="md-ellipsis">
      libffi's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libffi" class="md-nav__link">
    <span class="md-ellipsis">
      libffi:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgcrypts" class="md-nav__link">
    <span class="md-ellipsis">
      libgcrypt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgcrypt" class="md-nav__link">
    <span class="md-ellipsis">
      libgcrypt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgds" class="md-nav__link">
    <span class="md-ellipsis">
      libgd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgd" class="md-nav__link">
    <span class="md-ellipsis">
      libgd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgeotiffs" class="md-nav__link">
    <span class="md-ellipsis">
      libgeotiff's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgeotiff" class="md-nav__link">
    <span class="md-ellipsis">
      libgeotiff:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libglvnds" class="md-nav__link">
    <span class="md-ellipsis">
      libglvnd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libglvnd" class="md-nav__link">
    <span class="md-ellipsis">
      libglvnd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgpg-errors" class="md-nav__link">
    <span class="md-ellipsis">
      libgpg-error's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgpg-error" class="md-nav__link">
    <span class="md-ellipsis">
      libgpg-error:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgpuarrays" class="md-nav__link">
    <span class="md-ellipsis">
      libgpuarray's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgpuarray" class="md-nav__link">
    <span class="md-ellipsis">
      libgpuarray:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgtextutilss" class="md-nav__link">
    <span class="md-ellipsis">
      libgtextutils's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libgtextutils" class="md-nav__link">
    <span class="md-ellipsis">
      libgtextutils:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libiconvs" class="md-nav__link">
    <span class="md-ellipsis">
      libiconv's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libiconv" class="md-nav__link">
    <span class="md-ellipsis">
      libiconv:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libjpeg-turbos" class="md-nav__link">
    <span class="md-ellipsis">
      libjpeg-turbo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libjpeg-turbo" class="md-nav__link">
    <span class="md-ellipsis">
      libjpeg-turbo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libmathevals" class="md-nav__link">
    <span class="md-ellipsis">
      libmatheval's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libmatheval" class="md-nav__link">
    <span class="md-ellipsis">
      libmatheval:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libpciaccesss" class="md-nav__link">
    <span class="md-ellipsis">
      libpciaccess's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libpciaccess" class="md-nav__link">
    <span class="md-ellipsis">
      libpciaccess:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libpngs" class="md-nav__link">
    <span class="md-ellipsis">
      libpng's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libpng" class="md-nav__link">
    <span class="md-ellipsis">
      libpng:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libreadlines" class="md-nav__link">
    <span class="md-ellipsis">
      libreadline's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libreadline" class="md-nav__link">
    <span class="md-ellipsis">
      libreadline:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libspatialites" class="md-nav__link">
    <span class="md-ellipsis">
      libspatialite's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libspatialite" class="md-nav__link">
    <span class="md-ellipsis">
      libspatialite:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libtools" class="md-nav__link">
    <span class="md-ellipsis">
      libtool's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libtool" class="md-nav__link">
    <span class="md-ellipsis">
      libtool:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libunistrings" class="md-nav__link">
    <span class="md-ellipsis">
      libunistring's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libunistring" class="md-nav__link">
    <span class="md-ellipsis">
      libunistring:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libunwinds" class="md-nav__link">
    <span class="md-ellipsis">
      libunwind's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libunwind" class="md-nav__link">
    <span class="md-ellipsis">
      libunwind:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libvpxs" class="md-nav__link">
    <span class="md-ellipsis">
      libvpx's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libvpx" class="md-nav__link">
    <span class="md-ellipsis">
      libvpx:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxcs" class="md-nav__link">
    <span class="md-ellipsis">
      libxc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxc" class="md-nav__link">
    <span class="md-ellipsis">
      libxc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxslts" class="md-nav__link">
    <span class="md-ellipsis">
      libxslt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxslt" class="md-nav__link">
    <span class="md-ellipsis">
      libxslt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxsmms" class="md-nav__link">
    <span class="md-ellipsis">
      libxsmm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libxsmm" class="md-nav__link">
    <span class="md-ellipsis">
      libxsmm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libzstds" class="md-nav__link">
    <span class="md-ellipsis">
      libzstd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#libzstd" class="md-nav__link">
    <span class="md-ellipsis">
      libzstd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lighttpds" class="md-nav__link">
    <span class="md-ellipsis">
      lighttpd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lighttpd" class="md-nav__link">
    <span class="md-ellipsis">
      lighttpd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#likwids" class="md-nav__link">
    <span class="md-ellipsis">
      likwid's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#likwid" class="md-nav__link">
    <span class="md-ellipsis">
      likwid:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lp_solves" class="md-nav__link">
    <span class="md-ellipsis">
      lp_solve's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lp_solve" class="md-nav__link">
    <span class="md-ellipsis">
      lp_solve:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lwgrps" class="md-nav__link">
    <span class="md-ellipsis">
      lwgrp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lwgrp" class="md-nav__link">
    <span class="md-ellipsis">
      lwgrp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lz4s" class="md-nav__link">
    <span class="md-ellipsis">
      lz4's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lz4" class="md-nav__link">
    <span class="md-ellipsis">
      lz4:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magmas_2" class="md-nav__link">
    <span class="md-ellipsis">
      magma's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#magma_2" class="md-nav__link">
    <span class="md-ellipsis">
      magma:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#makedepends" class="md-nav__link">
    <span class="md-ellipsis">
      makedepend's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#makedepend" class="md-nav__link">
    <span class="md-ellipsis">
      makedepend:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mantas" class="md-nav__link">
    <span class="md-ellipsis">
      manta's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#manta" class="md-nav__link">
    <span class="md-ellipsis">
      manta:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mapdamages" class="md-nav__link">
    <span class="md-ellipsis">
      mapDamage's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mapdamage" class="md-nav__link">
    <span class="md-ellipsis">
      mapDamage:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#merantks" class="md-nav__link">
    <span class="md-ellipsis">
      meRanTK's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#merantk" class="md-nav__link">
    <span class="md-ellipsis">
      meRanTK:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#medakas" class="md-nav__link">
    <span class="md-ellipsis">
      medaka's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#medaka" class="md-nav__link">
    <span class="md-ellipsis">
      medaka:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megalodons" class="md-nav__link">
    <span class="md-ellipsis">
      megalodon's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#megalodon" class="md-nav__link">
    <span class="md-ellipsis">
      megalodon:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metawraps" class="md-nav__link">
    <span class="md-ellipsis">
      metaWRAP's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#metawrap" class="md-nav__link">
    <span class="md-ellipsis">
      metaWRAP:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mirdeep2s" class="md-nav__link">
    <span class="md-ellipsis">
      miRDeep2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mirdeep2" class="md-nav__link">
    <span class="md-ellipsis">
      miRDeep2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#midls" class="md-nav__link">
    <span class="md-ellipsis">
      midl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#midl" class="md-nav__link">
    <span class="md-ellipsis">
      midl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mimallocs" class="md-nav__link">
    <span class="md-ellipsis">
      mimalloc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mimalloc" class="md-nav__link">
    <span class="md-ellipsis">
      mimalloc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniasms" class="md-nav__link">
    <span class="md-ellipsis">
      miniasm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#miniasm" class="md-nav__link">
    <span class="md-ellipsis">
      miniasm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minieigens" class="md-nav__link">
    <span class="md-ellipsis">
      minieigen's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minieigen" class="md-nav__link">
    <span class="md-ellipsis">
      minieigen:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimap2s" class="md-nav__link">
    <span class="md-ellipsis">
      minimap2's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#minimap2" class="md-nav__link">
    <span class="md-ellipsis">
      minimap2:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mjpegtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      mjpegtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mjpegtools" class="md-nav__link">
    <span class="md-ellipsis">
      mjpegtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mlpacks" class="md-nav__link">
    <span class="md-ellipsis">
      mlpack's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mlpack" class="md-nav__link">
    <span class="md-ellipsis">
      mlpack:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mo_tidls" class="md-nav__link">
    <span class="md-ellipsis">
      mo_tidl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mo_tidl" class="md-nav__link">
    <span class="md-ellipsis">
      mo_tidl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modbam2beds" class="md-nav__link">
    <span class="md-ellipsis">
      modbam2bed's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modbam2bed" class="md-nav__link">
    <span class="md-ellipsis">
      modbam2bed:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpi4pys" class="md-nav__link">
    <span class="md-ellipsis">
      mpi4py's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpi4py" class="md-nav__link">
    <span class="md-ellipsis">
      mpi4py:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpifileutilss" class="md-nav__link">
    <span class="md-ellipsis">
      mpifileutils's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#mpifileutils" class="md-nav__link">
    <span class="md-ellipsis">
      mpifileutils:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muparsers" class="md-nav__link">
    <span class="md-ellipsis">
      muParser's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#muparser" class="md-nav__link">
    <span class="md-ellipsis">
      muParser:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanos" class="md-nav__link">
    <span class="md-ellipsis">
      nano's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nano" class="md-nav__link">
    <span class="md-ellipsis">
      nano:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanoqcs" class="md-nav__link">
    <span class="md-ellipsis">
      nanoQC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanoqc" class="md-nav__link">
    <span class="md-ellipsis">
      nanoQC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanofilts" class="md-nav__link">
    <span class="md-ellipsis">
      nanofilt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanofilt" class="md-nav__link">
    <span class="md-ellipsis">
      nanofilt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanogets" class="md-nav__link">
    <span class="md-ellipsis">
      nanoget's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanoget" class="md-nav__link">
    <span class="md-ellipsis">
      nanoget:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanomaths" class="md-nav__link">
    <span class="md-ellipsis">
      nanomath's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanomath" class="md-nav__link">
    <span class="md-ellipsis">
      nanomath:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanopolishs" class="md-nav__link">
    <span class="md-ellipsis">
      nanopolish's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nanopolish" class="md-nav__link">
    <span class="md-ellipsis">
      nanopolish:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-vdbs" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-vdb's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncbi-vdb" class="md-nav__link">
    <span class="md-ellipsis">
      ncbi-vdb:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nccmps" class="md-nav__link">
    <span class="md-ellipsis">
      nccmp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nccmp" class="md-nav__link">
    <span class="md-ellipsis">
      nccmp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncviews_1" class="md-nav__link">
    <span class="md-ellipsis">
      ncview's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ncview_1" class="md-nav__link">
    <span class="md-ellipsis">
      ncview:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nes" class="md-nav__link">
    <span class="md-ellipsis">
      ne's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ne" class="md-nav__link">
    <span class="md-ellipsis">
      ne:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nearlines" class="md-nav__link">
    <span class="md-ellipsis">
      nearline's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdfs" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-cs" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-C++'s:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-c" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-C++:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-c4s" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-C++4's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-c4" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-C++4:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-fortrans" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-Fortran's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#netcdf-fortran" class="md-nav__link">
    <span class="md-ellipsis">
      netCDF-Fortran:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nettles" class="md-nav__link">
    <span class="md-ellipsis">
      nettle's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nettle" class="md-nav__link">
    <span class="md-ellipsis">
      nettle:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#networkxs" class="md-nav__link">
    <span class="md-ellipsis">
      networkx's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#networkx" class="md-nav__link">
    <span class="md-ellipsis">
      networkx:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nodejss" class="md-nav__link">
    <span class="md-ellipsis">
      nodejs's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nodejs" class="md-nav__link">
    <span class="md-ellipsis">
      nodejs:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsegs" class="md-nav__link">
    <span class="md-ellipsis">
      nseg's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nseg" class="md-nav__link">
    <span class="md-ellipsis">
      nseg:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsyncs" class="md-nav__link">
    <span class="md-ellipsis">
      nsync's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nsync" class="md-nav__link">
    <span class="md-ellipsis">
      nsync:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nullarbors" class="md-nav__link">
    <span class="md-ellipsis">
      nullarbor's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nullarbor" class="md-nav__link">
    <span class="md-ellipsis">
      nullarbor:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ont-guppy-gpus" class="md-nav__link">
    <span class="md-ellipsis">
      ont-guppy-gpu's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ont-guppy-gpu" class="md-nav__link">
    <span class="md-ellipsis">
      ont-guppy-gpu:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#padlocs" class="md-nav__link">
    <span class="md-ellipsis">
      padloc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#padloc" class="md-nav__link">
    <span class="md-ellipsis">
      padloc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pairtoolss" class="md-nav__link">
    <span class="md-ellipsis">
      pairtools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pairtools" class="md-nav__link">
    <span class="md-ellipsis">
      pairtools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#panaroos" class="md-nav__link">
    <span class="md-ellipsis">
      panaroo's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#panaroo" class="md-nav__link">
    <span class="md-ellipsis">
      panaroo:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pandocs" class="md-nav__link">
    <span class="md-ellipsis">
      pandoc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pandoc" class="md-nav__link">
    <span class="md-ellipsis">
      pandoc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallel-fastq-dumps" class="md-nav__link">
    <span class="md-ellipsis">
      parallel-fastq-dump's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parallel-fastq-dump" class="md-nav__link">
    <span class="md-ellipsis">
      parallel-fastq-dump:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parasails" class="md-nav__link">
    <span class="md-ellipsis">
      parasail's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#parasail" class="md-nav__link">
    <span class="md-ellipsis">
      parasail:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#patchelfs" class="md-nav__link">
    <span class="md-ellipsis">
      patchelf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#patchelf" class="md-nav__link">
    <span class="md-ellipsis">
      patchelf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pauvres" class="md-nav__link">
    <span class="md-ellipsis">
      pauvre's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pauvre" class="md-nav__link">
    <span class="md-ellipsis">
      pauvre:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pfunits" class="md-nav__link">
    <span class="md-ellipsis">
      pfunit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pfunit" class="md-nav__link">
    <span class="md-ellipsis">
      pfunit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phyxs" class="md-nav__link">
    <span class="md-ellipsis">
      phyx's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phyx" class="md-nav__link">
    <span class="md-ellipsis">
      phyx:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#picards" class="md-nav__link">
    <span class="md-ellipsis">
      picard's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#picard" class="md-nav__link">
    <span class="md-ellipsis">
      picard:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pixmans" class="md-nav__link">
    <span class="md-ellipsis">
      pixman's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pixman" class="md-nav__link">
    <span class="md-ellipsis">
      pixman:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pplacers" class="md-nav__link">
    <span class="md-ellipsis">
      pplacer's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pplacer" class="md-nav__link">
    <span class="md-ellipsis">
      pplacer:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#preseqs" class="md-nav__link">
    <span class="md-ellipsis">
      preseq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#preseq" class="md-nav__link">
    <span class="md-ellipsis">
      preseq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigals_1" class="md-nav__link">
    <span class="md-ellipsis">
      prodigal's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigal_1" class="md-nav__link">
    <span class="md-ellipsis">
      prodigal:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigal-gvs" class="md-nav__link">
    <span class="md-ellipsis">
      prodigal-gv's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prodigal-gv" class="md-nav__link">
    <span class="md-ellipsis">
      prodigal-gv:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prokkas" class="md-nav__link">
    <span class="md-ellipsis">
      prokka's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#prokka" class="md-nav__link">
    <span class="md-ellipsis">
      prokka:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#proovreads" class="md-nav__link">
    <span class="md-ellipsis">
      proovread's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#proovread" class="md-nav__link">
    <span class="md-ellipsis">
      proovread:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protobufs" class="md-nav__link">
    <span class="md-ellipsis">
      protobuf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protobuf" class="md-nav__link">
    <span class="md-ellipsis">
      protobuf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protobuf-pythons" class="md-nav__link">
    <span class="md-ellipsis">
      protobuf-python's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protobuf-python" class="md-nav__link">
    <span class="md-ellipsis">
      protobuf-python:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#psmcs" class="md-nav__link">
    <span class="md-ellipsis">
      psmc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#psmc" class="md-nav__link">
    <span class="md-ellipsis">
      psmc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pstoedits" class="md-nav__link">
    <span class="md-ellipsis">
      pstoedit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pstoedit" class="md-nav__link">
    <span class="md-ellipsis">
      pstoedit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pullseqs" class="md-nav__link">
    <span class="md-ellipsis">
      pullseq's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pullseq" class="md-nav__link">
    <span class="md-ellipsis">
      pullseq:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#purge_dupss" class="md-nav__link">
    <span class="md-ellipsis">
      purge_dups's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#purge_dups" class="md-nav__link">
    <span class="md-ellipsis">
      purge_dups:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#purge_haplotigss" class="md-nav__link">
    <span class="md-ellipsis">
      purge_haplotigs's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#purge_haplotigs" class="md-nav__link">
    <span class="md-ellipsis">
      purge_haplotigs:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pvs" class="md-nav__link">
    <span class="md-ellipsis">
      pv's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pv" class="md-nav__link">
    <span class="md-ellipsis">
      pv:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyanis" class="md-nav__link">
    <span class="md-ellipsis">
      pyani's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyani" class="md-nav__link">
    <span class="md-ellipsis">
      pyani:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pycoqcs" class="md-nav__link">
    <span class="md-ellipsis">
      pycoQC's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pycoqc" class="md-nav__link">
    <span class="md-ellipsis">
      pycoQC:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pymol-open-sources" class="md-nav__link">
    <span class="md-ellipsis">
      pymol-open-source's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pymol-open-source" class="md-nav__link">
    <span class="md-ellipsis">
      pymol-open-source:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyspoas" class="md-nav__link">
    <span class="md-ellipsis">
      pyspoa's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#pyspoa" class="md-nav__link">
    <span class="md-ellipsis">
      pyspoa:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qcats" class="md-nav__link">
    <span class="md-ellipsis">
      qcat's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qcat" class="md-nav__link">
    <span class="md-ellipsis">
      qcat:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rdocks" class="md-nav__link">
    <span class="md-ellipsis">
      rDock's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rdock" class="md-nav__link">
    <span class="md-ellipsis">
      rDock:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#randfolds" class="md-nav__link">
    <span class="md-ellipsis">
      randfold's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#randfold" class="md-nav__link">
    <span class="md-ellipsis">
      randfold:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rasusas" class="md-nav__link">
    <span class="md-ellipsis">
      rasusa's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rasusa" class="md-nav__link">
    <span class="md-ellipsis">
      rasusa:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#razers3s" class="md-nav__link">
    <span class="md-ellipsis">
      razers3's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#razers3" class="md-nav__link">
    <span class="md-ellipsis">
      razers3:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#re2cs" class="md-nav__link">
    <span class="md-ellipsis">
      re2c's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#re2c" class="md-nav__link">
    <span class="md-ellipsis">
      re2c:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnaquasts" class="md-nav__link">
    <span class="md-ellipsis">
      rnaQUAST's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rnaquast" class="md-nav__link">
    <span class="md-ellipsis">
      rnaQUAST:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsyncs" class="md-nav__link">
    <span class="md-ellipsis">
      rsync's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rsync" class="md-nav__link">
    <span class="md-ellipsis">
      rsync:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rust-fmlrcs" class="md-nav__link">
    <span class="md-ellipsis">
      rust-fmlrc's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rust-fmlrc" class="md-nav__link">
    <span class="md-ellipsis">
      rust-fmlrc:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samblasters" class="md-nav__link">
    <span class="md-ellipsis">
      samblaster's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samblaster" class="md-nav__link">
    <span class="md-ellipsis">
      samblaster:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samclips" class="md-nav__link">
    <span class="md-ellipsis">
      samclip's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#samclip" class="md-nav__link">
    <span class="md-ellipsis">
      samclip:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbts" class="md-nav__link">
    <span class="md-ellipsis">
      sbt's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sbt" class="md-nav__link">
    <span class="md-ellipsis">
      sbt:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqmagicks" class="md-nav__link">
    <span class="md-ellipsis">
      seqmagick's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqmagick" class="md-nav__link">
    <span class="md-ellipsis">
      seqmagick:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqtks" class="md-nav__link">
    <span class="md-ellipsis">
      seqtk's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#seqtk" class="md-nav__link">
    <span class="md-ellipsis">
      seqtk:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shumlibs" class="md-nav__link">
    <span class="md-ellipsis">
      shumlib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#shumlib" class="md-nav__link">
    <span class="md-ellipsis">
      shumlib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sismonrs" class="md-nav__link">
    <span class="md-ellipsis">
      sismonr's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sismonr" class="md-nav__link">
    <span class="md-ellipsis">
      sismonr:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slow5toolss" class="md-nav__link">
    <span class="md-ellipsis">
      slow5tools's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slow5tools" class="md-nav__link">
    <span class="md-ellipsis">
      slow5tools:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smafas" class="md-nav__link">
    <span class="md-ellipsis">
      smafa's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smafa" class="md-nav__link">
    <span class="md-ellipsis">
      smafa:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smooves" class="md-nav__link">
    <span class="md-ellipsis">
      smoove's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#smoove" class="md-nav__link">
    <span class="md-ellipsis">
      smoove:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snakemakes" class="md-nav__link">
    <span class="md-ellipsis">
      snakemake's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snakemake" class="md-nav__link">
    <span class="md-ellipsis">
      snakemake:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snaphus" class="md-nav__link">
    <span class="md-ellipsis">
      snaphu's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snaphu" class="md-nav__link">
    <span class="md-ellipsis">
      snaphu:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snappys" class="md-nav__link">
    <span class="md-ellipsis">
      snappy's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snappy" class="md-nav__link">
    <span class="md-ellipsis">
      snappy:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snp-sitess" class="md-nav__link">
    <span class="md-ellipsis">
      snp-sites's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snp-sites" class="md-nav__link">
    <span class="md-ellipsis">
      snp-sites:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snpeffs" class="md-nav__link">
    <span class="md-ellipsis">
      snpEff's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#snpeff" class="md-nav__link">
    <span class="md-ellipsis">
      snpEff:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#somaliers" class="md-nav__link">
    <span class="md-ellipsis">
      somalier's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#somalier" class="md-nav__link">
    <span class="md-ellipsis">
      somalier:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spalns" class="md-nav__link">
    <span class="md-ellipsis">
      spaln's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spaln" class="md-nav__link">
    <span class="md-ellipsis">
      spaln:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spdlogs" class="md-nav__link">
    <span class="md-ellipsis">
      spdlog's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spdlog" class="md-nav__link">
    <span class="md-ellipsis">
      spdlog:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spglibs" class="md-nav__link">
    <span class="md-ellipsis">
      spglib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spglib" class="md-nav__link">
    <span class="md-ellipsis">
      spglib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#splats" class="md-nav__link">
    <span class="md-ellipsis">
      splat's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#splat" class="md-nav__link">
    <span class="md-ellipsis">
      splat:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spoas" class="md-nav__link">
    <span class="md-ellipsis">
      spoa's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#spoa" class="md-nav__link">
    <span class="md-ellipsis">
      spoa:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sratoolkits" class="md-nav__link">
    <span class="md-ellipsis">
      sratoolkit's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sratoolkit" class="md-nav__link">
    <span class="md-ellipsis">
      sratoolkit:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#srun-wrappers" class="md-nav__link">
    <span class="md-ellipsis">
      srun-wrapper's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#srun-wrapper" class="md-nav__link">
    <span class="md-ellipsis">
      srun-wrapper:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sublimes" class="md-nav__link">
    <span class="md-ellipsis">
      sublime's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#sublime" class="md-nav__link">
    <span class="md-ellipsis">
      sublime:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swarms" class="md-nav__link">
    <span class="md-ellipsis">
      swarm's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swarm" class="md-nav__link">
    <span class="md-ellipsis">
      swarm:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swissknifes" class="md-nav__link">
    <span class="md-ellipsis">
      swissknife's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#swissknife" class="md-nav__link">
    <span class="md-ellipsis">
      swissknife:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trnascan-ses" class="md-nav__link">
    <span class="md-ellipsis">
      tRNAscan-SE's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trnascan-se" class="md-nav__link">
    <span class="md-ellipsis">
      tRNAscan-SE:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tabixs" class="md-nav__link">
    <span class="md-ellipsis">
      tabix's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tabix" class="md-nav__link">
    <span class="md-ellipsis">
      tabix:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tabixpps" class="md-nav__link">
    <span class="md-ellipsis">
      tabixpp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tabixpp" class="md-nav__link">
    <span class="md-ellipsis">
      tabixpp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tbbs" class="md-nav__link">
    <span class="md-ellipsis">
      tbb's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tbb" class="md-nav__link">
    <span class="md-ellipsis">
      tbb:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tbl2asns" class="md-nav__link">
    <span class="md-ellipsis">
      tbl2asn's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tbl2asn" class="md-nav__link">
    <span class="md-ellipsis">
      tbl2asn:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tmuxs" class="md-nav__link">
    <span class="md-ellipsis">
      tmux's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tmux" class="md-nav__link">
    <span class="md-ellipsis">
      tmux:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trfs" class="md-nav__link">
    <span class="md-ellipsis">
      trf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trf" class="md-nav__link">
    <span class="md-ellipsis">
      trf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimals" class="md-nav__link">
    <span class="md-ellipsis">
      trimAl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#trimal" class="md-nav__link">
    <span class="md-ellipsis">
      trimAl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#um2netcdfs" class="md-nav__link">
    <span class="md-ellipsis">
      um2netcdf's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#um2netcdf" class="md-nav__link">
    <span class="md-ellipsis">
      um2netcdf:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unimaps" class="md-nav__link">
    <span class="md-ellipsis">
      unimap's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unimap" class="md-nav__link">
    <span class="md-ellipsis">
      unimap:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unrars" class="md-nav__link">
    <span class="md-ellipsis">
      unrar's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#unrar" class="md-nav__link">
    <span class="md-ellipsis">
      unrar:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#util-linuxs" class="md-nav__link">
    <span class="md-ellipsis">
      util-linux's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#util-linux" class="md-nav__link">
    <span class="md-ellipsis">
      util-linux:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcflibs" class="md-nav__link">
    <span class="md-ellipsis">
      vcflib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vcflib" class="md-nav__link">
    <span class="md-ellipsis">
      vcflib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vgs" class="md-nav__link">
    <span class="md-ellipsis">
      vg's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#vg" class="md-nav__link">
    <span class="md-ellipsis">
      vg:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wgsims" class="md-nav__link">
    <span class="md-ellipsis">
      wgsim's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wgsim" class="md-nav__link">
    <span class="md-ellipsis">
      wgsim:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wheels" class="md-nav__link">
    <span class="md-ellipsis">
      wheel's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wheel" class="md-nav__link">
    <span class="md-ellipsis">
      wheel:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wtdbgs" class="md-nav__link">
    <span class="md-ellipsis">
      wtdbg's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wtdbg" class="md-nav__link">
    <span class="md-ellipsis">
      wtdbg:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wxwidgetss" class="md-nav__link">
    <span class="md-ellipsis">
      wxWidgets's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#wxwidgets" class="md-nav__link">
    <span class="md-ellipsis">
      wxWidgets:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xbitmapss" class="md-nav__link">
    <span class="md-ellipsis">
      xbitmaps's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xbitmaps" class="md-nav__link">
    <span class="md-ellipsis">
      xbitmaps:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xkbcommons" class="md-nav__link">
    <span class="md-ellipsis">
      xkbcommon's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#xkbcommon" class="md-nav__link">
    <span class="md-ellipsis">
      xkbcommon:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yacrds" class="md-nav__link">
    <span class="md-ellipsis">
      yacrd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yacrd" class="md-nav__link">
    <span class="md-ellipsis">
      yacrd:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yajls" class="md-nav__link">
    <span class="md-ellipsis">
      yajl's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yajl" class="md-nav__link">
    <span class="md-ellipsis">
      yajl:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yaks" class="md-nav__link">
    <span class="md-ellipsis">
      yak's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yak" class="md-nav__link">
    <span class="md-ellipsis">
      yak:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yaml-cpps" class="md-nav__link">
    <span class="md-ellipsis">
      yaml-cpp's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#yaml-cpp" class="md-nav__link">
    <span class="md-ellipsis">
      yaml-cpp:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zlibs" class="md-nav__link">
    <span class="md-ellipsis">
      zlib's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zlib" class="md-nav__link">
    <span class="md-ellipsis">
      zlib:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zstds" class="md-nav__link">
    <span class="md-ellipsis">
      zstd's:
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#zstd" class="md-nav__link">
    <span class="md-ellipsis">
      zstd:
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                

                  

  
    <a href="https://github.com/nesi/support-docs-concept/edit/main/docs/Glossary.md" title="Edit this page" class="md-content__button md-icon">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"/></svg>
    </a>
  
  


  <h1>Glossary</h1>

<h3 id="cpus">CPUs:<a class="headerlink" href="#cpus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Slectronic circuitry executes instructions of a computer program.
</code></pre></div>
<h3 id="cpus_1">CPU's:<a class="headerlink" href="#cpus_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Slectronic circuitry executes instructions of a computer program.
</code></pre></div>
<h3 id="cpu">CPU:<a class="headerlink" href="#cpu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Slectronic circuitry executes instructions of a computer program.
</code></pre></div>
<h3 id="cuda">CUDA:<a class="headerlink" href="#cuda" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
</code></pre></div>
<h3 id="hdf">HDF:<a class="headerlink" href="#hdf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines.
</code></pre></div>
<h3 id="hpcs">HPCs:<a class="headerlink" href="#hpcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="hpc">HPC:<a class="headerlink" href="#hpc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="mpi">MPI:<a class="headerlink" href="#mpi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A standardised message-passing standard designed to function on parallel computing architectures.
</code></pre></div>
<h3 id="nesis">NeSI's:<a class="headerlink" href="#nesis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>New Zealand national high performance computing platform.
</code></pre></div>
<h3 id="nesi">NeSI:<a class="headerlink" href="#nesi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>New Zealand national high performance computing platform.
</code></pre></div>
<h3 id="niwas">NIWA's:<a class="headerlink" href="#niwas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Crown Research Institute, conducts research across a broad range of disciplines in the environmental sciences.
</code></pre></div>
<h3 id="niwa">NIWA:<a class="headerlink" href="#niwa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Crown Research Institute, conducts research across a broad range of disciplines in the environmental sciences.
</code></pre></div>
<h3 id="scp">SCP:<a class="headerlink" href="#scp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Means of securely transferring files between over an SSH connection.
</code></pre></div>
<h3 id="ssh">SSH:<a class="headerlink" href="#ssh" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A network communication protocol that enables two computers to communicate
</code></pre></div>
<h3 id="vpn">VPN:<a class="headerlink" href="#vpn" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Method of extending access to a private network.
</code></pre></div>
<h3 id="supercomputers">supercomputers:<a class="headerlink" href="#supercomputers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="supercomputer">supercomputer:<a class="headerlink" href="#supercomputer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="supercomputings">supercomputings:<a class="headerlink" href="#supercomputings" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="supercomputing">supercomputing:<a class="headerlink" href="#supercomputing" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Like a regular computer, but larger. Primarily used for heating data centers.
</code></pre></div>
<h3 id="abaquss">ABAQUS's:<a class="headerlink" href="#abaquss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA.
</code></pre></div>
<h3 id="abaqus">ABAQUS:<a class="headerlink" href="#abaqus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA.
</code></pre></div>
<h3 id="abricates">ABRicate's:<a class="headerlink" href="#abricates" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mass screening of contigs for antimicrobial and virulence genes
</code></pre></div>
<h3 id="abricate">ABRicate:<a class="headerlink" href="#abricate" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mass screening of contigs for antimicrobial and virulence genes
</code></pre></div>
<h3 id="abysss">ABySS's:<a class="headerlink" href="#abysss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler
</code></pre></div>
<h3 id="abyss">ABySS:<a class="headerlink" href="#abyss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler
</code></pre></div>
<h3 id="actcs">ACTC's:<a class="headerlink" href="#actcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ACTC converts independent triangles into triangle strips or fans.
</code></pre></div>
<h3 id="actc">ACTC:<a class="headerlink" href="#actc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ACTC converts independent triangles into triangle strips or fans.
</code></pre></div>
<h3 id="agats">AGAT's:<a class="headerlink" href="#agats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Suite of tools to handle gene annotations in any GTF/GFF format.
</code></pre></div>
<h3 id="agat">AGAT:<a class="headerlink" href="#agat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Suite of tools to handle gene annotations in any GTF/GFF format.
</code></pre></div>
<h3 id="ages">AGE's:<a class="headerlink" href="#ages" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Alignment of sequences with structural variants.
</code></pre></div>
<h3 id="age">AGE:<a class="headerlink" href="#age" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Alignment of sequences with structural variants.
</code></pre></div>
<h3 id="amoss">AMOS's:<a class="headerlink" href="#amoss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of tools for genome assembly
</code></pre></div>
<h3 id="amos">AMOS:<a class="headerlink" href="#amos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of tools for genome assembly
</code></pre></div>
<h3 id="amrfinderpluss">AMRFinderPlus's:<a class="headerlink" href="#amrfinderpluss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCBI Antimicrobial Resistance Gene Finder Plus
</code></pre></div>
<h3 id="amrfinderplus">AMRFinderPlus:<a class="headerlink" href="#amrfinderplus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCBI Antimicrobial Resistance Gene Finder Plus
</code></pre></div>
<h3 id="anicalculators">ANIcalculator's:<a class="headerlink" href="#anicalculators" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Calculate the bidirectional average nucleotide identity (gANI) and
</code></pre></div>
<h3 id="anicalculator">ANIcalculator:<a class="headerlink" href="#anicalculator" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Calculate the bidirectional average nucleotide identity (gANI) and
</code></pre></div>
<h3 id="annovars">ANNOVAR's:<a class="headerlink" href="#annovars" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient software tool to utilize update-to-date information to functionally
</code></pre></div>
<h3 id="annovar">ANNOVAR:<a class="headerlink" href="#annovar" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient software tool to utilize update-to-date information to functionally
</code></pre></div>
<h3 id="antlrs">ANTLR's:<a class="headerlink" href="#antlrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.
</code></pre></div>
<h3 id="antlr">ANTLR:<a class="headerlink" href="#antlr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.
</code></pre></div>
<h3 id="antss">ANTS's:<a class="headerlink" href="#antss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ants is a versatile, easy to use Python library for developing ancillary applications.                   This package is restricted to the UM user group.
</code></pre></div>
<h3 id="ants">ANTS:<a class="headerlink" href="#ants" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ants is a versatile, easy to use Python library for developing ancillary applications.                   This package is restricted to the UM user group.
</code></pre></div>
<h3 id="antss_1">ANTs's:<a class="headerlink" href="#antss_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,
</code></pre></div>
<h3 id="ants_1">ANTs:<a class="headerlink" href="#ants_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,
</code></pre></div>
<h3 id="aoccs">AOCC's:<a class="headerlink" href="#aoccs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AMD Optimized C/C++ &amp; Fortran compilers (AOCC) based on LLVM 13.0
</code></pre></div>
<h3 id="aocc">AOCC:<a class="headerlink" href="#aocc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AMD Optimized C/C++ &amp; Fortran compilers (AOCC) based on LLVM 13.0
</code></pre></div>
<h3 id="aocl-bliss">AOCL-BLIS's:<a class="headerlink" href="#aocl-bliss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of BLIS for AMD EPYC family of processors..
</code></pre></div>
<h3 id="aocl-blis">AOCL-BLIS:<a class="headerlink" href="#aocl-blis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of BLIS for AMD EPYC family of processors..
</code></pre></div>
<h3 id="aocl-fftws">AOCL-FFTW's:<a class="headerlink" href="#aocl-fftws" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of FFTW for AMD EPYC family of processors.
</code></pre></div>
<h3 id="aocl-fftw">AOCL-FFTW:<a class="headerlink" href="#aocl-fftw" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of FFTW for AMD EPYC family of processors.
</code></pre></div>
<h3 id="aocl-scalapacks">AOCL-ScaLAPACK's:<a class="headerlink" href="#aocl-scalapacks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of ScaLAPACK for AMD EPYC family of processors.
</code></pre></div>
<h3 id="aocl-scalapack">AOCL-ScaLAPACK:<a class="headerlink" href="#aocl-scalapack" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Optimized version of ScaLAPACK for AMD EPYC family of processors.
</code></pre></div>
<h3 id="aprs">APR's:<a class="headerlink" href="#aprs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Portable Runtime (APR) libraries.
</code></pre></div>
<h3 id="apr">APR:<a class="headerlink" href="#apr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Portable Runtime (APR) libraries.
</code></pre></div>
<h3 id="apr-utils">APR-util's:<a class="headerlink" href="#apr-utils" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Portable Runtime (APR) util libraries.
</code></pre></div>
<h3 id="apr-util">APR-util:<a class="headerlink" href="#apr-util" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Portable Runtime (APR) util libraries.
</code></pre></div>
<h3 id="arcsis">ARCSI's:<a class="headerlink" href="#arcsis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Atmospheric and Radiometric Correction of Satellite Imagery (ARCSI) software provides a command line tool for the atmospheric correction of Earth Observation imagery. The aim of ARCSI is to provide as automatic as possible method of retrieving the atmospheric correction parameters and using them to parameterise 6S.Universal Command Line Environment for AWS.
</code></pre></div>
<h3 id="arcsi">ARCSI:<a class="headerlink" href="#arcsi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Atmospheric and Radiometric Correction of Satellite Imagery (ARCSI) software provides a command line tool for the atmospheric correction of Earth Observation imagery. The aim of ARCSI is to provide as automatic as possible method of retrieving the atmospheric correction parameters and using them to parameterise 6S.Universal Command Line Environment for AWS.
</code></pre></div>
<h3 id="aribas">ARIBA's:<a class="headerlink" href="#aribas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Antimicrobial Resistance Identification By Assembly
</code></pre></div>
<h3 id="ariba">ARIBA:<a class="headerlink" href="#ariba" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Antimicrobial Resistance Identification By Assembly
</code></pre></div>
<h3 id="atks">ATK's:<a class="headerlink" href="#atks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ATK provides the set of accessibility interfaces that are implemented by other toolkits and applications.
</code></pre></div>
<h3 id="atk">ATK:<a class="headerlink" href="#atk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ATK provides the set of accessibility interfaces that are implemented by other toolkits and applications.
</code></pre></div>
<h3 id="augustuss">AUGUSTUS's:<a class="headerlink" href="#augustuss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
</code></pre></div>
<h3 id="augustus">AUGUSTUS:<a class="headerlink" href="#augustus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
</code></pre></div>
<h3 id="adapterremovals">AdapterRemoval's:<a class="headerlink" href="#adapterremovals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ssearches for and removes remnant adapter sequences
</code></pre></div>
<h3 id="adapterremoval">AdapterRemoval:<a class="headerlink" href="#adapterremoval" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ssearches for and removes remnant adapter sequences
</code></pre></div>
<h3 id="advisors">Advisor's:<a class="headerlink" href="#advisors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Vectorization Optimization and Thread Prototyping
</code></pre></div>
<h3 id="advisor">Advisor:<a class="headerlink" href="#advisor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Vectorization Optimization and Thread Prototyping
</code></pre></div>
<h3 id="alphafolds">AlphaFold's:<a class="headerlink" href="#alphafolds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
</code></pre></div>
<h3 id="alphafold">AlphaFold:<a class="headerlink" href="#alphafold" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known
</code></pre></div>
<h3 id="alphafold2dbs">AlphaFold2DB's:<a class="headerlink" href="#alphafold2dbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AlphaFold2 databases
</code></pre></div>
<h3 id="alphafold2db">AlphaFold2DB:<a class="headerlink" href="#alphafold2db" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AlphaFold2 databases
</code></pre></div>
<h3 id="alwaysintelmkls">AlwaysIntelMKL's:<a class="headerlink" href="#alwaysintelmkls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Overrides the MKL internal utility function mkl_serv_intel_cpu_true
</code></pre></div>
<h3 id="alwaysintelmkl">AlwaysIntelMKL:<a class="headerlink" href="#alwaysintelmkl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Overrides the MKL internal utility function mkl_serv_intel_cpu_true
</code></pre></div>
<h3 id="ambers">Amber's:<a class="headerlink" href="#ambers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Amber (originally Assisted Model Building with Energy
</code></pre></div>
<h3 id="amber">Amber:<a class="headerlink" href="#amber" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Amber (originally Assisted Model Building with Energy
</code></pre></div>
<h3 id="anaconda2s">Anaconda2's:<a class="headerlink" href="#anaconda2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.  IMPORTANT: This version of Anaconda Python comes with Intel MKL support to speed up            certain types of mathematical computations, such as linear algebra or FFT.            The module sets             MKL_NUM_THREADS=1             to run MKL on a single thread by default, avoiding accidental oversubscription            of cores. The number of threads can be increased for large problems, please            refer to the Intel MKL documentation for guidance.
</code></pre></div>
<h3 id="anaconda2">Anaconda2:<a class="headerlink" href="#anaconda2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.  IMPORTANT: This version of Anaconda Python comes with Intel MKL support to speed up            certain types of mathematical computations, such as linear algebra or FFT.            The module sets             MKL_NUM_THREADS=1             to run MKL on a single thread by default, avoiding accidental oversubscription            of cores. The number of threads can be increased for large problems, please            refer to the Intel MKL documentation for guidance.
</code></pre></div>
<h3 id="anaconda3s">Anaconda3's:<a class="headerlink" href="#anaconda3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.
</code></pre></div>
<h3 id="anaconda3">Anaconda3:<a class="headerlink" href="#anaconda3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture.
</code></pre></div>
<h3 id="apptainers">Apptainer's:<a class="headerlink" href="#apptainers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apptainer is a portable application stack packaging and runtime utility.
</code></pre></div>
<h3 id="apptainer">Apptainer:<a class="headerlink" href="#apptainer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apptainer is a portable application stack packaging and runtime utility.
</code></pre></div>
<h3 id="armadillos">Armadillo's:<a class="headerlink" href="#armadillos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use.
</code></pre></div>
<h3 id="armadillo">Armadillo:<a class="headerlink" href="#armadillo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use.
</code></pre></div>
<h3 id="arrows">Arrow's:<a class="headerlink" href="#arrows" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Arrow, a cross-language development platform for in-memory data.
</code></pre></div>
<h3 id="arrow">Arrow:<a class="headerlink" href="#arrow" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Arrow, a cross-language development platform for in-memory data.
</code></pre></div>
<h3 id="aspera-clis">Aspera-CLI's:<a class="headerlink" href="#aspera-clis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IBM Aspera Command-Line Interface (the Aspera CLI) is
</code></pre></div>
<h3 id="aspera-cli">Aspera-CLI:<a class="headerlink" href="#aspera-cli" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IBM Aspera Command-Line Interface (the Aspera CLI) is
</code></pre></div>
<h3 id="augustuss_1">Augustus's:<a class="headerlink" href="#augustuss_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.
</code></pre></div>
<h3 id="augustus_1">Augustus:<a class="headerlink" href="#augustus_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.
</code></pre></div>
<h3 id="autodock-gpus">AutoDock-GPU's:<a class="headerlink" href="#autodock-gpus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly
</code></pre></div>
<h3 id="autodock-gpu">AutoDock-GPU:<a class="headerlink" href="#autodock-gpu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrasingly
</code></pre></div>
<h3 id="autodock_vinas">AutoDock_Vina's:<a class="headerlink" href="#autodock_vinas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AutoDock Vina is an open-source program for doing molecular docking.
</code></pre></div>
<h3 id="autodock_vina">AutoDock_Vina:<a class="headerlink" href="#autodock_vina" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AutoDock Vina is an open-source program for doing molecular docking.
</code></pre></div>
<h3 id="bbmaps">BBMap's:<a class="headerlink" href="#bbmaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BBMap short read aligner, and other bioinformatic tools.
</code></pre></div>
<h3 id="bbmap">BBMap:<a class="headerlink" href="#bbmap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BBMap short read aligner, and other bioinformatic tools.
</code></pre></div>
<h3 id="bcftoolss">BCFtools's:<a class="headerlink" href="#bcftoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF.
</code></pre></div>
<h3 id="bcftools">BCFtools:<a class="headerlink" href="#bcftools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF.
</code></pre></div>
<h3 id="bcl-converts">BCL-Convert's:<a class="headerlink" href="#bcl-converts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Converts per cycle binary data output by Illumina sequencers containing basecall
</code></pre></div>
<h3 id="bcl-convert">BCL-Convert:<a class="headerlink" href="#bcl-convert" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Converts per cycle binary data output by Illumina sequencers containing basecall
</code></pre></div>
<h3 id="beasts">BEAST's:<a class="headerlink" href="#beasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bayesian MCMC phylogenetic analysis of molecular sequences for reconstructing
</code></pre></div>
<h3 id="beast">BEAST:<a class="headerlink" href="#beast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bayesian MCMC phylogenetic analysis of molecular sequences for reconstructing
</code></pre></div>
<h3 id="bedopss">BEDOPS's:<a class="headerlink" href="#bedopss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BEDOPS is an open-source command-line toolkit that performs highly
</code></pre></div>
<h3 id="bedops">BEDOPS:<a class="headerlink" href="#bedops" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BEDOPS is an open-source command-line toolkit that performs highly
</code></pre></div>
<h3 id="bedtoolss">BEDTools's:<a class="headerlink" href="#bedtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
</code></pre></div>
<h3 id="bedtools">BEDTools:<a class="headerlink" href="#bedtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
</code></pre></div>
<h3 id="beefs">BEEF's:<a class="headerlink" href="#beefs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BEEF is a library implementing the Bayesian Error Estimation Functional, a description of which can be found here:  http://dx.doi.org/10.1103/PhysRevB.85.235149
</code></pre></div>
<h3 id="beef">BEEF:<a class="headerlink" href="#beef" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BEEF is a library implementing the Bayesian Error Estimation Functional, a description of which can be found here:  http://dx.doi.org/10.1103/PhysRevB.85.235149
</code></pre></div>
<h3 id="blasrs">BLASR's:<a class="headerlink" href="#blasrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLASR (Basic Local Alignment with Successive Refinement) rapidly maps
</code></pre></div>
<h3 id="blasr">BLASR:<a class="headerlink" href="#blasr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLASR (Basic Local Alignment with Successive Refinement) rapidly maps
</code></pre></div>
<h3 id="blasts">BLAST's:<a class="headerlink" href="#blasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Basic Local Alignment Search Tool, or BLAST, is an algorithm
</code></pre></div>
<h3 id="blast">BLAST:<a class="headerlink" href="#blast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Basic Local Alignment Search Tool, or BLAST, is an algorithm
</code></pre></div>
<h3 id="blastdbs">BLASTDB's:<a class="headerlink" href="#blastdbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLAST databases downloaded from NCBI.
</code></pre></div>
<h3 id="blastdb">BLASTDB:<a class="headerlink" href="#blastdb" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLAST databases downloaded from NCBI.
</code></pre></div>
<h3 id="blats">BLAT's:<a class="headerlink" href="#blats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more.
</code></pre></div>
<h3 id="blat">BLAT:<a class="headerlink" href="#blat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more.
</code></pre></div>
<h3 id="bliss">BLIS's:<a class="headerlink" href="#bliss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLIS is a portable software framework for instantiating high-performance
</code></pre></div>
<h3 id="blis">BLIS:<a class="headerlink" href="#blis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BLIS is a portable software framework for instantiating high-performance
</code></pre></div>
<h3 id="bolt-lmms">BOLT-LMM's:<a class="headerlink" href="#bolt-lmms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BOLT-LMM algorithm for mixed model association testing,
</code></pre></div>
<h3 id="bolt-lmm">BOLT-LMM:<a class="headerlink" href="#bolt-lmm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BOLT-LMM algorithm for mixed model association testing,
</code></pre></div>
<h3 id="brakers">BRAKER's:<a class="headerlink" href="#brakers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET
</code></pre></div>
<h3 id="braker">BRAKER:<a class="headerlink" href="#braker" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET
</code></pre></div>
<h3 id="buscos">BUSCO's:<a class="headerlink" href="#buscos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs
</code></pre></div>
<h3 id="busco">BUSCO:<a class="headerlink" href="#busco" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs
</code></pre></div>
<h3 id="bwas">BWA's:<a class="headerlink" href="#bwas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
</code></pre></div>
<h3 id="bwa">BWA:<a class="headerlink" href="#bwa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Burrows-Wheeler Aligner (BWA) is an efficient program that aligns
</code></pre></div>
<h3 id="bamtoolss">BamTools's:<a class="headerlink" href="#bamtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BamTools provides both a programmer&#39;s API and an end-user&#39;s toolkit for handling BAM files.
</code></pre></div>
<h3 id="bamtools">BamTools:<a class="headerlink" href="#bamtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>BamTools provides both a programmer&#39;s API and an end-user&#39;s toolkit for handling BAM files.
</code></pre></div>
<h3 id="bandages">Bandage's:<a class="headerlink" href="#bandages" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bandage is a program for visualising de novo assembly graphs
</code></pre></div>
<h3 id="bandage">Bandage:<a class="headerlink" href="#bandage" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bandage is a program for visualising de novo assembly graphs
</code></pre></div>
<h3 id="basilisks">Basilisk's:<a class="headerlink" href="#basilisks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Basilisk is a Free Software program for the solution of partial differential equations on adaptive Cartesian meshes.
</code></pre></div>
<h3 id="basilisk">Basilisk:<a class="headerlink" href="#basilisk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Basilisk is a Free Software program for the solution of partial differential equations on adaptive Cartesian meshes.
</code></pre></div>
<h3 id="baypasss">BayPass's:<a class="headerlink" href="#baypasss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-Wide Scan for Adaptive Differentiation and Association Analysis with population-specific covariables
</code></pre></div>
<h3 id="baypass">BayPass:<a class="headerlink" href="#baypass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-Wide Scan for Adaptive Differentiation and Association Analysis with population-specific covariables
</code></pre></div>
<h3 id="bayescans">BayeScan's:<a class="headerlink" href="#bayescans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identify candidate loci under natural selection from genetic data,
</code></pre></div>
<h3 id="bayescan">BayeScan:<a class="headerlink" href="#bayescan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identify candidate loci under natural selection from genetic data,
</code></pre></div>
<h3 id="bayesasss">BayesAss's:<a class="headerlink" href="#bayesasss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for inference of recent immigration rates between populations using unlinked multilocus genotypes
</code></pre></div>
<h3 id="bayesass">BayesAss:<a class="headerlink" href="#bayesass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for inference of recent immigration rates between populations using unlinked multilocus genotypes
</code></pre></div>
<h3 id="bazels">Bazel's:<a class="headerlink" href="#bazels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bazel is a build tool that builds code quickly and reliably.
</code></pre></div>
<h3 id="bazel">Bazel:<a class="headerlink" href="#bazel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bazel is a build tool that builds code quickly and reliably.
</code></pre></div>
<h3 id="beagles">Beagle's:<a class="headerlink" href="#beagles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Package for phasing genotypes and for imputing ungenotyped markers.
</code></pre></div>
<h3 id="beagle">Beagle:<a class="headerlink" href="#beagle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Package for phasing genotypes and for imputing ungenotyped markers.
</code></pre></div>
<h3 id="berkeleygws">BerkeleyGW's:<a class="headerlink" href="#berkeleygws" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
</code></pre></div>
<h3 id="berkeleygw">BerkeleyGW:<a class="headerlink" href="#berkeleygw" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
</code></pre></div>
<h3 id="bifrosts">Bifrost's:<a class="headerlink" href="#bifrosts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Highly parallel construction, indexing and querying of colored and compacted de Bruijn graphs.
</code></pre></div>
<h3 id="bifrost">Bifrost:<a class="headerlink" href="#bifrost" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Highly parallel construction, indexing and querying of colored and compacted de Bruijn graphs.
</code></pre></div>
<h3 id="bio-db-bigfiles">Bio-DB-BigFile's:<a class="headerlink" href="#bio-db-bigfiles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Read BigWig and BigBed genome feature databases
</code></pre></div>
<h3 id="bio-db-bigfile">Bio-DB-BigFile:<a class="headerlink" href="#bio-db-bigfile" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Read BigWig and BigBed genome feature databases
</code></pre></div>
<h3 id="bio-db-htss">Bio-DB-HTS's:<a class="headerlink" href="#bio-db-htss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
</code></pre></div>
<h3 id="bio-db-hts">Bio-DB-HTS:<a class="headerlink" href="#bio-db-hts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Read files using HTSlib including BAM/CRAM, Tabix and BCF database files
</code></pre></div>
<h3 id="bioconductors">BioConductor's:<a class="headerlink" href="#bioconductors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for the analysis of high-throughput genomic data in R.
</code></pre></div>
<h3 id="bioconductor">BioConductor:<a class="headerlink" href="#bioconductor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for the analysis of high-throughput genomic data in R.
</code></pre></div>
<h3 id="bismarks">Bismark's:<a class="headerlink" href="#bismarks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to map bisulfite converted sequence reads and
</code></pre></div>
<h3 id="bismark">Bismark:<a class="headerlink" href="#bismark" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to map bisulfite converted sequence reads and
</code></pre></div>
<h3 id="bisons">Bison's:<a class="headerlink" href="#bisons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
</code></pre></div>
<h3 id="bison">Bison:<a class="headerlink" href="#bison" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bison is a general-purpose parser generator that converts an annotated context-free grammar into a deterministic LR or generalized LR (GLR) parser employing LALR(1) parser tables.
</code></pre></div>
<h3 id="blenderpys">BlenderPy's:<a class="headerlink" href="#blenderpys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Blender provides a pipeline for 3D modeling, rigging, animation, simulation, rendering,
</code></pre></div>
<h3 id="blenderpy">BlenderPy:<a class="headerlink" href="#blenderpy" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Blender provides a pipeline for 3D modeling, rigging, animation, simulation, rendering,
</code></pre></div>
<h3 id="boosts">Boost's:<a class="headerlink" href="#boosts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Boost provides free peer-reviewed portable C++ source libraries.
</code></pre></div>
<h3 id="boost">Boost:<a class="headerlink" href="#boost" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Boost provides free peer-reviewed portable C++ source libraries.
</code></pre></div>
<h3 id="bowties">Bowtie's:<a class="headerlink" href="#bowties" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast, memory-efficient short read aligner.
</code></pre></div>
<h3 id="bowtie">Bowtie:<a class="headerlink" href="#bowtie" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast, memory-efficient short read aligner.
</code></pre></div>
<h3 id="bowtie2s">Bowtie2's:<a class="headerlink" href="#bowtie2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast and memory-efficient tool for aligning
</code></pre></div>
<h3 id="bowtie2">Bowtie2:<a class="headerlink" href="#bowtie2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast and memory-efficient tool for aligning
</code></pre></div>
<h3 id="bpipes">Bpipe's:<a class="headerlink" href="#bpipes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A platform for running big bioinformatics jobs that consist of a series of processing stages
</code></pre></div>
<h3 id="bpipe">Bpipe:<a class="headerlink" href="#bpipe" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A platform for running big bioinformatics jobs that consist of a series of processing stages
</code></pre></div>
<h3 id="brackens">Bracken's:<a class="headerlink" href="#brackens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hghly accurate statistical method that computes the abundance of
</code></pre></div>
<h3 id="bracken">Bracken:<a class="headerlink" href="#bracken" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hghly accurate statistical method that computes the abundance of
</code></pre></div>
<h3 id="breakdancers">BreakDancer's:<a class="headerlink" href="#breakdancers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-wide detection of structural variants from next generation paired-end sequencing reads.
</code></pre></div>
<h3 id="breakdancer">BreakDancer:<a class="headerlink" href="#breakdancer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-wide detection of structural variants from next generation paired-end sequencing reads.
</code></pre></div>
<h3 id="breakseq2s">BreakSeq2's:<a class="headerlink" href="#breakseq2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nucleotide-resolution analysis of structural variants
</code></pre></div>
<h3 id="breakseq2">BreakSeq2:<a class="headerlink" href="#breakseq2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nucleotide-resolution analysis of structural variants
</code></pre></div>
<h3 id="ccls">CCL's:<a class="headerlink" href="#ccls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clozure CL (often called CCL for short) is a free Common Lisp implementation
</code></pre></div>
<h3 id="ccl">CCL:<a class="headerlink" href="#ccl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clozure CL (often called CCL for short) is a free Common Lisp implementation
</code></pre></div>
<h3 id="cd-hits">CD-HIT's:<a class="headerlink" href="#cd-hits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CD-HIT is a very widely used program for clustering and
</code></pre></div>
<h3 id="cd-hit">CD-HIT:<a class="headerlink" href="#cd-hit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CD-HIT is a very widely used program for clustering and
</code></pre></div>
<h3 id="cdos">CDO's:<a class="headerlink" href="#cdos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.
</code></pre></div>
<h3 id="cdo">CDO:<a class="headerlink" href="#cdo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.
</code></pre></div>
<h3 id="cfitsios">CFITSIO's:<a class="headerlink" href="#cfitsios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
</code></pre></div>
<h3 id="cfitsio">CFITSIO:<a class="headerlink" href="#cfitsio" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
</code></pre></div>
<h3 id="cgals">CGAL's:<a class="headerlink" href="#cgals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The goal of the CGAL Open Source Project is to provide easy access to efficient
</code></pre></div>
<h3 id="cgal">CGAL:<a class="headerlink" href="#cgal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The goal of the CGAL Open Source Project is to provide easy access to efficient
</code></pre></div>
<h3 id="cmakes">CMake's:<a class="headerlink" href="#cmakes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.
</code></pre></div>
<h3 id="cmake">CMake:<a class="headerlink" href="#cmake" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.
</code></pre></div>
<h3 id="cnvnators">CNVnator's:<a class="headerlink" href="#cnvnators" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Copy Number Variation discovery and genotyping from depth of read mapping.
</code></pre></div>
<h3 id="cnvnator">CNVnator:<a class="headerlink" href="#cnvnator" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Copy Number Variation discovery and genotyping from depth of read mapping.
</code></pre></div>
<h3 id="cnvpytors">CNVpytor's:<a class="headerlink" href="#cnvpytors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python package and command line tool for CNV/CNA analysis from depth-of-coverage by mapped read
</code></pre></div>
<h3 id="cnvpytor">CNVpytor:<a class="headerlink" href="#cnvpytor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python package and command line tool for CNV/CNA analysis from depth-of-coverage by mapped read
</code></pre></div>
<h3 id="comsols">COMSOL's:<a class="headerlink" href="#comsols" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>COMSOL is a multiphysics solver that provides a unified workflow for electrical, mechanical, fluid, and chemical applications.
</code></pre></div>
<h3 id="comsol">COMSOL:<a class="headerlink" href="#comsol" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>COMSOL is a multiphysics solver that provides a unified workflow for electrical, mechanical, fluid, and chemical applications.
</code></pre></div>
<h3 id="concocts">CONCOCT's:<a class="headerlink" href="#concocts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for unsupervised binning of metagenomic contigs by using nucleotide composition,
</code></pre></div>
<h3 id="concoct">CONCOCT:<a class="headerlink" href="#concoct" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for unsupervised binning of metagenomic contigs by using nucleotide composition,
</code></pre></div>
<h3 id="cp2ks">CP2K's:<a class="headerlink" href="#cp2ks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
</code></pre></div>
<h3 id="cp2k">CP2K:<a class="headerlink" href="#cp2k" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
</code></pre></div>
<h3 id="craminos">CRAMINO's:<a class="headerlink" href="#craminos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool for quick quality assessment of cram and bam files, intended for long read sequencing
</code></pre></div>
<h3 id="cramino">CRAMINO:<a class="headerlink" href="#cramino" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool for quick quality assessment of cram and bam files, intended for long read sequencing
</code></pre></div>
<h3 id="ctpls">CTPL's:<a class="headerlink" href="#ctpls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ Thread Pool Library
</code></pre></div>
<h3 id="ctpl">CTPL:<a class="headerlink" href="#ctpl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ Thread Pool Library
</code></pre></div>
<h3 id="cudas">CUDA's:<a class="headerlink" href="#cudas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.
</code></pre></div>
<h3 id="cunits">CUnit's:<a class="headerlink" href="#cunits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Automated testing framework for C.
</code></pre></div>
<h3 id="cunit">CUnit:<a class="headerlink" href="#cunit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Automated testing framework for C.
</code></pre></div>
<h3 id="canus">Canu's:<a class="headerlink" href="#canus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence assembler designed for high-noise single-molecule sequencing.
</code></pre></div>
<h3 id="canu">Canu:<a class="headerlink" href="#canu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence assembler designed for high-noise single-molecule sequencing.
</code></pre></div>
<h3 id="capnprotos">CapnProto's:<a class="headerlink" href="#capnprotos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast data interchange format and capability-based RPC system.
</code></pre></div>
<h3 id="capnproto">CapnProto:<a class="headerlink" href="#capnproto" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast data interchange format and capability-based RPC system.
</code></pre></div>
<h3 id="catch2s">Catch2's:<a class="headerlink" href="#catch2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A modern, C++-native, header-only,  test framework for unit-tests, TDD and BDD
</code></pre></div>
<h3 id="catch2">Catch2:<a class="headerlink" href="#catch2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A modern, C++-native, header-only,  test framework for unit-tests, TDD and BDD
</code></pre></div>
<h3 id="cellrangers">CellRanger's:<a class="headerlink" href="#cellrangers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cell Ranger is a set of analysis pipelines that process Chromium
</code></pre></div>
<h3 id="cellranger">CellRanger:<a class="headerlink" href="#cellranger" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cell Ranger is a set of analysis pipelines that process Chromium
</code></pre></div>
<h3 id="centrifuges">Centrifuge's:<a class="headerlink" href="#centrifuges" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Classifier for metagenomic sequences
</code></pre></div>
<h3 id="centrifuge">Centrifuge:<a class="headerlink" href="#centrifuge" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Classifier for metagenomic sequences
</code></pre></div>
<h3 id="cereals">Cereal's:<a class="headerlink" href="#cereals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++11 serialization library
</code></pre></div>
<h3 id="cereal">Cereal:<a class="headerlink" href="#cereal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++11 serialization library
</code></pre></div>
<h3 id="charms">Charm++'s:<a class="headerlink" href="#charms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An asynchronous message-driven C++ communication library targeted towards tightly coupled, high-performance parallel machines.
</code></pre></div>
<h3 id="charm">Charm++:<a class="headerlink" href="#charm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An asynchronous message-driven C++ communication library targeted towards tightly coupled, high-performance parallel machines.
</code></pre></div>
<h3 id="checkms">CheckM's:<a class="headerlink" href="#checkms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
</code></pre></div>
<h3 id="checkm">CheckM:<a class="headerlink" href="#checkm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
</code></pre></div>
<h3 id="checkm2s">CheckM2's:<a class="headerlink" href="#checkm2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid assessment of genome bin quality using machine learning
</code></pre></div>
<h3 id="checkm2">CheckM2:<a class="headerlink" href="#checkm2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid assessment of genome bin quality using machine learning
</code></pre></div>
<h3 id="checkvs">CheckV's:<a class="headerlink" href="#checkvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assess the quality of metagenome-assembled viral genomes.
</code></pre></div>
<h3 id="checkv">CheckV:<a class="headerlink" href="#checkv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assess the quality of metagenome-assembled viral genomes.
</code></pre></div>
<h3 id="circlators">Circlator's:<a class="headerlink" href="#circlators" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to circularize genome assemblies
</code></pre></div>
<h3 id="circlator">Circlator:<a class="headerlink" href="#circlator" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to circularize genome assemblies
</code></pre></div>
<h3 id="circoss">Circos's:<a class="headerlink" href="#circoss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Package for visualizing data in a circular layout - this makes Circos ideal for exploring
</code></pre></div>
<h3 id="circos">Circos:<a class="headerlink" href="#circos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Package for visualizing data in a circular layout - this makes Circos ideal for exploring
</code></pre></div>
<h3 id="clair3s">Clair3's:<a class="headerlink" href="#clair3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Syumphonizing pileup and full-alignment for high-performance long-read variant calling.
</code></pre></div>
<h3 id="clair3">Clair3:<a class="headerlink" href="#clair3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Syumphonizing pileup and full-alignment for high-performance long-read variant calling.
</code></pre></div>
<h3 id="clustal-omegas">Clustal-Omega's:<a class="headerlink" href="#clustal-omegas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clustal Omega is a multiple sequence alignment
</code></pre></div>
<h3 id="clustal-omega">Clustal-Omega:<a class="headerlink" href="#clustal-omega" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clustal Omega is a multiple sequence alignment
</code></pre></div>
<h3 id="clustalw2s">ClustalW2's:<a class="headerlink" href="#clustalw2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.
</code></pre></div>
<h3 id="clustalw2">ClustalW2:<a class="headerlink" href="#clustalw2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.
</code></pre></div>
<h3 id="corsets">Corset's:<a class="headerlink" href="#corsets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clusters contigs and counts reads from de novo assembled transcriptomes.
</code></pre></div>
<h3 id="corset">Corset:<a class="headerlink" href="#corset" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Clusters contigs and counts reads from de novo assembled transcriptomes.
</code></pre></div>
<h3 id="coverms">CoverM's:<a class="headerlink" href="#coverms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DNA read coverage and relative abundance calculator focused on metagenomics applications
</code></pre></div>
<h3 id="coverm">CoverM:<a class="headerlink" href="#coverm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DNA read coverage and relative abundance calculator focused on metagenomics applications
</code></pre></div>
<h3 id="cppunits">CppUnit's:<a class="headerlink" href="#cppunits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ port of the JUnit framework for unit testing.
</code></pre></div>
<h3 id="cppunit">CppUnit:<a class="headerlink" href="#cppunit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ port of the JUnit framework for unit testing.
</code></pre></div>
<h3 id="craycces">CrayCCE's:<a class="headerlink" href="#craycces" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: February 2023).
</code></pre></div>
<h3 id="craycce">CrayCCE:<a class="headerlink" href="#craycce" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: February 2023).
</code></pre></div>
<h3 id="craygnus">CrayGNU's:<a class="headerlink" href="#craygnus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: 23.02).
</code></pre></div>
<h3 id="craygnu">CrayGNU:<a class="headerlink" href="#craygnu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: 23.02).
</code></pre></div>
<h3 id="crayintels">CrayIntel's:<a class="headerlink" href="#crayintels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: February 2023 with Intel 19 compiler).
</code></pre></div>
<h3 id="crayintel">CrayIntel:<a class="headerlink" href="#crayintel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: February 2023 with Intel 19 compiler).
</code></pre></div>
<h3 id="cubeguis">CubeGUI's:<a class="headerlink" href="#cubeguis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Graphical report explorer report explorer for Scalasca and Score-P
</code></pre></div>
<h3 id="cubegui">CubeGUI:<a class="headerlink" href="#cubegui" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Graphical report explorer report explorer for Scalasca and Score-P
</code></pre></div>
<h3 id="cubelibs">CubeLib's:<a class="headerlink" href="#cubelibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cube general purpose C++ library component and command-line tools.
</code></pre></div>
<h3 id="cubelib">CubeLib:<a class="headerlink" href="#cubelib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cube general purpose C++ library component and command-line tools.
</code></pre></div>
<h3 id="cubewriters">CubeWriter's:<a class="headerlink" href="#cubewriters" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cube high-performance C writer library component.
</code></pre></div>
<h3 id="cubewriter">CubeWriter:<a class="headerlink" href="#cubewriter" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cube high-performance C writer library component.
</code></pre></div>
<h3 id="cufflinkss">Cufflinks's:<a class="headerlink" href="#cufflinkss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transcript assembly, differential expression, and differential regulation for RNA-Seq
</code></pre></div>
<h3 id="cufflinks">Cufflinks:<a class="headerlink" href="#cufflinks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transcript assembly, differential expression, and differential regulation for RNA-Seq
</code></pre></div>
<h3 id="cytoscapes">Cytoscape's:<a class="headerlink" href="#cytoscapes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cytoscape is an open source software platform for visualizing molecular interaction networks and
</code></pre></div>
<h3 id="cytoscape">Cytoscape:<a class="headerlink" href="#cytoscape" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cytoscape is an open source software platform for visualizing molecular interaction networks and
</code></pre></div>
<h3 id="d-geniess">D-Genies's:<a class="headerlink" href="#d-geniess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>D-Genies also allows to display dot plots from other aligners by uploading their PAF or MAF alignment file.
</code></pre></div>
<h3 id="d-genies">D-Genies:<a class="headerlink" href="#d-genies" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>D-Genies also allows to display dot plots from other aligners by uploading their PAF or MAF alignment file.
</code></pre></div>
<h3 id="das_tools">DAS_Tool's:<a class="headerlink" href="#das_tools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DAS Tool is an automated method that integrates the results of a flexible number of binning
</code></pre></div>
<h3 id="das_tool">DAS_Tool:<a class="headerlink" href="#das_tool" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DAS Tool is an automated method that integrates the results of a flexible number of binning
</code></pre></div>
<h3 id="dbs">DB's:<a class="headerlink" href="#dbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Berkeley DB enables the development of custom data management solutions,
</code></pre></div>
<h3 id="db">DB:<a class="headerlink" href="#db" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Berkeley DB enables the development of custom data management solutions,
</code></pre></div>
<h3 id="dbg2olcs">DBG2OLC's:<a class="headerlink" href="#dbg2olcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation
</code></pre></div>
<h3 id="dbg2olc">DBG2OLC:<a class="headerlink" href="#dbg2olc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation
</code></pre></div>
<h3 id="diamonds">DIAMOND's:<a class="headerlink" href="#diamonds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence aligner for protein and translated DNA searches
</code></pre></div>
<h3 id="diamond">DIAMOND:<a class="headerlink" href="#diamond" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence aligner for protein and translated DNA searches
</code></pre></div>
<h3 id="discovardenovos">DISCOVARdenovo's:<a class="headerlink" href="#discovardenovos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembler suitable for large genomes based on Illumina reads of length 250 or longer.
</code></pre></div>
<h3 id="discovardenovo">DISCOVARdenovo:<a class="headerlink" href="#discovardenovo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembler suitable for large genomes based on Illumina reads of length 250 or longer.
</code></pre></div>
<h3 id="drams">DRAM's:<a class="headerlink" href="#drams" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for annotating metagenomic assembled genomes and VirSorter identified viral contigs..
</code></pre></div>
<h3 id="dram">DRAM:<a class="headerlink" href="#dram" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for annotating metagenomic assembled genomes and VirSorter identified viral contigs..
</code></pre></div>
<h3 id="dalilites">DaliLite's:<a class="headerlink" href="#dalilites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,
</code></pre></div>
<h3 id="dalilite">DaliLite:<a class="headerlink" href="#dalilite" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,
</code></pre></div>
<h3 id="deconseqs">DeconSeq's:<a class="headerlink" href="#deconseqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool that can be used to automatically detect and efficiently remove sequence contaminations
</code></pre></div>
<h3 id="deconseq">DeconSeq:<a class="headerlink" href="#deconseq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool that can be used to automatically detect and efficiently remove sequence contaminations
</code></pre></div>
<h3 id="deeplabcuts">DeepLabCut's:<a class="headerlink" href="#deeplabcuts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient method for 3D markerless pose estimation based on transfer learning with deep neural networks.
</code></pre></div>
<h3 id="deeplabcut">DeepLabCut:<a class="headerlink" href="#deeplabcut" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient method for 3D markerless pose estimation based on transfer learning with deep neural networks.
</code></pre></div>
<h3 id="defaultmoduless">DefaultModules's:<a class="headerlink" href="#defaultmoduless" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Defines the set of modules loaded by default
</code></pre></div>
<h3 id="defaultmodules">DefaultModules:<a class="headerlink" href="#defaultmodules" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Defines the set of modules loaded by default
</code></pre></div>
<h3 id="delft3ds">Delft3D's:<a class="headerlink" href="#delft3ds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Integrated simulation of sediment transport and morphology, waves, water quality and ecology.
</code></pre></div>
<h3 id="delft3d">Delft3D:<a class="headerlink" href="#delft3d" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Integrated simulation of sediment transport and morphology, waves, water quality and ecology.
</code></pre></div>
<h3 id="delft3d_fms">Delft3D_FM's:<a class="headerlink" href="#delft3d_fms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>3D modeling suite to investigate hydrodynamics, sediment transport and morphology and water quality for fluvial, estuarine and coastal environments
</code></pre></div>
<h3 id="delft3d_fm">Delft3D_FM:<a class="headerlink" href="#delft3d_fm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>3D modeling suite to investigate hydrodynamics, sediment transport and morphology and water quality for fluvial, estuarine and coastal environments
</code></pre></div>
<h3 id="dellys">Delly's:<a class="headerlink" href="#dellys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Structural variant discovery by integrated paired-end and split-read analysis
</code></pre></div>
<h3 id="delly">Delly:<a class="headerlink" href="#delly" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Structural variant discovery by integrated paired-end and split-read analysis
</code></pre></div>
<h3 id="dorados">Dorado's:<a class="headerlink" href="#dorados" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
</code></pre></div>
<h3 id="dorado">Dorado:<a class="headerlink" href="#dorado" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.
</code></pre></div>
<h3 id="dsuites">Dsuite's:<a class="headerlink" href="#dsuites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast calculation of the ABBA-BABA statistics across many populations/species
</code></pre></div>
<h3 id="dsuite">Dsuite:<a class="headerlink" href="#dsuite" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast calculation of the ABBA-BABA statistics across many populations/species
</code></pre></div>
<h3 id="edtas">EDTA's:<a class="headerlink" href="#edtas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Automated whole-genome de-novo TE annotation and benchmarking the annotation performance of TE libraries.
</code></pre></div>
<h3 id="edta">EDTA:<a class="headerlink" href="#edta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Automated whole-genome de-novo TE annotation and benchmarking the annotation performance of TE libraries.
</code></pre></div>
<h3 id="eigensofts">EIGENSOFT's:<a class="headerlink" href="#eigensofts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.
</code></pre></div>
<h3 id="eigensoft">EIGENSOFT:<a class="headerlink" href="#eigensoft" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.
</code></pre></div>
<h3 id="elpas">ELPA's:<a class="headerlink" href="#elpas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eigenvalue SoLvers for Petaflop-Applications .
</code></pre></div>
<h3 id="elpa">ELPA:<a class="headerlink" href="#elpa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eigenvalue SoLvers for Petaflop-Applications .
</code></pre></div>
<h3 id="emans">EMAN's:<a class="headerlink" href="#emans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EMAN is a powerful image processing library as well as a complete software suite
</code></pre></div>
<h3 id="eman">EMAN:<a class="headerlink" href="#eman" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EMAN is a powerful image processing library as well as a complete software suite
</code></pre></div>
<h3 id="eman2s">EMAN2's:<a class="headerlink" href="#eman2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Greyscale scientific image processing suite with a primary focus on processing data from transmission electron microscopes
</code></pre></div>
<h3 id="eman2">EMAN2:<a class="headerlink" href="#eman2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Greyscale scientific image processing suite with a primary focus on processing data from transmission electron microscopes
</code></pre></div>
<h3 id="embosss">EMBOSS's:<a class="headerlink" href="#embosss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EMBOSS is &#39;The European Molecular Biology Open Software Suite&#39;.
</code></pre></div>
<h3 id="emboss">EMBOSS:<a class="headerlink" href="#emboss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EMBOSS is &#39;The European Molecular Biology Open Software Suite&#39;.
</code></pre></div>
<h3 id="emirges">EMIRGE's:<a class="headerlink" href="#emirges" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reconstructs full length ribosomal genes from short read
</code></pre></div>
<h3 id="emirge">EMIRGE:<a class="headerlink" href="#emirge" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reconstructs full length ribosomal genes from short read
</code></pre></div>
<h3 id="enmtmls">ENMTML's:<a class="headerlink" href="#enmtmls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R package for integrated construction of Ecological Niche Models.
</code></pre></div>
<h3 id="enmtml">ENMTML:<a class="headerlink" href="#enmtml" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R package for integrated construction of Ecological Niche Models.
</code></pre></div>
<h3 id="esmfs">ESMF's:<a class="headerlink" href="#esmfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Earth System Modeling Framework (ESMF) is software for building and coupling weather,  climate, and related models.
</code></pre></div>
<h3 id="esmf">ESMF:<a class="headerlink" href="#esmf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Earth System Modeling Framework (ESMF) is software for building and coupling weather,  climate, and related models.
</code></pre></div>
<h3 id="etes">ETE's:<a class="headerlink" href="#etes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Python framework for the analysis and visualization of phylogenetic trees
</code></pre></div>
<h3 id="ete">ETE:<a class="headerlink" href="#ete" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Python framework for the analysis and visualization of phylogenetic trees
</code></pre></div>
<h3 id="easybuilds">EasyBuild's:<a class="headerlink" href="#easybuilds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EasyBuild is a software build and installation framework
</code></pre></div>
<h3 id="easybuild">EasyBuild:<a class="headerlink" href="#easybuild" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>EasyBuild is a software build and installation framework
</code></pre></div>
<h3 id="eigens">Eigen's:<a class="headerlink" href="#eigens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eigen is a C++ template library for linear algebra:
</code></pre></div>
<h3 id="eigen">Eigen:<a class="headerlink" href="#eigen" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eigen is a C++ template library for linear algebra:
</code></pre></div>
<h3 id="elmers">Elmer's:<a class="headerlink" href="#elmers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Elmer is an open source multiphysical simulation software mainly developed by
</code></pre></div>
<h3 id="elmer">Elmer:<a class="headerlink" href="#elmer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Elmer is an open source multiphysical simulation software mainly developed by
</code></pre></div>
<h3 id="embrees">Embree's:<a class="headerlink" href="#embrees" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree&#39;s performance-optimized ray tracing kernels.
</code></pre></div>
<h3 id="embree">Embree:<a class="headerlink" href="#embree" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Embree is a collection of high-performance ray tracing kernels, developed at Intel. The target users of Embree are graphics application engineers who want to improve the performance of their photo-realistic rendering application by leveraging Embree&#39;s performance-optimized ray tracing kernels.
</code></pre></div>
<h3 id="energypluss">EnergyPlus's:<a class="headerlink" href="#energypluss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Energy simulation program used to model energy consumption and water use in buildings.
</code></pre></div>
<h3 id="energyplus">EnergyPlus:<a class="headerlink" href="#energyplus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Energy simulation program used to model energy consumption and water use in buildings.
</code></pre></div>
<h3 id="erlangotps">ErlangOTP's:<a class="headerlink" href="#erlangotps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Erlang is a programming language used to build massively scalable
</code></pre></div>
<h3 id="erlangotp">ErlangOTP:<a class="headerlink" href="#erlangotp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Erlang is a programming language used to build massively scalable
</code></pre></div>
<h3 id="eukrep-eukccs">EukRep-EukCC's:<a class="headerlink" href="#eukrep-eukccs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Completeness and contamination estimator for metagenomic assembled microbial eukaryotic genomes.
</code></pre></div>
<h3 id="eukrep-eukcc">EukRep-EukCC:<a class="headerlink" href="#eukrep-eukcc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Completeness and contamination estimator for metagenomic assembled microbial eukaryotic genomes.
</code></pre></div>
<h3 id="exabayess">ExaBayes's:<a class="headerlink" href="#exabayess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bayesian tree inference, particularly suitable for large-scale analyses.
</code></pre></div>
<h3 id="exabayes">ExaBayes:<a class="headerlink" href="#exabayes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bayesian tree inference, particularly suitable for large-scale analyses.
</code></pre></div>
<h3 id="examls">ExaML's:<a class="headerlink" href="#examls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Exascale Maximum Likelihood for phylogenetic inference using MPI.
</code></pre></div>
<h3 id="examl">ExaML:<a class="headerlink" href="#examl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Exascale Maximum Likelihood for phylogenetic inference using MPI.
</code></pre></div>
<h3 id="extraes">Extrae's:<a class="headerlink" href="#extraes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches
</code></pre></div>
<h3 id="extrae">Extrae:<a class="headerlink" href="#extrae" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches
</code></pre></div>
<h3 id="falcons">FALCON's:<a class="headerlink" href="#falcons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Falcon: a set of tools for fast aligning long reads for consensus and assembly
</code></pre></div>
<h3 id="falcon">FALCON:<a class="headerlink" href="#falcon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Falcon: a set of tools for fast aligning long reads for consensus and assembly
</code></pre></div>
<h3 id="fastx-toolkits">FASTX-Toolkit's:<a class="headerlink" href="#fastx-toolkits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for Short-Reads FASTA/FASTQ files preprocessing.
</code></pre></div>
<h3 id="fastx-toolkit">FASTX-Toolkit:<a class="headerlink" href="#fastx-toolkit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for Short-Reads FASTA/FASTQ files preprocessing.
</code></pre></div>
<h3 id="fcms">FCM's:<a class="headerlink" href="#fcms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FCM Build - A powerful build system for modern Fortran software applications. FCM Version Control - Wrappers to the Subversion version control system, usage conventions and processes for scientific software development.
</code></pre></div>
<h3 id="fcm">FCM:<a class="headerlink" href="#fcm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FCM Build - A powerful build system for modern Fortran software applications. FCM Version Control - Wrappers to the Subversion version control system, usage conventions and processes for scientific software development.
</code></pre></div>
<h3 id="fdss">FDS's:<a class="headerlink" href="#fdss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.
</code></pre></div>
<h3 id="fds">FDS:<a class="headerlink" href="#fds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.
</code></pre></div>
<h3 id="fftws">FFTW's:<a class="headerlink" href="#fftws" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.
</code></pre></div>
<h3 id="fftw">FFTW:<a class="headerlink" href="#fftw" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.
</code></pre></div>
<h3 id="ffmpegs">FFmpeg's:<a class="headerlink" href="#ffmpegs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A complete, cross-platform solution to record, convert and stream audio and video.
</code></pre></div>
<h3 id="ffmpeg">FFmpeg:<a class="headerlink" href="#ffmpeg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A complete, cross-platform solution to record, convert and stream audio and video.
</code></pre></div>
<h3 id="fltks">FLTK's:<a class="headerlink" href="#fltks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
</code></pre></div>
<h3 id="fltk">FLTK:<a class="headerlink" href="#fltk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
</code></pre></div>
<h3 id="ftgls">FTGL's:<a class="headerlink" href="#ftgls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FTGL is a free cross-platform Open Source C++ library that uses Freetype2 to simplify rendering fonts in OpenGL applications. FTGL supports bitmaps, pixmaps, texture maps, outlines, polygon mesh, and extruded polygon rendering modes.
</code></pre></div>
<h3 id="ftgl">FTGL:<a class="headerlink" href="#ftgl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FTGL is a free cross-platform Open Source C++ library that uses Freetype2 to simplify rendering fonts in OpenGL applications. FTGL supports bitmaps, pixmaps, texture maps, outlines, polygon mesh, and extruded polygon rendering modes.
</code></pre></div>
<h3 id="fastanis">FastANI's:<a class="headerlink" href="#fastanis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for fast alignment-free computation of
</code></pre></div>
<h3 id="fastani">FastANI:<a class="headerlink" href="#fastani" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for fast alignment-free computation of
</code></pre></div>
<h3 id="fastmes">FastME's:<a class="headerlink" href="#fastmes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.
</code></pre></div>
<h3 id="fastme">FastME:<a class="headerlink" href="#fastme" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.
</code></pre></div>
<h3 id="fastqcs">FastQC's:<a class="headerlink" href="#fastqcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools (in Java) for working with next generation sequencing data in the BAM format.
</code></pre></div>
<h3 id="fastqc">FastQC:<a class="headerlink" href="#fastqc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools (in Java) for working with next generation sequencing data in the BAM format.
</code></pre></div>
<h3 id="fastq_screens">FastQ_Screen's:<a class="headerlink" href="#fastq_screens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastQ Screen allows you to screen a library of sequences in FastQ
</code></pre></div>
<h3 id="fastq_screen">FastQ_Screen:<a class="headerlink" href="#fastq_screen" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastQ Screen allows you to screen a library of sequences in FastQ
</code></pre></div>
<h3 id="fasttrees">FastTree's:<a class="headerlink" href="#fasttrees" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
</code></pre></div>
<h3 id="fasttree">FastTree:<a class="headerlink" href="#fasttree" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide
</code></pre></div>
<h3 id="file-renames">File-Rename's:<a class="headerlink" href="#file-renames" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Perl version of the rename utility, with support for regular expressions.
</code></pre></div>
<h3 id="file-rename">File-Rename:<a class="headerlink" href="#file-rename" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Perl version of the rename utility, with support for regular expressions.
</code></pre></div>
<h3 id="filtlongs">Filtlong's:<a class="headerlink" href="#filtlongs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for filtering long reads by quality.
</code></pre></div>
<h3 id="filtlong">Filtlong:<a class="headerlink" href="#filtlong" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for filtering long reads by quality.
</code></pre></div>
<h3 id="fimtypers">FimTyper's:<a class="headerlink" href="#fimtypers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identifies the FimH type in total or partial sequenced isolates of E. coli..
</code></pre></div>
<h3 id="fimtyper">FimTyper:<a class="headerlink" href="#fimtyper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identifies the FimH type in total or partial sequenced isolates of E. coli..
</code></pre></div>
<h3 id="flexiblass">FlexiBLAS's:<a class="headerlink" href="#flexiblass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
</code></pre></div>
<h3 id="flexiblas">FlexiBLAS:<a class="headerlink" href="#flexiblas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
</code></pre></div>
<h3 id="flyes">Flye's:<a class="headerlink" href="#flyes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
</code></pre></div>
<h3 id="flye">Flye:<a class="headerlink" href="#flye" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
</code></pre></div>
<h3 id="foxs">FoX's:<a class="headerlink" href="#foxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FoX is an XML library written in Fortran 95.
</code></pre></div>
<h3 id="fox">FoX:<a class="headerlink" href="#fox" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FoX is an XML library written in Fortran 95.
</code></pre></div>
<h3 id="fraggenescans">FragGeneScan's:<a class="headerlink" href="#fraggenescans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FragGeneScan is an application for finding (fragmented) genes in short reads.
</code></pre></div>
<h3 id="fraggenescan">FragGeneScan:<a class="headerlink" href="#fraggenescan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FragGeneScan is an application for finding (fragmented) genes in short reads.
</code></pre></div>
<h3 id="freebayess">FreeBayes's:<a class="headerlink" href="#freebayess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.
</code></pre></div>
<h3 id="freebayes">FreeBayes:<a class="headerlink" href="#freebayes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.
</code></pre></div>
<h3 id="freesurfers">FreeSurfer's:<a class="headerlink" href="#freesurfers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
</code></pre></div>
<h3 id="freesurfer">FreeSurfer:<a class="headerlink" href="#freesurfer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data.
</code></pre></div>
<h3 id="fribidis">FriBidi's:<a class="headerlink" href="#fribidis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Free Implementation of the Unicode Bidirectional Algorithm.
</code></pre></div>
<h3 id="fribidi">FriBidi:<a class="headerlink" href="#fribidi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Free Implementation of the Unicode Bidirectional Algorithm.
</code></pre></div>
<h3 id="gatks">GATK's:<a class="headerlink" href="#gatks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
</code></pre></div>
<h3 id="gatk">GATK:<a class="headerlink" href="#gatk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
</code></pre></div>
<h3 id="gccs">GCC's:<a class="headerlink" href="#gccs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
</code></pre></div>
<h3 id="gcc">GCC:<a class="headerlink" href="#gcc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
</code></pre></div>
<h3 id="gcccores">GCCcore's:<a class="headerlink" href="#gcccores" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
</code></pre></div>
<h3 id="gcccore">GCCcore:<a class="headerlink" href="#gcccore" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
</code></pre></div>
<h3 id="gds">GD's:<a class="headerlink" href="#gds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Interface to Gd Graphics Library
</code></pre></div>
<h3 id="gd">GD:<a class="headerlink" href="#gd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Interface to Gd Graphics Library
</code></pre></div>
<h3 id="gdals">GDAL's:<a class="headerlink" href="#gdals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful command-line utilities for  data translation and processing.  NOTE: The GDAL IO cache by default uses 5% of total memory. This seems not necessary. This module sets GDAL_CACHEMAX=256 (256MB),   which should have no performance impact. Feel free to change if necessary, using &#39;export GDAL_CACHEMAX=xxx&#39; (in your job script)   after loading the GDAL module.
</code></pre></div>
<h3 id="gdal">GDAL:<a class="headerlink" href="#gdal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful command-line utilities for  data translation and processing.  NOTE: The GDAL IO cache by default uses 5% of total memory. This seems not necessary. This module sets GDAL_CACHEMAX=256 (256MB),   which should have no performance impact. Feel free to change if necessary, using &#39;export GDAL_CACHEMAX=xxx&#39; (in your job script)   after loading the GDAL module.
</code></pre></div>
<h3 id="gemmas">GEMMA's:<a class="headerlink" href="#gemmas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-wide Efficient Mixed Model Association
</code></pre></div>
<h3 id="gemma">GEMMA:<a class="headerlink" href="#gemma" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-wide Efficient Mixed Model Association
</code></pre></div>
<h3 id="geoss">GEOS's:<a class="headerlink" href="#geoss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)
</code></pre></div>
<h3 id="geos">GEOS:<a class="headerlink" href="#geos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)
</code></pre></div>
<h3 id="glms">GLM's:<a class="headerlink" href="#glms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on
</code></pre></div>
<h3 id="glm">GLM:<a class="headerlink" href="#glm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on
</code></pre></div>
<h3 id="glpks">GLPK's:<a class="headerlink" href="#glpks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Linear Programming Kit is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.
</code></pre></div>
<h3 id="glpk">GLPK:<a class="headerlink" href="#glpk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Linear Programming Kit is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.
</code></pre></div>
<h3 id="glibs">GLib's:<a class="headerlink" href="#glibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GLib is one of the base libraries of the GTK+ project
</code></pre></div>
<h3 id="glib">GLib:<a class="headerlink" href="#glib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GLib is one of the base libraries of the GTK+ project
</code></pre></div>
<h3 id="gmap-gsnaps">GMAP-GSNAP's:<a class="headerlink" href="#gmap-gsnaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
</code></pre></div>
<h3 id="gmap-gsnap">GMAP-GSNAP:<a class="headerlink" href="#gmap-gsnap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences
</code></pre></div>
<h3 id="gmps">GMP's:<a class="headerlink" href="#gmps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMP is a free library for arbitrary precision arithmetic,
</code></pre></div>
<h3 id="gmp">GMP:<a class="headerlink" href="#gmp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMP is a free library for arbitrary precision arithmetic,
</code></pre></div>
<h3 id="gmts">GMT's:<a class="headerlink" href="#gmts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools.
</code></pre></div>
<h3 id="gmt">GMT:<a class="headerlink" href="#gmt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools.
</code></pre></div>
<h3 id="golds">GOLD's:<a class="headerlink" href="#golds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A genetic algorithm for docking flexible ligands into protein binding sites
</code></pre></div>
<h3 id="gold">GOLD:<a class="headerlink" href="#gold" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A genetic algorithm for docking flexible ligands into protein binding sites
</code></pre></div>
<h3 id="gobject-introspections">GObject-Introspection's:<a class="headerlink" href="#gobject-introspections" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GObject introspection is a middleware layer between C libraries
</code></pre></div>
<h3 id="gobject-introspection">GObject-Introspection:<a class="headerlink" href="#gobject-introspection" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GObject introspection is a middleware layer between C libraries
</code></pre></div>
<h3 id="gradss">GRADS's:<a class="headerlink" href="#gradss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Grid Analysis and Display System (GrADS) is an interactive desktop tool that is used for easy access, manipulation, and visualization of earth science data.
</code></pre></div>
<h3 id="grads">GRADS:<a class="headerlink" href="#grads" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Grid Analysis and Display System (GrADS) is an interactive desktop tool that is used for easy access, manipulation, and visualization of earth science data.
</code></pre></div>
<h3 id="grasss">GRASS's:<a class="headerlink" href="#grasss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization
</code></pre></div>
<h3 id="grass">GRASS:<a class="headerlink" href="#grass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Geographic Resources Analysis Support System - used for geospatial data management and analysis, image processing, graphics and maps production, spatial modeling, and visualization
</code></pre></div>
<h3 id="gridsss">GRIDSS's:<a class="headerlink" href="#gridsss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements.
</code></pre></div>
<h3 id="gridss">GRIDSS:<a class="headerlink" href="#gridss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements.
</code></pre></div>
<h3 id="gromacss">GROMACS's:<a class="headerlink" href="#gromacss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
</code></pre></div>
<h3 id="gromacs">GROMACS:<a class="headerlink" href="#gromacs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
</code></pre></div>
<h3 id="gsls">GSL's:<a class="headerlink" href="#gsls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Scientific Library (GSL) is a numerical library for C and C++  programmers.  The library provides a wide range of mathematical routines  such as random number generators, special functions and least-squares fitting.
</code></pre></div>
<h3 id="gsl">GSL:<a class="headerlink" href="#gsl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Scientific Library (GSL) is a numerical library for C and C++  programmers.  The library provides a wide range of mathematical routines  such as random number generators, special functions and least-squares fitting.
</code></pre></div>
<h3 id="gst-plugins-bases">GST-plugins-base's:<a class="headerlink" href="#gst-plugins-bases" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GStreamer plug-ins and elements.
</code></pre></div>
<h3 id="gst-plugins-base">GST-plugins-base:<a class="headerlink" href="#gst-plugins-base" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GStreamer plug-ins and elements.
</code></pre></div>
<h3 id="gstreamers">GStreamer's:<a class="headerlink" href="#gstreamers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.
</code></pre></div>
<h3 id="gstreamer">GStreamer:<a class="headerlink" href="#gstreamer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.
</code></pre></div>
<h3 id="gtdb-tks">GTDB-Tk's:<a class="headerlink" href="#gtdb-tks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
</code></pre></div>
<h3 id="gtdb-tk">GTDB-Tk:<a class="headerlink" href="#gtdb-tk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
</code></pre></div>
<h3 id="gtks">GTK+'s:<a class="headerlink" href="#gtks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GTK+ is the primary library used to construct user interfaces in GNOME.
</code></pre></div>
<h3 id="gtk">GTK+:<a class="headerlink" href="#gtk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GTK+ is the primary library used to construct user interfaces in GNOME.
</code></pre></div>
<h3 id="gtss">GTS's:<a class="headerlink" href="#gtss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.
</code></pre></div>
<h3 id="gts">GTS:<a class="headerlink" href="#gts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.
</code></pre></div>
<h3 id="genemark-ess">GeneMark-ES's:<a class="headerlink" href="#genemark-ess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eukaryotic gene prediction suite with automatic training
</code></pre></div>
<h3 id="genemark-es">GeneMark-ES:<a class="headerlink" href="#genemark-es" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eukaryotic gene prediction suite with automatic training
</code></pre></div>
<h3 id="genomethreaders">GenomeThreader's:<a class="headerlink" href="#genomethreaders" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GenomeThreader is a software tool to compute gene structure predictions.
</code></pre></div>
<h3 id="genomethreader">GenomeThreader:<a class="headerlink" href="#genomethreader" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GenomeThreader is a software tool to compute gene structure predictions.
</code></pre></div>
<h3 id="gerriss">Gerris's:<a class="headerlink" href="#gerriss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow. This module also includes GfsView, a visualisation application for Gerris output.
</code></pre></div>
<h3 id="gerris">Gerris:<a class="headerlink" href="#gerris" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow. This module also includes GfsView, a visualisation application for Gerris output.
</code></pre></div>
<h3 id="getorganelles">GetOrganelle's:<a class="headerlink" href="#getorganelles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit to assemble organelle genome from genomic skimming data.
</code></pre></div>
<h3 id="getorganelle">GetOrganelle:<a class="headerlink" href="#getorganelle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit to assemble organelle genome from genomic skimming data.
</code></pre></div>
<h3 id="glimmerhmms">GlimmerHMM's:<a class="headerlink" href="#glimmerhmms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gene finder based on a Generalized Hidden Markov Model.
</code></pre></div>
<h3 id="glimmerhmm">GlimmerHMM:<a class="headerlink" href="#glimmerhmm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gene finder based on a Generalized Hidden Markov Model.
</code></pre></div>
<h3 id="gos">Go's:<a class="headerlink" href="#gos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source programming language
</code></pre></div>
<h3 id="go">Go:<a class="headerlink" href="#go" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source programming language
</code></pre></div>
<h3 id="graphvizs">Graphviz's:<a class="headerlink" href="#graphvizs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Graphviz is open source graph visualization software. Graph visualization
</code></pre></div>
<h3 id="graphviz">Graphviz:<a class="headerlink" href="#graphviz" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Graphviz is open source graph visualization software. Graph visualization
</code></pre></div>
<h3 id="gubbinss">Gubbins's:<a class="headerlink" href="#gubbinss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genealogies Unbiased By recomBinations In Nucleotide Sequences
</code></pre></div>
<h3 id="gubbins">Gubbins:<a class="headerlink" href="#gubbins" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genealogies Unbiased By recomBinations In Nucleotide Sequences
</code></pre></div>
<h3 id="guiles">Guile's:<a class="headerlink" href="#guiles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Guile is the GNU Ubiquitous Intelligent Language for Extensions,
</code></pre></div>
<h3 id="guile">Guile:<a class="headerlink" href="#guile" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Guile is the GNU Ubiquitous Intelligent Language for Extensions,
</code></pre></div>
<h3 id="hdfs">HDF's:<a class="headerlink" href="#hdfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines.
</code></pre></div>
<h3 id="hdf-eoss">HDF-EOS's:<a class="headerlink" href="#hdf-eoss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.
</code></pre></div>
<h3 id="hdf-eos">HDF-EOS:<a class="headerlink" href="#hdf-eos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.
</code></pre></div>
<h3 id="hdf-eos5s">HDF-EOS5's:<a class="headerlink" href="#hdf-eos5s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.
</code></pre></div>
<h3 id="hdf-eos5">HDF-EOS5:<a class="headerlink" href="#hdf-eos5" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF-EOS (Hierarchical Data Format - Earth Observing System) is a self-describing file format for transfer of various types of data between different machines based upon HDF. HDF-EOS is a standard format to store data collected from EOS satellites: Terra, Aqua and Aura.
</code></pre></div>
<h3 id="hdf5s">HDF5's:<a class="headerlink" href="#hdf5s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF5 is a unique technology suite that makes possible the management of
</code></pre></div>
<h3 id="hdf5">HDF5:<a class="headerlink" href="#hdf5" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HDF5 is a unique technology suite that makes possible the management of
</code></pre></div>
<h3 id="hisat2s">HISAT2's:<a class="headerlink" href="#hisat2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
</code></pre></div>
<h3 id="hisat2">HISAT2:<a class="headerlink" href="#hisat2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
</code></pre></div>
<h3 id="hmmers">HMMER's:<a class="headerlink" href="#hmmers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HMMER is used for searching sequence databases for homologs of protein sequences,
</code></pre></div>
<h3 id="hmmer">HMMER:<a class="headerlink" href="#hmmer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HMMER is used for searching sequence databases for homologs of protein sequences,
</code></pre></div>
<h3 id="hmmer2s">HMMER2's:<a class="headerlink" href="#hmmer2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HMMER is used for searching sequence databases for homologs of protein sequences,
</code></pre></div>
<h3 id="hmmer2">HMMER2:<a class="headerlink" href="#hmmer2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HMMER is used for searching sequence databases for homologs of protein sequences,
</code></pre></div>
<h3 id="hopss">HOPS's:<a class="headerlink" href="#hopss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline which focuses on screening MALT data for the presence of a user-specified list of target species.
</code></pre></div>
<h3 id="hops">HOPS:<a class="headerlink" href="#hops" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline which focuses on screening MALT data for the presence of a user-specified list of target species.
</code></pre></div>
<h3 id="htseqs">HTSeq's:<a class="headerlink" href="#htseqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HTSeq is a Python library to facilitate processing and analysis
</code></pre></div>
<h3 id="htseq">HTSeq:<a class="headerlink" href="#htseq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HTSeq is a Python library to facilitate processing and analysis
</code></pre></div>
<h3 id="htslibs">HTSlib's:<a class="headerlink" href="#htslibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A C library for reading/writing high-throughput sequencing data.
</code></pre></div>
<h3 id="htslib">HTSlib:<a class="headerlink" href="#htslib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A C library for reading/writing high-throughput sequencing data.
</code></pre></div>
<h3 id="harfbuzzs">HarfBuzz's:<a class="headerlink" href="#harfbuzzs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HarfBuzz is an OpenType text shaping engine.
</code></pre></div>
<h3 id="harfbuzz">HarfBuzz:<a class="headerlink" href="#harfbuzz" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HarfBuzz is an OpenType text shaping engine.
</code></pre></div>
<h3 id="hpcgridrunners">HpcGridRunner's:<a class="headerlink" href="#hpcgridrunners" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HPC GridRunner is a simple command-line interface to high throughput computing using a variety of different grid computing platforms, including LSF, SGE, SLURM, and PBS.
</code></pre></div>
<h3 id="hpcgridrunner">HpcGridRunner:<a class="headerlink" href="#hpcgridrunner" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>HPC GridRunner is a simple command-line interface to high throughput computing using a variety of different grid computing platforms, including LSF, SGE, SLURM, and PBS.
</code></pre></div>
<h3 id="humanns">Humann's:<a class="headerlink" href="#humanns" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for efficiently and accurately determining the coverage and abundance of microbial pathways in a community from metagenomic data.
</code></pre></div>
<h3 id="humann">Humann:<a class="headerlink" href="#humann" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for efficiently and accurately determining the coverage and abundance of microbial pathways in a community from metagenomic data.
</code></pre></div>
<h3 id="hybpipers">HybPiper's:<a class="headerlink" href="#hybpipers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extracting Coding Sequence and Introns for Phylogenetics from High-Throughput Sequencing Reads Using Target Enrichment.
</code></pre></div>
<h3 id="hybpiper">HybPiper:<a class="headerlink" href="#hybpiper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extracting Coding Sequence and Introns for Phylogenetics from High-Throughput Sequencing Reads Using Target Enrichment.
</code></pre></div>
<h3 id="hypres">Hypre's:<a class="headerlink" href="#hypres" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hypre is a library for solving large, sparse linear systems of equations on massively
</code></pre></div>
<h3 id="hypre">Hypre:<a class="headerlink" href="#hypre" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hypre is a library for solving large, sparse linear systems of equations on massively
</code></pre></div>
<h3 id="icus">ICU's:<a class="headerlink" href="#icus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.
</code></pre></div>
<h3 id="icu">ICU:<a class="headerlink" href="#icu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.
</code></pre></div>
<h3 id="idbas">IDBA's:<a class="headerlink" href="#idbas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Iterative de Bruijn graph assembler for second-generation sequencing reads.
</code></pre></div>
<h3 id="idba">IDBA:<a class="headerlink" href="#idba" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Iterative de Bruijn graph assembler for second-generation sequencing reads.
</code></pre></div>
<h3 id="idba-uds">IDBA-UD's:<a class="headerlink" href="#idba-uds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
</code></pre></div>
<h3 id="idba-ud">IDBA-UD:<a class="headerlink" href="#idba-ud" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
</code></pre></div>
<h3 id="idls">IDL's:<a class="headerlink" href="#idls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IDL is the trusted scientific programming language used across disciplines to extract meaningful visualizations from complex numerical data.
</code></pre></div>
<h3 id="idl">IDL:<a class="headerlink" href="#idl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>IDL is the trusted scientific programming language used across disciplines to extract meaningful visualizations from complex numerical data.
</code></pre></div>
<h3 id="igvs">IGV's:<a class="headerlink" href="#igvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Integrative Genomics Viewer (IGV) is a high-performance visualization
</code></pre></div>
<h3 id="igv">IGV:<a class="headerlink" href="#igv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Integrative Genomics Viewer (IGV) is a high-performance visualization
</code></pre></div>
<h3 id="imputes">IMPUTE's:<a class="headerlink" href="#imputes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genotype imputation and haplotype phasing.
</code></pre></div>
<h3 id="impute">IMPUTE:<a class="headerlink" href="#impute" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genotype imputation and haplotype phasing.
</code></pre></div>
<h3 id="iq-trees">IQ-TREE's:<a class="headerlink" href="#iq-trees" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient phylogenomic software by maximum likelihood
</code></pre></div>
<h3 id="iq-tree">IQ-TREE:<a class="headerlink" href="#iq-tree" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient phylogenomic software by maximum likelihood
</code></pre></div>
<h3 id="irkernels">IRkernel's:<a class="headerlink" href="#irkernels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R packages for providing R kernel for Jupyter.
</code></pre></div>
<h3 id="irkernel">IRkernel:<a class="headerlink" href="#irkernel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R packages for providing R kernel for Jupyter.
</code></pre></div>
<h3 id="isa-ls">ISA-L's:<a class="headerlink" href="#isa-ls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intelligent Storage Acceleration Library
</code></pre></div>
<h3 id="isa-l">ISA-L:<a class="headerlink" href="#isa-l" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intelligent Storage Acceleration Library
</code></pre></div>
<h3 id="imagemagicks">ImageMagick's:<a class="headerlink" href="#imagemagicks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ImageMagick is a software suite to create, edit, compose, or convert bitmap images
</code></pre></div>
<h3 id="imagemagick">ImageMagick:<a class="headerlink" href="#imagemagick" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ImageMagick is a software suite to create, edit, compose, or convert bitmap images
</code></pre></div>
<h3 id="infernals">Infernal's:<a class="headerlink" href="#infernals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Infernal (&#39;INFERence of RNA ALignment&#39;) is for searching DNA sequence databases
</code></pre></div>
<h3 id="infernal">Infernal:<a class="headerlink" href="#infernal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Infernal (&#39;INFERence of RNA ALignment&#39;) is for searching DNA sequence databases
</code></pre></div>
<h3 id="inspectors">Inspector's:<a class="headerlink" href="#inspectors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Inspector XE is an easy to use memory error checker and thread checker for serial
</code></pre></div>
<h3 id="inspector">Inspector:<a class="headerlink" href="#inspector" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Inspector XE is an easy to use memory error checker and thread checker for serial
</code></pre></div>
<h3 id="interproscans">InterProScan's:<a class="headerlink" href="#interproscans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence analysis application (nucleotide and protein sequences) that combines
</code></pre></div>
<h3 id="interproscan">InterProScan:<a class="headerlink" href="#interproscan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence analysis application (nucleotide and protein sequences) that combines
</code></pre></div>
<h3 id="jagss">JAGS's:<a class="headerlink" href="#jagss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Just Another Gibbs Sampler - a program for the statistical analysis of Bayesian hierarchical models by Markov Chain Monte Carlo.
</code></pre></div>
<h3 id="jags">JAGS:<a class="headerlink" href="#jags" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Just Another Gibbs Sampler - a program for the statistical analysis of Bayesian hierarchical models by Markov Chain Monte Carlo.
</code></pre></div>
<h3 id="junits">JUnit's:<a class="headerlink" href="#junits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A programmer-oriented testing framework for Java.
</code></pre></div>
<h3 id="junit">JUnit:<a class="headerlink" href="#junit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A programmer-oriented testing framework for Java.
</code></pre></div>
<h3 id="jaspers">JasPer's:<a class="headerlink" href="#jaspers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard.
</code></pre></div>
<h3 id="jasper">JasPer:<a class="headerlink" href="#jasper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard.
</code></pre></div>
<h3 id="javas">Java's:<a class="headerlink" href="#javas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java Platform, Standard Edition (Java SE) lets you develop and deploy  Java applications on desktops and servers.
</code></pre></div>
<h3 id="java">Java:<a class="headerlink" href="#java" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java Platform, Standard Edition (Java SE) lets you develop and deploy  Java applications on desktops and servers.
</code></pre></div>
<h3 id="jellyfishs">Jellyfish's:<a class="headerlink" href="#jellyfishs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
</code></pre></div>
<h3 id="jellyfish">Jellyfish:<a class="headerlink" href="#jellyfish" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.
</code></pre></div>
<h3 id="jsoncpps">JsonCpp's:<a class="headerlink" href="#jsoncpps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JsonCpp is a C++ library that allows manipulating JSON values,
</code></pre></div>
<h3 id="jsoncpp">JsonCpp:<a class="headerlink" href="#jsoncpp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JsonCpp is a C++ library that allows manipulating JSON values,
</code></pre></div>
<h3 id="julias">Julia's:<a class="headerlink" href="#julias" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level, high-performance dynamic language for technical computing.  This version was compiled from source with USE_INTEL_JITEVENTS=1 to enable profiling with VTune.
</code></pre></div>
<h3 id="julia">Julia:<a class="headerlink" href="#julia" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level, high-performance dynamic language for technical computing.  This version was compiled from source with USE_INTEL_JITEVENTS=1 to enable profiling with VTune.
</code></pre></div>
<h3 id="jupyterlabs">JupyterLab's:<a class="headerlink" href="#jupyterlabs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture.
</code></pre></div>
<h3 id="jupyterlab">JupyterLab:<a class="headerlink" href="#jupyterlab" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture.
</code></pre></div>
<h3 id="kats">KAT's:<a class="headerlink" href="#kats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.
</code></pre></div>
<h3 id="kat">KAT:<a class="headerlink" href="#kat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.
</code></pre></div>
<h3 id="kealibs">KEALib's:<a class="headerlink" href="#kealibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>KEALib provides an implementation of the GDAL data model. The format supports raster attribute tables, image pyramids, meta-data and in-built statistics while also handling very large files and compression throughout. Based on the HDF5 standard, it also provides a base from which other formats can be derived and is a good choice for long term data archiving. An independent software library (libkea) provides complete access to the KEA image format and a GDAL driver allowing KEA images to be used from any GDAL supported software.
</code></pre></div>
<h3 id="kealib">KEALib:<a class="headerlink" href="#kealib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>KEALib provides an implementation of the GDAL data model. The format supports raster attribute tables, image pyramids, meta-data and in-built statistics while also handling very large files and compression throughout. Based on the HDF5 standard, it also provides a base from which other formats can be derived and is a good choice for long term data archiving. An independent software library (libkea) provides complete access to the KEA image format and a GDAL driver allowing KEA images to be used from any GDAL supported software.
</code></pre></div>
<h3 id="kmcs">KMC's:<a class="headerlink" href="#kmcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.
</code></pre></div>
<h3 id="kmc">KMC:<a class="headerlink" href="#kmc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.
</code></pre></div>
<h3 id="kaijus">Kaiju's:<a class="headerlink" href="#kaijus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Kaiju is a program for sensitive taxonomic classification of high-throughput
</code></pre></div>
<h3 id="kaiju">Kaiju:<a class="headerlink" href="#kaiju" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Kaiju is a program for sensitive taxonomic classification of high-throughput
</code></pre></div>
<h3 id="kent_toolss">Kent_tools's:<a class="headerlink" href="#kent_toolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of tools used by the UCSC genome browser.
</code></pre></div>
<h3 id="kent_tools">Kent_tools:<a class="headerlink" href="#kent_tools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of tools used by the UCSC genome browser.
</code></pre></div>
<h3 id="korfsnaps">KorfSNAP's:<a class="headerlink" href="#korfsnaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Semi-HMM-based Nucleic Acid Parser
</code></pre></div>
<h3 id="korfsnap">KorfSNAP:<a class="headerlink" href="#korfsnap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Semi-HMM-based Nucleic Acid Parser
</code></pre></div>
<h3 id="kraken2s">Kraken2's:<a class="headerlink" href="#kraken2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Taxonomic sequence classifier.
</code></pre></div>
<h3 id="kraken2">Kraken2:<a class="headerlink" href="#kraken2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Taxonomic sequence classifier.
</code></pre></div>
<h3 id="kronatoolss">KronaTools's:<a class="headerlink" href="#kronatoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Krona Tools is a set of scripts to create Krona charts from
</code></pre></div>
<h3 id="kronatools">KronaTools:<a class="headerlink" href="#kronatools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Krona Tools is a set of scripts to create Krona charts from
</code></pre></div>
<h3 id="kyotocabinets">KyotoCabinet's:<a class="headerlink" href="#kyotocabinets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library of routines for managing a database.
</code></pre></div>
<h3 id="kyotocabinet">KyotoCabinet:<a class="headerlink" href="#kyotocabinet" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library of routines for managing a database.
</code></pre></div>
<h3 id="lames">LAME's:<a class="headerlink" href="#lames" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
</code></pre></div>
<h3 id="lame">LAME:<a class="headerlink" href="#lame" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
</code></pre></div>
<h3 id="lammpss">LAMMPS's:<a class="headerlink" href="#lammpss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality.
</code></pre></div>
<h3 id="lammps">LAMMPS:<a class="headerlink" href="#lammps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or, more generically, as a parallel particle simulator at the atomic, meso, or continuum scale. LAMMPS runs on single processors or in parallel using message-passing techniques and a spatial-decomposition of the simulation domain. The code is designed to be easy to modify or extend with new functionality.
</code></pre></div>
<h3 id="lasts">LAST's:<a class="headerlink" href="#lasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAST finds similar regions between sequences.
</code></pre></div>
<h3 id="last">LAST:<a class="headerlink" href="#last" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LAST finds similar regions between sequences.
</code></pre></div>
<h3 id="lastzs">LASTZ's:<a class="headerlink" href="#lastzs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to
</code></pre></div>
<h3 id="lastz">LASTZ:<a class="headerlink" href="#lastz" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to
</code></pre></div>
<h3 id="ldcs">LDC's:<a class="headerlink" href="#ldcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>D programming language compiler
</code></pre></div>
<h3 id="ldc">LDC:<a class="headerlink" href="#ldc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>D programming language compiler
</code></pre></div>
<h3 id="lefses">LEfSe's:<a class="headerlink" href="#lefses" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Determines the features most likely to explain differences between classes by coupling standard tests for statistical significance
</code></pre></div>
<h3 id="lefse">LEfSe:<a class="headerlink" href="#lefse" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Determines the features most likely to explain differences between classes by coupling standard tests for statistical significance
</code></pre></div>
<h3 id="linkss">LINKS's:<a class="headerlink" href="#linkss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Alignment-free scaffolding of genome assembly drafts with long reads
</code></pre></div>
<h3 id="links">LINKS:<a class="headerlink" href="#links" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Alignment-free scaffolding of genome assembly drafts with long reads
</code></pre></div>
<h3 id="llvms">LLVM's:<a class="headerlink" href="#llvms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  (&quot;LLVM IR&quot;). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.   This build includes the clang C/C++ compiler frontend.
</code></pre></div>
<h3 id="llvm">LLVM:<a class="headerlink" href="#llvm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  (&quot;LLVM IR&quot;). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.   This build includes the clang C/C++ compiler frontend.
</code></pre></div>
<h3 id="lmdbs">LMDB's:<a class="headerlink" href="#lmdbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
</code></pre></div>
<h3 id="lmdb">LMDB:<a class="headerlink" href="#lmdb" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance
</code></pre></div>
<h3 id="lsd2s">LSD2's:<a class="headerlink" href="#lsd2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Least-squares methods to estimate rates and dates from phylogenies
</code></pre></div>
<h3 id="lsd2">LSD2:<a class="headerlink" href="#lsd2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Least-squares methods to estimate rates and dates from phylogenies
</code></pre></div>
<h3 id="ltr_retrievers">LTR_retriever's:<a class="headerlink" href="#ltr_retrievers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package.
</code></pre></div>
<h3 id="ltr_retriever">LTR_retriever:<a class="headerlink" href="#ltr_retriever" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package.
</code></pre></div>
<h3 id="lumpys">LUMPY's:<a class="headerlink" href="#lumpys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A probabilistic framework for structural variant discovery.
</code></pre></div>
<h3 id="lumpy">LUMPY:<a class="headerlink" href="#lumpy" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A probabilistic framework for structural variant discovery.
</code></pre></div>
<h3 id="lzos">LZO's:<a class="headerlink" href="#lzos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Portable lossless data compression library
</code></pre></div>
<h3 id="lzo">LZO:<a class="headerlink" href="#lzo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Portable lossless data compression library
</code></pre></div>
<h3 id="libtiffs">LibTIFF's:<a class="headerlink" href="#libtiffs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tiff: Library and tools for reading and writing TIFF data files
</code></pre></div>
<h3 id="libtiff">LibTIFF:<a class="headerlink" href="#libtiff" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tiff: Library and tools for reading and writing TIFF data files
</code></pre></div>
<h3 id="libavs">Libav's:<a class="headerlink" href="#libavs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libraries for dealing with multimedia formats of all sorts.  Forked from FFmpeg
</code></pre></div>
<h3 id="libav">Libav:<a class="headerlink" href="#libav" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libraries for dealing with multimedia formats of all sorts.  Forked from FFmpeg
</code></pre></div>
<h3 id="libints">Libint's:<a class="headerlink" href="#libints" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.
</code></pre></div>
<h3 id="libint">Libint:<a class="headerlink" href="#libint" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.
</code></pre></div>
<h3 id="littlecmss">LittleCMS's:<a class="headerlink" href="#littlecmss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Color management engine.
</code></pre></div>
<h3 id="littlecms">LittleCMS:<a class="headerlink" href="#littlecms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Color management engine.
</code></pre></div>
<h3 id="lokis">Loki's:<a class="headerlink" href="#lokis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Loki is a C++ library of designs, containing flexible implementations of common design patterns and
</code></pre></div>
<h3 id="loki">Loki:<a class="headerlink" href="#loki" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Loki is a C++ library of designs, containing flexible implementations of common design patterns and
</code></pre></div>
<h3 id="longstitchs">LongStitch's:<a class="headerlink" href="#longstitchs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A genome assembly correction and scaffolding pipeline using long reads
</code></pre></div>
<h3 id="longstitch">LongStitch:<a class="headerlink" href="#longstitch" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A genome assembly correction and scaffolding pipeline using long reads
</code></pre></div>
<h3 id="m4s">M4's:<a class="headerlink" href="#m4s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
</code></pre></div>
<h3 id="m4">M4:<a class="headerlink" href="#m4" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
</code></pre></div>
<h3 id="maffts">MAFFT's:<a class="headerlink" href="#maffts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Multiple sequence alignment program offering a range of methods.
</code></pre></div>
<h3 id="mafft">MAFFT:<a class="headerlink" href="#mafft" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Multiple sequence alignment program offering a range of methods.
</code></pre></div>
<h3 id="magmas">MAGMA's:<a class="headerlink" href="#magmas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for gene analysis and generalized gene-set analysis of GWAS data.
</code></pre></div>
<h3 id="magma">MAGMA:<a class="headerlink" href="#magma" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for gene analysis and generalized gene-set analysis of GWAS data.
</code></pre></div>
<h3 id="makers">MAKER's:<a class="headerlink" href="#makers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome annotation pipeline
</code></pre></div>
<h3 id="maker">MAKER:<a class="headerlink" href="#maker" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome annotation pipeline
</code></pre></div>
<h3 id="matios">MATIO's:<a class="headerlink" href="#matios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>matio is an C library for reading and writing Matlab MAT files.
</code></pre></div>
<h3 id="matio">MATIO:<a class="headerlink" href="#matio" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>matio is an C library for reading and writing Matlab MAT files.
</code></pre></div>
<h3 id="matlabs">MATLAB's:<a class="headerlink" href="#matlabs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level language and interactive environment for numerical computing.
</code></pre></div>
<h3 id="matlab">MATLAB:<a class="headerlink" href="#matlab" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level language and interactive environment for numerical computing.
</code></pre></div>
<h3 id="mcls">MCL's:<a class="headerlink" href="#mcls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MCL algorithm is short for the Markov Cluster Algorithm, a fast
</code></pre></div>
<h3 id="mcl">MCL:<a class="headerlink" href="#mcl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MCL algorithm is short for the Markov Cluster Algorithm, a fast
</code></pre></div>
<h3 id="mcrs">MCR's:<a class="headerlink" href="#mcrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Matlab Compiler Runtime is required for running compiled MATLAB executables without MATLAB itself.
</code></pre></div>
<h3 id="mcr">MCR:<a class="headerlink" href="#mcr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Matlab Compiler Runtime is required for running compiled MATLAB executables without MATLAB itself.
</code></pre></div>
<h3 id="megahits">MEGAHIT's:<a class="headerlink" href="#megahits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An ultra-fast single-node solution for large and complex
</code></pre></div>
<h3 id="megahit">MEGAHIT:<a class="headerlink" href="#megahit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An ultra-fast single-node solution for large and complex
</code></pre></div>
<h3 id="metabolics">METABOLIC's:<a class="headerlink" href="#metabolics" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Metabolic And Biogeochemistry anaLyses In microbes
</code></pre></div>
<h3 id="metabolic">METABOLIC:<a class="headerlink" href="#metabolic" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Metabolic And Biogeochemistry anaLyses In microbes
</code></pre></div>
<h3 id="metiss">METIS's:<a class="headerlink" href="#metiss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
</code></pre></div>
<h3 id="metis">METIS:<a class="headerlink" href="#metis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
</code></pre></div>
<h3 id="mmseqs2s">MMseqs2's:<a class="headerlink" href="#mmseqs2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MMseqs2: ultra fast and sensitive search and clustering suite
</code></pre></div>
<h3 id="mmseqs2">MMseqs2:<a class="headerlink" href="#mmseqs2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MMseqs2: ultra fast and sensitive search and clustering suite
</code></pre></div>
<h3 id="mob-suites">MOB-suite's:<a class="headerlink" href="#mob-suites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software tools for clustering, reconstruction and typing of plasmids from draft assemblies.
</code></pre></div>
<h3 id="mob-suite">MOB-suite:<a class="headerlink" href="#mob-suite" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software tools for clustering, reconstruction and typing of plasmids from draft assemblies.
</code></pre></div>
<h3 id="modflows">MODFLOW's:<a class="headerlink" href="#modflows" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MODFLOW is the U.S. Geological Survey modular finite-difference flow model, which is a computer code that solves the groundwater flow equation. The program is used by hydrogeologists to simulate the flow of groundwater through aquifers.
</code></pre></div>
<h3 id="modflow">MODFLOW:<a class="headerlink" href="#modflow" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MODFLOW is the U.S. Geological Survey modular finite-difference flow model, which is a computer code that solves the groundwater flow equation. The program is used by hydrogeologists to simulate the flow of groundwater through aquifers.
</code></pre></div>
<h3 id="mpfrs">MPFR's:<a class="headerlink" href="#mpfrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MPFR library is a C library for multiple-precision
</code></pre></div>
<h3 id="mpfr">MPFR:<a class="headerlink" href="#mpfr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MPFR library is a C library for multiple-precision
</code></pre></div>
<h3 id="msmcs">MSMC's:<a class="headerlink" href="#msmcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Multiple Sequentially Markovian Coalescent, infers population size and gene flow from multiple genome sequences
</code></pre></div>
<h3 id="msmc">MSMC:<a class="headerlink" href="#msmc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Multiple Sequentially Markovian Coalescent, infers population size and gene flow from multiple genome sequences
</code></pre></div>
<h3 id="mumpss">MUMPS's:<a class="headerlink" href="#mumpss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A parallel sparse direct solver
</code></pre></div>
<h3 id="mumps">MUMPS:<a class="headerlink" href="#mumps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A parallel sparse direct solver
</code></pre></div>
<h3 id="mummers">MUMmer's:<a class="headerlink" href="#mummers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUMmer is a system for rapidly aligning entire genomes,
</code></pre></div>
<h3 id="mummer">MUMmer:<a class="headerlink" href="#mummer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUMmer is a system for rapidly aligning entire genomes,
</code></pre></div>
<h3 id="muscles">MUSCLE's:<a class="headerlink" href="#muscles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUSCLE is a program for creating multiple alignments of amino acid or nucleotide
</code></pre></div>
<h3 id="muscle">MUSCLE:<a class="headerlink" href="#muscle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUSCLE is a program for creating multiple alignments of amino acid or nucleotide
</code></pre></div>
<h3 id="musts">MUST's:<a class="headerlink" href="#musts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.
</code></pre></div>
<h3 id="must">MUST:<a class="headerlink" href="#must" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MUST detects usage errors of the Message Passing Interface (MPI) and reports them to the user.
</code></pre></div>
<h3 id="masurcas">MaSuRCA's:<a class="headerlink" href="#masurcas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph
</code></pre></div>
<h3 id="masurca">MaSuRCA:<a class="headerlink" href="#masurca" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph
</code></pre></div>
<h3 id="magmas_1">Magma's:<a class="headerlink" href="#magmas_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Magma is a large, well-supported software package designed for computations in algebra, number theory, algebraic geometry and algebraic combinatorics. It provides a mathematically rigorous environment     for defining and working with structures such as groups, rings, fields, modules, algebras, schemes, curves, graphs, designs, codes and many others. Magma also supports a number of databases designed     to aid computational research in those areas of mathematics which are algebraic in nature.
</code></pre></div>
<h3 id="magma_1">Magma:<a class="headerlink" href="#magma_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Magma is a large, well-supported software package designed for computations in algebra, number theory, algebraic geometry and algebraic combinatorics. It provides a mathematically rigorous environment     for defining and working with structures such as groups, rings, fields, modules, algebras, schemes, curves, graphs, designs, codes and many others. Magma also supports a number of databases designed     to aid computational research in those areas of mathematics which are algebraic in nature.
</code></pre></div>
<h3 id="markerminers">MarkerMiner's:<a class="headerlink" href="#markerminers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow for effective discovery of SCN loci in flowering plants angiosperms
</code></pre></div>
<h3 id="markerminer">MarkerMiner:<a class="headerlink" href="#markerminer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow for effective discovery of SCN loci in flowering plants angiosperms
</code></pre></div>
<h3 id="mashs">Mash's:<a class="headerlink" href="#mashs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast genome and metagenome distance estimation using MinHash
</code></pre></div>
<h3 id="mash">Mash:<a class="headerlink" href="#mash" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast genome and metagenome distance estimation using MinHash
</code></pre></div>
<h3 id="mavens">Maven's:<a class="headerlink" href="#mavens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Binary maven install, Apache Maven is a software project management and comprehension tool. Based on
</code></pre></div>
<h3 id="maven">Maven:<a class="headerlink" href="#maven" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Binary maven install, Apache Maven is a software project management and comprehension tool. Based on
</code></pre></div>
<h3 id="maxbins">MaxBin's:<a class="headerlink" href="#maxbins" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MaxBin is software for binning assembled metagenomic sequences based on
</code></pre></div>
<h3 id="maxbin">MaxBin:<a class="headerlink" href="#maxbin" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MaxBin is software for binning assembled metagenomic sequences based on
</code></pre></div>
<h3 id="meraculouss">Meraculous's:<a class="headerlink" href="#meraculouss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eukaryotic genome assembler for Illumina sequence data.
</code></pre></div>
<h3 id="meraculous">Meraculous:<a class="headerlink" href="#meraculous" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Eukaryotic genome assembler for Illumina sequence data.
</code></pre></div>
<h3 id="mesas">Mesa's:<a class="headerlink" href="#mesas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.   Note that this build enables CPU-based rendering with OpenSWR and LLVM.  The module is intended to be used with visualisation software, such as  ParaView, on nodes where no GPU hardware is available.   Both on-screen and off-screen rendering are supported.  IMPORTANT: The OpenSWR software rasteriser can use multiple threads for            best performance. The number of threads is controlled by the            environment variable KNOB_MAX_WORKER_THREADS. The module sets             KNOB_MAX_WORKER_THREADS=1             by default to avoid accidental oversubscription of nodes.
</code></pre></div>
<h3 id="mesa">Mesa:<a class="headerlink" href="#mesa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.   Note that this build enables CPU-based rendering with OpenSWR and LLVM.  The module is intended to be used with visualisation software, such as  ParaView, on nodes where no GPU hardware is available.   Both on-screen and off-screen rendering are supported.  IMPORTANT: The OpenSWR software rasteriser can use multiple threads for            best performance. The number of threads is controlled by the            environment variable KNOB_MAX_WORKER_THREADS. The module sets             KNOB_MAX_WORKER_THREADS=1             by default to avoid accidental oversubscription of nodes.
</code></pre></div>
<h3 id="mesons">Meson's:<a class="headerlink" href="#mesons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.
</code></pre></div>
<h3 id="meson">Meson:<a class="headerlink" href="#meson" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.
</code></pre></div>
<h3 id="metabats">MetaBAT's:<a class="headerlink" href="#metabats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An efficient tool for accurately reconstructing single genomes from complex microbial communities
</code></pre></div>
<h3 id="metabat">MetaBAT:<a class="headerlink" href="#metabat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An efficient tool for accurately reconstructing single genomes from complex microbial communities
</code></pre></div>
<h3 id="metaeuks">MetaEuk's:<a class="headerlink" href="#metaeuks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic
</code></pre></div>
<h3 id="metaeuk">MetaEuk:<a class="headerlink" href="#metaeuk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaEuk is a modular toolkit designed for large-scale gene discovery and annotation in eukaryotic
</code></pre></div>
<h3 id="metageneannotators">MetaGeneAnnotator's:<a class="headerlink" href="#metageneannotators" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaGeneAnnotator is a gene-finding program for prokaryote and phage.
</code></pre></div>
<h3 id="metageneannotator">MetaGeneAnnotator:<a class="headerlink" href="#metageneannotator" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaGeneAnnotator is a gene-finding program for prokaryote and phage.
</code></pre></div>
<h3 id="metaphlans">MetaPhlAn's:<a class="headerlink" href="#metaphlans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaPhlAn is a computational tool for profiling the composition of microbial
</code></pre></div>
<h3 id="metaphlan">MetaPhlAn:<a class="headerlink" href="#metaphlan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaPhlAn is a computational tool for profiling the composition of microbial
</code></pre></div>
<h3 id="metaphlan2s">MetaPhlAn2's:<a class="headerlink" href="#metaphlan2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaPhlAn is a computational tool for profiling the composition of microbial
</code></pre></div>
<h3 id="metaphlan2">MetaPhlAn2:<a class="headerlink" href="#metaphlan2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MetaPhlAn is a computational tool for profiling the composition of microbial
</code></pre></div>
<h3 id="metasvs">MetaSV's:<a class="headerlink" href="#metasvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Structural-variant caller
</code></pre></div>
<h3 id="metasv">MetaSV:<a class="headerlink" href="#metasv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Structural-variant caller
</code></pre></div>
<h3 id="metavelvets">MetaVelvet's:<a class="headerlink" href="#metavelvets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A short read assember for metagenomics
</code></pre></div>
<h3 id="metavelvet">MetaVelvet:<a class="headerlink" href="#metavelvet" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A short read assember for metagenomics
</code></pre></div>
<h3 id="metaxa2s">Metaxa2's:<a class="headerlink" href="#metaxa2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Taxonomic classification of rRNA.
</code></pre></div>
<h3 id="metaxa2">Metaxa2:<a class="headerlink" href="#metaxa2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Taxonomic classification of rRNA.
</code></pre></div>
<h3 id="miniconda3s">Miniconda3's:<a class="headerlink" href="#miniconda3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A platform for Python-based data analytics
</code></pre></div>
<h3 id="miniconda3">Miniconda3:<a class="headerlink" href="#miniconda3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A platform for Python-based data analytics
</code></pre></div>
<h3 id="minimac3s">Minimac3's:<a class="headerlink" href="#minimac3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Low memory and more computationally efficient implementation of the genotype imputation algorithms.
</code></pre></div>
<h3 id="minimac3">Minimac3:<a class="headerlink" href="#minimac3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Low memory and more computationally efficient implementation of the genotype imputation algorithms.
</code></pre></div>
<h3 id="mitozs">MitoZ's:<a class="headerlink" href="#mitozs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit which aims to automatically filter pair-end raw data,
</code></pre></div>
<h3 id="mitoz">MitoZ:<a class="headerlink" href="#mitoz" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit which aims to automatically filter pair-end raw data,
</code></pre></div>
<h3 id="molcass">Molcas's:<a class="headerlink" href="#molcass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Molcas is an ab initio quantum chemistry software package
</code></pre></div>
<h3 id="molcas">Molcas:<a class="headerlink" href="#molcas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Molcas is an ab initio quantum chemistry software package
</code></pre></div>
<h3 id="molpros">Molpro's:<a class="headerlink" href="#molpros" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Molpro is a complete system of ab initio programs for molecular electronic structure calculations.
</code></pre></div>
<h3 id="molpro">Molpro:<a class="headerlink" href="#molpro" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Molpro is a complete system of ab initio programs for molecular electronic structure calculations.
</code></pre></div>
<h3 id="monos">Mono's:<a class="headerlink" href="#monos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source, cross-platform, implementation of C# and the CLR that is
</code></pre></div>
<h3 id="mono">Mono:<a class="headerlink" href="#mono" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source, cross-platform, implementation of C# and the CLR that is
</code></pre></div>
<h3 id="monocle3s">Monocle3's:<a class="headerlink" href="#monocle3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An analysis toolkit for single-cell RNA-seq.
</code></pre></div>
<h3 id="monocle3">Monocle3:<a class="headerlink" href="#monocle3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An analysis toolkit for single-cell RNA-seq.
</code></pre></div>
<h3 id="mothurs">Mothur's:<a class="headerlink" href="#mothurs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mothur is a single piece of open-source, expandable software
</code></pre></div>
<h3 id="mothur">Mothur:<a class="headerlink" href="#mothur" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mothur is a single piece of open-source, expandable software
</code></pre></div>
<h3 id="motioncorrs">MotionCorr's:<a class="headerlink" href="#motioncorrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Motion Correction for Dose-Fractionation Stack, by Dr. Xueming Li of the Cheng Laboratory at UCSF. The actual executable is named dosefgpu_driftcorr
</code></pre></div>
<h3 id="motioncorr">MotionCorr:<a class="headerlink" href="#motioncorr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Motion Correction for Dose-Fractionation Stack, by Dr. Xueming Li of the Cheng Laboratory at UCSF. The actual executable is named dosefgpu_driftcorr
</code></pre></div>
<h3 id="mrbayess">MrBayes's:<a class="headerlink" href="#mrbayess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MrBayes is a program for the Bayesian estimation of phylogeny.
</code></pre></div>
<h3 id="mrbayes">MrBayes:<a class="headerlink" href="#mrbayes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MrBayes is a program for the Bayesian estimation of phylogeny.
</code></pre></div>
<h3 id="mules">Mule's:<a class="headerlink" href="#mules" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mule is an API written in Python which allows you to access and manipulate files produced by the UM (Unified Model, of the Met Office (UK)).
</code></pre></div>
<h3 id="mule">Mule:<a class="headerlink" href="#mule" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mule is an API written in Python which allows you to access and manipulate files produced by the UM (Unified Model, of the Met Office (UK)).
</code></pre></div>
<h3 id="multiqcs">MultiQC's:<a class="headerlink" href="#multiqcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Aggregate results from bioinformatics analyses across many samples into a single
</code></pre></div>
<h3 id="multiqc">MultiQC:<a class="headerlink" href="#multiqc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Aggregate results from bioinformatics analyses across many samples into a single
</code></pre></div>
<h3 id="namds">NAMD's:<a class="headerlink" href="#namds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NAMD is a parallel molecular dynamics code designed for high-performance simulation of
</code></pre></div>
<h3 id="namd">NAMD:<a class="headerlink" href="#namd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NAMD is a parallel molecular dynamics code designed for high-performance simulation of
</code></pre></div>
<h3 id="nasms">NASM's:<a class="headerlink" href="#nasms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NASM: General-purpose x86 assembler
</code></pre></div>
<h3 id="nasm">NASM:<a class="headerlink" href="#nasm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NASM: General-purpose x86 assembler
</code></pre></div>
<h3 id="ncargs">NCARG's:<a class="headerlink" href="#ncargs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCAR Graphics is a Fortran and C based software package for
</code></pre></div>
<h3 id="ncarg">NCARG:<a class="headerlink" href="#ncarg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCAR Graphics is a Fortran and C based software package for
</code></pre></div>
<h3 id="nccls">NCCL's:<a class="headerlink" href="#nccls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
</code></pre></div>
<h3 id="nccl">NCCL:<a class="headerlink" href="#nccl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
</code></pre></div>
<h3 id="ncls">NCL's:<a class="headerlink" href="#ncls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCL is an interpreted language designed specifically for scientific data analysis and   visualization.
</code></pre></div>
<h3 id="ncl">NCL:<a class="headerlink" href="#ncl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NCL is an interpreted language designed specifically for scientific data analysis and   visualization.
</code></pre></div>
<h3 id="ncos">NCO's:<a class="headerlink" href="#ncos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5
</code></pre></div>
<h3 id="nco">NCO:<a class="headerlink" href="#nco" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5
</code></pre></div>
<h3 id="ncviews">NCVIEW's:<a class="headerlink" href="#ncviews" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc.
</code></pre></div>
<h3 id="ncview">NCVIEW:<a class="headerlink" href="#ncview" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc.
</code></pre></div>
<h3 id="ngss">NGS's:<a class="headerlink" href="#ngss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from
</code></pre></div>
<h3 id="ngs">NGS:<a class="headerlink" href="#ngs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from
</code></pre></div>
<h3 id="nlopts">NLopt's:<a class="headerlink" href="#nlopts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms
</code></pre></div>
<h3 id="nlopt">NLopt:<a class="headerlink" href="#nlopt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NLopt is a free/open-source library for nonlinear optimization, providing a common interface for a number of different free optimization routines available online as well as original implementations of various other algorithms
</code></pre></div>
<h3 id="nsprs">NSPR's:<a class="headerlink" href="#nsprs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
</code></pre></div>
<h3 id="nspr">NSPR:<a class="headerlink" href="#nspr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level
</code></pre></div>
<h3 id="nsss">NSS's:<a class="headerlink" href="#nsss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Network Security Services (NSS) is a set of libraries designed to support cross-platform development
</code></pre></div>
<h3 id="nss">NSS:<a class="headerlink" href="#nss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Network Security Services (NSS) is a set of libraries designed to support cross-platform development
</code></pre></div>
<h3 id="nvhpcs">NVHPC's:<a class="headerlink" href="#nvhpcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)
</code></pre></div>
<h3 id="nvhpc">NVHPC:<a class="headerlink" href="#nvhpc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)
</code></pre></div>
<h3 id="nwchems">NWChem's:<a class="headerlink" href="#nwchems" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NWChem aims to provide its users with computational chemistry tools that are scalable both in
</code></pre></div>
<h3 id="nwchem">NWChem:<a class="headerlink" href="#nwchem" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NWChem aims to provide its users with computational chemistry tools that are scalable both in
</code></pre></div>
<h3 id="nanocomps">NanoComp's:<a class="headerlink" href="#nanocomps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Comparing runs of Oxford Nanopore sequencing data and alignments
</code></pre></div>
<h3 id="nanocomp">NanoComp:<a class="headerlink" href="#nanocomp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Comparing runs of Oxford Nanopore sequencing data and alignments
</code></pre></div>
<h3 id="nanolyses">NanoLyse's:<a class="headerlink" href="#nanolyses" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Removing reads mapping to the lambda genome.
</code></pre></div>
<h3 id="nanolyse">NanoLyse:<a class="headerlink" href="#nanolyse" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Removing reads mapping to the lambda genome.
</code></pre></div>
<h3 id="nanoplots">NanoPlot's:<a class="headerlink" href="#nanoplots" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Plotting suite for Oxford Nanopore sequencing data and alignments.
</code></pre></div>
<h3 id="nanoplot">NanoPlot:<a class="headerlink" href="#nanoplot" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Plotting suite for Oxford Nanopore sequencing data and alignments.
</code></pre></div>
<h3 id="nanostats">NanoStat's:<a class="headerlink" href="#nanostats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.
</code></pre></div>
<h3 id="nanostat">NanoStat:<a class="headerlink" href="#nanostat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.
</code></pre></div>
<h3 id="newton-xs">Newton-X's:<a class="headerlink" href="#newton-xs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NX is a general-purpose program package for simulating the dynamics of electronically excited molecules and molecular assemblies.
</code></pre></div>
<h3 id="newton-x">Newton-X:<a class="headerlink" href="#newton-x" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NX is a general-purpose program package for simulating the dynamics of electronically excited molecules and molecular assemblies.
</code></pre></div>
<h3 id="nextgenmaps">NextGenMap's:<a class="headerlink" href="#nextgenmaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NextGenMap is a flexible highly sensitive short read mapping tool that
</code></pre></div>
<h3 id="nextgenmap">NextGenMap:<a class="headerlink" href="#nextgenmap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NextGenMap is a flexible highly sensitive short read mapping tool that
</code></pre></div>
<h3 id="nextflows">Nextflow's:<a class="headerlink" href="#nextflows" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nextflow is a reactive workflow framework and a programming DSL
</code></pre></div>
<h3 id="nextflow">Nextflow:<a class="headerlink" href="#nextflow" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nextflow is a reactive workflow framework and a programming DSL
</code></pre></div>
<h3 id="ninjas">Ninja's:<a class="headerlink" href="#ninjas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ninja is a small build system with a focus on speed.
</code></pre></div>
<h3 id="ninja">Ninja:<a class="headerlink" href="#ninja" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ninja is a small build system with a focus on speed.
</code></pre></div>
<h3 id="nsight-computes">Nsight-Compute's:<a class="headerlink" href="#nsight-computes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA® Nsight™ Compute is an interactive kernel profiler for CUDA applications. It provides detailed
</code></pre></div>
<h3 id="nsight-compute">Nsight-Compute:<a class="headerlink" href="#nsight-compute" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA® Nsight™ Compute is an interactive kernel profiler for CUDA applications. It provides detailed
</code></pre></div>
<h3 id="nsight-systemss">Nsight-Systems's:<a class="headerlink" href="#nsight-systemss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an
</code></pre></div>
<h3 id="nsight-systems">Nsight-Systems:<a class="headerlink" href="#nsight-systems" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA® Nsight™ Systems is a system-wide performance analysis tool designed to visualize an
</code></pre></div>
<h3 id="oasis3-mcts">OASIS3-MCT's:<a class="headerlink" href="#oasis3-mcts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OASIS coupler is a software allowing synchronized exchanges of coupling information between numerical codes representing different components of the climate system.
</code></pre></div>
<h3 id="oasis3-mct">OASIS3-MCT:<a class="headerlink" href="#oasis3-mct" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OASIS coupler is a software allowing synchronized exchanges of coupling information between numerical codes representing different components of the climate system.
</code></pre></div>
<h3 id="obitoolss">OBITools's:<a class="headerlink" href="#obitoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manipulate various data and sequence files.
</code></pre></div>
<h3 id="obitools">OBITools:<a class="headerlink" href="#obitools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manipulate various data and sequence files.
</code></pre></div>
<h3 id="ocis">OCI's:<a class="headerlink" href="#ocis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Oracle Call Interface (OCI) is the comprehensive, high performance, native C language interface to Oracle Database for custom or packaged applications.  NOTE: This package is only available on Maui Ancil nodes that provide database access.
</code></pre></div>
<h3 id="oci">OCI:<a class="headerlink" href="#oci" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Oracle Call Interface (OCI) is the comprehensive, high performance, native C language interface to Oracle Database for custom or packaged applications.  NOTE: This package is only available on Maui Ancil nodes that provide database access.
</code></pre></div>
<h3 id="omas">OMA's:<a class="headerlink" href="#omas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Orthologous MAtrix project is a method and database for the inference
</code></pre></div>
<h3 id="oma">OMA:<a class="headerlink" href="#oma" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Orthologous MAtrix project is a method and database for the inference
</code></pre></div>
<h3 id="opari2s">OPARI2's:<a class="headerlink" href="#opari2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>source-to-source instrumentation tool for OpenMP and hybrid codes.
</code></pre></div>
<h3 id="opari2">OPARI2:<a class="headerlink" href="#opari2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>source-to-source instrumentation tool for OpenMP and hybrid codes.
</code></pre></div>
<h3 id="orcas">ORCA's:<a class="headerlink" href="#orcas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry
</code></pre></div>
<h3 id="orca">ORCA:<a class="headerlink" href="#orca" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry
</code></pre></div>
<h3 id="osprays">OSPRay's:<a class="headerlink" href="#osprays" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OSPRay features interactive CPU rendering capabilities geared towards Scientific Visualization applications. Advanced shading effects such as Ambient Occlusion, shadows, and transparency can be rendered interactively, enabling new insights into data exploration.
</code></pre></div>
<h3 id="ospray">OSPRay:<a class="headerlink" href="#ospray" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OSPRay features interactive CPU rendering capabilities geared towards Scientific Visualization applications. Advanced shading effects such as Ambient Occlusion, shadows, and transparency can be rendered interactively, enabling new insights into data exploration.
</code></pre></div>
<h3 id="osu-micro-benchmarkss">OSU-Micro-Benchmarks's:<a class="headerlink" href="#osu-micro-benchmarkss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OSU Micro-Benchmarks for MPI
</code></pre></div>
<h3 id="osu-micro-benchmarks">OSU-Micro-Benchmarks:<a class="headerlink" href="#osu-micro-benchmarks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OSU Micro-Benchmarks for MPI
</code></pre></div>
<h3 id="octaves">Octave's:<a class="headerlink" href="#octaves" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Octave is a high-level interpreted language, primarily intended for numerical computations.
</code></pre></div>
<h3 id="octave">Octave:<a class="headerlink" href="#octave" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Octave is a high-level interpreted language, primarily intended for numerical computations.
</code></pre></div>
<h3 id="octopuss">Octopus's:<a class="headerlink" href="#octopuss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Octopus is a scientific program aimed at the ab initio virtual experimentation
</code></pre></div>
<h3 id="octopus">Octopus:<a class="headerlink" href="#octopus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Octopus is a scientific program aimed at the ab initio virtual experimentation
</code></pre></div>
<h3 id="openblass">OpenBLAS's:<a class="headerlink" href="#openblass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
</code></pre></div>
<h3 id="openblas">OpenBLAS:<a class="headerlink" href="#openblas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.
</code></pre></div>
<h3 id="openbabels">OpenBabel's:<a class="headerlink" href="#openbabels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Open Babel is a chemical toolbox designed to speak the many
</code></pre></div>
<h3 id="openbabel">OpenBabel:<a class="headerlink" href="#openbabel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Open Babel is a chemical toolbox designed to speak the many
</code></pre></div>
<h3 id="opencmisss">OpenCMISS's:<a class="headerlink" href="#opencmisss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCMISS is a set of libraries and applications which provide modelling and
</code></pre></div>
<h3 id="opencmiss">OpenCMISS:<a class="headerlink" href="#opencmiss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCMISS is a set of libraries and applications which provide modelling and
</code></pre></div>
<h3 id="opencvs">OpenCV's:<a class="headerlink" href="#opencvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCV (Open Source Computer Vision Library) is an open source computer vision
</code></pre></div>
<h3 id="opencv">OpenCV:<a class="headerlink" href="#opencv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenCV (Open Source Computer Vision Library) is an open source computer vision
</code></pre></div>
<h3 id="openfasts">OpenFAST's:<a class="headerlink" href="#openfasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Wind turbine multiphysics simulation tool
</code></pre></div>
<h3 id="openfast">OpenFAST:<a class="headerlink" href="#openfast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Wind turbine multiphysics simulation tool
</code></pre></div>
<h3 id="openfoams">OpenFOAM's:<a class="headerlink" href="#openfoams" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenFOAM is a free, open source CFD software package.
</code></pre></div>
<h3 id="openfoam">OpenFOAM:<a class="headerlink" href="#openfoam" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenFOAM is a free, open source CFD software package.
</code></pre></div>
<h3 id="openjpegs">OpenJPEG's:<a class="headerlink" href="#openjpegs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source JPEG 2000 codec written in C
</code></pre></div>
<h3 id="openjpeg">OpenJPEG:<a class="headerlink" href="#openjpeg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source JPEG 2000 codec written in C
</code></pre></div>
<h3 id="openmpis">OpenMPI's:<a class="headerlink" href="#openmpis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Open MPI Project is an open source MPI-3 implementation. This version is built with CUDA support enabled.
</code></pre></div>
<h3 id="openmpi">OpenMPI:<a class="headerlink" href="#openmpi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Open MPI Project is an open source MPI-3 implementation. This version is built with CUDA support enabled.
</code></pre></div>
<h3 id="openssls">OpenSSL's:<a class="headerlink" href="#openssls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
</code></pre></div>
<h3 id="openssl">OpenSSL:<a class="headerlink" href="#openssl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
</code></pre></div>
<h3 id="openseess">OpenSees's:<a class="headerlink" href="#openseess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.
</code></pre></div>
<h3 id="opensees">OpenSees:<a class="headerlink" href="#opensees" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.
</code></pre></div>
<h3 id="openseespys">OpenSeesPy's:<a class="headerlink" href="#openseespys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.
</code></pre></div>
<h3 id="openseespy">OpenSeesPy:<a class="headerlink" href="#openseespy" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.
</code></pre></div>
<h3 id="openslides">OpenSlide's:<a class="headerlink" href="#openslides" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSlide is a C library that provides a simple interface to
</code></pre></div>
<h3 id="openslide">OpenSlide:<a class="headerlink" href="#openslide" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OpenSlide is a C library that provides a simple interface to
</code></pre></div>
<h3 id="orfms">OrfM's:<a class="headerlink" href="#orfms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A simple and not slow open reading frame (ORF) caller.
</code></pre></div>
<h3 id="orfm">OrfM:<a class="headerlink" href="#orfm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A simple and not slow open reading frame (ORF) caller.
</code></pre></div>
<h3 id="orthofillers">OrthoFiller's:<a class="headerlink" href="#orthofillers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identifies missing annotations for evolutionarily conserved genes.
</code></pre></div>
<h3 id="orthofiller">OrthoFiller:<a class="headerlink" href="#orthofiller" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Identifies missing annotations for evolutionarily conserved genes.
</code></pre></div>
<h3 id="orthofinders">OrthoFinder's:<a class="headerlink" href="#orthofinders" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics
</code></pre></div>
<h3 id="orthofinder">OrthoFinder:<a class="headerlink" href="#orthofinder" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics
</code></pre></div>
<h3 id="orthomcls">OrthoMCL's:<a class="headerlink" href="#orthomcls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-scale algorithm for grouping orthologous protein sequences.
</code></pre></div>
<h3 id="orthomcl">OrthoMCL:<a class="headerlink" href="#orthomcl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome-scale algorithm for grouping orthologous protein sequences.
</code></pre></div>
<h3 id="paleomixs">PALEOMIX's:<a class="headerlink" href="#paleomixs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pipelines and tools designed to aid the rapid processing of High-Throughput Sequencing (HTS) data.
</code></pre></div>
<h3 id="paleomix">PALEOMIX:<a class="headerlink" href="#paleomix" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pipelines and tools designed to aid the rapid processing of High-Throughput Sequencing (HTS) data.
</code></pre></div>
<h3 id="pamls">PAML's:<a class="headerlink" href="#pamls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PAML is a package of programs for phylogenetic
</code></pre></div>
<h3 id="paml">PAML:<a class="headerlink" href="#paml" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PAML is a package of programs for phylogenetic
</code></pre></div>
<h3 id="papis">PAPI's:<a class="headerlink" href="#papis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PAPI provides the tool designer and application engineer with a consistent interface and
</code></pre></div>
<h3 id="papi">PAPI:<a class="headerlink" href="#papi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PAPI provides the tool designer and application engineer with a consistent interface and
</code></pre></div>
<h3 id="pbjellys">PBJelly's:<a class="headerlink" href="#pbjellys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or
</code></pre></div>
<h3 id="pbjelly">PBJelly:<a class="headerlink" href="#pbjelly" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or
</code></pre></div>
<h3 id="pcres">PCRE's:<a class="headerlink" href="#pcres" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The PCRE library is a set of functions that implement regular expression pattern matching using
</code></pre></div>
<h3 id="pcre">PCRE:<a class="headerlink" href="#pcre" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The PCRE library is a set of functions that implement regular expression pattern matching using
</code></pre></div>
<h3 id="pcre2s">PCRE2's:<a class="headerlink" href="#pcre2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5.
</code></pre></div>
<h3 id="pcre2">PCRE2:<a class="headerlink" href="#pcre2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5.
</code></pre></div>
<h3 id="pdts">PDT's:<a class="headerlink" href="#pdts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program
</code></pre></div>
<h3 id="pdt">PDT:<a class="headerlink" href="#pdt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program Database Toolkit (PDT) is a framework for analyzing source code written in several programming languages and for making rich program
</code></pre></div>
<h3 id="pears">PEAR's:<a class="headerlink" href="#pears" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Memory-efficient,fully parallelized and highly accurate pair-end read merger.
</code></pre></div>
<h3 id="pear">PEAR:<a class="headerlink" href="#pear" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Memory-efficient,fully parallelized and highly accurate pair-end read merger.
</code></pre></div>
<h3 id="pests">PEST++'s:<a class="headerlink" href="#pests" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PEST++ is a software suite aimed at supporting
</code></pre></div>
<h3 id="pest">PEST++:<a class="headerlink" href="#pest" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PEST++ is a software suite aimed at supporting
</code></pre></div>
<h3 id="petscs">PETSc's:<a class="headerlink" href="#petscs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
</code></pre></div>
<h3 id="petsc">PETSc:<a class="headerlink" href="#petsc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
</code></pre></div>
<h3 id="pffts">PFFT's:<a class="headerlink" href="#pffts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PFFT is a software library for computing massively parallel, fast Fourier
</code></pre></div>
<h3 id="pfft">PFFT:<a class="headerlink" href="#pfft" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PFFT is a software library for computing massively parallel, fast Fourier
</code></pre></div>
<h3 id="pgis">PGI's:<a class="headerlink" href="#pgis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C, C++ and Fortran compilers from The Portland Group - PGI
</code></pre></div>
<h3 id="pgi">PGI:<a class="headerlink" href="#pgi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C, C++ and Fortran compilers from The Portland Group - PGI
</code></pre></div>
<h3 id="phasiuss">PHASIUS's:<a class="headerlink" href="#phasiuss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to visualize phase block structure from (many) BAM or CRAM files together with BED annotation
</code></pre></div>
<h3 id="phasius">PHASIUS:<a class="headerlink" href="#phasius" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to visualize phase block structure from (many) BAM or CRAM files together with BED annotation
</code></pre></div>
<h3 id="plinks">PLINK's:<a class="headerlink" href="#plinks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PLINK is a free, open-source whole genome association analysis toolset,
</code></pre></div>
<h3 id="plink">PLINK:<a class="headerlink" href="#plink" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PLINK is a free, open-source whole genome association analysis toolset,
</code></pre></div>
<h3 id="plumeds">PLUMED's:<a class="headerlink" href="#plumeds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PLUMED is an open source library for free energy calculations in molecular systems which
</code></pre></div>
<h3 id="plumed">PLUMED:<a class="headerlink" href="#plumed" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PLUMED is an open source library for free energy calculations in molecular systems which
</code></pre></div>
<h3 id="pranks">PRANK's:<a class="headerlink" href="#pranks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Probabilistic multiple alignment program for DNA, codon and amino-acid sequences. .
</code></pre></div>
<h3 id="prank">PRANK:<a class="headerlink" href="#prank" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Probabilistic multiple alignment program for DNA, codon and amino-acid sequences. .
</code></pre></div>
<h3 id="projs">PROJ's:<a class="headerlink" href="#projs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program proj is a standard Unix filter function which converts  geographic longitude and latitude coordinates into cartesian coordinates
</code></pre></div>
<h3 id="proj">PROJ:<a class="headerlink" href="#proj" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program proj is a standard Unix filter function which converts  geographic longitude and latitude coordinates into cartesian coordinates
</code></pre></div>
<h3 id="pangos">Pango's:<a class="headerlink" href="#pangos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
</code></pre></div>
<h3 id="pango">Pango:<a class="headerlink" href="#pango" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
</code></pre></div>
<h3 id="parmetiss">ParMETIS's:<a class="headerlink" href="#parmetiss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs,
</code></pre></div>
<h3 id="parmetis">ParMETIS:<a class="headerlink" href="#parmetis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs,
</code></pre></div>
<h3 id="paraviews">ParaView's:<a class="headerlink" href="#paraviews" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ParaView is a scientific parallel visualizer.  This version supports CPU-only rendering without X context using the OSMesa library, it does not support GPU rendering, and it does not provide a GUI.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.  Ray tracing using the OSPRay library is also supported.
</code></pre></div>
<h3 id="paraview">ParaView:<a class="headerlink" href="#paraview" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ParaView is a scientific parallel visualizer.  This version supports CPU-only rendering without X context using the OSMesa library, it does not support GPU rendering, and it does not provide a GUI.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.  Ray tracing using the OSPRay library is also supported.
</code></pre></div>
<h3 id="parallels">Parallel's:<a class="headerlink" href="#parallels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parallel: Build and execute shell commands in parallel
</code></pre></div>
<h3 id="parallel">Parallel:<a class="headerlink" href="#parallel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parallel: Build and execute shell commands in parallel
</code></pre></div>
<h3 id="parallelios">ParallelIO's:<a class="headerlink" href="#parallelios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level Parallel I/O Library for structured grid applications
</code></pre></div>
<h3 id="parallelio">ParallelIO:<a class="headerlink" href="#parallelio" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A high-level Parallel I/O Library for structured grid applications
</code></pre></div>
<h3 id="paravers">Paraver's:<a class="headerlink" href="#paravers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Performance visualization and analysis tool based on traces.
</code></pre></div>
<h3 id="paraver">Paraver:<a class="headerlink" href="#paraver" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Performance visualization and analysis tool based on traces.
</code></pre></div>
<h3 id="peregrines">Peregrine's:<a class="headerlink" href="#peregrines" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome assembler for long reads (length &gt; 10kb, accuracy &gt; 99%).
</code></pre></div>
<h3 id="peregrine">Peregrine:<a class="headerlink" href="#peregrine" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome assembler for long reads (length &gt; 10kb, accuracy &gt; 99%).
</code></pre></div>
<h3 id="perls">Perl's:<a class="headerlink" href="#perls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Larry Wall&#39;s Practical Extraction and Report Language
</code></pre></div>
<h3 id="perl">Perl:<a class="headerlink" href="#perl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Larry Wall&#39;s Practical Extraction and Report Language
</code></pre></div>
<h3 id="phymls">PhyML's:<a class="headerlink" href="#phymls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Phylogenetic estimation using Maximum Likelihood
</code></pre></div>
<h3 id="phyml">PhyML:<a class="headerlink" href="#phyml" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Phylogenetic estimation using Maximum Likelihood
</code></pre></div>
<h3 id="phylophlans">PhyloPhlAn's:<a class="headerlink" href="#phylophlans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes.
</code></pre></div>
<h3 id="phylophlan">PhyloPhlAn:<a class="headerlink" href="#phylophlan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Integrated pipeline for large-scale phylogenetic profiling of genomes and metagenomes.
</code></pre></div>
<h3 id="pilons">Pilon's:<a class="headerlink" href="#pilons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pilon is an automated genome assembly improvement and variant detection tool
</code></pre></div>
<h3 id="pilon">Pilon:<a class="headerlink" href="#pilon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pilon is an automated genome assembly improvement and variant detection tool
</code></pre></div>
<h3 id="pnetcdfs">PnetCDF's:<a class="headerlink" href="#pnetcdfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Parallel netCDF: A Parallel I/O Library for NetCDF File Access
</code></pre></div>
<h3 id="pnetcdf">PnetCDF:<a class="headerlink" href="#pnetcdf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Parallel netCDF: A Parallel I/O Library for NetCDF File Access
</code></pre></div>
<h3 id="porechops">Porechop's:<a class="headerlink" href="#porechops" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
</code></pre></div>
<h3 id="porechop">Porechop:<a class="headerlink" href="#porechop" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
</code></pre></div>
<h3 id="postgresqls">PostgreSQL's:<a class="headerlink" href="#postgresqls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Client-side programs and libraries for accessing PostgreSQL databases.
</code></pre></div>
<h3 id="postgresql">PostgreSQL:<a class="headerlink" href="#postgresql" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Client-side programs and libraries for accessing PostgreSQL databases.
</code></pre></div>
<h3 id="prodigals">Prodigal's:<a class="headerlink" href="#prodigals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
</code></pre></div>
<h3 id="prodigal">Prodigal:<a class="headerlink" href="#prodigal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
</code></pre></div>
<h3 id="prothints">ProtHint's:<a class="headerlink" href="#prothints" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for predicting and scoring hints (in the form of introns, start and
</code></pre></div>
<h3 id="prothint">ProtHint:<a class="headerlink" href="#prothint" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline for predicting and scoring hints (in the form of introns, start and
</code></pre></div>
<h3 id="proteinorthos">Proteinortho's:<a class="headerlink" href="#proteinorthos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Proteinortho is a tool to detect orthologous genes within different species.
</code></pre></div>
<h3 id="proteinortho">Proteinortho:<a class="headerlink" href="#proteinortho" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Proteinortho is a tool to detect orthologous genes within different species.
</code></pre></div>
<h3 id="pyopengls">PyOpenGL's:<a class="headerlink" href="#pyopengls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.
</code></pre></div>
<h3 id="pyopengl">PyOpenGL:<a class="headerlink" href="#pyopengl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.
</code></pre></div>
<h3 id="pyqts">PyQt's:<a class="headerlink" href="#pyqts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.
</code></pre></div>
<h3 id="pyqt">PyQt:<a class="headerlink" href="#pyqt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.
</code></pre></div>
<h3 id="pytorchs">PyTorch's:<a class="headerlink" href="#pytorchs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tensors and Dynamic neural networks in Python with strong GPU acceleration.
</code></pre></div>
<h3 id="pytorch">PyTorch:<a class="headerlink" href="#pytorch" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tensors and Dynamic neural networks in Python with strong GPU acceleration.
</code></pre></div>
<h3 id="pythons">Python's:<a class="headerlink" href="#pythons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python is a programming language that lets you work more quickly and integrate your systems more effectively.
</code></pre></div>
<h3 id="python">Python:<a class="headerlink" href="#python" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python is a programming language that lets you work more quickly and integrate your systems more effectively.
</code></pre></div>
<h3 id="python-gpus">Python-GPU's:<a class="headerlink" href="#python-gpus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The python packages which depend on CUDA: pycuda, pygpu, and scikit-cuda.
</code></pre></div>
<h3 id="python-gpu">Python-GPU:<a class="headerlink" href="#python-gpu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The python packages which depend on CUDA: pycuda, pygpu, and scikit-cuda.
</code></pre></div>
<h3 id="python-geos">Python-Geo's:<a class="headerlink" href="#python-geos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python packages for geospatial data I/O, mostly based on the OSGEO libraries GDAL and OGR
</code></pre></div>
<h3 id="python-geo">Python-Geo:<a class="headerlink" href="#python-geo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python packages for geospatial data I/O, mostly based on the OSGEO libraries GDAL and OGR
</code></pre></div>
<h3 id="qiime2s">QIIME2's:<a class="headerlink" href="#qiime2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source bioinformatics pipeline for microbiome analysis
</code></pre></div>
<h3 id="qiime2">QIIME2:<a class="headerlink" href="#qiime2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source bioinformatics pipeline for microbiome analysis
</code></pre></div>
<h3 id="quasts">QUAST's:<a class="headerlink" href="#quasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Evaluates genome assemblies
</code></pre></div>
<h3 id="quast">QUAST:<a class="headerlink" href="#quast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Evaluates genome assemblies
</code></pre></div>
<h3 id="qt5s">Qt5's:<a class="headerlink" href="#qt5s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Qt is a comprehensive cross-platform C++ application framework.
</code></pre></div>
<h3 id="qt5">Qt5:<a class="headerlink" href="#qt5" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Qt is a comprehensive cross-platform C++ application framework.
</code></pre></div>
<h3 id="quantumespressos">QuantumESPRESSO's:<a class="headerlink" href="#quantumespressos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Quantum ESPRESSO  is an integrated suite of computer codes
</code></pre></div>
<h3 id="quantumespresso">QuantumESPRESSO:<a class="headerlink" href="#quantumespresso" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Quantum ESPRESSO  is an integrated suite of computer codes
</code></pre></div>
<h3 id="quicktrees">QuickTree's:<a class="headerlink" href="#quicktrees" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient implementation of the Neighbor-Joining algorithm, capable of reconstructing phylogenies from huge alignments .
</code></pre></div>
<h3 id="quicktree">QuickTree:<a class="headerlink" href="#quicktree" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient implementation of the Neighbor-Joining algorithm, capable of reconstructing phylogenies from huge alignments .
</code></pre></div>
<h3 id="rs">R's:<a class="headerlink" href="#rs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R is a free software environment for statistical computing and graphics.
</code></pre></div>
<h3 id="r">R:<a class="headerlink" href="#r" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R is a free software environment for statistical computing and graphics.
</code></pre></div>
<h3 id="r-geos">R-Geo's:<a class="headerlink" href="#r-geos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R packages for Geometric and Geospatial data which depend
</code></pre></div>
<h3 id="r-geo">R-Geo:<a class="headerlink" href="#r-geo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R packages for Geometric and Geospatial data which depend
</code></pre></div>
<h3 id="r-bundle-bioconductors">R-bundle-Bioconductor's:<a class="headerlink" href="#r-bundle-bioconductors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bioconductor provides tools for the analysis and comprehension
</code></pre></div>
<h3 id="r-bundle-bioconductor">R-bundle-Bioconductor:<a class="headerlink" href="#r-bundle-bioconductor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Bioconductor provides tools for the analysis and comprehension
</code></pre></div>
<h3 id="rangs-gshhss">RANGS-GSHHS's:<a class="headerlink" href="#rangs-gshhss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A binary file set RANGS (Regionally Accessible Nested Global Shorelines) based on GSHHS (Global Self-consistent Hierarchical High-resolution Shorelines) data.  Note: RANGS-GSHHS is used with NCL.
</code></pre></div>
<h3 id="rangs-gshhs">RANGS-GSHHS:<a class="headerlink" href="#rangs-gshhs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A binary file set RANGS (Regionally Accessible Nested Global Shorelines) based on GSHHS (Global Self-consistent Hierarchical High-resolution Shorelines) data.  Note: RANGS-GSHHS is used with NCL.
</code></pre></div>
<h3 id="raxmls">RAxML's:<a class="headerlink" href="#raxmls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
</code></pre></div>
<h3 id="raxml">RAxML:<a class="headerlink" href="#raxml" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
</code></pre></div>
<h3 id="raxml-ngs">RAxML-NG's:<a class="headerlink" href="#raxml-ngs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)
</code></pre></div>
<h3 id="raxml-ng">RAxML-NG:<a class="headerlink" href="#raxml-ng" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML)
</code></pre></div>
<h3 id="rdp-classifiers">RDP-Classifier's:<a class="headerlink" href="#rdp-classifiers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic
</code></pre></div>
<h3 id="rdp-classifier">RDP-Classifier:<a class="headerlink" href="#rdp-classifier" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic
</code></pre></div>
<h3 id="recons">RECON's:<a class="headerlink" href="#recons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo identification and classification of repeat sequence families from genomic sequences
</code></pre></div>
<h3 id="recon">RECON:<a class="headerlink" href="#recon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo identification and classification of repeat sequence families from genomic sequences
</code></pre></div>
<h3 id="rmblasts">RMBlast's:<a class="headerlink" href="#rmblasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RMBlast supports RepeatMasker searches by adding a few necessary features to the stock NCBI blastn program. These include:
</code></pre></div>
<h3 id="rmblast">RMBlast:<a class="headerlink" href="#rmblast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RMBlast supports RepeatMasker searches by adding a few necessary features to the stock NCBI blastn program. These include:
</code></pre></div>
<h3 id="rnammers">RNAmmer's:<a class="headerlink" href="#rnammers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>consistent and rapid annotation of ribosomal RNA genes.
</code></pre></div>
<h3 id="rnammer">RNAmmer:<a class="headerlink" href="#rnammer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>consistent and rapid annotation of ribosomal RNA genes.
</code></pre></div>
<h3 id="rocms">ROCm's:<a class="headerlink" href="#rocms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Platform for GPU Enabled HPC and UltraScale Computing
</code></pre></div>
<h3 id="rocm">ROCm:<a class="headerlink" href="#rocm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Platform for GPU Enabled HPC and UltraScale Computing
</code></pre></div>
<h3 id="roots">ROOT's:<a class="headerlink" href="#roots" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ROOT system provides a set of OO frameworks with all the functionality
</code></pre></div>
<h3 id="root">ROOT:<a class="headerlink" href="#root" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ROOT system provides a set of OO frameworks with all the functionality
</code></pre></div>
<h3 id="rsems">RSEM's:<a class="headerlink" href="#rsems" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Estimates gene and isoform expression levels from RNA-Seq data
</code></pre></div>
<h3 id="rsem">RSEM:<a class="headerlink" href="#rsem" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Estimates gene and isoform expression levels from RNA-Seq data
</code></pre></div>
<h3 id="rsgislibs">RSGISLib's:<a class="headerlink" href="#rsgislibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Remote Sensing and GIS software library (RSGISLib) is a
</code></pre></div>
<h3 id="rsgislib">RSGISLib:<a class="headerlink" href="#rsgislib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Remote Sensing and GIS software library (RSGISLib) is a
</code></pre></div>
<h3 id="racons">Racon's:<a class="headerlink" href="#racons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.
</code></pre></div>
<h3 id="racon">Racon:<a class="headerlink" href="#racon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.
</code></pre></div>
<h3 id="ragouts">Ragout's:<a class="headerlink" href="#ragouts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for chromosome assembly using multiple references.
</code></pre></div>
<h3 id="ragout">Ragout:<a class="headerlink" href="#ragout" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for chromosome assembly using multiple references.
</code></pre></div>
<h3 id="rapidnjs">RapidNJ's:<a class="headerlink" href="#rapidnjs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An algorithmic engineered implementation of canonical neighbour-joining.
</code></pre></div>
<h3 id="rapidnj">RapidNJ:<a class="headerlink" href="#rapidnj" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An algorithmic engineered implementation of canonical neighbour-joining.
</code></pre></div>
<h3 id="ravens">Raven's:<a class="headerlink" href="#ravens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo genome assembler for long uncorrected reads.
</code></pre></div>
<h3 id="raven">Raven:<a class="headerlink" href="#raven" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo genome assembler for long uncorrected reads.
</code></pre></div>
<h3 id="rcorrectors">Rcorrector's:<a class="headerlink" href="#rcorrectors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>kmer-based error correction method for RNA-seq data.
</code></pre></div>
<h3 id="rcorrector">Rcorrector:<a class="headerlink" href="#rcorrector" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>kmer-based error correction method for RNA-seq data.
</code></pre></div>
<h3 id="relions">Relion's:<a class="headerlink" href="#relions" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on)
</code></pre></div>
<h3 id="relion">Relion:<a class="headerlink" href="#relion" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on)
</code></pre></div>
<h3 id="repeatmaskers">RepeatMasker's:<a class="headerlink" href="#repeatmaskers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Screens DNA sequences for interspersed repeats and low complexity DNA sequences.
</code></pre></div>
<h3 id="repeatmasker">RepeatMasker:<a class="headerlink" href="#repeatmasker" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Screens DNA sequences for interspersed repeats and low complexity DNA sequences.
</code></pre></div>
<h3 id="repeatmodelers">RepeatModeler's:<a class="headerlink" href="#repeatmodelers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo transposable element (TE) family identification and modeling package.
</code></pre></div>
<h3 id="repeatmodeler">RepeatModeler:<a class="headerlink" href="#repeatmodeler" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo transposable element (TE) family identification and modeling package.
</code></pre></div>
<h3 id="repeatscouts">RepeatScout's:<a class="headerlink" href="#repeatscouts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo identification of repeat families in large genomes
</code></pre></div>
<h3 id="repeatscout">RepeatScout:<a class="headerlink" href="#repeatscout" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>De novo identification of repeat families in large genomes
</code></pre></div>
<h3 id="riskscapes">Riskscape's:<a class="headerlink" href="#riskscapes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RiskScape is an open-source spatial data processing application used for multi-hazard risk analysis. RiskScape is highly customisable, letting modellers tailor the risk analysis to suit the problem domain and input data being modelled.
</code></pre></div>
<h3 id="riskscape">Riskscape:<a class="headerlink" href="#riskscape" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RiskScape is an open-source spatial data processing application used for multi-hazard risk analysis. RiskScape is highly customisable, letting modellers tailor the risk analysis to suit the problem domain and input data being modelled.
</code></pre></div>
<h3 id="roarys">Roary's:<a class="headerlink" href="#roarys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid large-scale prokaryote pan genome analysis
</code></pre></div>
<h3 id="roary">Roary:<a class="headerlink" href="#roary" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid large-scale prokaryote pan genome analysis
</code></pre></div>
<h3 id="rosettas">Rosetta's:<a class="headerlink" href="#rosettas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rosetta is the premier software suite for modeling macromolecular structures. As a flexible,
</code></pre></div>
<h3 id="rosetta">Rosetta:<a class="headerlink" href="#rosetta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rosetta is the premier software suite for modeling macromolecular structures. As a flexible,
</code></pre></div>
<h3 id="rstudios">Rstudio's:<a class="headerlink" href="#rstudios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RStudio is a set of integrated tools designed to help you be more productive with R.
</code></pre></div>
<h3 id="rstudio">Rstudio:<a class="headerlink" href="#rstudio" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RStudio is a set of integrated tools designed to help you be more productive with R.
</code></pre></div>
<h3 id="rubys">Ruby's:<a class="headerlink" href="#rubys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ruby is a dynamic, open source programming language with
</code></pre></div>
<h3 id="ruby">Ruby:<a class="headerlink" href="#ruby" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ruby is a dynamic, open source programming language with
</code></pre></div>
<h3 id="rusts">Rust's:<a class="headerlink" href="#rusts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Systems programming language that runs blazingly fast, prevents segfaults,
</code></pre></div>
<h3 id="rust">Rust:<a class="headerlink" href="#rust" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Systems programming language that runs blazingly fast, prevents segfaults,
</code></pre></div>
<h3 id="sages">SAGE's:<a class="headerlink" href="#sages" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ppackage containing programs for use in the genetic analysis of
</code></pre></div>
<h3 id="sage">SAGE:<a class="headerlink" href="#sage" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ppackage containing programs for use in the genetic analysis of
</code></pre></div>
<h3 id="samtoolss">SAMtools's:<a class="headerlink" href="#samtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Samtools is a suite of programs for interacting with high-throughput sequencing data.
</code></pre></div>
<h3 id="samtools">SAMtools:<a class="headerlink" href="#samtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Samtools is a suite of programs for interacting with high-throughput sequencing data.
</code></pre></div>
<h3 id="sass">SAS's:<a class="headerlink" href="#sass" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SAS is a statistical software suite developed by SAS Institute for data management, advanced analytics, multivariate analysis, business intelligence, criminal investigation, and predictive analytics. - Homepage: https://www.sas.com/en_nz/home.html/
</code></pre></div>
<h3 id="sas">SAS:<a class="headerlink" href="#sas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SAS is a statistical software suite developed by SAS Institute for data management, advanced analytics, multivariate analysis, business intelligence, criminal investigation, and predictive analytics. - Homepage: https://www.sas.com/en_nz/home.html/
</code></pre></div>
<h3 id="scotchs">SCOTCH's:<a class="headerlink" href="#scotchs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
</code></pre></div>
<h3 id="scotch">SCOTCH:<a class="headerlink" href="#scotch" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.
</code></pre></div>
<h3 id="sconss">SCons's:<a class="headerlink" href="#sconss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SCons is a software construction tool.
</code></pre></div>
<h3 id="scons">SCons:<a class="headerlink" href="#scons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SCons is a software construction tool.
</code></pre></div>
<h3 id="sdl2s">SDL2's:<a class="headerlink" href="#sdl2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Simple DirectMedia Layer, a cross-platform multimedia library
</code></pre></div>
<h3 id="sdl2">SDL2:<a class="headerlink" href="#sdl2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Simple DirectMedia Layer, a cross-platform multimedia library
</code></pre></div>
<h3 id="sepps">SEPP's:<a class="headerlink" href="#sepps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SATe-enabled Phylogenetic Placement. Phylogenetic placement of short reads into reference alignments and trees.
</code></pre></div>
<h3 id="sepp">SEPP:<a class="headerlink" href="#sepp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SATe-enabled Phylogenetic Placement. Phylogenetic placement of short reads into reference alignments and trees.
</code></pre></div>
<h3 id="shapeit4s">SHAPEIT4's:<a class="headerlink" href="#shapeit4s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Estimation of haplotypes (aka phasing)
</code></pre></div>
<h3 id="shapeit4">SHAPEIT4:<a class="headerlink" href="#shapeit4" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Estimation of haplotypes (aka phasing)
</code></pre></div>
<h3 id="sionlibs">SIONlib's:<a class="headerlink" href="#sionlibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Scalable I/O library for parallel access to task-local files.
</code></pre></div>
<h3 id="sionlib">SIONlib:<a class="headerlink" href="#sionlib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Scalable I/O library for parallel access to task-local files.
</code></pre></div>
<h3 id="sips">SIP's:<a class="headerlink" href="#sips" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.
</code></pre></div>
<h3 id="sip">SIP:<a class="headerlink" href="#sip" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.
</code></pre></div>
<h3 id="skesas">SKESA's:<a class="headerlink" href="#skesas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.
</code></pre></div>
<h3 id="skesa">SKESA:<a class="headerlink" href="#skesa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.
</code></pre></div>
<h3 id="smrt-links">SMRT-Link's:<a class="headerlink" href="#smrt-links" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PacBio’s open-source software suite is designed for use with Single Molecule,
</code></pre></div>
<h3 id="smrt-link">SMRT-Link:<a class="headerlink" href="#smrt-link" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PacBio’s open-source software suite is designed for use with Single Molecule,
</code></pre></div>
<h3 id="snvoter-nanomethphases">SNVoter-NanoMethPhase's:<a class="headerlink" href="#snvoter-nanomethphases" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SNVoter - A top up tool to enhance SNV calling from Nanopore sequencing data &amp;
</code></pre></div>
<h3 id="snvoter-nanomethphase">SNVoter-NanoMethPhase:<a class="headerlink" href="#snvoter-nanomethphase" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SNVoter - A top up tool to enhance SNV calling from Nanopore sequencing data &amp;
</code></pre></div>
<h3 id="soapdenovo2s">SOAPdenovo2's:<a class="headerlink" href="#soapdenovo2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Short Oligonucleotide Analysis Package - novel short-read assembly
</code></pre></div>
<h3 id="soapdenovo2">SOAPdenovo2:<a class="headerlink" href="#soapdenovo2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Short Oligonucleotide Analysis Package - novel short-read assembly
</code></pre></div>
<h3 id="socis">SOCI's:<a class="headerlink" href="#socis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Database access library for C++ that makes the illusion of embedding SQL queries in the
</code></pre></div>
<h3 id="soci">SOCI:<a class="headerlink" href="#soci" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Database access library for C++ that makes the illusion of embedding SQL queries in the
</code></pre></div>
<h3 id="spadess">SPAdes's:<a class="headerlink" href="#spadess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome assembler for single-cell and isolates data sets
</code></pre></div>
<h3 id="spades">SPAdes:<a class="headerlink" href="#spades" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genome assembler for single-cell and isolates data sets
</code></pre></div>
<h3 id="specfem3ds">SPECFEM3D's:<a class="headerlink" href="#specfem3ds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SPECFEM3D Cartesian simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic, poroelastic or seismic wave propagation in any type of conforming mesh of hexahedra (structured or not.) It can, for instance, model seismic waves propagating in sedimentary basins or any other regional geological model following earthquakes. It can also be used for non-destructive testing or for ocean acoustics.
</code></pre></div>
<h3 id="specfem3d">SPECFEM3D:<a class="headerlink" href="#specfem3d" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SPECFEM3D Cartesian simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic, poroelastic or seismic wave propagation in any type of conforming mesh of hexahedra (structured or not.) It can, for instance, model seismic waves propagating in sedimentary basins or any other regional geological model following earthquakes. It can also be used for non-destructive testing or for ocean acoustics.
</code></pre></div>
<h3 id="spiders">SPIDER's:<a class="headerlink" href="#spiders" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>System for Processing Image Data from Electron microscopy and Related fields
</code></pre></div>
<h3 id="spider">SPIDER:<a class="headerlink" href="#spider" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>System for Processing Image Data from Electron microscopy and Related fields
</code></pre></div>
<h3 id="sqlites">SQLite's:<a class="headerlink" href="#sqlites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SQLite: SQL Database Engine in a C Library
</code></pre></div>
<h3 id="sqlite">SQLite:<a class="headerlink" href="#sqlite" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SQLite: SQL Database Engine in a C Library
</code></pre></div>
<h3 id="sqlpluss">SQLplus's:<a class="headerlink" href="#sqlpluss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SQL*Plus is an interactive and batch query tool that is installed with every Oracle Database installation. It has a command-line user interface and a Windows Graphical User Interface (GUI).
</code></pre></div>
<h3 id="sqlplus">SQLplus:<a class="headerlink" href="#sqlplus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SQL*Plus is an interactive and batch query tool that is installed with every Oracle Database installation. It has a command-line user interface and a Windows Graphical User Interface (GUI).
</code></pre></div>
<h3 id="ssaha2s">SSAHA2's:<a class="headerlink" href="#ssaha2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pairwise sequence alignment program designed for the efficient mapping of sequencing
</code></pre></div>
<h3 id="ssaha2">SSAHA2:<a class="headerlink" href="#ssaha2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pairwise sequence alignment program designed for the efficient mapping of sequencing
</code></pre></div>
<h3 id="stars">STAR's:<a class="headerlink" href="#stars" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast universal RNA-seq aligner
</code></pre></div>
<h3 id="star">STAR:<a class="headerlink" href="#star" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast universal RNA-seq aligner
</code></pre></div>
<h3 id="star-fusions">STAR-Fusion's:<a class="headerlink" href="#star-fusions" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set
</code></pre></div>
<h3 id="star-fusion">STAR-Fusion:<a class="headerlink" href="#star-fusion" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set
</code></pre></div>
<h3 id="sundialss">SUNDIALS's:<a class="headerlink" href="#sundialss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers
</code></pre></div>
<h3 id="sundials">SUNDIALS:<a class="headerlink" href="#sundials" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers
</code></pre></div>
<h3 id="survivors">SURVIVOR's:<a class="headerlink" href="#survivors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,
</code></pre></div>
<h3 id="survivor">SURVIVOR:<a class="headerlink" href="#survivor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool set for simulating/evaluating SVs, merging and comparing SVs within and among samples,
</code></pre></div>
<h3 id="swigs">SWIG's:<a class="headerlink" href="#swigs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.
</code></pre></div>
<h3 id="swig">SWIG:<a class="headerlink" href="#swig" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.
</code></pre></div>
<h3 id="salmons">Salmon's:<a class="headerlink" href="#salmons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Salmon is a wicked-fast program to produce a highly-accurate,
</code></pre></div>
<h3 id="salmon">Salmon:<a class="headerlink" href="#salmon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Salmon is a wicked-fast program to produce a highly-accurate,
</code></pre></div>
<h3 id="sambambas">Sambamba's:<a class="headerlink" href="#sambambas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for working with SAM/BAM data
</code></pre></div>
<h3 id="sambamba">Sambamba:<a class="headerlink" href="#sambamba" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for working with SAM/BAM data
</code></pre></div>
<h3 id="scalapacks">ScaLAPACK's:<a class="headerlink" href="#scalapacks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
</code></pre></div>
<h3 id="scalapack">ScaLAPACK:<a class="headerlink" href="#scalapack" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
</code></pre></div>
<h3 id="scalascas">Scalasca's:<a class="headerlink" href="#scalascas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool that supports the performance optimization of
</code></pre></div>
<h3 id="scalasca">Scalasca:<a class="headerlink" href="#scalasca" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool that supports the performance optimization of
</code></pre></div>
<h3 id="score-ps">Score-P's:<a class="headerlink" href="#score-ps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Measurement infrastructure is a highly scalable and easy-to-use
</code></pre></div>
<h3 id="score-p">Score-P:<a class="headerlink" href="#score-p" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Measurement infrastructure is a highly scalable and easy-to-use
</code></pre></div>
<h3 id="seqan3s">SeqAn3's:<a class="headerlink" href="#seqan3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ library of efficient algorithms and data structures for the
</code></pre></div>
<h3 id="seqan3">SeqAn3:<a class="headerlink" href="#seqan3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ library of efficient algorithms and data structures for the
</code></pre></div>
<h3 id="seqkits">SeqKit's:<a class="headerlink" href="#seqkits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast toolkit for FASTA/Q file manipulation
</code></pre></div>
<h3 id="seqkit">SeqKit:<a class="headerlink" href="#seqkit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ultrafast toolkit for FASTA/Q file manipulation
</code></pre></div>
<h3 id="seqmonks">SeqMonk's:<a class="headerlink" href="#seqmonks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to visualise and analyse high throughput mapped sequence data.
</code></pre></div>
<h3 id="seqmonk">SeqMonk:<a class="headerlink" href="#seqmonk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool to visualise and analyse high throughput mapped sequence data.
</code></pre></div>
<h3 id="sibelias">SiBELia's:<a class="headerlink" href="#sibelias" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A comparative genomics tool for analysing genomic variations that correlate with pathogens, or
</code></pre></div>
<h3 id="sibelia">SiBELia:<a class="headerlink" href="#sibelia" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A comparative genomics tool for analysing genomic variations that correlate with pathogens, or
</code></pre></div>
<h3 id="siestas">Siesta's:<a class="headerlink" href="#siestas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SIESTA is both a method and its computer program implementation, to perform efficient electronic
</code></pre></div>
<h3 id="siesta">Siesta:<a class="headerlink" href="#siesta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SIESTA is both a method and its computer program implementation, to perform efficient electronic
</code></pre></div>
<h3 id="signalps">SignalP's:<a class="headerlink" href="#signalps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SignalP predicts the presence  and  location of signal peptide  cleavage sites
</code></pre></div>
<h3 id="signalp">SignalP:<a class="headerlink" href="#signalp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SignalP predicts the presence  and  location of signal peptide  cleavage sites
</code></pre></div>
<h3 id="sniffless">Sniffles's:<a class="headerlink" href="#sniffless" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A fast structural variant caller for long-read sequencing.
</code></pre></div>
<h3 id="sniffles">Sniffles:<a class="headerlink" href="#sniffles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A fast structural variant caller for long-read sequencing.
</code></pre></div>
<h3 id="sortmernas">SortMeRNA's:<a class="headerlink" href="#sortmernas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.
</code></pre></div>
<h3 id="sortmerna">SortMeRNA:<a class="headerlink" href="#sortmerna" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.
</code></pre></div>
<h3 id="sourcetrackers">SourceTracker's:<a class="headerlink" href="#sourcetrackers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SourceTracker is a Bayesian approach to estimating the proportion of a novel community that comes
</code></pre></div>
<h3 id="sourcetracker">SourceTracker:<a class="headerlink" href="#sourcetracker" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SourceTracker is a Bayesian approach to estimating the proportion of a novel community that comes
</code></pre></div>
<h3 id="spacks">Spack's:<a class="headerlink" href="#spacks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific
</code></pre></div>
<h3 id="spack">Spack:<a class="headerlink" href="#spack" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific
</code></pre></div>
<h3 id="sparks">Spark's:<a class="headerlink" href="#sparks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spark is Hadoop MapReduce done in memory
</code></pre></div>
<h3 id="spark">Spark:<a class="headerlink" href="#spark" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spark is Hadoop MapReduce done in memory
</code></pre></div>
<h3 id="squeezemetas">SqueezeMeta's:<a class="headerlink" href="#squeezemetas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>fully automated metagenomics pipeline, from reads to bins.
</code></pre></div>
<h3 id="squeezemeta">SqueezeMeta:<a class="headerlink" href="#squeezemeta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>fully automated metagenomics pipeline, from reads to bins.
</code></pre></div>
<h3 id="stackss">Stacks's:<a class="headerlink" href="#stackss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
</code></pre></div>
<h3 id="stacks">Stacks:<a class="headerlink" href="#stacks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
</code></pre></div>
<h3 id="stringties">StringTie's:<a class="headerlink" href="#stringties" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.
</code></pre></div>
<h3 id="stringtie">StringTie:<a class="headerlink" href="#stringtie" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.
</code></pre></div>
<h3 id="structures">Structure's:<a class="headerlink" href="#structures" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The program structure is a free software package for using multi-locus genotype data to investigate
</code></pre></div>
<h3 id="structure">Structure:<a class="headerlink" href="#structure" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The program structure is a free software package for using multi-locus genotype data to investigate
</code></pre></div>
<h3 id="subreads">Subread's:<a class="headerlink" href="#subreads" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High performance read alignment, quantification and mutation discovery
</code></pre></div>
<h3 id="subread">Subread:<a class="headerlink" href="#subread" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High performance read alignment, quantification and mutation discovery
</code></pre></div>
<h3 id="subversions">Subversion's:<a class="headerlink" href="#subversions" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Subversion is an open source version control system.
</code></pre></div>
<h3 id="subversion">Subversion:<a class="headerlink" href="#subversion" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Subversion is an open source version control system.
</code></pre></div>
<h3 id="suitesparses">SuiteSparse's:<a class="headerlink" href="#suitesparses" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SuiteSparse is a collection of libraries manipulate sparse matrices.
</code></pre></div>
<h3 id="suitesparse">SuiteSparse:<a class="headerlink" href="#suitesparse" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SuiteSparse is a collection of libraries manipulate sparse matrices.
</code></pre></div>
<h3 id="superlus">SuperLU's:<a class="headerlink" href="#superlus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Solution of large, sparse, nonsymmetric systems of linear equations.
</code></pre></div>
<h3 id="superlu">SuperLU:<a class="headerlink" href="#superlu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Solution of large, sparse, nonsymmetric systems of linear equations.
</code></pre></div>
<h3 id="supernovas">Supernova's:<a class="headerlink" href="#supernovas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Supernova is a software package for de novo assembly from Chromium Linked-Reads
</code></pre></div>
<h3 id="supernova">Supernova:<a class="headerlink" href="#supernova" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Supernova is a software package for de novo assembly from Chromium Linked-Reads
</code></pre></div>
<h3 id="tmhmms">TMHMM's:<a class="headerlink" href="#tmhmms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prediction of transmembrane helices in proteins
</code></pre></div>
<h3 id="tmhmm">TMHMM:<a class="headerlink" href="#tmhmm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prediction of transmembrane helices in proteins
</code></pre></div>
<h3 id="tsebras">TSEBRA's:<a class="headerlink" href="#tsebras" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transcript Selector for BRAKER
</code></pre></div>
<h3 id="tsebra">TSEBRA:<a class="headerlink" href="#tsebra" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transcript Selector for BRAKER
</code></pre></div>
<h3 id="turbomoles">TURBOMOLE's:<a class="headerlink" href="#turbomoles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program Package For Electronic Structure Calculations.
</code></pre></div>
<h3 id="turbomole">TURBOMOLE:<a class="headerlink" href="#turbomole" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program Package For Electronic Structure Calculations.
</code></pre></div>
<h3 id="twl-ninjas">TWL-NINJA's:<a class="headerlink" href="#twl-ninjas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nearly Infinite Neighbor Joining Application.
</code></pre></div>
<h3 id="twl-ninja">TWL-NINJA:<a class="headerlink" href="#twl-ninja" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nearly Infinite Neighbor Joining Application.
</code></pre></div>
<h3 id="tcls">Tcl's:<a class="headerlink" href="#tcls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language,
</code></pre></div>
<h3 id="tcl">Tcl:<a class="headerlink" href="#tcl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language,
</code></pre></div>
<h3 id="tensorflows">TensorFlow's:<a class="headerlink" href="#tensorflows" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source software library for Machine Intelligence
</code></pre></div>
<h3 id="tensorflow">TensorFlow:<a class="headerlink" href="#tensorflow" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open-source software library for Machine Intelligence
</code></pre></div>
<h3 id="tensorrts">TensorRT's:<a class="headerlink" href="#tensorrts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA TensorRT is a platform for high-performance deep learning inference
</code></pre></div>
<h3 id="tensorrt">TensorRT:<a class="headerlink" href="#tensorrt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NVIDIA TensorRT is a platform for high-performance deep learning inference
</code></pre></div>
<h3 id="theanos">Theano's:<a class="headerlink" href="#theanos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Theano is a Python library that allows you to define, optimize,
</code></pre></div>
<h3 id="theano">Theano:<a class="headerlink" href="#theano" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Theano is a Python library that allows you to define, optimize,
</code></pre></div>
<h3 id="tks">Tk's:<a class="headerlink" href="#tks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
</code></pre></div>
<h3 id="tk">Tk:<a class="headerlink" href="#tk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
</code></pre></div>
<h3 id="tophats">TopHat's:<a class="headerlink" href="#tophats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TopHat is a fast splice junction mapper for RNA-Seq reads.
</code></pre></div>
<h3 id="tophat">TopHat:<a class="headerlink" href="#tophat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TopHat is a fast splice junction mapper for RNA-Seq reads.
</code></pre></div>
<h3 id="transdecoders">TransDecoder's:<a class="headerlink" href="#transdecoders" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TransDecoder identifies candidate coding regions within transcript sequences.
</code></pre></div>
<h3 id="transdecoder">TransDecoder:<a class="headerlink" href="#transdecoder" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TransDecoder identifies candidate coding regions within transcript sequences.
</code></pre></div>
<h3 id="treemixs">TreeMix's:<a class="headerlink" href="#treemixs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a
</code></pre></div>
<h3 id="treemix">TreeMix:<a class="headerlink" href="#treemix" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a
</code></pre></div>
<h3 id="trilinoss">Trilinos's:<a class="headerlink" href="#trilinoss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Trilinos Project is an effort to develop algorithms and enabling technologies
</code></pre></div>
<h3 id="trilinos">Trilinos:<a class="headerlink" href="#trilinos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Trilinos Project is an effort to develop algorithms and enabling technologies
</code></pre></div>
<h3 id="trimgalores">TrimGalore's:<a class="headerlink" href="#trimgalores" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A wrapper of FastQC and cutadapt to automate quality and adapter trimming
</code></pre></div>
<h3 id="trimgalore">TrimGalore:<a class="headerlink" href="#trimgalore" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A wrapper of FastQC and cutadapt to automate quality and adapter trimming
</code></pre></div>
<h3 id="trimmomatics">Trimmomatic's:<a class="headerlink" href="#trimmomatics" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Trimmomatic performs a variety of useful trimming tasks for illumina
</code></pre></div>
<h3 id="trimmomatic">Trimmomatic:<a class="headerlink" href="#trimmomatic" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Trimmomatic performs a variety of useful trimming tasks for illumina
</code></pre></div>
<h3 id="trinitys">Trinity's:<a class="headerlink" href="#trinitys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Trinity represents a novel method for the efficient and robust de novo reconstruction
</code></pre></div>
<h3 id="trinity">Trinity:<a class="headerlink" href="#trinity" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Trinity represents a novel method for the efficient and robust de novo reconstruction
</code></pre></div>
<h3 id="trinotates">Trinotate's:<a class="headerlink" href="#trinotates" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ library of efficient algorithms and data structures for the
</code></pre></div>
<h3 id="trinotate">Trinotate:<a class="headerlink" href="#trinotate" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ library of efficient algorithms and data structures for the
</code></pre></div>
<h3 id="trycyclers">Trycycler's:<a class="headerlink" href="#trycyclers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for generating consensus long-read assemblies for bacterial genomes.
</code></pre></div>
<h3 id="trycycler">Trycycler:<a class="headerlink" href="#trycycler" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for generating consensus long-read assemblies for bacterial genomes.
</code></pre></div>
<h3 id="tuiviews">TuiView's:<a class="headerlink" href="#tuiviews" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TuiView is a lightweight raster GIS with powerful raster attribute table manipulation
</code></pre></div>
<h3 id="tuiview">TuiView:<a class="headerlink" href="#tuiview" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TuiView is a lightweight raster GIS with powerful raster attribute table manipulation
</code></pre></div>
<h3 id="turbovncs">TurboVNC's:<a class="headerlink" href="#turbovncs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide
</code></pre></div>
<h3 id="turbovnc">TurboVNC:<a class="headerlink" href="#turbovnc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide
</code></pre></div>
<h3 id="ucxs">UCX's:<a class="headerlink" href="#ucxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Unified Communication X
</code></pre></div>
<h3 id="ucx">UCX:<a class="headerlink" href="#ucx" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Unified Communication X
</code></pre></div>
<h3 id="udunitss">UDUNITS's:<a class="headerlink" href="#udunitss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
</code></pre></div>
<h3 id="udunits">UDUNITS:<a class="headerlink" href="#udunits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.
</code></pre></div>
<h3 id="usearchs">USEARCH's:<a class="headerlink" href="#usearchs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are
</code></pre></div>
<h3 id="usearch">USEARCH:<a class="headerlink" href="#usearch" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are
</code></pre></div>
<h3 id="unicyclers">Unicycler's:<a class="headerlink" href="#unicyclers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets
</code></pre></div>
<h3 id="unicycler">Unicycler:<a class="headerlink" href="#unicycler" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets
</code></pre></div>
<h3 id="vasps">VASP's:<a class="headerlink" href="#vasps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.
</code></pre></div>
<h3 id="vasp">VASP:<a class="headerlink" href="#vasp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.
</code></pre></div>
<h3 id="vcf-kits">VCF-kit's:<a class="headerlink" href="#vcf-kits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VCF-kit is a command-line based collection of utilities for performing analysis on
</code></pre></div>
<h3 id="vcf-kit">VCF-kit:<a class="headerlink" href="#vcf-kit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VCF-kit is a command-line based collection of utilities for performing analysis on
</code></pre></div>
<h3 id="vcftoolss">VCFtools's:<a class="headerlink" href="#vcftoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The aim of VCFtools is to provide
</code></pre></div>
<h3 id="vcftools">VCFtools:<a class="headerlink" href="#vcftools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The aim of VCFtools is to provide
</code></pre></div>
<h3 id="veps">VEP's:<a class="headerlink" href="#veps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Variant Effect Predictor (VEP) determines the effect of your
</code></pre></div>
<h3 id="vep">VEP:<a class="headerlink" href="#vep" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Variant Effect Predictor (VEP) determines the effect of your
</code></pre></div>
<h3 id="vestas">VESTA's:<a class="headerlink" href="#vestas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies. Run &#39;VESTA-gui&#39; to launch.
</code></pre></div>
<h3 id="vesta">VESTA:<a class="headerlink" href="#vesta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies. Run &#39;VESTA-gui&#39; to launch.
</code></pre></div>
<h3 id="vibrants">VIBRANT's:<a class="headerlink" href="#vibrants" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Virus Identification By iteRative ANnoTation
</code></pre></div>
<h3 id="vibrant">VIBRANT:<a class="headerlink" href="#vibrant" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Virus Identification By iteRative ANnoTation
</code></pre></div>
<h3 id="vmds">VMD's:<a class="headerlink" href="#vmds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
</code></pre></div>
<h3 id="vmd">VMD:<a class="headerlink" href="#vmd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
</code></pre></div>
<h3 id="vsearchs">VSEARCH's:<a class="headerlink" href="#vsearchs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source alternative to the metagenomics tool USEARCH.
</code></pre></div>
<h3 id="vsearch">VSEARCH:<a class="headerlink" href="#vsearch" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An open source alternative to the metagenomics tool USEARCH.
</code></pre></div>
<h3 id="vtks">VTK's:<a class="headerlink" href="#vtks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
</code></pre></div>
<h3 id="vtk">VTK:<a class="headerlink" href="#vtk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.
</code></pre></div>
<h3 id="vtunes">VTune's:<a class="headerlink" href="#vtunes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.
</code></pre></div>
<h3 id="vtune">VTune:<a class="headerlink" href="#vtune" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.
</code></pre></div>
<h3 id="valgrinds">Valgrind's:<a class="headerlink" href="#valgrinds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Valgrind: Debugging and profiling tools
</code></pre></div>
<h3 id="valgrind">Valgrind:<a class="headerlink" href="#valgrind" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Valgrind: Debugging and profiling tools
</code></pre></div>
<h3 id="varscans">VarScan's:<a class="headerlink" href="#varscans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Variant calling and somatic mutation/CNV detection for next-generation sequencing data
</code></pre></div>
<h3 id="varscan">VarScan:<a class="headerlink" href="#varscan" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Variant calling and somatic mutation/CNV detection for next-generation sequencing data
</code></pre></div>
<h3 id="velvets">Velvet's:<a class="headerlink" href="#velvets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence assembler for very short reads
</code></pre></div>
<h3 id="velvet">Velvet:<a class="headerlink" href="#velvet" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Sequence assembler for very short reads
</code></pre></div>
<h3 id="velvetoptimisers">VelvetOptimiser's:<a class="headerlink" href="#velvetoptimisers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Perl script for optimising the three primary parameter options of the Velvet de novo sequence assembler.
</code></pre></div>
<h3 id="velvetoptimiser">VelvetOptimiser:<a class="headerlink" href="#velvetoptimiser" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Perl script for optimising the three primary parameter options of the Velvet de novo sequence assembler.
</code></pre></div>
<h3 id="viennarnas">ViennaRNA's:<a class="headerlink" href="#viennarnas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Vienna RNA Package consists of a C code library and several
</code></pre></div>
<h3 id="viennarna">ViennaRNA:<a class="headerlink" href="#viennarna" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Vienna RNA Package consists of a C code library and several
</code></pre></div>
<h3 id="vims">Vim's:<a class="headerlink" href="#vims" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor &#39;Vi&#39;, with a more complete feature set.
</code></pre></div>
<h3 id="vim">Vim:<a class="headerlink" href="#vim" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor &#39;Vi&#39;, with a more complete feature set.
</code></pre></div>
<h3 id="virhostmatchers">VirHostMatcher's:<a class="headerlink" href="#virhostmatchers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for computing various oligonucleotide frequency (ONF) based distance/dissimialrity measures.
</code></pre></div>
<h3 id="virhostmatcher">VirHostMatcher:<a class="headerlink" href="#virhostmatcher" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for computing various oligonucleotide frequency (ONF) based distance/dissimialrity measures.
</code></pre></div>
<h3 id="virsorters">VirSorter's:<a class="headerlink" href="#virsorters" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VirSorter: mining viral signal from microbial genomic data.
</code></pre></div>
<h3 id="virsorter">VirSorter:<a class="headerlink" href="#virsorter" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VirSorter: mining viral signal from microbial genomic data.
</code></pre></div>
<h3 id="virtualgls">VirtualGL's:<a class="headerlink" href="#virtualgls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VirtualGL is an open source toolkit that gives any Linux or
</code></pre></div>
<h3 id="virtualgl">VirtualGL:<a class="headerlink" href="#virtualgl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VirtualGL is an open source toolkit that gives any Linux or
</code></pre></div>
<h3 id="visits">VisIt's:<a class="headerlink" href="#visits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool.  This version supports interactive CPU-only rendering with the VisIt GUI using the Mesa library. It does not support GPU rendering.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.
</code></pre></div>
<h3 id="visit">VisIt:<a class="headerlink" href="#visit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool.  This version supports interactive CPU-only rendering with the VisIt GUI using the Mesa library. It does not support GPU rendering.  Use the GALLIUM_DRIVER environment variable to choose a software renderer, it is recommended to use  GALLIUM_DRIVER=swr  for best performance.
</code></pre></div>
<h3 id="waafles">WAAFLE's:<a class="headerlink" href="#waafles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow to Annotate Assemblies and Find LGT Events.
</code></pre></div>
<h3 id="waafle">WAAFLE:<a class="headerlink" href="#waafle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow to Annotate Assemblies and Find LGT Events.
</code></pre></div>
<h3 id="whatshaps">WhatsHap's:<a class="headerlink" href="#whatshaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.
</code></pre></div>
<h3 id="whatshap">WhatsHap:<a class="headerlink" href="#whatshap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for phasing genomic variants using DNA sequencing reads, also called read-based phasing or haplotype assembly.
</code></pre></div>
<h3 id="wise2s">Wise2's:<a class="headerlink" href="#wise2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Aligning proteins or protein HMMs to DNA
</code></pre></div>
<h3 id="wise2">Wise2:<a class="headerlink" href="#wise2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Aligning proteins or protein HMMs to DNA
</code></pre></div>
<h3 id="xconvs">XCONV's:<a class="headerlink" href="#xconvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Xconv is a program designed to convert model output into a format suitable for use in various plotting packages. Xconv is designed to be simple to use with a point and click, windows based interface.
</code></pre></div>
<h3 id="xconv">XCONV:<a class="headerlink" href="#xconv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Xconv is a program designed to convert model output into a format suitable for use in various plotting packages. Xconv is designed to be simple to use with a point and click, windows based interface.
</code></pre></div>
<h3 id="xgkss">XGKS's:<a class="headerlink" href="#xgkss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>XGKS is a level 2C implementation of the ANSI Graphical Kernel System
</code></pre></div>
<h3 id="xgks">XGKS:<a class="headerlink" href="#xgks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>XGKS is a level 2C implementation of the ANSI Graphical Kernel System
</code></pre></div>
<h3 id="xhmms">XHMM's:<a class="headerlink" href="#xhmms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Calls copy number variation (CNV) from normalized read-depth data from exome capture or other targeted sequencing experiments.
</code></pre></div>
<h3 id="xhmm">XHMM:<a class="headerlink" href="#xhmm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Calls copy number variation (CNV) from normalized read-depth data from exome capture or other targeted sequencing experiments.
</code></pre></div>
<h3 id="xioss">XIOS's:<a class="headerlink" href="#xioss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>XIOS stands for XML-IO-Server and is a library dedicated to I/O management in climate codes. This version uses netCDF4 with the parallel HDF5 library, and it contains a patch for various problems in the mesh connectivity algorithm
</code></pre></div>
<h3 id="xios">XIOS:<a class="headerlink" href="#xios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>XIOS stands for XML-IO-Server and is a library dedicated to I/O management in climate codes. This version uses netCDF4 with the parallel HDF5 library, and it contains a patch for various problems in the mesh connectivity algorithm
</code></pre></div>
<h3 id="xmds2s">XMDS2's:<a class="headerlink" href="#xmds2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast integrator of stochastic partial differential equations.
</code></pre></div>
<h3 id="xmds2">XMDS2:<a class="headerlink" href="#xmds2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast integrator of stochastic partial differential equations.
</code></pre></div>
<h3 id="xsds">XSD's:<a class="headerlink" href="#xsds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.
</code></pre></div>
<h3 id="xsd">XSD:<a class="headerlink" href="#xsd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.
</code></pre></div>
<h3 id="xzs">XZ's:<a class="headerlink" href="#xzs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>xz: XZ utilities
</code></pre></div>
<h3 id="xz">XZ:<a class="headerlink" href="#xz" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>xz: XZ utilities
</code></pre></div>
<h3 id="xerces-cs">Xerces-C++'s:<a class="headerlink" href="#xerces-cs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Xerces-C++ is a validating XML parser written in a portable
</code></pre></div>
<h3 id="xerces-c">Xerces-C++:<a class="headerlink" href="#xerces-c" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Xerces-C++ is a validating XML parser written in a portable
</code></pre></div>
<h3 id="yaxts">YAXT's:<a class="headerlink" href="#yaxts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet Another eXchange Tool - Library that performs halo exchange with MPI for domain decomposed simulations.
</code></pre></div>
<h3 id="yaxt">YAXT:<a class="headerlink" href="#yaxt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet Another eXchange Tool - Library that performs halo exchange with MPI for domain decomposed simulations.
</code></pre></div>
<h3 id="yades">Yade's:<a class="headerlink" href="#yades" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yade is an extensible open-source framework for discrete numerical models,
</code></pre></div>
<h3 id="yade">Yade:<a class="headerlink" href="#yade" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yade is an extensible open-source framework for discrete numerical models,
</code></pre></div>
<h3 id="yasms">Yasm's:<a class="headerlink" href="#yasms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yasm: Complete rewrite of the NASM assembler with BSD license
</code></pre></div>
<h3 id="yasm">Yasm:<a class="headerlink" href="#yasm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yasm: Complete rewrite of the NASM assembler with BSD license
</code></pre></div>
<h3 id="zeromqs">ZeroMQ's:<a class="headerlink" href="#zeromqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
</code></pre></div>
<h3 id="zeromq">ZeroMQ:<a class="headerlink" href="#zeromq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
</code></pre></div>
<h3 id="zips">Zip's:<a class="headerlink" href="#zips" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Zip is a compression and file packaging/archive utility.
</code></pre></div>
<h3 id="zip">Zip:<a class="headerlink" href="#zip" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Zip is a compression and file packaging/archive utility.
</code></pre></div>
<h3 id="zonations">Zonation's:<a class="headerlink" href="#zonations" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spatial conservation prioritisation framework for large-scale conservation planning.
</code></pre></div>
<h3 id="zonation">Zonation:<a class="headerlink" href="#zonation" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spatial conservation prioritisation framework for large-scale conservation planning.
</code></pre></div>
<h3 id="angsds">angsd's:<a class="headerlink" href="#angsds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for analysing NGS data.
</code></pre></div>
<h3 id="angsd">angsd:<a class="headerlink" href="#angsd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Program for analysing NGS data.
</code></pre></div>
<h3 id="ants_2">ant's:<a class="headerlink" href="#ants_2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
</code></pre></div>
<h3 id="ant">ant:<a class="headerlink" href="#ant" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
</code></pre></div>
<h3 id="antismashs">antiSMASH's:<a class="headerlink" href="#antismashs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary
</code></pre></div>
<h3 id="antismash">antiSMASH:<a class="headerlink" href="#antismash" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary
</code></pre></div>
<h3 id="any2fastas">any2fasta's:<a class="headerlink" href="#any2fastas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Convert various sequence formats to FASTA
</code></pre></div>
<h3 id="any2fasta">any2fasta:<a class="headerlink" href="#any2fasta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Convert various sequence formats to FASTA
</code></pre></div>
<h3 id="argtables">argtable's:<a class="headerlink" href="#argtables" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Argtable is an ANSI C library for parsing GNU style
</code></pre></div>
<h3 id="argtable">argtable:<a class="headerlink" href="#argtable" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Argtable is an ANSI C library for parsing GNU style
</code></pre></div>
<h3 id="aria2s">aria2's:<a class="headerlink" href="#aria2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>aria2 is a lightweight multi-protocol &amp; multi-source command-line download utility.
</code></pre></div>
<h3 id="aria2">aria2:<a class="headerlink" href="#aria2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>aria2 is a lightweight multi-protocol &amp; multi-source command-line download utility.
</code></pre></div>
<h3 id="arpack-ngs">arpack-ng's:<a class="headerlink" href="#arpack-ngs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
</code></pre></div>
<h3 id="arpack-ng">arpack-ng:<a class="headerlink" href="#arpack-ng" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.
</code></pre></div>
<h3 id="at-spi2-atks">at-spi2-atk's:<a class="headerlink" href="#at-spi2-atks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AT-SPI 2 toolkit bridge
</code></pre></div>
<h3 id="at-spi2-atk">at-spi2-atk:<a class="headerlink" href="#at-spi2-atk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>AT-SPI 2 toolkit bridge
</code></pre></div>
<h3 id="attrs">attr's:<a class="headerlink" href="#attrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Commands for Manipulating Filesystem Extended Attributes
</code></pre></div>
<h3 id="attr">attr:<a class="headerlink" href="#attr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Commands for Manipulating Filesystem Extended Attributes
</code></pre></div>
<h3 id="azul-zulus">azul-zulu's:<a class="headerlink" href="#azul-zulus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java Development Kit (JDK), and a compliant implementation of the Java Standard Edition (SE) specification.
</code></pre></div>
<h3 id="azul-zulu">azul-zulu:<a class="headerlink" href="#azul-zulu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java Development Kit (JDK), and a compliant implementation of the Java Standard Edition (SE) specification.
</code></pre></div>
<h3 id="bamutils">bamUtil's:<a class="headerlink" href="#bamutils" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Repository that contains several programs
</code></pre></div>
<h3 id="bamutil">bamUtil:<a class="headerlink" href="#bamutil" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Repository that contains several programs
</code></pre></div>
<h3 id="barrnaps">barrnap's:<a class="headerlink" href="#barrnaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Barrnap predicts the location of ribosomal RNA genes in genomes.
</code></pre></div>
<h3 id="barrnap">barrnap:<a class="headerlink" href="#barrnap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Barrnap predicts the location of ribosomal RNA genes in genomes.
</code></pre></div>
<h3 id="bcl2fastq2s">bcl2fastq2's:<a class="headerlink" href="#bcl2fastq2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by
</code></pre></div>
<h3 id="bcl2fastq2">bcl2fastq2:<a class="headerlink" href="#bcl2fastq2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by
</code></pre></div>
<h3 id="binutilss">binutils's:<a class="headerlink" href="#binutilss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>binutils: GNU binary utilities
</code></pre></div>
<h3 id="binutils">binutils:<a class="headerlink" href="#binutils" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>binutils: GNU binary utilities
</code></pre></div>
<h3 id="bioawks">bioawk's:<a class="headerlink" href="#bioawks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extension to awk, adding the support of several common biological data formats
</code></pre></div>
<h3 id="bioawk">bioawk:<a class="headerlink" href="#bioawk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extension to awk, adding the support of several common biological data formats
</code></pre></div>
<h3 id="blasr_libcpps">blasr_libcpp's:<a class="headerlink" href="#blasr_libcpps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for
</code></pre></div>
<h3 id="blasr_libcpp">blasr_libcpp:<a class="headerlink" href="#blasr_libcpp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for
</code></pre></div>
<h3 id="breseqs">breseq's:<a class="headerlink" href="#breseqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>breseq is a computational pipeline for the analysis of short-read re-sequencing data
</code></pre></div>
<h3 id="breseq">breseq:<a class="headerlink" href="#breseq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>breseq is a computational pipeline for the analysis of short-read re-sequencing data
</code></pre></div>
<h3 id="bsddb3s">bsddb3's:<a class="headerlink" href="#bsddb3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bsddb3 is a nearly complete Python binding of the
</code></pre></div>
<h3 id="bsddb3">bsddb3:<a class="headerlink" href="#bsddb3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bsddb3 is a nearly complete Python binding of the
</code></pre></div>
<h3 id="bzip2s">bzip2's:<a class="headerlink" href="#bzip2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bzip2 is a freely available, patent free, high-quality data compressor. It typically
</code></pre></div>
<h3 id="bzip2">bzip2:<a class="headerlink" href="#bzip2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>bzip2 is a freely available, patent free, high-quality data compressor. It typically
</code></pre></div>
<h3 id="c-aress">c-ares's:<a class="headerlink" href="#c-aress" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>c-ares is a C library for asynchronous DNS requests (including name resolves)
</code></pre></div>
<h3 id="c-ares">c-ares:<a class="headerlink" href="#c-ares" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>c-ares is a C library for asynchronous DNS requests (including name resolves)
</code></pre></div>
<h3 id="cairos">cairo's:<a class="headerlink" href="#cairos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cairo is a 2D graphics library with support for multiple output devices.
</code></pre></div>
<h3 id="cairo">cairo:<a class="headerlink" href="#cairo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Cairo is a 2D graphics library with support for multiple output devices.
</code></pre></div>
<h3 id="cdbfastas">cdbfasta's:<a class="headerlink" href="#cdbfastas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fasta file indexing and retrival tool
</code></pre></div>
<h3 id="cdbfasta">cdbfasta:<a class="headerlink" href="#cdbfasta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fasta file indexing and retrival tool
</code></pre></div>
<h3 id="chewbbacas">chewBBACA's:<a class="headerlink" href="#chewbbacas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A complete suite for gene-by-gene schema creation and strain identification..
</code></pre></div>
<h3 id="chewbbaca">chewBBACA:<a class="headerlink" href="#chewbbaca" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A complete suite for gene-by-gene schema creation and strain identification..
</code></pre></div>
<h3 id="choppers">chopper's:<a class="headerlink" href="#choppers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rust implementation of NanoFilt+NanoLyse
</code></pre></div>
<h3 id="chopper">chopper:<a class="headerlink" href="#chopper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rust implementation of NanoFilt+NanoLyse
</code></pre></div>
<h3 id="cromwells">cromwell's:<a class="headerlink" href="#cromwells" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow Management System geared towards scientific workflows.
</code></pre></div>
<h3 id="cromwell">cromwell:<a class="headerlink" href="#cromwell" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Workflow Management System geared towards scientific workflows.
</code></pre></div>
<h3 id="ctagss">ctags's:<a class="headerlink" href="#ctagss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ctags generates an index (or tag) file of language objects found in source files that allows these
</code></pre></div>
<h3 id="ctags">ctags:<a class="headerlink" href="#ctags" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Ctags generates an index (or tag) file of language objects found in source files that allows these
</code></pre></div>
<h3 id="ctffinds">ctffind's:<a class="headerlink" href="#ctffinds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ctffind is a program for finding CTFs of electron micrographs
</code></pre></div>
<h3 id="ctffind">ctffind:<a class="headerlink" href="#ctffind" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ctffind is a program for finding CTFs of electron micrographs
</code></pre></div>
<h3 id="cudnns">cuDNN's:<a class="headerlink" href="#cudnns" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for     deep neural networks.
</code></pre></div>
<h3 id="cudnn">cuDNN:<a class="headerlink" href="#cudnn" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for     deep neural networks.
</code></pre></div>
<h3 id="cutadapts">cutadapt's:<a class="headerlink" href="#cutadapts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>cutadapt removes adapter sequences
</code></pre></div>
<h3 id="cutadapt">cutadapt:<a class="headerlink" href="#cutadapt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>cutadapt removes adapter sequences
</code></pre></div>
<h3 id="cutesvs">cuteSV's:<a class="headerlink" href="#cutesvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast and scalable long-read-based SV detection
</code></pre></div>
<h3 id="cutesv">cuteSV:<a class="headerlink" href="#cutesv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast and scalable long-read-based SV detection
</code></pre></div>
<h3 id="cwltools">cwltool's:<a class="headerlink" href="#cwltools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Common Workflow Language tool description reference implementation
</code></pre></div>
<h3 id="cwltool">cwltool:<a class="headerlink" href="#cwltool" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Common Workflow Language tool description reference implementation
</code></pre></div>
<h3 id="cyvcf2s">cyvcf2's:<a class="headerlink" href="#cyvcf2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>cython + htslib == fast VCF and BCF processing
</code></pre></div>
<h3 id="cyvcf2">cyvcf2:<a class="headerlink" href="#cyvcf2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>cython + htslib == fast VCF and BCF processing
</code></pre></div>
<h3 id="dammits">dammit's:<a class="headerlink" href="#dammits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>de novo transcriptome annotator..
</code></pre></div>
<h3 id="dammit">dammit:<a class="headerlink" href="#dammit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>de novo transcriptome annotator..
</code></pre></div>
<h3 id="deeptoolss">deepTools's:<a class="headerlink" href="#deeptoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>deepTools is a suite of python tools particularly developed for the efficient analysis of
</code></pre></div>
<h3 id="deeptools">deepTools:<a class="headerlink" href="#deeptools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>deepTools is a suite of python tools particularly developed for the efficient analysis of
</code></pre></div>
<h3 id="devtoolss">devtools's:<a class="headerlink" href="#devtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R functions that simplify and expedite common tasks in package development.
</code></pre></div>
<h3 id="devtools">devtools:<a class="headerlink" href="#devtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>R functions that simplify and expedite common tasks in package development.
</code></pre></div>
<h3 id="double-conversions">double-conversion's:<a class="headerlink" href="#double-conversions" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.
</code></pre></div>
<h3 id="double-conversion">double-conversion:<a class="headerlink" href="#double-conversion" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.
</code></pre></div>
<h3 id="dreps">drep's:<a class="headerlink" href="#dreps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid and accurate comparison and de-replication of microbial genomes
</code></pre></div>
<h3 id="drep">drep:<a class="headerlink" href="#drep" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Rapid and accurate comparison and de-replication of microbial genomes
</code></pre></div>
<h3 id="dtcmps">dtcmp's:<a class="headerlink" href="#dtcmps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DTCMP Library provides pre-defined and user-defined
</code></pre></div>
<h3 id="dtcmp">dtcmp:<a class="headerlink" href="#dtcmp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>DTCMP Library provides pre-defined and user-defined
</code></pre></div>
<h3 id="dupholds">duphold's:<a class="headerlink" href="#dupholds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>uphold your DUP and DEL calls
</code></pre></div>
<h3 id="duphold">duphold:<a class="headerlink" href="#duphold" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>uphold your DUP and DEL calls
</code></pre></div>
<h3 id="duplex-toolss">duplex-tools's:<a class="headerlink" href="#duplex-toolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Range of tools to support operations on Duplex Sequencing read pairs.
</code></pre></div>
<h3 id="duplex-tools">duplex-tools:<a class="headerlink" href="#duplex-tools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Range of tools to support operations on Duplex Sequencing read pairs.
</code></pre></div>
<h3 id="ednas">eDNA's:<a class="headerlink" href="#ednas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A suite of tools to conduct metabarcoding analyses targeting any group of organisms. Includes utilities
</code></pre></div>
<h3 id="edna">eDNA:<a class="headerlink" href="#edna" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A suite of tools to conduct metabarcoding analyses targeting any group of organisms. Includes utilities
</code></pre></div>
<h3 id="eccodess">ecCodes's:<a class="headerlink" href="#eccodess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ecCodes is a package developed by ECMWF which provides an application programming interface and
</code></pre></div>
<h3 id="eccodes">ecCodes:<a class="headerlink" href="#eccodes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ecCodes is a package developed by ECMWF which provides an application programming interface and
</code></pre></div>
<h3 id="eccodess_1">eccodes's:<a class="headerlink" href="#eccodess_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats:  - WMO FM-92 GRIB edition 1 and edition 2 - WMO FM-94 BUFR edition 3 and edition 4  - WMO GTS abbreviated header (only decoding)  A useful set of command line tools provide quick access to the messages.
</code></pre></div>
<h3 id="eccodes_1">eccodes:<a class="headerlink" href="#eccodes_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats:  - WMO FM-92 GRIB edition 1 and edition 2 - WMO FM-94 BUFR edition 3 and edition 4  - WMO GTS abbreviated header (only decoding)  A useful set of command line tools provide quick access to the messages.
</code></pre></div>
<h3 id="ectypers">ectyper's:<a class="headerlink" href="#ectypers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Standalone versatile serotyping module for Escherichia coli..
</code></pre></div>
<h3 id="ectyper">ectyper:<a class="headerlink" href="#ectyper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Standalone versatile serotyping module for Escherichia coli..
</code></pre></div>
<h3 id="edlibs">edlib's:<a class="headerlink" href="#edlibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
</code></pre></div>
<h3 id="edlib">edlib:<a class="headerlink" href="#edlib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Lightweight, super fast library for sequence alignment using edit (Levenshtein) distance.
</code></pre></div>
<h3 id="eggnog-mappers">eggnog-mapper's:<a class="headerlink" href="#eggnog-mappers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for fast functional annotation of novel sequences (genes or proteins)
</code></pre></div>
<h3 id="eggnog-mapper">eggnog-mapper:<a class="headerlink" href="#eggnog-mapper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for fast functional annotation of novel sequences (genes or proteins)
</code></pre></div>
<h3 id="ensmallens">ensmallen's:<a class="headerlink" href="#ensmallens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ header-only library for numerical optimization
</code></pre></div>
<h3 id="ensmallen">ensmallen:<a class="headerlink" href="#ensmallen" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ header-only library for numerical optimization
</code></pre></div>
<h3 id="entrez-directs">entrez-direct's:<a class="headerlink" href="#entrez-directs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>an advanced method for accessing the NCBI&#39;s set of interconnected databases
</code></pre></div>
<h3 id="entrez-direct">entrez-direct:<a class="headerlink" href="#entrez-direct" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>an advanced method for accessing the NCBI&#39;s set of interconnected databases
</code></pre></div>
<h3 id="exonerates">exonerate's:<a class="headerlink" href="#exonerates" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic tool for pairwise sequence comparison
</code></pre></div>
<h3 id="exonerate">exonerate:<a class="headerlink" href="#exonerate" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic tool for pairwise sequence comparison
</code></pre></div>
<h3 id="expats">expat's:<a class="headerlink" href="#expats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Expat is an XML parser library written in C. It is a stream-oriented parser  in which an application registers handlers for things the parser might find  in the XML document (like start tags)
</code></pre></div>
<h3 id="expat">expat:<a class="headerlink" href="#expat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Expat is an XML parser library written in C. It is a stream-oriented parser  in which an application registers handlers for things the parser might find  in the XML document (like start tags)
</code></pre></div>
<h3 id="faststructures">fastStructure's:<a class="headerlink" href="#faststructures" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x.
</code></pre></div>
<h3 id="faststructure">fastStructure:<a class="headerlink" href="#faststructure" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x.
</code></pre></div>
<h3 id="fastps">fastp's:<a class="headerlink" href="#fastps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool designed to provide fast all-in-one preprocessing for FastQ files.
</code></pre></div>
<h3 id="fastp">fastp:<a class="headerlink" href="#fastp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A tool designed to provide fast all-in-one preprocessing for FastQ files.
</code></pre></div>
<h3 id="fastq-toolss">fastq-tools's:<a class="headerlink" href="#fastq-toolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A collection of small and efficient programs for performing some common and
</code></pre></div>
<h3 id="fastq-tools">fastq-tools:<a class="headerlink" href="#fastq-tools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A collection of small and efficient programs for performing some common and
</code></pre></div>
<h3 id="fcgenes">fcGENE's:<a class="headerlink" href="#fcgenes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Format converting tool for genotype Data.
</code></pre></div>
<h3 id="fcgene">fcGENE:<a class="headerlink" href="#fcgene" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Format converting tool for genotype Data.
</code></pre></div>
<h3 id="fgbios">fgbio's:<a class="headerlink" href="#fgbios" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools to analyze genomic data with a focus on Next Generation Sequencing.
</code></pre></div>
<h3 id="fgbio">fgbio:<a class="headerlink" href="#fgbio" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools to analyze genomic data with a focus on Next Generation Sequencing.
</code></pre></div>
<h3 id="fineradstructures">fineRADstructure's:<a class="headerlink" href="#fineradstructures" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A package for population structure inference from RAD-seq data
</code></pre></div>
<h3 id="fineradstructure">fineRADstructure:<a class="headerlink" href="#fineradstructure" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A package for population structure inference from RAD-seq data
</code></pre></div>
<h3 id="flatbufferss">flatbuffers's:<a class="headerlink" href="#flatbufferss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FlatBuffers: Memory Efficient Serialization Library
</code></pre></div>
<h3 id="flatbuffers">flatbuffers:<a class="headerlink" href="#flatbuffers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FlatBuffers: Memory Efficient Serialization Library
</code></pre></div>
<h3 id="flexs">flex's:<a class="headerlink" href="#flexs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text.
</code></pre></div>
<h3 id="flex">flex:<a class="headerlink" href="#flex" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text.
</code></pre></div>
<h3 id="fmlrcs">fmlrc's:<a class="headerlink" href="#fmlrcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for performing  hybrid correction of long read sequencing
</code></pre></div>
<h3 id="fmlrc">fmlrc:<a class="headerlink" href="#fmlrc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for performing  hybrid correction of long read sequencing
</code></pre></div>
<h3 id="fmts">fmt's:<a class="headerlink" href="#fmts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Formatting library providing a fast and safe alternative to C stdio and C++ iostreams.
</code></pre></div>
<h3 id="fmt">fmt:<a class="headerlink" href="#fmt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Formatting library providing a fast and safe alternative to C stdio and C++ iostreams.
</code></pre></div>
<h3 id="forges">forge's:<a class="headerlink" href="#forges" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Arm Forge combines Arm DDT, the leading debugger for time-saving high performance application debugging, and Arm MAP, the trusted performance profiler for invaluable optimization advice.
</code></pre></div>
<h3 id="forge">forge:<a class="headerlink" href="#forge" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Arm Forge combines Arm DDT, the leading debugger for time-saving high performance application debugging, and Arm MAP, the trusted performance profiler for invaluable optimization advice.
</code></pre></div>
<h3 id="fosss">foss's:<a class="headerlink" href="#fosss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain, including
</code></pre></div>
<h3 id="foss">foss:<a class="headerlink" href="#foss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain, including
</code></pre></div>
<h3 id="funcx-endpoints">funcx-endpoint's:<a class="headerlink" href="#funcx-endpoints" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>funcX is a distributed Function as a Service (FaaS) platform that enables flexible,
</code></pre></div>
<h3 id="funcx-endpoint">funcx-endpoint:<a class="headerlink" href="#funcx-endpoint" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>funcX is a distributed Function as a Service (FaaS) platform that enables flexible,
</code></pre></div>
<h3 id="fxtracts">fxtract's:<a class="headerlink" href="#fxtracts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extract sequences from a fastx (fasta or fastq) file given a subsequence.
</code></pre></div>
<h3 id="fxtract">fxtract:<a class="headerlink" href="#fxtract" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Extract sequences from a fastx (fasta or fastq) file given a subsequence.
</code></pre></div>
<h3 id="g2clibs">g2clib's:<a class="headerlink" href="#g2clibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library contains GRIB2 encoder/decoder (&#39;C&#39; version).
</code></pre></div>
<h3 id="g2clib">g2clib:<a class="headerlink" href="#g2clib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library contains GRIB2 encoder/decoder (&#39;C&#39; version).
</code></pre></div>
<h3 id="g2libs">g2lib's:<a class="headerlink" href="#g2libs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library contains GRIB2 encoder/decoder and search/indexing routines.
</code></pre></div>
<h3 id="g2lib">g2lib:<a class="headerlink" href="#g2lib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library contains GRIB2 encoder/decoder and search/indexing routines.
</code></pre></div>
<h3 id="ga4ghs">ga4gh's:<a class="headerlink" href="#ga4ghs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A reference implementation of the GA4GH API
</code></pre></div>
<h3 id="ga4gh">ga4gh:<a class="headerlink" href="#ga4gh" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A reference implementation of the GA4GH API
</code></pre></div>
<h3 id="geanys">geany's:<a class="headerlink" href="#geanys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A GTK+ based text editor with with basic features of an integrated development environment.
</code></pre></div>
<h3 id="geany">geany:<a class="headerlink" href="#geany" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A GTK+ based text editor with with basic features of an integrated development environment.
</code></pre></div>
<h3 id="genometoolss">genometools's:<a class="headerlink" href="#genometoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GenomeTools: A Comprehensive Software Library for Efficient Processing of
</code></pre></div>
<h3 id="genometools">genometools:<a class="headerlink" href="#genometools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GenomeTools: A Comprehensive Software Library for Efficient Processing of
</code></pre></div>
<h3 id="gettexts">gettext's:<a class="headerlink" href="#gettexts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU &#39;gettext&#39; is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation
</code></pre></div>
<h3 id="gettext">gettext:<a class="headerlink" href="#gettext" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU &#39;gettext&#39; is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation
</code></pre></div>
<h3 id="gffreads">gffread's:<a class="headerlink" href="#gffreads" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GFF/GTF parsing utility providing format conversions,
</code></pre></div>
<h3 id="gffread">gffread:<a class="headerlink" href="#gffread" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GFF/GTF parsing utility providing format conversions,
</code></pre></div>
<h3 id="giflibs">giflib's:<a class="headerlink" href="#giflibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>giflib is a library for reading and writing gif images.
</code></pre></div>
<h3 id="giflib">giflib:<a class="headerlink" href="#giflib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>giflib is a library for reading and writing gif images.
</code></pre></div>
<h3 id="gimkls">gimkl's:<a class="headerlink" href="#gimkls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL
</code></pre></div>
<h3 id="gimkl">gimkl:<a class="headerlink" href="#gimkl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL
</code></pre></div>
<h3 id="gimpis">gimpi's:<a class="headerlink" href="#gimpis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.
</code></pre></div>
<h3 id="gimpi">gimpi:<a class="headerlink" href="#gimpi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.
</code></pre></div>
<h3 id="gits">git's:<a class="headerlink" href="#gits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Git is a free and open source distributed version control system designed
</code></pre></div>
<h3 id="git">git:<a class="headerlink" href="#git" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Git is a free and open source distributed version control system designed
</code></pre></div>
<h3 id="globus-automate-clients">globus-automate-client's:<a class="headerlink" href="#globus-automate-clients" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Client for the Globus Flows service.
</code></pre></div>
<h3 id="globus-automate-client">globus-automate-client:<a class="headerlink" href="#globus-automate-client" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Client for the Globus Flows service.
</code></pre></div>
<h3 id="globus-compute-endpoints">globus-compute-endpoint's:<a class="headerlink" href="#globus-compute-endpoints" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Globus Compute is a distributed Function as a Service (FaaS) platform that enables flexible,
</code></pre></div>
<h3 id="globus-compute-endpoint">globus-compute-endpoint:<a class="headerlink" href="#globus-compute-endpoint" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Globus Compute is a distributed Function as a Service (FaaS) platform that enables flexible,
</code></pre></div>
<h3 id="gmshs">gmsh's:<a class="headerlink" href="#gmshs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor..
</code></pre></div>
<h3 id="gmsh">gmsh:<a class="headerlink" href="#gmsh" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor..
</code></pre></div>
<h3 id="gnuplots">gnuplot's:<a class="headerlink" href="#gnuplots" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Portable interactive, function plotting utility
</code></pre></div>
<h3 id="gnuplot">gnuplot:<a class="headerlink" href="#gnuplot" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Portable interactive, function plotting utility
</code></pre></div>
<h3 id="gompis">gompi's:<a class="headerlink" href="#gompis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain,
</code></pre></div>
<h3 id="gompi">gompi:<a class="headerlink" href="#gompi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU Compiler Collection (GCC) based compiler toolchain,
</code></pre></div>
<h3 id="google-sparsehashs">google-sparsehash's:<a class="headerlink" href="#google-sparsehashs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library
</code></pre></div>
<h3 id="google-sparsehash">google-sparsehash:<a class="headerlink" href="#google-sparsehash" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library
</code></pre></div>
<h3 id="googletests">googletest's:<a class="headerlink" href="#googletests" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Google&#39;s C++ test framework
</code></pre></div>
<h3 id="googletest">googletest:<a class="headerlink" href="#googletest" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Google&#39;s C++ test framework
</code></pre></div>
<h3 id="gperfs">gperf's:<a class="headerlink" href="#gperfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pperfect hash function generator.
</code></pre></div>
<h3 id="gperf">gperf:<a class="headerlink" href="#gperf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pperfect hash function generator.
</code></pre></div>
<h3 id="grib_apis">grib_api's:<a class="headerlink" href="#grib_apis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of command line tools is also provided to give quick access to GRIB messages.  Note that JPEG and Python support have been disabled for this build.
</code></pre></div>
<h3 id="grib_api">grib_api:<a class="headerlink" href="#grib_api" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of command line tools is also provided to give quick access to GRIB messages.  Note that JPEG and Python support have been disabled for this build.
</code></pre></div>
<h3 id="grive2s">grive2's:<a class="headerlink" href="#grive2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command line tool for Google Drive.
</code></pre></div>
<h3 id="grive2">grive2:<a class="headerlink" href="#grive2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command line tool for Google Drive.
</code></pre></div>
<h3 id="gsorts">gsort's:<a class="headerlink" href="#gsorts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool to sort genomic files according to a genomefile.
</code></pre></div>
<h3 id="gsort">gsort:<a class="headerlink" href="#gsort" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool to sort genomic files according to a genomefile.
</code></pre></div>
<h3 id="h5pps">h5pp's:<a class="headerlink" href="#h5pps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A simple C++17 wrapper for HDF5.
</code></pre></div>
<h3 id="h5pp">h5pp:<a class="headerlink" href="#h5pp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A simple C++17 wrapper for HDF5.
</code></pre></div>
<h3 id="help2mans">help2man's:<a class="headerlink" href="#help2mans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>help2man produces simple manual pages from the &#39;--help&#39; and &#39;--version&#39; output of other commands.
</code></pre></div>
<h3 id="help2man">help2man:<a class="headerlink" href="#help2man" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>help2man produces simple manual pages from the &#39;--help&#39; and &#39;--version&#39; output of other commands.
</code></pre></div>
<h3 id="hifiasms">hifiasm's:<a class="headerlink" href="#hifiasms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.
</code></pre></div>
<h3 id="hifiasm">hifiasm:<a class="headerlink" href="#hifiasm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.
</code></pre></div>
<h3 id="hunspells">hunspell's:<a class="headerlink" href="#hunspells" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spell checker and morphological analyzer library and program designed for languages
</code></pre></div>
<h3 id="hunspell">hunspell:<a class="headerlink" href="#hunspell" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spell checker and morphological analyzer library and program designed for languages
</code></pre></div>
<h3 id="hypothesiss">hypothesis's:<a class="headerlink" href="#hypothesiss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized
</code></pre></div>
<h3 id="hypothesis">hypothesis:<a class="headerlink" href="#hypothesis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized
</code></pre></div>
<h3 id="iccs">icc's:<a class="headerlink" href="#iccs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C and C++ compilers
</code></pre></div>
<h3 id="icc">icc:<a class="headerlink" href="#icc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C and C++ compilers
</code></pre></div>
<h3 id="icciforts">iccifort's:<a class="headerlink" href="#icciforts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C, C++ &amp; Fortran compilers
</code></pre></div>
<h3 id="iccifort">iccifort:<a class="headerlink" href="#iccifort" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C, C++ &amp; Fortran compilers
</code></pre></div>
<h3 id="iforts">ifort's:<a class="headerlink" href="#iforts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Fortran compiler
</code></pre></div>
<h3 id="ifort">ifort:<a class="headerlink" href="#ifort" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Fortran compiler
</code></pre></div>
<h3 id="iimpis">iimpi's:<a class="headerlink" href="#iimpis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C/C++ and Fortran compilers, alongside Intel MPI.
</code></pre></div>
<h3 id="iimpi">iimpi:<a class="headerlink" href="#iimpi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C/C++ and Fortran compilers, alongside Intel MPI.
</code></pre></div>
<h3 id="imkls">imkl's:<a class="headerlink" href="#imkls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Math Kernel Library is a library of highly optimized,
</code></pre></div>
<h3 id="imkl">imkl:<a class="headerlink" href="#imkl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel Math Kernel Library is a library of highly optimized,
</code></pre></div>
<h3 id="imkl-fftws">imkl-FFTW's:<a class="headerlink" href="#imkl-fftws" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FFTW interfaces using Intel oneAPI Math Kernel Library
</code></pre></div>
<h3 id="imkl-fftw">imkl-FFTW:<a class="headerlink" href="#imkl-fftw" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FFTW interfaces using Intel oneAPI Math Kernel Library
</code></pre></div>
<h3 id="impis">impi's:<a class="headerlink" href="#impis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel MPI Library, compatible with MPICH ABI
</code></pre></div>
<h3 id="impi">impi:<a class="headerlink" href="#impi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel MPI Library, compatible with MPICH ABI
</code></pre></div>
<h3 id="intels">intel's:<a class="headerlink" href="#intels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).
</code></pre></div>
<h3 id="intel">intel:<a class="headerlink" href="#intel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).
</code></pre></div>
<h3 id="intel-compilerss">intel-compilers's:<a class="headerlink" href="#intel-compilerss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C, C++ &amp; Fortran compilers (classic and oneAPI)
</code></pre></div>
<h3 id="intel-compilers">intel-compilers:<a class="headerlink" href="#intel-compilers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel C, C++ &amp; Fortran compilers (classic and oneAPI)
</code></pre></div>
<h3 id="ipyrads">ipyrad's:<a class="headerlink" href="#ipyrads" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic
</code></pre></div>
<h3 id="ipyrad">ipyrad:<a class="headerlink" href="#ipyrad" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic
</code></pre></div>
<h3 id="ispcs">ispc's:<a class="headerlink" href="#ispcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for &#39;single program, multiple data&#39; (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.
</code></pre></div>
<h3 id="ispc">ispc:<a class="headerlink" href="#ispc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for &#39;single program, multiple data&#39; (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.
</code></pre></div>
<h3 id="jbigkits">jbigkit's:<a class="headerlink" href="#jbigkits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JBIG-KIT is a software implementation of the JBIG1 data compression standard
</code></pre></div>
<h3 id="jbigkit">jbigkit:<a class="headerlink" href="#jbigkit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JBIG-KIT is a software implementation of the JBIG1 data compression standard
</code></pre></div>
<h3 id="jcvis">jcvi's:<a class="headerlink" href="#jcvis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of Python libraries to parse bioinformatics files, or perform computation related to assembly, annotation, and comparative genomics.
</code></pre></div>
<h3 id="jcvi">jcvi:<a class="headerlink" href="#jcvi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Collection of Python libraries to parse bioinformatics files, or perform computation related to assembly, annotation, and comparative genomics.
</code></pre></div>
<h3 id="jemallocs">jemalloc's:<a class="headerlink" href="#jemallocs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
</code></pre></div>
<h3 id="jemalloc">jemalloc:<a class="headerlink" href="#jemalloc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A general purpose malloc(3) implementation that emphasizes fragmentation avoidance and
</code></pre></div>
<h3 id="jqs">jq's:<a class="headerlink" href="#jqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Lightweight and flexible command-line JSON processor.
</code></pre></div>
<h3 id="jq">jq:<a class="headerlink" href="#jq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Lightweight and flexible command-line JSON processor.
</code></pre></div>
<h3 id="json-cs">json-c's:<a class="headerlink" href="#json-cs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JSON-C implements a reference counting object model that allows you to easily construct JSON objects
</code></pre></div>
<h3 id="json-c">json-c:<a class="headerlink" href="#json-c" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>JSON-C implements a reference counting object model that allows you to easily construct JSON objects
</code></pre></div>
<h3 id="jvarkits">jvarkit's:<a class="headerlink" href="#jvarkits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java utilities for Bioinformatics
</code></pre></div>
<h3 id="jvarkit">jvarkit:<a class="headerlink" href="#jvarkit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Java utilities for Bioinformatics
</code></pre></div>
<h3 id="kalign2s">kalign2's:<a class="headerlink" href="#kalign2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Kalign is a fast multiple sequence alignment program for biological sequences.
</code></pre></div>
<h3 id="kalign2">kalign2:<a class="headerlink" href="#kalign2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Kalign is a fast multiple sequence alignment program for biological sequences.
</code></pre></div>
<h3 id="kallistos">kallisto's:<a class="headerlink" href="#kallistos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
</code></pre></div>
<h3 id="kallisto">kallisto:<a class="headerlink" href="#kallisto" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
</code></pre></div>
<h3 id="kinetos">kineto's:<a class="headerlink" href="#kinetos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters
</code></pre></div>
<h3 id="kineto">kineto:<a class="headerlink" href="#kineto" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters
</code></pre></div>
<h3 id="kmas">kma's:<a class="headerlink" href="#kmas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>KMA is a mapping method designed to map raw reads directly against redundant databases,
</code></pre></div>
<h3 id="kma">kma:<a class="headerlink" href="#kma" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>KMA is a mapping method designed to map raw reads directly against redundant databases,
</code></pre></div>
<h3 id="libflames">libFLAME's:<a class="headerlink" href="#libflames" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libFLAME is a portable library for dense matrix computations,
</code></pre></div>
<h3 id="libflame">libFLAME:<a class="headerlink" href="#libflame" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libFLAME is a portable library for dense matrix computations,
</code></pre></div>
<h3 id="libglus">libGLU's:<a class="headerlink" href="#libglus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.
</code></pre></div>
<h3 id="libglu">libGLU:<a class="headerlink" href="#libglu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL.
</code></pre></div>
<h3 id="libkmls">libKML's:<a class="headerlink" href="#libkmls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reference implementation of OGC KML 2.2
</code></pre></div>
<h3 id="libkml">libKML:<a class="headerlink" href="#libkml" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reference implementation of OGC KML 2.2
</code></pre></div>
<h3 id="libcircles">libcircle's:<a class="headerlink" href="#libcircles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>API for distributing embarrassingly parallel workloads using self-stabilization.
</code></pre></div>
<h3 id="libcircle">libcircle:<a class="headerlink" href="#libcircle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>API for distributing embarrassingly parallel workloads using self-stabilization.
</code></pre></div>
<h3 id="libconfigs">libconfig's:<a class="headerlink" href="#libconfigs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Library for processing structured configuration files
</code></pre></div>
<h3 id="libconfig">libconfig:<a class="headerlink" href="#libconfig" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A Library for processing structured configuration files
</code></pre></div>
<h3 id="libdeflates">libdeflate's:<a class="headerlink" href="#libdeflates" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.
</code></pre></div>
<h3 id="libdeflate">libdeflate:<a class="headerlink" href="#libdeflate" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Heavily optimized library for DEFLATE/zlib/gzip compression and decompression.
</code></pre></div>
<h3 id="libdrms">libdrm's:<a class="headerlink" href="#libdrms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Direct Rendering Manager runtime library.
</code></pre></div>
<h3 id="libdrm">libdrm:<a class="headerlink" href="#libdrm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Direct Rendering Manager runtime library.
</code></pre></div>
<h3 id="libdwarfs">libdwarf's:<a class="headerlink" href="#libdwarfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The DWARF Debugging Information Format is of interest to programmers working on compilers
</code></pre></div>
<h3 id="libdwarf">libdwarf:<a class="headerlink" href="#libdwarf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The DWARF Debugging Information Format is of interest to programmers working on compilers
</code></pre></div>
<h3 id="libepoxys">libepoxy's:<a class="headerlink" href="#libepoxys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library for handling OpenGL function pointer management
</code></pre></div>
<h3 id="libepoxy">libepoxy:<a class="headerlink" href="#libepoxy" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library for handling OpenGL function pointer management
</code></pre></div>
<h3 id="libevents">libevent's:<a class="headerlink" href="#libevents" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.
</code></pre></div>
<h3 id="libevent">libevent:<a class="headerlink" href="#libevent" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.
</code></pre></div>
<h3 id="libffis">libffi's:<a class="headerlink" href="#libffis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The libffi library provides a portable, high level programming interface to various calling
</code></pre></div>
<h3 id="libffi">libffi:<a class="headerlink" href="#libffi" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The libffi library provides a portable, high level programming interface to various calling
</code></pre></div>
<h3 id="libgcrypts">libgcrypt's:<a class="headerlink" href="#libgcrypts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libgpg-error is a small library that defines common error values for all GnuPG components.
</code></pre></div>
<h3 id="libgcrypt">libgcrypt:<a class="headerlink" href="#libgcrypt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libgpg-error is a small library that defines common error values for all GnuPG components.
</code></pre></div>
<h3 id="libgds">libgd's:<a class="headerlink" href="#libgds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GD is an open source code library for the dynamic creation of images by programmers.
</code></pre></div>
<h3 id="libgd">libgd:<a class="headerlink" href="#libgd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GD is an open source code library for the dynamic creation of images by programmers.
</code></pre></div>
<h3 id="libgeotiffs">libgeotiff's:<a class="headerlink" href="#libgeotiffs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library for reading and writing coordinate system information from/to GeoTIFF files
</code></pre></div>
<h3 id="libgeotiff">libgeotiff:<a class="headerlink" href="#libgeotiff" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Library for reading and writing coordinate system information from/to GeoTIFF files
</code></pre></div>
<h3 id="libglvnds">libglvnd's:<a class="headerlink" href="#libglvnds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.
</code></pre></div>
<h3 id="libglvnd">libglvnd:<a class="headerlink" href="#libglvnd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.
</code></pre></div>
<h3 id="libgpg-errors">libgpg-error's:<a class="headerlink" href="#libgpg-errors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libgpg-error is a small library that defines common error values for all GnuPG components.
</code></pre></div>
<h3 id="libgpg-error">libgpg-error:<a class="headerlink" href="#libgpg-error" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libgpg-error is a small library that defines common error values for all GnuPG components.
</code></pre></div>
<h3 id="libgpuarrays">libgpuarray's:<a class="headerlink" href="#libgpuarrays" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Arrays on GPU device memory, for Theano
</code></pre></div>
<h3 id="libgpuarray">libgpuarray:<a class="headerlink" href="#libgpuarray" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Arrays on GPU device memory, for Theano
</code></pre></div>
<h3 id="libgtextutilss">libgtextutils's:<a class="headerlink" href="#libgtextutilss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream
</code></pre></div>
<h3 id="libgtextutils">libgtextutils:<a class="headerlink" href="#libgtextutils" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream
</code></pre></div>
<h3 id="libiconvs">libiconv's:<a class="headerlink" href="#libiconvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libiconv converts from one character encoding to another through Unicode conversion
</code></pre></div>
<h3 id="libiconv">libiconv:<a class="headerlink" href="#libiconv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libiconv converts from one character encoding to another through Unicode conversion
</code></pre></div>
<h3 id="libjpeg-turbos">libjpeg-turbo's:<a class="headerlink" href="#libjpeg-turbos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG
</code></pre></div>
<h3 id="libjpeg-turbo">libjpeg-turbo:<a class="headerlink" href="#libjpeg-turbo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG
</code></pre></div>
<h3 id="libmathevals">libmatheval's:<a class="headerlink" href="#libmathevals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU libmatheval is a library (callable from C and Fortran) to parse
</code></pre></div>
<h3 id="libmatheval">libmatheval:<a class="headerlink" href="#libmatheval" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU libmatheval is a library (callable from C and Fortran) to parse
</code></pre></div>
<h3 id="libpciaccesss">libpciaccess's:<a class="headerlink" href="#libpciaccesss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic PCI access library.
</code></pre></div>
<h3 id="libpciaccess">libpciaccess:<a class="headerlink" href="#libpciaccess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic PCI access library.
</code></pre></div>
<h3 id="libpngs">libpng's:<a class="headerlink" href="#libpngs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libpng is the official PNG reference library
</code></pre></div>
<h3 id="libpng">libpng:<a class="headerlink" href="#libpng" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>libpng is the official PNG reference library
</code></pre></div>
<h3 id="libreadlines">libreadline's:<a class="headerlink" href="#libreadlines" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Readline library provides a set of functions for use by applications that
</code></pre></div>
<h3 id="libreadline">libreadline:<a class="headerlink" href="#libreadline" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The GNU Readline library provides a set of functions for use by applications that
</code></pre></div>
<h3 id="libspatialites">libspatialite's:<a class="headerlink" href="#libspatialites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SpatiaLite is an open source library intended to extend the SQLite core to support
</code></pre></div>
<h3 id="libspatialite">libspatialite:<a class="headerlink" href="#libspatialite" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SpatiaLite is an open source library intended to extend the SQLite core to support
</code></pre></div>
<h3 id="libtools">libtool's:<a class="headerlink" href="#libtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries
</code></pre></div>
<h3 id="libtool">libtool:<a class="headerlink" href="#libtool" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries
</code></pre></div>
<h3 id="libunistrings">libunistring's:<a class="headerlink" href="#libunistrings" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>This library provides functions for manipulating Unicode strings and for manipulating C strings
</code></pre></div>
<h3 id="libunistring">libunistring:<a class="headerlink" href="#libunistring" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>This library provides functions for manipulating Unicode strings and for manipulating C strings
</code></pre></div>
<h3 id="libunwinds">libunwind's:<a class="headerlink" href="#libunwinds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Define a portable and efficient C programming API to determine the call-chain of a program.
</code></pre></div>
<h3 id="libunwind">libunwind:<a class="headerlink" href="#libunwind" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Define a portable and efficient C programming API to determine the call-chain of a program.
</code></pre></div>
<h3 id="libvpxs">libvpx's:<a class="headerlink" href="#libvpxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The WebM Project is dedicated to developing a high-quality, open video format for the web that&#39;s freely available to everyone.
</code></pre></div>
<h3 id="libvpx">libvpx:<a class="headerlink" href="#libvpx" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The WebM Project is dedicated to developing a high-quality, open video format for the web that&#39;s freely available to everyone.
</code></pre></div>
<h3 id="libxcs">libxc's:<a class="headerlink" href="#libxcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.
</code></pre></div>
<h3 id="libxc">libxc:<a class="headerlink" href="#libxc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.
</code></pre></div>
<h3 id="libxslts">libxslt's:<a class="headerlink" href="#libxslts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libxslt is the XSLT C library developed for the GNOME project
</code></pre></div>
<h3 id="libxslt">libxslt:<a class="headerlink" href="#libxslt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Libxslt is the XSLT C library developed for the GNOME project
</code></pre></div>
<h3 id="libxsmms">libxsmm's:<a class="headerlink" href="#libxsmms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications
</code></pre></div>
<h3 id="libxsmm">libxsmm:<a class="headerlink" href="#libxsmm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications
</code></pre></div>
<h3 id="libzstds">libzstd's:<a class="headerlink" href="#libzstds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast lossless compression algorithm.
</code></pre></div>
<h3 id="libzstd">libzstd:<a class="headerlink" href="#libzstd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast lossless compression algorithm.
</code></pre></div>
<h3 id="lighttpds">lighttpd's:<a class="headerlink" href="#lighttpds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A web server.
</code></pre></div>
<h3 id="lighttpd">lighttpd:<a class="headerlink" href="#lighttpd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A web server.
</code></pre></div>
<h3 id="likwids">likwid's:<a class="headerlink" href="#likwids" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command line tools for Linux to support programmers in developing high
</code></pre></div>
<h3 id="likwid">likwid:<a class="headerlink" href="#likwid" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command line tools for Linux to support programmers in developing high
</code></pre></div>
<h3 id="lp_solves">lp_solve's:<a class="headerlink" href="#lp_solves" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mixed Integer Linear Programming (MILP) solver
</code></pre></div>
<h3 id="lp_solve">lp_solve:<a class="headerlink" href="#lp_solve" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Mixed Integer Linear Programming (MILP) solver
</code></pre></div>
<h3 id="lwgrps">lwgrp's:<a class="headerlink" href="#lwgrps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The light-weight group library defines data structures and collective operations to
</code></pre></div>
<h3 id="lwgrp">lwgrp:<a class="headerlink" href="#lwgrp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The light-weight group library defines data structures and collective operations to
</code></pre></div>
<h3 id="lz4s">lz4's:<a class="headerlink" href="#lz4s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core.
</code></pre></div>
<h3 id="lz4">lz4:<a class="headerlink" href="#lz4" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.  It features an extremely fast decoder, with speed in multiple GB/s per core.
</code></pre></div>
<h3 id="magmas_2">magma's:<a class="headerlink" href="#magmas_2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MAGMA project aims to develop a dense linear algebra library similar to
</code></pre></div>
<h3 id="magma_2">magma:<a class="headerlink" href="#magma_2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The MAGMA project aims to develop a dense linear algebra library similar to
</code></pre></div>
<h3 id="makedepends">makedepend's:<a class="headerlink" href="#makedepends" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.
</code></pre></div>
<h3 id="makedepend">makedepend:<a class="headerlink" href="#makedepend" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.
</code></pre></div>
<h3 id="mantas">manta's:<a class="headerlink" href="#mantas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.
</code></pre></div>
<h3 id="manta">manta:<a class="headerlink" href="#manta" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.
</code></pre></div>
<h3 id="mapdamages">mapDamage's:<a class="headerlink" href="#mapdamages" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tracks and quantifies DNA damage patterns among ancient
</code></pre></div>
<h3 id="mapdamage">mapDamage:<a class="headerlink" href="#mapdamage" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tracks and quantifies DNA damage patterns among ancient
</code></pre></div>
<h3 id="merantks">meRanTK's:<a class="headerlink" href="#merantks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High performance toolkit for complete analysis of methylated RNA data.
</code></pre></div>
<h3 id="merantk">meRanTK:<a class="headerlink" href="#merantk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>High performance toolkit for complete analysis of methylated RNA data.
</code></pre></div>
<h3 id="medakas">medaka's:<a class="headerlink" href="#medakas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Medaka is a tool to create a consensus sequence from nanopore sequencing data.
</code></pre></div>
<h3 id="medaka">medaka:<a class="headerlink" href="#medaka" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Medaka is a tool to create a consensus sequence from nanopore sequencing data.
</code></pre></div>
<h3 id="megalodons">megalodon's:<a class="headerlink" href="#megalodons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads
</code></pre></div>
<h3 id="megalodon">megalodon:<a class="headerlink" href="#megalodon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool to extract high accuracy modified base and sequence variant calls from raw nanopore reads
</code></pre></div>
<h3 id="metawraps">metaWRAP's:<a class="headerlink" href="#metawraps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flexible pipeline for genome-resolved metagenomic data analysis.
</code></pre></div>
<h3 id="metawrap">metaWRAP:<a class="headerlink" href="#metawrap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Flexible pipeline for genome-resolved metagenomic data analysis.
</code></pre></div>
<h3 id="mirdeep2s">miRDeep2's:<a class="headerlink" href="#mirdeep2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs
</code></pre></div>
<h3 id="mirdeep2">miRDeep2:<a class="headerlink" href="#mirdeep2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs
</code></pre></div>
<h3 id="midls">midl's:<a class="headerlink" href="#midls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).
</code></pre></div>
<h3 id="midl">midl:<a class="headerlink" href="#midl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).
</code></pre></div>
<h3 id="mimallocs">mimalloc's:<a class="headerlink" href="#mimallocs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>mimalloc is a general purpose allocator with excellent performance characteristics.
</code></pre></div>
<h3 id="mimalloc">mimalloc:<a class="headerlink" href="#mimalloc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>mimalloc is a general purpose allocator with excellent performance characteristics.
</code></pre></div>
<h3 id="miniasms">miniasm's:<a class="headerlink" href="#miniasms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast OLC-based de novo assembler for noisy long reads.
</code></pre></div>
<h3 id="miniasm">miniasm:<a class="headerlink" href="#miniasm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast OLC-based de novo assembler for noisy long reads.
</code></pre></div>
<h3 id="minieigens">minieigen's:<a class="headerlink" href="#minieigens" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A small wrapper for core parts of Eigen, c++ library for linear algebra.
</code></pre></div>
<h3 id="minieigen">minieigen:<a class="headerlink" href="#minieigen" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A small wrapper for core parts of Eigen, c++ library for linear algebra.
</code></pre></div>
<h3 id="minimap2s">minimap2's:<a class="headerlink" href="#minimap2s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Minimap2 is a fast sequence mapping and alignment
</code></pre></div>
<h3 id="minimap2">minimap2:<a class="headerlink" href="#minimap2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Minimap2 is a fast sequence mapping and alignment
</code></pre></div>
<h3 id="mjpegtoolss">mjpegtools's:<a class="headerlink" href="#mjpegtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The mjpeg programs are a set of tools that can do recording of videos and playback, simple cut-and-paste editing and the MPEG compression of audio and video under Linux.
</code></pre></div>
<h3 id="mjpegtools">mjpegtools:<a class="headerlink" href="#mjpegtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The mjpeg programs are a set of tools that can do recording of videos and playback, simple cut-and-paste editing and the MPEG compression of audio and video under Linux.
</code></pre></div>
<h3 id="mlpacks">mlpack's:<a class="headerlink" href="#mlpacks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast, and flexible C++ machine learning library with bindings to other languages
</code></pre></div>
<h3 id="mlpack">mlpack:<a class="headerlink" href="#mlpack" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast, and flexible C++ machine learning library with bindings to other languages
</code></pre></div>
<h3 id="mo_tidls">mo_tidl's:<a class="headerlink" href="#mo_tidls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).
</code></pre></div>
<h3 id="mo_tidl">mo_tidl:<a class="headerlink" href="#mo_tidl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Met Office library is a collection of routines written in IDL at the Met Office, originally written in PV-WAVE in the Hadley Centre in the early 1990s, with development continuing to the present day. Its main purpose is to support analysis and visualisation of PP data produced by the Unified Model (UM).
</code></pre></div>
<h3 id="modbam2beds">modbam2bed's:<a class="headerlink" href="#modbam2beds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A program to aggregate modified base counts stored in a modified-base BAM file to a bedMethyl file.
</code></pre></div>
<h3 id="modbam2bed">modbam2bed:<a class="headerlink" href="#modbam2bed" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A program to aggregate modified base counts stored in a modified-base BAM file to a bedMethyl file.
</code></pre></div>
<h3 id="mpi4pys">mpi4py's:<a class="headerlink" href="#mpi4pys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>This package provides Python bindings for the Message Passing Interface (MPI) standard. It is implemented on top of the MPI-1/2/3 specification and exposes an API which grounds on the standard MPI-2 C++ bindings.
</code></pre></div>
<h3 id="mpi4py">mpi4py:<a class="headerlink" href="#mpi4py" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>This package provides Python bindings for the Message Passing Interface (MPI) standard. It is implemented on top of the MPI-1/2/3 specification and exposes an API which grounds on the standard MPI-2 C++ bindings.
</code></pre></div>
<h3 id="mpifileutilss">mpifileutils's:<a class="headerlink" href="#mpifileutilss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MPI-Based File Utilities For Distributed Systems
</code></pre></div>
<h3 id="mpifileutils">mpifileutils:<a class="headerlink" href="#mpifileutils" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>MPI-Based File Utilities For Distributed Systems
</code></pre></div>
<h3 id="muparsers">muParser's:<a class="headerlink" href="#muparsers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>muParser is an extensible high performance math expression
</code></pre></div>
<h3 id="muparser">muParser:<a class="headerlink" href="#muparser" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>muParser is an extensible high performance math expression
</code></pre></div>
<h3 id="nanos">nano's:<a class="headerlink" href="#nanos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nano is a text editor.
</code></pre></div>
<h3 id="nano">nano:<a class="headerlink" href="#nano" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nano is a text editor.
</code></pre></div>
<h3 id="nanoqcs">nanoQC's:<a class="headerlink" href="#nanoqcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Create fastQC-like plots for Oxford Nanopore sequencing data.
</code></pre></div>
<h3 id="nanoqc">nanoQC:<a class="headerlink" href="#nanoqc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Create fastQC-like plots for Oxford Nanopore sequencing data.
</code></pre></div>
<h3 id="nanofilts">nanofilt's:<a class="headerlink" href="#nanofilts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Filtering and trimming of long read sequencing data.
</code></pre></div>
<h3 id="nanofilt">nanofilt:<a class="headerlink" href="#nanofilt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Filtering and trimming of long read sequencing data.
</code></pre></div>
<h3 id="nanogets">nanoget's:<a class="headerlink" href="#nanogets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Functions to extract information from Oxford Nanopore sequencing data and alignments
</code></pre></div>
<h3 id="nanoget">nanoget:<a class="headerlink" href="#nanoget" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Functions to extract information from Oxford Nanopore sequencing data and alignments
</code></pre></div>
<h3 id="nanomaths">nanomath's:<a class="headerlink" href="#nanomaths" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A few simple math function for other Oxford Nanopore processing scripts
</code></pre></div>
<h3 id="nanomath">nanomath:<a class="headerlink" href="#nanomath" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A few simple math function for other Oxford Nanopore processing scripts
</code></pre></div>
<h3 id="nanopolishs">nanopolish's:<a class="headerlink" href="#nanopolishs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software package for signal-level analysis of Oxford Nanopore sequencing data.
</code></pre></div>
<h3 id="nanopolish">nanopolish:<a class="headerlink" href="#nanopolish" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software package for signal-level analysis of Oxford Nanopore sequencing data.
</code></pre></div>
<h3 id="ncbi-vdbs">ncbi-vdb's:<a class="headerlink" href="#ncbi-vdbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
</code></pre></div>
<h3 id="ncbi-vdb">ncbi-vdb:<a class="headerlink" href="#ncbi-vdb" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
</code></pre></div>
<h3 id="nccmps">nccmp's:<a class="headerlink" href="#nccmps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage). Parallel comparisons are done in local memory without requiring temporary files. Highly recommended for regression testing scientific models or datasets in a test-driven development environment.
</code></pre></div>
<h3 id="nccmp">nccmp:<a class="headerlink" href="#nccmp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage). Parallel comparisons are done in local memory without requiring temporary files. Highly recommended for regression testing scientific models or datasets in a test-driven development environment.
</code></pre></div>
<h3 id="ncviews_1">ncview's:<a class="headerlink" href="#ncviews_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Visual browser for netCDF format files.
</code></pre></div>
<h3 id="ncview_1">ncview:<a class="headerlink" href="#ncview_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Visual browser for netCDF format files.
</code></pre></div>
<h3 id="nes">ne's:<a class="headerlink" href="#nes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ne is a free (GPL&#39;d) text editor based on the POSIX standard
</code></pre></div>
<h3 id="ne">ne:<a class="headerlink" href="#ne" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>ne is a free (GPL&#39;d) text editor based on the POSIX standard
</code></pre></div>
<h3 id="nearlines">nearline's:<a class="headerlink" href="#nearlines" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NeSI nearline client provides &#39;End User&#39; access to NeSI&#39;s archive filesystem Nearline which is connected to a Tape library.
</code></pre></div>
<h3 id="netcdfs">netCDF's:<a class="headerlink" href="#netcdfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries
</code></pre></div>
<h3 id="netcdf">netCDF:<a class="headerlink" href="#netcdf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries
</code></pre></div>
<h3 id="netcdf-cs">netCDF-C++'s:<a class="headerlink" href="#netcdf-cs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries
</code></pre></div>
<h3 id="netcdf-c">netCDF-C++:<a class="headerlink" href="#netcdf-c" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries
</code></pre></div>
<h3 id="netcdf-c4s">netCDF-C++4's:<a class="headerlink" href="#netcdf-c4s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.
</code></pre></div>
<h3 id="netcdf-c4">netCDF-C++4:<a class="headerlink" href="#netcdf-c4" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.
</code></pre></div>
<h3 id="netcdf-fortrans">netCDF-Fortran's:<a class="headerlink" href="#netcdf-fortrans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.
</code></pre></div>
<h3 id="netcdf-fortran">netCDF-Fortran:<a class="headerlink" href="#netcdf-fortran" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.
</code></pre></div>
<h3 id="nettles">nettle's:<a class="headerlink" href="#nettles" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nettle is a cryptographic library that is designed to fit easily
</code></pre></div>
<h3 id="nettle">nettle:<a class="headerlink" href="#nettle" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Nettle is a cryptographic library that is designed to fit easily
</code></pre></div>
<h3 id="networkxs">networkx's:<a class="headerlink" href="#networkxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
</code></pre></div>
<h3 id="networkx">networkx:<a class="headerlink" href="#networkx" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
</code></pre></div>
<h3 id="nodejss">nodejs's:<a class="headerlink" href="#nodejss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Node.js is a platform built on Chrome&#39;s JavaScript runtime  for easily building fast, scalable network applications. Node.js uses an  event-driven, non-blocking I/O model that makes it lightweight and efficient,  perfect for data-intensive real-time applications that run across distributed devices.
</code></pre></div>
<h3 id="nodejs">nodejs:<a class="headerlink" href="#nodejs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Node.js is a platform built on Chrome&#39;s JavaScript runtime  for easily building fast, scalable network applications. Node.js uses an  event-driven, non-blocking I/O model that makes it lightweight and efficient,  perfect for data-intensive real-time applications that run across distributed devices.
</code></pre></div>
<h3 id="nsegs">nseg's:<a class="headerlink" href="#nsegs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Used to mask nucleic acid sequences
</code></pre></div>
<h3 id="nseg">nseg:<a class="headerlink" href="#nseg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Used to mask nucleic acid sequences
</code></pre></div>
<h3 id="nsyncs">nsync's:<a class="headerlink" href="#nsyncs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nsync is a C library that exports various synchronization primitives, such as mutexes
</code></pre></div>
<h3 id="nsync">nsync:<a class="headerlink" href="#nsync" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>nsync is a C library that exports various synchronization primitives, such as mutexes
</code></pre></div>
<h3 id="nullarbors">nullarbor's:<a class="headerlink" href="#nullarbors" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reads to report pipeline for bacterial isolate NGS data.
</code></pre></div>
<h3 id="nullarbor">nullarbor:<a class="headerlink" href="#nullarbor" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Reads to report pipeline for bacterial isolate NGS data.
</code></pre></div>
<h3 id="ont-guppy-gpus">ont-guppy-gpu's:<a class="headerlink" href="#ont-guppy-gpus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Data processing toolkit that contains the Oxford Nanopore Technologies&#39; basecalling algorithms,
</code></pre></div>
<h3 id="ont-guppy-gpu">ont-guppy-gpu:<a class="headerlink" href="#ont-guppy-gpu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Data processing toolkit that contains the Oxford Nanopore Technologies&#39; basecalling algorithms,
</code></pre></div>
<h3 id="padlocs">padloc's:<a class="headerlink" href="#padlocs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prokaryotic Antiviral Defence LOCator
</code></pre></div>
<h3 id="padloc">padloc:<a class="headerlink" href="#padloc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prokaryotic Antiviral Defence LOCator
</code></pre></div>
<h3 id="pairtoolss">pairtools's:<a class="headerlink" href="#pairtoolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CLI tools to process mapped Hi-C data
</code></pre></div>
<h3 id="pairtools">pairtools:<a class="headerlink" href="#pairtools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>CLI tools to process mapped Hi-C data
</code></pre></div>
<h3 id="panaroos">panaroo's:<a class="headerlink" href="#panaroos" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A pangenome analysis pipeline.
</code></pre></div>
<h3 id="panaroo">panaroo:<a class="headerlink" href="#panaroo" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A pangenome analysis pipeline.
</code></pre></div>
<h3 id="pandocs">pandoc's:<a class="headerlink" href="#pandocs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Almost universal document converter
</code></pre></div>
<h3 id="pandoc">pandoc:<a class="headerlink" href="#pandoc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Almost universal document converter
</code></pre></div>
<h3 id="parallel-fastq-dumps">parallel-fastq-dump's:<a class="headerlink" href="#parallel-fastq-dumps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parallel fastq-dump wrapper
</code></pre></div>
<h3 id="parallel-fastq-dump">parallel-fastq-dump:<a class="headerlink" href="#parallel-fastq-dump" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parallel fastq-dump wrapper
</code></pre></div>
<h3 id="parasails">parasail's:<a class="headerlink" href="#parasails" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parasail is a SIMD C (C99) library containing implementations
</code></pre></div>
<h3 id="parasail">parasail:<a class="headerlink" href="#parasail" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>parasail is a SIMD C (C99) library containing implementations
</code></pre></div>
<h3 id="patchelfs">patchelf's:<a class="headerlink" href="#patchelfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
</code></pre></div>
<h3 id="patchelf">patchelf:<a class="headerlink" href="#patchelf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.
</code></pre></div>
<h3 id="pauvres">pauvre's:<a class="headerlink" href="#pauvres" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for plotting Oxford Nanopore and other long-read data.
</code></pre></div>
<h3 id="pauvre">pauvre:<a class="headerlink" href="#pauvre" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tools for plotting Oxford Nanopore and other long-read data.
</code></pre></div>
<h3 id="pfunits">pfunit's:<a class="headerlink" href="#pfunits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.
</code></pre></div>
<h3 id="pfunit">pfunit:<a class="headerlink" href="#pfunit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.
</code></pre></div>
<h3 id="phyxs">phyx's:<a class="headerlink" href="#phyxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>phyx performs phylogenetics analyses on trees and sequences.
</code></pre></div>
<h3 id="phyx">phyx:<a class="headerlink" href="#phyx" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>phyx performs phylogenetics analyses on trees and sequences.
</code></pre></div>
<h3 id="picards">picard's:<a class="headerlink" href="#picards" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools (in Java) for working with next generation sequencing data in the BAM format.
</code></pre></div>
<h3 id="picard">picard:<a class="headerlink" href="#picard" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A set of tools (in Java) for working with next generation sequencing data in the BAM format.
</code></pre></div>
<h3 id="pixmans">pixman's:<a class="headerlink" href="#pixmans" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pixman is a low-level software library for pixel manipulation, providing features such as image
</code></pre></div>
<h3 id="pixman">pixman:<a class="headerlink" href="#pixman" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pixman is a low-level software library for pixel manipulation, providing features such as image
</code></pre></div>
<h3 id="pplacers">pplacer's:<a class="headerlink" href="#pplacers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Places query sequences on a fixed reference phylogenetic tree
</code></pre></div>
<h3 id="pplacer">pplacer:<a class="headerlink" href="#pplacer" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Places query sequences on a fixed reference phylogenetic tree
</code></pre></div>
<h3 id="preseqs">preseq's:<a class="headerlink" href="#preseqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software for predicting library complexity and genome coverage in high-throughput sequencing.
</code></pre></div>
<h3 id="preseq">preseq:<a class="headerlink" href="#preseq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Software for predicting library complexity and genome coverage in high-throughput sequencing.
</code></pre></div>
<h3 id="prodigals_1">prodigal's:<a class="headerlink" href="#prodigals_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
</code></pre></div>
<h3 id="prodigal_1">prodigal:<a class="headerlink" href="#prodigal_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)
</code></pre></div>
<h3 id="prodigal-gvs">prodigal-gv's:<a class="headerlink" href="#prodigal-gvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A fork of Prodigal meant to improve gene calling
</code></pre></div>
<h3 id="prodigal-gv">prodigal-gv:<a class="headerlink" href="#prodigal-gv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A fork of Prodigal meant to improve gene calling
</code></pre></div>
<h3 id="prokkas">prokka's:<a class="headerlink" href="#prokkas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prokka is a software tool for the rapid annotation of prokaryotic genomes.
</code></pre></div>
<h3 id="prokka">prokka:<a class="headerlink" href="#prokka" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Prokka is a software tool for the rapid annotation of prokaryotic genomes.
</code></pre></div>
<h3 id="proovreads">proovread's:<a class="headerlink" href="#proovreads" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PacBio hybrid error correction through iterative short read consensus
</code></pre></div>
<h3 id="proovread">proovread:<a class="headerlink" href="#proovread" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PacBio hybrid error correction through iterative short read consensus
</code></pre></div>
<h3 id="protobufs">protobuf's:<a class="headerlink" href="#protobufs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Google Protocol Buffers
</code></pre></div>
<h3 id="protobuf">protobuf:<a class="headerlink" href="#protobuf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Google Protocol Buffers
</code></pre></div>
<h3 id="protobuf-pythons">protobuf-python's:<a class="headerlink" href="#protobuf-pythons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python Protocol Buffers runtime library.
</code></pre></div>
<h3 id="protobuf-python">protobuf-python:<a class="headerlink" href="#protobuf-python" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python Protocol Buffers runtime library.
</code></pre></div>
<h3 id="psmcs">psmc's:<a class="headerlink" href="#psmcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Infers population size history from a diploid sequence using the PSMC model.
</code></pre></div>
<h3 id="psmc">psmc:<a class="headerlink" href="#psmc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Infers population size history from a diploid sequence using the PSMC model.
</code></pre></div>
<h3 id="pstoedits">pstoedit's:<a class="headerlink" href="#pstoedits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pstoedit translates PostScript and PDF graphics into other vector formats.
</code></pre></div>
<h3 id="pstoedit">pstoedit:<a class="headerlink" href="#pstoedit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>pstoedit translates PostScript and PDF graphics into other vector formats.
</code></pre></div>
<h3 id="pullseqs">pullseq's:<a class="headerlink" href="#pullseqs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Utility program for extracting sequences from a fasta/fastq file
</code></pre></div>
<h3 id="pullseq">pullseq:<a class="headerlink" href="#pullseq" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Utility program for extracting sequences from a fasta/fastq file
</code></pre></div>
<h3 id="purge_dupss">purge_dups's:<a class="headerlink" href="#purge_dupss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>purge haplotigs and overlaps in an assembly based on read depth
</code></pre></div>
<h3 id="purge_dups">purge_dups:<a class="headerlink" href="#purge_dups" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>purge haplotigs and overlaps in an assembly based on read depth
</code></pre></div>
<h3 id="purge_haplotigss">purge_haplotigs's:<a class="headerlink" href="#purge_haplotigss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline to help with curating heterozygous diploid genome assemblies
</code></pre></div>
<h3 id="purge_haplotigs">purge_haplotigs:<a class="headerlink" href="#purge_haplotigs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Pipeline to help with curating heterozygous diploid genome assemblies
</code></pre></div>
<h3 id="pvs">pv's:<a class="headerlink" href="#pvs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Monitors the progress of data through a unix pipeline.
</code></pre></div>
<h3 id="pv">pv:<a class="headerlink" href="#pv" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Monitors the progress of data through a unix pipeline.
</code></pre></div>
<h3 id="pyanis">pyani's:<a class="headerlink" href="#pyanis" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Whole-genome classification using Average Nucleotide Identity
</code></pre></div>
<h3 id="pyani">pyani:<a class="headerlink" href="#pyani" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Whole-genome classification using Average Nucleotide Identity
</code></pre></div>
<h3 id="pycoqcs">pycoQC's:<a class="headerlink" href="#pycoqcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.
</code></pre></div>
<h3 id="pycoqc">pycoQC:<a class="headerlink" href="#pycoqc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.
</code></pre></div>
<h3 id="pymol-open-sources">pymol-open-source's:<a class="headerlink" href="#pymol-open-sources" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyMOL (open source version) molecular visualization system.
</code></pre></div>
<h3 id="pymol-open-source">pymol-open-source:<a class="headerlink" href="#pymol-open-source" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>PyMOL (open source version) molecular visualization system.
</code></pre></div>
<h3 id="pyspoas">pyspoa's:<a class="headerlink" href="#pyspoas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python bindings to spoa.
</code></pre></div>
<h3 id="pyspoa">pyspoa:<a class="headerlink" href="#pyspoa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Python bindings to spoa.
</code></pre></div>
<h3 id="qcats">qcat's:<a class="headerlink" href="#qcats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files
</code></pre></div>
<h3 id="qcat">qcat:<a class="headerlink" href="#qcat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command-line tool for demultiplexing Oxford Nanopore reads from FASTQ files
</code></pre></div>
<h3 id="rdocks">rDock's:<a class="headerlink" href="#rdocks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>rDock is a fast and versatile Open Source docking program that
</code></pre></div>
<h3 id="rdock">rDock:<a class="headerlink" href="#rdock" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>rDock is a fast and versatile Open Source docking program that
</code></pre></div>
<h3 id="randfolds">randfold's:<a class="headerlink" href="#randfolds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Minimum free energy of folding randomization test software
</code></pre></div>
<h3 id="randfold">randfold:<a class="headerlink" href="#randfold" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Minimum free energy of folding randomization test software
</code></pre></div>
<h3 id="rasusas">rasusa's:<a class="headerlink" href="#rasusas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Randomly subsample sequencing reads to a specified coverage.
</code></pre></div>
<h3 id="rasusa">rasusa:<a class="headerlink" href="#rasusa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Randomly subsample sequencing reads to a specified coverage.
</code></pre></div>
<h3 id="razers3s">razers3's:<a class="headerlink" href="#razers3s" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for mapping millions of short genomic reads onto a reference genome.
</code></pre></div>
<h3 id="razers3">razers3:<a class="headerlink" href="#razers3" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for mapping millions of short genomic reads onto a reference genome.
</code></pre></div>
<h3 id="re2cs">re2c's:<a class="headerlink" href="#re2cs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
</code></pre></div>
<h3 id="re2c">re2c:<a class="headerlink" href="#re2c" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>re2c is a free and open-source lexer generator for C and C++. Its main goal is generating
</code></pre></div>
<h3 id="rnaquasts">rnaQUAST's:<a class="headerlink" href="#rnaquasts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for evaluating RNA-Seq assemblies using reference genome and gene database
</code></pre></div>
<h3 id="rnaquast">rnaQUAST:<a class="headerlink" href="#rnaquast" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for evaluating RNA-Seq assemblies using reference genome and gene database
</code></pre></div>
<h3 id="rsyncs">rsync's:<a class="headerlink" href="#rsyncs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>rsync is an open source utility that provides fast incremental file transfer.  rsync is freely available under the GNU General Public License
</code></pre></div>
<h3 id="rsync">rsync:<a class="headerlink" href="#rsync" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>rsync is an open source utility that provides fast incremental file transfer.  rsync is freely available under the GNU General Public License
</code></pre></div>
<h3 id="rust-fmlrcs">rust-fmlrc's:<a class="headerlink" href="#rust-fmlrcs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FM-index Long Read Corrector (Rust implementation)
</code></pre></div>
<h3 id="rust-fmlrc">rust-fmlrc:<a class="headerlink" href="#rust-fmlrc" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>FM-index Long Read Corrector (Rust implementation)
</code></pre></div>
<h3 id="samblasters">samblaster's:<a class="headerlink" href="#samblasters" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>samblaster is a fast and flexible program for marking duplicates in read-id grouped paired-end SAM files.
</code></pre></div>
<h3 id="samblaster">samblaster:<a class="headerlink" href="#samblaster" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>samblaster is a fast and flexible program for marking duplicates in read-id grouped paired-end SAM files.
</code></pre></div>
<h3 id="samclips">samclip's:<a class="headerlink" href="#samclips" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Filter SAM file for soft and hard clipped alignments.
</code></pre></div>
<h3 id="samclip">samclip:<a class="headerlink" href="#samclip" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Filter SAM file for soft and hard clipped alignments.
</code></pre></div>
<h3 id="sbts">sbt's:<a class="headerlink" href="#sbts" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>sbt is a build tool for Scala, Java, and more.
</code></pre></div>
<h3 id="sbt">sbt:<a class="headerlink" href="#sbt" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>sbt is a build tool for Scala, Java, and more.
</code></pre></div>
<h3 id="seqmagicks">seqmagick's:<a class="headerlink" href="#seqmagicks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Seqmagick is a utility built in the spirit of imagemagick to expose the
</code></pre></div>
<h3 id="seqmagick">seqmagick:<a class="headerlink" href="#seqmagick" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Seqmagick is a utility built in the spirit of imagemagick to expose the
</code></pre></div>
<h3 id="seqtks">seqtk's:<a class="headerlink" href="#seqtks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.
</code></pre></div>
<h3 id="seqtk">seqtk:<a class="headerlink" href="#seqtk" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.
</code></pre></div>
<h3 id="shumlibs">shumlib's:<a class="headerlink" href="#shumlibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Shumlib is the collective name for a set of libraries which are used by the UM; the UK Met Office&#39;s Unified Model, that may be of use to external tools or applications where identical functionality is desired. The hope of the project is to enable developers to quickly and easily access parts of the UM code that are commonly duplicated elsewhere, at the same time benefiting from any improvements or optimisations that might be made in support of the UM itself.
</code></pre></div>
<h3 id="shumlib">shumlib:<a class="headerlink" href="#shumlib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Shumlib is the collective name for a set of libraries which are used by the UM; the UK Met Office&#39;s Unified Model, that may be of use to external tools or applications where identical functionality is desired. The hope of the project is to enable developers to quickly and easily access parts of the UM code that are commonly duplicated elsewhere, at the same time benefiting from any improvements or optimisations that might be made in support of the UM itself.
</code></pre></div>
<h3 id="sismonrs">sismonr's:<a class="headerlink" href="#sismonrs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Simulation of In Silico Multi-Omic Networks R package.
</code></pre></div>
<h3 id="sismonr">sismonr:<a class="headerlink" href="#sismonr" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Simulation of In Silico Multi-Omic Networks R package.
</code></pre></div>
<h3 id="slow5toolss">slow5tools's:<a class="headerlink" href="#slow5toolss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit for converting (FAST5 &lt;-&gt; SLOW5), compressing, viewing, indexing
</code></pre></div>
<h3 id="slow5tools">slow5tools:<a class="headerlink" href="#slow5tools" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Toolkit for converting (FAST5 &lt;-&gt; SLOW5), compressing, viewing, indexing
</code></pre></div>
<h3 id="smafas">smafa's:<a class="headerlink" href="#smafas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences
</code></pre></div>
<h3 id="smafa">smafa:<a class="headerlink" href="#smafa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences
</code></pre></div>
<h3 id="smooves">smoove's:<a class="headerlink" href="#smooves" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>simplifies and speeds calling and genotyping SVs for short reads.
</code></pre></div>
<h3 id="smoove">smoove:<a class="headerlink" href="#smoove" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>simplifies and speeds calling and genotyping SVs for short reads.
</code></pre></div>
<h3 id="snakemakes">snakemake's:<a class="headerlink" href="#snakemakes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.
</code></pre></div>
<h3 id="snakemake">snakemake:<a class="headerlink" href="#snakemake" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.
</code></pre></div>
<h3 id="snaphus">snaphu's:<a class="headerlink" href="#snaphus" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping
</code></pre></div>
<h3 id="snaphu">snaphu:<a class="headerlink" href="#snaphu" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping
</code></pre></div>
<h3 id="snappys">snappy's:<a class="headerlink" href="#snappys" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.
</code></pre></div>
<h3 id="snappy">snappy:<a class="headerlink" href="#snappy" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.
</code></pre></div>
<h3 id="snp-sitess">snp-sites's:<a class="headerlink" href="#snp-sitess" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Finds SNP sites from a multi-FASTA alignment file.
</code></pre></div>
<h3 id="snp-sites">snp-sites:<a class="headerlink" href="#snp-sites" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Finds SNP sites from a multi-FASTA alignment file.
</code></pre></div>
<h3 id="snpeffs">snpEff's:<a class="headerlink" href="#snpeffs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SnpEff is a variant annotation and effect prediction tool.
</code></pre></div>
<h3 id="snpeff">snpEff:<a class="headerlink" href="#snpeff" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>SnpEff is a variant annotation and effect prediction tool.
</code></pre></div>
<h3 id="somaliers">somalier's:<a class="headerlink" href="#somaliers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>extract informative sites, evaluate relatedness, and
</code></pre></div>
<h3 id="somalier">somalier:<a class="headerlink" href="#somalier" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>extract informative sites, evaluate relatedness, and
</code></pre></div>
<h3 id="spalns">spaln's:<a class="headerlink" href="#spalns" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Stand-alone program that maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.
</code></pre></div>
<h3 id="spaln">spaln:<a class="headerlink" href="#spaln" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Stand-alone program that maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.
</code></pre></div>
<h3 id="spdlogs">spdlog's:<a class="headerlink" href="#spdlogs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast C++ logging library.
</code></pre></div>
<h3 id="spdlog">spdlog:<a class="headerlink" href="#spdlog" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fast C++ logging library.
</code></pre></div>
<h3 id="spglibs">spglib's:<a class="headerlink" href="#spglibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spglib is a C library for finding and handling crystal symmetries.
</code></pre></div>
<h3 id="spglib">spglib:<a class="headerlink" href="#spglib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Spglib is a C library for finding and handling crystal symmetries.
</code></pre></div>
<h3 id="splats">splat's:<a class="headerlink" href="#splats" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Splat deploys (builds and installs) the components of heterogenous systems such as cylc suites, described by splat config files, into target directory trees.
</code></pre></div>
<h3 id="splat">splat:<a class="headerlink" href="#splat" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Splat deploys (builds and installs) the components of heterogenous systems such as cylc suites, described by splat config files, into target directory trees.
</code></pre></div>
<h3 id="spoas">spoa's:<a class="headerlink" href="#spoas" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>c++ implementation of the partial order alignment (POA) algorithm
</code></pre></div>
<h3 id="spoa">spoa:<a class="headerlink" href="#spoa" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>c++ implementation of the partial order alignment (POA) algorithm
</code></pre></div>
<h3 id="sratoolkits">sratoolkit's:<a class="headerlink" href="#sratoolkits" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The SRA Toolkit, and the source-code SRA System Development
</code></pre></div>
<h3 id="sratoolkit">sratoolkit:<a class="headerlink" href="#sratoolkit" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>The SRA Toolkit, and the source-code SRA System Development
</code></pre></div>
<h3 id="srun-wrappers">srun-wrapper's:<a class="headerlink" href="#srun-wrappers" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Provides an mpirun script which merely wraps the srun command.
</code></pre></div>
<h3 id="srun-wrapper">srun-wrapper:<a class="headerlink" href="#srun-wrapper" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Provides an mpirun script which merely wraps the srun command.
</code></pre></div>
<h3 id="sublimes">sublime's:<a class="headerlink" href="#sublimes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Text editor with GUI
</code></pre></div>
<h3 id="sublime">sublime:<a class="headerlink" href="#sublime" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Text editor with GUI
</code></pre></div>
<h3 id="swarms">swarm's:<a class="headerlink" href="#swarms" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A robust and fast clustering method for amplicon-based studies.
</code></pre></div>
<h3 id="swarm">swarm:<a class="headerlink" href="#swarm" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A robust and fast clustering method for amplicon-based studies.
</code></pre></div>
<h3 id="swissknifes">swissknife's:<a class="headerlink" href="#swissknifes" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Perl module for reading and writing UniProtKB data in plain text format.
</code></pre></div>
<h3 id="swissknife">swissknife:<a class="headerlink" href="#swissknife" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Perl module for reading and writing UniProtKB data in plain text format.
</code></pre></div>
<h3 id="trnascan-ses">tRNAscan-SE's:<a class="headerlink" href="#trnascan-ses" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transfer RNA detection
</code></pre></div>
<h3 id="trnascan-se">tRNAscan-SE:<a class="headerlink" href="#trnascan-se" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Transfer RNA detection
</code></pre></div>
<h3 id="tabixs">tabix's:<a class="headerlink" href="#tabixs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic indexer for TAB-delimited genome position files
</code></pre></div>
<h3 id="tabix">tabix:<a class="headerlink" href="#tabix" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Generic indexer for TAB-delimited genome position files
</code></pre></div>
<h3 id="tabixpps">tabixpp's:<a class="headerlink" href="#tabixpps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ wrapper to tabix indexer
</code></pre></div>
<h3 id="tabixpp">tabixpp:<a class="headerlink" href="#tabixpp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>C++ wrapper to tabix indexer
</code></pre></div>
<h3 id="tbbs">tbb's:<a class="headerlink" href="#tbbs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
</code></pre></div>
<h3 id="tbb">tbb:<a class="headerlink" href="#tbb" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
</code></pre></div>
<h3 id="tbl2asns">tbl2asn's:<a class="headerlink" href="#tbl2asns" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command-line program that automates the creation of
</code></pre></div>
<h3 id="tbl2asn">tbl2asn:<a class="headerlink" href="#tbl2asn" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Command-line program that automates the creation of
</code></pre></div>
<h3 id="tmuxs">tmux's:<a class="headerlink" href="#tmuxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tmux is a terminal multiplexer. It lets you switch easily
</code></pre></div>
<h3 id="tmux">tmux:<a class="headerlink" href="#tmux" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>tmux is a terminal multiplexer. It lets you switch easily
</code></pre></div>
<h3 id="trfs">trf's:<a class="headerlink" href="#trfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Locates tandem repeats in DNA sequences.
</code></pre></div>
<h3 id="trf">trf:<a class="headerlink" href="#trf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Locates tandem repeats in DNA sequences.
</code></pre></div>
<h3 id="trimals">trimAl's:<a class="headerlink" href="#trimals" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for automated alignment trimming in large-scale phylogenetic analyses
</code></pre></div>
<h3 id="trimal">trimAl:<a class="headerlink" href="#trimal" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Tool for automated alignment trimming in large-scale phylogenetic analyses
</code></pre></div>
<h3 id="um2netcdfs">um2netcdf's:<a class="headerlink" href="#um2netcdfs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>um2netcdf converts Unified Model output files in fieldsfile format into netCDF format.
</code></pre></div>
<h3 id="um2netcdf">um2netcdf:<a class="headerlink" href="#um2netcdf" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>um2netcdf converts Unified Model output files in fieldsfile format into netCDF format.
</code></pre></div>
<h3 id="unimaps">unimap's:<a class="headerlink" href="#unimaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fork of minimap2 optimized for assembly-to-reference
</code></pre></div>
<h3 id="unimap">unimap:<a class="headerlink" href="#unimap" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Fork of minimap2 optimized for assembly-to-reference
</code></pre></div>
<h3 id="unrars">unrar's:<a class="headerlink" href="#unrars" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAR is a powerful archive manager.
</code></pre></div>
<h3 id="unrar">unrar:<a class="headerlink" href="#unrar" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>RAR is a powerful archive manager.
</code></pre></div>
<h3 id="util-linuxs">util-linux's:<a class="headerlink" href="#util-linuxs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Set of Linux utilities
</code></pre></div>
<h3 id="util-linux">util-linux:<a class="headerlink" href="#util-linux" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Set of Linux utilities
</code></pre></div>
<h3 id="vcflibs">vcflib's:<a class="headerlink" href="#vcflibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.
</code></pre></div>
<h3 id="vcflib">vcflib:<a class="headerlink" href="#vcflib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Genetic variant detector designed to find polymorphisms smaller than the length of a short-read sequencing alignment.
</code></pre></div>
<h3 id="vgs">vg's:<a class="headerlink" href="#vgs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods
</code></pre></div>
<h3 id="vg">vg:<a class="headerlink" href="#vg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods
</code></pre></div>
<h3 id="wgsims">wgsim's:<a class="headerlink" href="#wgsims" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Wgsim is a small tool for simulating sequence reads from a reference genome.
</code></pre></div>
<h3 id="wgsim">wgsim:<a class="headerlink" href="#wgsim" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Wgsim is a small tool for simulating sequence reads from a reference genome.
</code></pre></div>
<h3 id="wheels">wheel's:<a class="headerlink" href="#wheels" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A built-package format for Python.
</code></pre></div>
<h3 id="wheel">wheel:<a class="headerlink" href="#wheel" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>A built-package format for Python.
</code></pre></div>
<h3 id="wtdbgs">wtdbg's:<a class="headerlink" href="#wtdbgs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies.
</code></pre></div>
<h3 id="wtdbg">wtdbg:<a class="headerlink" href="#wtdbg" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies.
</code></pre></div>
<h3 id="wxwidgetss">wxWidgets's:<a class="headerlink" href="#wxwidgetss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>widget toolkit and tools library for creating graphical user interfaces for cross-platform applications.
</code></pre></div>
<h3 id="wxwidgets">wxWidgets:<a class="headerlink" href="#wxwidgets" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>widget toolkit and tools library for creating graphical user interfaces for cross-platform applications.
</code></pre></div>
<h3 id="xbitmapss">xbitmaps's:<a class="headerlink" href="#xbitmapss" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>provides bitmaps for x
</code></pre></div>
<h3 id="xbitmaps">xbitmaps:<a class="headerlink" href="#xbitmaps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>provides bitmaps for x
</code></pre></div>
<h3 id="xkbcommons">xkbcommon's:<a class="headerlink" href="#xkbcommons" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>keyboard keymap compiler and support library
</code></pre></div>
<h3 id="xkbcommon">xkbcommon:<a class="headerlink" href="#xkbcommon" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>keyboard keymap compiler and support library
</code></pre></div>
<h3 id="yacrds">yacrd's:<a class="headerlink" href="#yacrds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Chimeric Read Detector for long reads
</code></pre></div>
<h3 id="yacrd">yacrd:<a class="headerlink" href="#yacrd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Chimeric Read Detector for long reads
</code></pre></div>
<h3 id="yajls">yajl's:<a class="headerlink" href="#yajls" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet Another JSON Library. Why does the world need another C library for parsing JSON? Good question.
</code></pre></div>
<h3 id="yajl">yajl:<a class="headerlink" href="#yajl" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet Another JSON Library. Why does the world need another C library for parsing JSON? Good question.
</code></pre></div>
<h3 id="yaks">yak's:<a class="headerlink" href="#yaks" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet another k-mer analyzer
</code></pre></div>
<h3 id="yak">yak:<a class="headerlink" href="#yak" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Yet another k-mer analyzer
</code></pre></div>
<h3 id="yaml-cpps">yaml-cpp's:<a class="headerlink" href="#yaml-cpps" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>YAML parser and emitter in C++
</code></pre></div>
<h3 id="yaml-cpp">yaml-cpp:<a class="headerlink" href="#yaml-cpp" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>YAML parser and emitter in C++
</code></pre></div>
<h3 id="zlibs">zlib's:<a class="headerlink" href="#zlibs" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
</code></pre></div>
<h3 id="zlib">zlib:<a class="headerlink" href="#zlib" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
</code></pre></div>
<h3 id="zstds">zstd's:<a class="headerlink" href="#zstds" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.
</code></pre></div>
<h3 id="zstd">zstd:<a class="headerlink" href="#zstd" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>Zstandard is a real-time compression algorithm, providing high compression ratios.  It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.
</code></pre></div>







  
    
  
  
    
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1-2.1-2M12.5 7v5.2l4 2.4-1 1L11 13V7h1.5M11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2v1.8Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">December 4, 2023</span>
  </span>

    
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Created">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M14.47 15.08 11 13V7h1.5v5.25l3.08 1.83c-.41.28-.79.62-1.11 1m-1.39 4.84c-.36.05-.71.08-1.08.08-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8c0 .37-.03.72-.08 1.08.69.1 1.33.32 1.92.64.1-.56.16-1.13.16-1.72 0-5.5-4.5-10-10-10S2 6.5 2 12s4.47 10 10 10c.59 0 1.16-.06 1.72-.16-.32-.59-.54-1.23-.64-1.92M18 15v3h-3v2h3v3h2v-3h3v-2h-3v-3h-2Z"/></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date">December 4, 2023</span>
  </span>

    
    
    
  </aside>


  




                



              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
    <div id="new-footer">
        <div id="partners">
          <div id="logos">
            <a href="https://nesi.org.nz" class="nesi-footer-logo" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/LogoNeSI-footer.png"></a>
            <a href="http://www.mbie.govt.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoMBIE.png"></a>
            <a href="https://www.auckland.ac.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoUoA.png"></a>
            <a href="https://www.niwa.co.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoNIWA.png"></a>
            <a href="https://www.otago.ac.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/media/LogoUoO.png"></a>
            <a href="http://www.landcareresearch.co.nz/" target="_blank"><img alt="" src="https://www.nesi.org.nz/sites/default/files/LogoManaakiWhenuaLandcareResearch.png"></a>
          </div>
        </div>
        <div id="inner">
          <span id="links"><a href="https://www.nesi.org.nz/copyright" target="_blank">Copyright © <span id="year">2023</span> · New Zealand eScience Infrastructure</a> | <a href="https://www.nesi.org.nz//disclaimer" target="_blank">Disclaimer</a> | <a href="https://www.nesi.org.nz//privacy" target="_blank">Privacy</a>
          </span>
        </div>
    </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.indexes", "navigation.top", "navigation.prune", "content.code.copy", "content.action.edit", "navigation.path"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.caa56a14.min.js"></script>
      
        <script src="../assets/javascripts/general.js"></script>
      
    
  </body>
</html>